(window["webpackJsonp"]=window["webpackJsonp"]||[]).push([["1e10e808"],{"801a":function(e,t,n){},"9fc0":function(e,t,n){"use strict";n.r(t);var a=function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("q-page",{attrs:{padding:""}},[n("div",{staticClass:"column q-mx-auto",staticStyle:{"min-width":"800px","max-width":"1900px"}},[n("h1",[e._v("Help center")]),n("div",{attrs:{id:"contact"}},[n("h2",[e._v("Contact")]),n("div",{staticClass:"authorgroup"},[n("p",[e._v(" For any further questions, suggestions, and feedback, please drop us an email: ")]),n("ul",[n("li",[e._v("for questions related to biological background contact Damian Szklarczyk at\n              "),n("span",{staticClass:"text-reversed"},[n("em",[e._v("hc.hzu.slmi@kyzcralkzs.naimad")])])]),n("li",[e._v("for legal or business proposals contact prof. Dr Christian von Mering at\n              "),n("span",{staticClass:"text-reversed"},[n("em",[e._v("hc.hzu.slmi@gnirem")])])]),n("li",[e._v("for technical issues contact Milan Simonovic at\n              "),n("span",{staticClass:"text-reversed"},[n("em",[e._v("hc.hzu.slmi@civonomis.nalim")])])])])])]),n("div",{attrs:{id:"getting_started"}},[n("h2",[e._v("Getting started")]),n("p",[e._v("\n          PaxDb contains estimated abundance values for a large number of proteins\n          in several different species.\n          Furthermore, you can find information about inter-species variation of protein\n          abundances.\n        ")]),n("h3",[e._v("Examples of Evolutionary Protein Abundance Conservation")]),n("p",[e._v("\n          These plots show the averaged abundances of protein families found to be conserved\n          over large evolutionary distances. Each data point is one protein\n          family, its abundance is plotted after aggregation/summing over\n          paralogous gene copies, and after averaging over datasets and\n          organisms in PaxDb.\n        ")]),n("div",[n("img",{attrs:{src:"/statics/help/correlation_human_mouse.png",alt:"evolutionary correlation: human vs mouse"}}),n("img",{attrs:{src:"/statics/help/correlations_eukaryotes_bacteria.png",alt:"evolutionary correlation: eukaryotic vs bacterial"}}),n("img",{attrs:{src:"/statics/help/correlations_multi_single.png",alt:"evolutionary correlation: single cell vs multi-cellular"}})]),n("p",[e._v("For more information, have a look at these publications:")]),n("ul",[n("li",[e._v("\n            Wang, M. et al. Proteomics 2015 "),n("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/pubmed/25656970",target:"_blank"}},[e._v("pubmed")])]),n("li",[e._v("\n            Wang, M. et al. Mol Cell Proteomics 2012 "),n("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/pubmed/22535208",target:"_blank"}},[e._v("pubmed")])]),n("li",[e._v("\n            Weiss et al., Proteomics 2010\n            "),n("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/pubmed/20077411",target:"_blank"}},[e._v("pubmed")])]),n("li",[e._v("\n            Schrimpf et al., PLoS Biology 2009\n            "),n("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/pubmed/19260763",target:"_blank"}},[e._v("pubmed")])])])]),n("div",{attrs:{id:"scoring"}},[n("h2",[e._v("Re-scaling and Quality Scoring")]),n("p",[e._v("\n          Protein abundance datasets in PaxDb are re-scaled to a common abundance metric ('parts per million'),\n          and\n          also ranked via a universally applicable, albeit somewhat indirect quality score. For the re-scaling,\n          the\n          datasets are first parsed or processed such that the data reflect proportional abundances of whole\n          protein\n          molecules (i.e. proportionality to counts of complete, individual protein molecules, not to molecular\n          weights, protein volumes, or digested peptides). In the case of spectral counting data, this is done via\n          an\n          in-house pipeline that takes into account protein sizes and estimated relative detectabilities of\n          peptides.\n          For other datasets, the procedures depend on the type of data and the type of quantitative information\n          that\n          is provided (datasets which cannot be converted to proportional abundances of entire protein molecules\n          are\n          discarded). Then, the proportional abundances are re-scaled linearly to add up to one million; this\n          means\n          the abundance of each protein of interest is finally expressed in 'parts per million', relative to all\n          other\n          proteins in a sample. While this metric cannot be directly converted to 'molecules per cell', it has the\n          advantage of being comparable/meaningful across cells of different volumes, or across tissues of\n          different\n          cellular and extracellular compositions.\n        ")]),n("p",[e._v("\n          For the quality scoring, we identified a test that can be applied to any organism and to any abundance\n          dataset, albeit at the cost of providing an indirect quality estimate only. This test relies on the\n          assumption that proteins which are interacting physically in a protein complex, or functionally in a\n          pathway\n          or metabolic process, should have a tendency to be expressed at similar abundance levels. This is merely\n          a\n          tendency, of course, and numerous exceptions to this assumption exist. However, globally, the abundance\n          ratios of functionally interacting proteins are clearly closer to one than those of randomly chosen\n          protein\n          pairs, and this signal can be used to provide a relative ranking between datasets, given a constant and\n          externally provided network of functional links between proteins. To compute this score, we first import\n          protein-protein interaction information from the STRING database ("),n("a",{attrs:{href:"https://string-db.org"}},[e._v("string-db.org")]),e._v('),\n          separately for each\n          organism in PaxDb. For a given protein abundance dataset, we then compute the absolute log abundance\n          ratios\n          of all pairs of proteins annotated to be functionally linked. The median of these absolute log abundance\n          ratios represents an indirect quality metric: the closer it is to zero, the better (i.e. the more there\n          is\n          consistency between abundance values and functional annotations such as protein complexes or pathways).\n          We\n          then compute a background expectation for this metric, by permuting the abundance values in a given\n          dataset\n          randomly, and re-computing the median log abundance ratios. The permutation is repeated several times,\n          yielding a distribution of medians. The actually observed median is then expressed as a Z-score distance\n          to\n          the random distribution of medians - this distance is termed the "interaction consistency score".\n        ')])]),n("div",{attrs:{id:"faq"}},[n("h2",{staticClass:"title"},[e._v("Frequently Asked Questions")]),n("dl",[n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q1"}},[e._v("\n            Is it possible to export protein abundances?\n          ")])]),n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q2"}},[e._v("\n            Can PaxDb show tissue-specific abundances?\n          ")])]),n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q3"}},[e._v("\n            What is ppm?\n          ")])]),n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q4"}},[e._v("\n            How to calculate the abundance of proteins based on spectrum count?\n          ")])]),n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q5"}},[e._v("\n            What is PaxDB score?\n          ")])]),n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q6"}},[e._v("\n            How to map proteins from PaxDb to GenBank/Entrez/UniProt/RefSeq/...?\n          ")])]),n("dt",[e._v("Q: "),n("a",{attrs:{href:"help/#q7"}},[e._v("\n            How to cite PaxDb?\n          ")])])]),n("div",{staticClass:"question"},[n("strong",[e._v("Q: ")]),n("a",{attrs:{name:"q1"}},[e._v(" Is it possible to export protein abundances?")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A: ")]),e._v("\n          At the moment, there is a limited export functionality available. When you search for multiple\n          proteins, you'll get the button to download the abundance info in a TSV (tab separated value) file.\n          Here's "),n("a",{attrs:{href:"/proteins?ids=633631,632751,634610"}},[e._v(" anexample")]),e._v(".\n\n          You can also always go and "),n("router-link",{attrs:{to:"/download"}},[e._v("download")]),e._v("\n          the entire dataset for your species and easily filter it in Excel or with a script.\n        ")],1),n("div",{staticClass:"question"},[n("strong",[e._v("Q: ")]),n("a",{attrs:{name:"q2"}},[e._v("Can PaxDb show tissue-specific abundances? ")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A: ")]),e._v("\n          Yes for A. thaliana, H. Sapiens, M. Musculus, and few others. As soon as more tissue-specific data gets\n          published, we'll include it in PaxDb.\n        ")]),n("div",{staticClass:"question"},[n("strong",[e._v("Q: ")]),n("a",{attrs:{name:"q3"}},[e._v(" What is ppm?\n        ")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A:")]),e._v('\n          The protein abundance here is present in ppm which is short for parts\n          per million. Abundance in "ppm" is essentially describing each protein\n          with reference to the entire expressed proteome.This means each\n          protein entity is enumerated relative to all other protein molecules\n          in the sample.\n        ')]),n("div",{staticClass:"question"},[n("strong",[e._v("Q: ")]),n("a",{attrs:{name:"q4"}},[e._v(" How to calculate the abundance of proteins based on spectrum count?\n        ")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A: ")]),e._v("\n          The method of calculating the abundance of proteins based on\n          spectrum count is in publication: Weiss et al., Proteomics 2010\n          ("),n("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/pubmed/20077411"}},[e._v("http://www.ncbi.nlm.nih.gov/pubmed/20077411")]),e._v(")\n        ")]),n("div",{staticClass:"question"},[n("strong",[e._v("Q: ")]),n("a",{attrs:{name:"q5"}},[e._v(" What is PaxDB score?")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A: ")]),e._v("\n          The PaxDB score is developed to estimate protein abundance data\n          quality. This is based on the assumption that the interacting proteins\n          tend to have similar abundance. Therefore, we compute the abundance\n          ratios for pairs of interacting proteins (the protein interaction\n          information imported from String DB ), compare the median of these\n          ratios with the distribution of randomized median and convert them\n          into a z-score, which is PaxDB score. The bigger the PaxDB score, the\n          better the data quality.\n        ")]),n("div",{staticClass:"question"},[n("strong",[e._v("Q:")]),n("a",{attrs:{name:"q6"}},[e._v(" How to map proteins from PaxDb to\n          GenBank/Entrez/UniProt/RefSeq/...?\n        ")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A: ")]),e._v("\n          A mapping between PaxDb and UniProt proteins can be found in the downloads section:\n          "),n("a",{attrs:{href:"/downloads/latest/mapping_files/uniprot_mappings/"}},[e._v("mapping_files/uniprot_mappings")]),e._v(".\n\n          Here's an example how to add UniProt ids to a dataset:\n          "),n("pre",[e._v("wget https://pax-db.org/data/abundances/9606-KIDNEY-integrated.txt\nwget https://pax-db.org/downloads/4.1/mapping_files/uniprot_mappings/9606_reviewed_uniprot_2_string.04_2015.tsv.gz\ngunzip 9606_reviewed_uniprot_2_string.04_2015.tsv.gz\ngrep -v '#' 9606-KIDNEY-integrated.txt | cut -f 2,3 | sed 's/9606.//â€˜ | sort > kidney.txt\ngrep -v '#' 9606_reviewed_uniprot_2_string.04_2015.tsv | awk '{print $3,$2}' | sort > mapping.txt\njoin -1 1 -2 1  -o '0,2.2,1.2'  kidney.txt mapping.txt > kidney_uniprot.txt\n          ")])]),n("div",{staticClass:"question"},[n("strong",[e._v("Q: ")]),n("a",{attrs:{name:"q7"}},[e._v(" How to cite PaxDb?\n        ")])]),n("div",{staticClass:"answer"},[n("strong",[e._v("A: ")]),n("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/pubmed/25656970"}},[e._v("\n            Wang, M. et al. Proteomics 2015, 10.1002/pmic.201400441")])])]),n("div",{attrs:{id:"developer"}},[n("h2",[e._v("Developer Documentation")]),n("div",{attrs:{id:"developer-downloads"}},[n("h3",[e._v("Downloads")]),n("p",[e._v("\n              Our "),n("router-link",{attrs:{to:"/download"}},[e._v("download page")]),e._v(" contains all abundance datasets in a tab-seperated format as\n              well as the\n              "),n("a",{attrs:{href:"/downloads/4.1/paxdb-protein-sequences-v4.1.zip"}},[e._v("protein sequences")]),e._v(" used for\n              spectral counting etc.\n              For a detailed list of available files please visit "),n("router-link",{attrs:{to:"/download"}},[e._v("the download page")]),e._v(".\n            ")],1),n("p",[e._v("\n              The abundance data contains identifiers in the format "),n("code",[e._v("species_id.protein_id")]),e._v(" where "),n("code",[e._v("species_id")]),e._v("\n              is the Ensembl species id.\n            ")])]),n("div",{attrs:{id:"developer-api"}},[n("h3",[e._v("API")]),n("p",[e._v("\n              We decided to remove our API from public use since there hadn't been any interest.\n            ")])])])])])},i=[],s={name:"HelpPage",meta:{title:"PAXdb - Help"}},o=s,r=(n("d6b5"),n("2877")),l=Object(r["a"])(o,a,i,!1,null,"fe87244e",null);t["default"]=l.exports},d6b5:function(e,t,n){"use strict";var a=n("801a"),i=n.n(a);i.a}}]);