(window["webpackJsonp"]=window["webpackJsonp"]||[]).push([["app"],{0:function(e,a,t){e.exports=t("2f39")},"0196":function(e,a,t){"use strict";t.d(a,"a",function(){return n});t("7514");function n(e){if(1===e.datasets.length)return e.datasets[0];var a=e.datasets.find(function(e){return!("WHOLE_ORGANISM"!==e.organ||!e.integrated)});return a||(a=e.datasets.find(function(e){return!!e.integrated})),a}},"0538":function(e,a,t){"use strict";var n=t("23d1"),i=t.n(n);i.a},"14b0":function(e,a,t){"use strict";var n=function(){var e=this,a=e.$createElement,t=e._self._c||a;return e.dataLoaded?t("div",{staticClass:"column"},[t("h2",[e._v("Protein family")]),e.protein.hasFamily?t("div",{staticClass:"column"},[t("transition",{attrs:{appear:"","enter-active-class":"animated fadeIn","leave-active-class":"animated fadeOut"}},[t("div",{staticClass:"column"},[t("div",{staticClass:"row q-gutter-lg"},[t("div",{staticClass:"col-xs-12 col-sm-4 col-md-4 col-lg-3"},[t("q-select",{attrs:{filled:"",options:e.taxonomicLevels,label:"taxonomic level"},model:{value:e.selectedLevel,callback:function(a){e.selectedLevel=a},expression:"selectedLevel"}})],1),t("div",{staticClass:"col-xs-12 col-sm-4 col-md-4 col-lg-3"},[t("q-select",{attrs:{filled:"",options:e.tissues,label:"tissue"},model:{value:e.selectedTissue,callback:function(a){e.selectedTissue=a},expression:"selectedTissue"}})],1)]),t("div",{staticStyle:{overflow:"auto",width:"100%"}},[t("div",{attrs:{id:"family_tree"}})])])]),t("q-inner-loading",{attrs:{showing:!e.dataLoaded}},[t("q-spinner-gears",{attrs:{size:"50px",color:"primary"}})],1),t("div",{staticClass:"text-danger",staticStyle:{display:"none"}},[e._v("failed to load protein family")]),t("button",{staticClass:"btn btn-primary",staticStyle:{display:"none","margin-left":"0.5em"},attrs:{tabindex:"-1",id:"reload_family_button",type:"button"}},[e._v("Reload")])],1):t("div",[e._v("\n    "+e._s(e.protein.name)+" has no family members\n  ")])]):e._e()},i=[],o=t("82b3"),r=o["a"],s=(t("c88b"),t("38a5"),t("2877")),c=Object(s["a"])(r,n,i,!1,null,"e21d29f8",null);a["a"]=c.exports},"16b0":function(e,a,t){"use strict";t("0d6d"),t("ac6a"),t("cadf"),t("06db"),t("456d");var n={4932:"S. cerevisiae",9606:"H. sapiens",6239:"C. elegans",3702:"A. thaliana",7227:"D. melanogaster",64091:"H. salinarum",10090:"M. musculus",449447:"M. aeruginosa",267671:"L. interrogans",511145:"E. coli",7460:"A. mellifera",4896:"S. pombe",7955:"D. rerio",7165:"A. gambiae",8364:"X. tropicalis",9031:"G. gallus",9615:"C. familiaris",9823:"S. scrofa",9913:"B. taurus",10116:"R. norvegicus",4081:"S. lycopersicum",39947:"O. sativa",83332:"M. tuberculosis",160490:"S. pyogenes",1148:"S. sp. 6803",198214:"S. flexneri 2457T",224308:"B. subtili",546414:"D. deserti",593117:"T. g. EJ3",122586:"N. meningitidis",158878:"S. aureus",169963:"L. monocytogenes",208964:"P. aeruginosa",211586:"S. oneidensis",226186:"B. thetaiotaomicron",260799:"B. anthracis ",272623:"L. lactis",272624:"L. pneumophila",3055:"C. reinhardtii",44689:"D. discoideum",5061:"A. niger",5691:"T. brucei",5833:"P. falciparum",85962:"H. pylori",9796:"E. caballus",99287:"S. enterica",9598:"P. troglodytes",4577:"Z. mays",353153:"T. cruzi",214684:"C. neoformans",214092:"Y. pestis",882:"D. vulgaris",243159:"A. ferrooxidans",283166:"B. henselae",192222:"C. jejuni",722438:"M. pneumoniae"},i=[{id:"9606",official_name:"Homo sapiens"},{id:"4932",official_name:"Saccharomyces cerevisiae"},{id:"511145",official_name:"Escherichia coli str. K-12 substr. MG1655"},{id:"243159",official_name:"Acidithiobacillus ferrooxidans ATCC 23270"},{id:"7165",official_name:"Anopheles gambiae"},{id:"7460",official_name:"Apis mellifera"},{id:"3702",official_name:"Arabidopsis thaliana"},{id:"5061",official_name:"Aspergillus niger"},{id:"260799",official_name:"Bacillus anthracis str. Sterne"},{id:"224308",official_name:"Bacillus subtilis subsp. subtilis str. 168"},{id:"226186",official_name:"Bacteroides thetaiotaomicron VPI-5482"},{id:"283166",official_name:"Bartonella henselae str. Houston-1"},{id:"9913",official_name:"Bos taurus"},{id:"6239",official_name:"Caenorhabditis elegans"},{id:"192222",official_name:"Campylobacter jejuni subsp. jejuni NCTC 11168"},{id:"9615",official_name:"Canis lupus familiaris"},{id:"3055",official_name:"Chlamydomonas reinhardtii"},{id:"214684",official_name:"Cryptococcus neoformans var. neoformans JEC21"},{id:"7955",official_name:"Danio rerio"},{id:"546414",official_name:"Deinococcus deserti VCD115"},{id:"882",official_name:"Desulfovibrio vulgaris str. Hildenborough"},{id:"44689",official_name:"Dictyostelium discoideum"},{id:"7227",official_name:"Drosophila melanogaster"},{id:"9796",official_name:"Equus caballus"},{id:"511145",official_name:"Escherichia coli str. K-12 substr. MG1655"},{id:"9031",official_name:"Gallus gallus"},{id:"64091",official_name:"Halobacterium sp. NRC-1"},{id:"85962",official_name:"Helicobacter pylori 26695"},{id:"9606",official_name:"Homo sapiens"},{id:"272623",official_name:"Lactococcus lactis subsp. lactis Il1403"},{id:"272624",official_name:"Legionella pneumophila subsp. pneumophila str. Philadelphia 1"},{id:"267671",official_name:"Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130"},{id:"169963",official_name:"Listeria monocytogenes EGD-e"},{id:"449447",official_name:"Microcystis aeruginosa NIES-843"},{id:"10090",official_name:"Mus musculus"},{id:"83332",official_name:"Mycobacterium tuberculosis H37Rv"},{id:"722438",official_name:"Mycoplasma pneumoniae FH"},{id:"122586",official_name:"Neisseria meningitidis MC58"},{id:"39947",official_name:"Oryza sativa Japonica Group"},{id:"9598",official_name:"Pan troglodytes"},{id:"5833",official_name:"Plasmodium falciparum"},{id:"208964",official_name:"Pseudomonas aeruginosa PAO1"},{id:"10116",official_name:"Rattus norvegicus"},{id:"4932",official_name:"Saccharomyces cerevisiae"},{id:"99287",official_name:"Salmonella enterica subsp. enterica serovar Typhimurium str. LT2"},{id:"4896",official_name:"Schizosaccharomyces pombe"},{id:"211586",official_name:"Shewanella oneidensis MR-1"},{id:"198214",official_name:"Shigella flexneri 2a str. 301"},{id:"4081",official_name:"Solanum lycopersicum"},{id:"158878",official_name:"Staphylococcus aureus subsp. aureus Mu50"},{id:"160490",official_name:"Streptococcus pyogenes M1 GAS"},{id:"9823",official_name:"Sus scrofa"},{id:"1148",official_name:"Synechocystis sp. PCC 6803"},{id:"593117",official_name:"Thermococcus gammatolerans EJ3"},{id:"5691",official_name:"Trypanosoma brucei"},{id:"353153",official_name:"Trypanosoma cruzi strain CL Brener"},{id:"8364",official_name:"Xenopus (Silurana) tropicalis"},{id:"214092",official_name:"Yersinia pestis CO92"},{id:"4577",official_name:"Zea mays"}],o={9606:"Homo sapiens",4932:"Saccharomyces cerevisiae",511145:"Escherichia coli str. K-12 substr. MG1655",243159:"Acidithiobacillus ferrooxidans ATCC 23270",7165:"Anopheles gambiae",7460:"Apis mellifera",3702:"Arabidopsis thaliana",5061:"Aspergillus niger",260799:"Bacillus anthracis str. Sterne",224308:"Bacillus subtilis subsp. subtilis str. 168",226186:"Bacteroides thetaiotaomicron VPI-5482",283166:"Bartonella henselae str. Houston-1",9913:"Bos taurus",6239:"Caenorhabditis elegans",192222:"Campylobacter jejuni subsp. jejuni NCTC 11168",9615:"Canis lupus familiaris",3055:"Chlamydomonas reinhardtii",214684:"Cryptococcus neoformans var. neoformans JEC21",7955:"Danio rerio",546414:"Deinococcus deserti VCD115",882:"Desulfovibrio vulgaris str. Hildenborough",44689:"Dictyostelium discoideum",7227:"Drosophila melanogaster",9796:"Equus caballus",9031:"Gallus gallus",64091:"Halobacterium sp. NRC-1",85962:"Helicobacter pylori 26695",272623:"Lactococcus lactis subsp. lactis Il1403",272624:"Legionella pneumophila subsp. pneumophila str. Philadelphia 1",267671:"Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130",169963:"Listeria monocytogenes EGD-e",449447:"Microcystis aeruginosa NIES-843",10090:"Mus musculus",83332:"Mycobacterium tuberculosis H37Rv",722438:"Mycoplasma pneumoniae FH",122586:"Neisseria meningitidis MC58",39947:"Oryza sativa Japonica Group",9598:"Pan troglodytes",5833:"Plasmodium falciparum",208964:"Pseudomonas aeruginosa PAO1",10116:"Rattus norvegicus",99287:"Salmonella enterica subsp. enterica serovar Typhimurium str. LT2",4896:"Schizosaccharomyces pombe",211586:"Shewanella oneidensis MR-1",198214:"Shigella flexneri 2a str. 301",4081:"Solanum lycopersicum",158878:"Staphylococcus aureus subsp. aureus Mu50",160490:"Streptococcus pyogenes M1 GAS",9823:"Sus scrofa",1148:"Synechocystis sp. PCC 6803",593117:"Thermococcus gammatolerans EJ3",5691:"Trypanosoma brucei",353153:"Trypanosoma cruzi strain CL Brener",8364:"Xenopus (Silurana) tropicalis",214092:"Yersinia pestis CO92",4577:"Zea mays"},r={"A. ferrooxidans ATCC 23270":"243159","A. gambiae":"7165","A. mellifera":"7460","A. niger":"5061","A. thaliana":"3702","Acidithiobacillus ferrooxidans ATCC 23270":"243159","Acidithiobacillus ferrooxidans ATCC23270":"243159","Acidithiobacillus ferrooxidans str. ATCC 23270":"243159","African malaria mosquito":"7165",Amphibia:"8364","Anopheles gambiae sensu stricto":"7165","Anopheles gambiae":"7165",Anthophoridae:"7460",Anura:"8364","Aphanocapsa sp. (strain N-1)":"1148","Aphanocapsa sp. N-1":"1148",Apidae:"7460",Apinae:"7460",Apini:"7460","Apis mellifera":"7460","Apis mellifica":"7460",Apis:"7460",Apoidea:"7460","Arabidopsis thaliana (L.) Heynh.":"3702","Arabidopsis thaliana":"3702","Aspergillus niger Tiegh.":"5061","Aspergillus niger":"5061","B. anthracis str. Sterne":"260799","B. henselae str. Houston-1":"283166","B. henselae":"283166","B. subtilis subsp. subtilis str. 168":"224308","B. subtilis subsp. subtilis":"224308","B. subtilis":"224308","B. taurus":"9913","B. thetaiotaomicron VPI-5482":"226186","B. thetaiotaomicron":"226186","Bacillus aeruginosus":"208964","Bacillus anthracis (strain Sterne)":"260799","Bacillus anthracis Sterne":"260799","Bacillus anthracis str. Sterne":"260799","Bacillus globigii":"224308","Bacillus halobius ruber":"64091","Bacillus natto":"224308","Bacillus pyocyaneus":"208964","Bacillus sp. 15:2":"224308","Bacillus sp. RSP-GLU":"224308","Bacillus sp. S2 BC-1":"224308","Bacillus subtilis 168":"224308","Bacillus subtilis subsp. subtilis 168":"224308","Bacillus subtilis subsp. subtilis str. 168":"224308","Bacillus subtilis subsp. subtilis str. BGSC 1A700":"224308","Bacillus subtilis subsp. subtilis":"224308","Bacillus subtilis":"224308","Bacillus subtilis/Bacillus globigii":"224308","Bacillus thetaiotaomicron":"226186","Bacillus typhimurium":"99287","Bacillus uniflagellatus":"224308","Bacterium aeruginosum":"208964","Bacterium halobium":"64091","Bacterium pyocyaneum":"208964","Bacteroides fragilis subsp. thetaiotaomicron":"226186","Bacteroides thetaiotaomicron VPI-5482":"226186","Bacteroides thetaiotaomicron str. VPI-5482":"226186","Bacteroides thetaiotaomicron strain VPI-5482":"226186","Bacteroides thetaiotaomicron":"226186","Bartonella henselae ATCC 49882":"283166","Bartonella henselae str. ATCC 49882":"283166","Bartonella henselae str. Houston-1":"283166","Bartonella henselae strain ATCC 49882":"283166","Bartonella henselae strain Houston-1":"283166","Bartonella henselae":"283166",Batrachia:"8364","Bos bovis":"9913","Bos primigenius taurus":"9913","Bos taurus":"9913",Bos:"9913","Bovidae sp. Adi Nefas":"9913",Bovidae:"9913",Bovinae:"9913","Brachydanio rerio frankei":"7955","Brachydanio rerio":"7955",Brachydanio:"7955","Buffalo rat":"10116","C. elegans":"6239","C. jejuni subsp. jejuni NCTC 11168":"192222","C. lupus familiaris":"9615","C. lupus":"9615","C. neoformans var. neoformans JEC21":"214684","C. reinhardtii":"3055","Caenorhabditis elegans":"6239","Campylobacter jejuni NCTC11168":"192222","Campylobacter jejuni subsp. jejuni NCTC 11168":"192222","Campylobacter jejuni subsp. jejuni str. NCTC 11168":"192222","Campylobacter jejuni subsp. jejuni strain NCTC 11168":"192222","Candida robusta":"4932",Canidae:"9615","Canis canis":"9615","Canis domesticus":"9615","Canis familiaris":"9615","Canis lupus familiaris":"9615","Canis lupus":"9615",Canis:"9615",Celestichthys:"7955",Cellia:"7165",Chlamydomonadaceae:"3055",Chlamydomonadales:"3055","Chlamydomonas reinhardtii P.A.Dangeard":"3055","Chlamydomonas reinhardtii":"3055","Chlamydomonas smithii R.W.Howshaw & H.Ettl":"3055","Chlamydomonas smithii":"3055",Chlamydomonas:"3055",Chlorophyceae:"3055","Cryptococcus neoformans JEC21":"214684","Cryptococcus neoformans var. neoformans JEC21":"214684","Cryptococcus neoformans var. neoformans serotype D JEC21":"214684","Cryptococcus neoformans var. neoformans strain JEC21":"214684",Cyprinidae:"7955",Cypriniformes:"7955",Cypriniphysi:"7955",Cyprinoidea:"7955",Cyprinoidei:"7955","Cyprinus rerio":"7955","D. deserti VCD115":"546414","D. deserti":"546414","D. discoideum":"44689","D. melanogaster":"7227","D. rerio":"7955","D. vulgaris str. Hildenborough":"882","Danio frankei":"7955","Danio rerio frankei":"7955","Danio rerio":"7955",Danio:"7955","Deinococcus deserti VCD115":"546414","Deinococcus deserti de Groot et al. 2005":"546414","Deinococcus deserti str. VCD115":"546414","Deinococcus deserti strain VCD115":"546414","Deinococcus deserti":"546414","Desulfovibrio vulgaris (STRAIN HILDENBOROUGH)":"882","Desulfovibrio vulgaris ATCC 29579":"882","Desulfovibrio vulgaris Hildenborough":"882","Desulfovibrio vulgaris str. Hildenborough":"882","Desulfovibrio vulgaris subsp. vulgaris (strain Hildenborough)":"882","Desulfovibrio vulgaris subsp. vulgaris ATCC 29579":"882","Desulfovibrio vulgaris subsp. vulgaris str. Hildenborough":"882","Dictyostelium discoideum":"44689","Diplokokkus intracellularis meningitidis":"122586","Drosophila melanogaster":"7227","E. caballus":"9796","E. coli str. K-12 substr. MG1655":"511145","E. subg. Equus":"9796",Equidae:"9796","Equus caballus":"9796","Equus ferus caballus":"9796","Equus przewalskii f. caballus":"9796","Equus przewalskii forma caballus":"9796","Equus subg. Equus":"9796",Equus:"9796","Escherichia coli K12 MG1655":"511145","Escherichia coli MG1655":"511145","Escherichia coli str. K-12 substr. MG1655":"511145","Escherichia coli str. K12 substr. MG1655":"511145","Escherichia coli str. MG1655":"511145","Escherichia coli strain MG1655":"511145",Euchlaena:"4577","Filobasidiella neoformans var. neoformans strain JEC21":"214684","Filterable agent of primary atypical pneumonia":"722438","Flavobacterium (subgen. Halobacterium) halobium":"64091","Flavobacterium (subgen. Halobacterium) salinarium":"64091","Flavobacterium (subgen. Halobacterium)":"64091","G. gallus":"9031","Gallus domesticus":"9031","Gallus gallus domesticus":"9031","Gallus gallus":"9031",Gallus:"9031","Gunn rats":"10116","H. pylori 26695":"85962","H. salinarum":"64091","H. sapiens":"9606","H. sp. NRC-1":"64091",Haloarchaeum:"64091","Halobacter salinaria":"64091",Halobacter:"64091","Halobacterium NRC_1":"64091","Halobacterium cutirubrum":"64091","Halobacterium halobium":"64091","Halobacterium salinarium":"64091","Halobacterium salinarum NRC-1":"64091","Halobacterium salinarum str. NRC-1":"64091","Halobacterium salinarum strain NRC-1":"64091","Halobacterium salinarum":"64091","Halobacterium sp. (strain NRC-1 / ATCC 700922 / JCM 11081)":"64091","Halobacterium sp. (strain NRC-1)":"64091","Halobacterium sp. ATCC 700922":"64091","Halobacterium sp. JCM 11081":"64091","Halobacterium sp. NRC-1":"64091","Halobacterium sp. NRC1":"64091",Halobacterium:"64091","Helicobacter pylori (strain 26695)":"85962","Helicobacter pylori 26695":"85962","Helicobacter pylori ATCC 700392":"85962","Helicobacter pylori KE26695":"85962","Helicobacter pylori str. 26695":"85962","Helicobacter pylori strain 26695":"85962","Homo sapiens":"9606",Homo:"9606","Japanese rice":"39947","L. interrogans serovar Copenhageni str. Fiocruz L1-130":"267671","L. interrogans serovar Copenhageni":"267671","L. lactis subsp. lactis Il1403":"272623","L. lactis subsp. lactis":"272623","L. monocytogenes EGD-e":"169963","L. pneumophila subsp. pneumophila str. Philadelphia 1":"272624","L. pneumophila subsp. pneumophila":"272624","LK3 transgenic mice":"10090","Lactobacillus xylosus":"272623","Lactococcus lactis (SUBSP. LACTIS)":"272623","Lactococcus lactis Il1403":"272623","Lactococcus lactis lactis":"272623","Lactococcus lactis subsp. lactis Il1403":"272623","Lactococcus lactis subsp. lactis str. Il1403":"272623","Lactococcus lactis subsp. lactis strain Il1403":"272623","Lactococcus lactis subsp. lactis":"272623",Laverania:"5833","Legionella pneumophila Philadelphia":"272624","Legionella pneumophila subsp. pneumophila str. Philadelphia 1":"272624","Legionella pneumophila subsp. pneumophila strain Philadelphia 1":"272624","Legionella pneumophila subsp. pneumophila":"272624","Leptospira interrogans copenhageni":"267671","Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130":"267671","Leptospira interrogans serovar Copenhageni strain Fiocruz L1-130":"267671","Leptospira interrogans serovar Copenhageni":"267671",Lissamphibia:"8364","Listeria monocytogenes ATCC BAA-679":"169963","Listeria monocytogenes EGD-e":"169963","Listeria monocytogenes EGDe":"169963","Listeria monocytogenes str. EGD-e":"169963","Listeria monocytogenes strain EGD-e":"169963","Lycopersicon (Mill.) Seithe":"4081","Lycopersicon esculentum Mill.":"4081","Lycopersicon esculentum var. esculentum":"4081","Lycopersicon esculentum":"4081","Lycopersicon lycopersicum":"4081",Lycopersicon:"4081","M. aeruginosa NIES-843":"449447","M. aeruginosa":"449447","M. musculus":"10090","M. pneumoniae FH":"722438","M. pneumoniae":"722438","M. tuberculosis H37Rv":"83332",Mesobatrachia:"8364","Micrococcus intracellularis":"122586","Micrococcus meningitidis cerebrospinalis":"122586","Micrococcus meningitidis":"122586","Micrococcus pyocyaneus":"208964","Micrococcus scarlatinae":"160490","Microcystis aeruginosa NIES-843":"449447","Microcystis aeruginosa":"449447",Microcystis:"449447","Mus muscaris":"10090","Mus musculus":"10090","Mus sp. 129SV":"10090",Mus:"10090","Mycobacterium tuberculosis H37Rv":"83332","Mycobacterium tuberculosis str. H37Rv":"83332","Mycobacterium tuberculosis strain H37Rv":"83332","Mycoplasma pneumoniae ATCC 15531":"722438","Mycoplasma pneumoniae FH":"722438","Mycoplasma pneumoniae NCTC 10119":"722438","Mycoplasma pneumoniae str. FH":"722438","Mycoplasma pneumoniae strain FH":"722438","Mycoplasma pneumoniae":"722438","N. meningitidis MC58":"122586","N. meningitidis serogroup B":"122586","N. meningitidis":"122586","Neisseria meningitidis (group B)":"122586","Neisseria meningitidis (serogroup B)":"122586","Neisseria meningitidis MC58":"122586","Neisseria meningitidis group B":"122586","Neisseria meningitidis serogroup B strain MC58":"122586","Neisseria meningitidis serogroup B":"122586","Neisseria meningitidis":"122586","Neisseria weichselbaumii":"122586","Norway rat":"10116","O. sativa Japonica Group":"39947","Oryza sativa (japonica cultivar-group)":"39947","Oryza sativa (javanica cultivar-group)":"39947","Oryza sativa Aromatic Japonica Group":"39947","Oryza sativa Japonica Group":"39947","Oryza sativa Temperate Japonica Group":"39947","Oryza sativa Tropical Japonica Group":"39947","Oryza sativa aromatic cultivar-group":"39947","Oryza sativa japonica":"39947","Oryza sativa subsp. japonica":"39947","Oryza sativa temperate japonica cultivar-group":"39947","Oryza sativa tropical japonica cultivar-group":"39947","Oryza sativa var. javanica Koern.":"39947","Oryza sativa var. javanica":"39947",Ostariophysi:"7955",Otocephala:"7955",Otophysi:"7955","P. (Laverania)":"5833","P. aeruginosa PAO1":"208964","P. aeruginosa":"208964","P. falciparum":"5833","P. troglodytes":"9598",Pachytichospora:"4932","Pan sp. WO2005014654":"9598","Pan sp. WO2007026120":"9598","Pan troglodytes":"9598",Pan:"9598",Pecora:"9913",Perissodactyla:"9796",Phasianinae:"9031",Pipidae:"8364",Pipoidea:"8364","Plasmodium (Laverania) falciparum":"5833","Plasmodium (Laverania)":"5833","Plasmodium falciparum":"5833","Pseudobacterium thetaiotaomicron":"226186","Pseudomonas aeruginosa PAO1":"208964","Pseudomonas aeruginosa str. PAO1":"208964","Pseudomonas aeruginosa":"208964","Pseudomonas polycolor":"208964","Pseudomonas pyocyanea":"208964","Pseudomonas salinaria":"64091",Pyretophorus:"7165","R. norvegicus":"10116","Rattus PC12 clone IS":"10116","Rattus norvegicus":"10116","Rattus rattiscus":"10116","Rattus sp. strain Wistar":"10116",Rattus:"10116","Rhabditis elegans":"6239","Rochalimaea henselae":"283166",Ruminantia:"9913","S. aureus subsp. aureus Mu50":"158878","S. cerevisiae":"4932","S. enterica subsp. enterica serovar Typhimurium str. LT2":"99287","S. enterica subsp. enterica serovar Typhimurium":"99287","S. flexneri 2a str. 301":"198214","S. flexneri 2a":"198214","S. flexneri":"198214","S. lycopersicum":"4081","S. oneidensis MR-1":"211586","S. oneidensis":"211586","S. pombe":"4896","S. pyogenes M1 GAS":"160490","S. pyogenes serotype M1":"160490","S. pyogenes":"160490","S. scrofa":"9823","S. sp. PCC 6803":"1148","Saccharomyces capensis":"4932","Saccharomyces cerevisiae":"4932","Saccharomyces italicus":"4932","Saccharomyces oviformis":"4932","Saccharomyces uvarum var. melibiosus":"4932",Saccharomyces:"4932",Salientia:"8364","Salmonella choleraesuis serotype typhimurium":"99287","Salmonella enterica 1,4,[5],12,:i:1,2":"99287","Salmonella enterica LT2":"99287","Salmonella enterica ser. typhimurium":"99287","Salmonella enterica serotype Typhimurium":"99287","Salmonella enterica serovar Typhimurium":"99287","Salmonella enterica subsp. enterica serovar 1,4,[5],12,:i:1,2":"99287","Salmonella enterica subsp. enterica serovar Typhimurium LT2":"99287","Salmonella enterica subsp. enterica serovar Typhimurium str. LT2":"99287","Salmonella enterica subsp. enterica serovar Typhimurium strain LT2":"99287","Salmonella enterica subsp. enterica serovar Typhimurium strain LT2-LTL2":"99287","Salmonella enterica subsp. enterica serovar Typhimurium":"99287","Salmonella typhi-murium":"99287","Salmonella typhimurium LT2":"99287","Salmonella typhimurium":"99287","Schizoplasma pneumoniae":"722438","Schizosaccharomyces malidevorans":"4896","Schizosaccharomyces pombe":"4896",Schizotrypanum:"353153","Serratia cutirubrum":"64091","Serratia salinaria":"64091","Shewanella oneidensis ATCC 700550":"211586","Shewanella oneidensis MR-1":"211586","Shewanella oneidensis str. MR-1":"211586","Shewanella oneidensis strain MR-1":"211586","Shewanella oneidensis":"211586","Shewanella putrefaciens MR-1":"211586","Shewanella sp. MR-1":"211586","Shewanella sp. SCH0401":"211586","Shigella flexneri 2a str. 301":"198214","Shigella flexneri 2a":"198214","Shigella flexneri serotype 2a str. 301":"198214","Shigella flexneri serotype 2a":"198214","Shigella flexneri":"198214","Shigella paradysenteriae":"198214",Shigella:"198214","Silurana tropicalis":"8364",Silurana:"8364",Siluraninae:"8364","Solanum esculentum Dunal":"4081","Solanum esculentum":"4081","Solanum lycopersicum L.":"4081","Solanum lycopersicum":"4081","Sphaerocillus thetaiotaomicron":"226186","Sprague-Dawley rat":"10116","Staphylococcus aureus (strain Mu50 / ATCC 700699)":"158878","Staphylococcus aureus Mu50":"158878","Staphylococcus aureus subsp. aureus Mu50":"158878","Streptococcus diacetilactis":"272623","Streptococcus erysipelatos":"160490","Streptococcus hemolyticus":"160490","Streptococcus lactis subsp. diacetilactis":"272623","Streptococcus lactis subsp. lactis":"272623","Streptococcus pyogenes (serotype 1 / M1 GAS)":"160490","Streptococcus pyogenes ATCC 700294":"160490","Streptococcus pyogenes M1 GAS":"160490","Streptococcus pyogenes SF370":"160490","Streptococcus pyogenes serotype M1":"160490","Streptococcus pyogenes strain SF370":"160490","Streptococcus pyogenes":"160490","Streptococcus scarlatinae":"160490",Suidae:"9823",Suiformes:"9823",Suina:"9823","Sus scrofa":"9823",Sus:"9823","Synechocystis PCC6803":"1148","Synechocystis sp. (ATCC 27184)":"1148","Synechocystis sp. (PCC 6803)":"1148","Synechocystis sp. (strain PCC 6803)":"1148","Synechocystis sp. ATCC 27184":"1148","Synechocystis sp. PCC 6803 A":"1148","Synechocystis sp. PCC 6803 B":"1148","Synechocystis sp. PCC 6803":"1148","Synechocystis sp. PCC6803":"1148",Synechocystis:"1148","T. brucei":"5691","T. cruzi strain CL Brener":"353153","T. cruzi":"353153","T. gammatolerans EJ3":"593117","T. gammatolerans":"593117","Thermococcus gammatolerans EJ3":"593117","Thermococcus gammatolerans Jolivet et al. 2003":"593117","Thermococcus gammatolerans str. EJ3":"593117","Thermococcus gammatolerans strain EJ3":"593117","Thermococcus gammatolerans":"593117","Trypanosoma (Trypanozoon) brucei":"5691","Trypanosoma brucei subgroup":"5691","Trypanosoma brucei":"5691","Trypanosoma cruzi strain CL Brener":"353153","Trypanosoma cruzi":"353153",Trypanozoon:"5691","Vibrio subtilis":"224308",Volvocales:"3055",Volvocida:"3055","Wistar rats":"10116","X. (Silurana) tropicalis":"8364",Xenopodinae:"8364","Xenopus (Silurana) tropicalis":"8364","Xenopus Silurana":"8364","Xenopus laevis tropicalis":"8364","Xenopus tropicalis group":"8364","Xenopus tropicalis":"8364",Xenopus:"8364","Y. pestis CO92":"214092","Yersinia pestis CO92":"214092","Yersinia pestis str. CO92":"214092","Yersinia pestis strain CO92":"214092","Yersinia pestis":"214092","Z. mays":"4577","Zea L.":"4577","Zea mays L.":"4577","Zea mays var. japonica":"4577","Zea mays":"4577",Zea:"4577",amphibians:"8364",anurans:"8364","bacterium ASFP-37":"208964","bacterium ASFP-38":"208964","bacterium ASFP-45":"208964","bacterium ASFP-46":"208964","bacterium ASFP-48":"208964",bantam:"9031","beagle dog":"9615","beagle dogs":"9615",bee:"7460",bees:"7460",boars:"9823",bovine:"9913","brown rat":"10116","bumble bees and honey bees":"7460",cattle:"9913",chicken:"9031",chickens:"9031",chimpanzee:"9598",chimpanzees:"9598",cow:"9913",dog:"9615","dog, coyote, wolf, fox":"9615",dogs:"9615","domestic cattle":"9913","domestic cow":"9913","domestic horse":"9796","dwarf Leghorn chickens":"9031",equine:"9796","fission yeast":"4896","frogs & toads":"8364","frogs and toads":"8364",frogs:"8364","fruit fly":"7227","g. species complex":"7165","gambiae species complex":"7165","gray wolf":"9615","grey wolf":"9615","honey bee":"7460","honey bees":"7460",honeybee:"7460",horse:"9796",horses:"9796","house mouse":"10090",human:"9606","laboratory rat":"10116","lager beer yeast":"4932","leopard danio":"7955",maize:"4577","malaria parasite P. falciparum":"5833",man:"9606",mice:"10090",mouse:"10090","mouse-ear cress":"3702",nematode:"6239","nude mice":"10090","odd-toed ungulates":"9796","oxen, cattle":"9913",pig:"9823",pigs:"9823","pipid frogs":"8364",rat:"10116",rats:"10116","red junglefowl":"9031",swine:"9823","thale cress":"3702","thale-cress":"3702",tomato:"4081","tongueless frogs":"8364","transgenic mice":"10090","western clawed frog":"8364","wild boar":"9823",yeast:"4932","zebra danio":"7955","zebra fish":"7955",zebrafish:"7955","zitter rats":"10116"},s={"cellular organisms":["882","1148","3055","3702","4081","4577","4896","4932","5061","5691","5833","6239","7165","7227","7460","7955","8364","9031","9598","9606","9615","9796","9823","9913","10090","10116","39947","44689","64091","83332","85962","99287","122586","158878","160490","169963","192222","198214","208964","211586","214092","214684","224308","226186","243159","260799","267671","272623","272624","283166","353153","449447","511145","546414","593117","722438"],Cyanobacteria:["1148","449447"],Deproteobacteria:["192222","85962","882"],Gammaproteobacteria:["198214","208964","211586","214092","243159","272624","511145","99287"],Proteobacteria:["122586","192222","198214","208964","211586","214092","243159","272624","283166","511145","85962","882","99287"],Firmicutes:["158878","160490","169963","224308","260799","272623"],Bacteria:["1148","122586","158878","160490","169963","192222","198214","208964","211586","214092","224308","226186","243159","260799","267671","272623","272624","283166","449447","511145","546414","722438","83332","85962","882","99287"],Archaea:["64091","593117"],Eukaryotes:["10090","10116","214684","3055","353153","3702","39947","44689","4081","4577","4896","4932","5061","5691","5833","6239","7165","7227","7460","7955","8364","9031","9598","9606","9615","9796","9823","9913"],Apicomplexa:["5833"],Kinetoplasts:["353153","5691"],Fungi:["4932","4896","5061","214684"],Plants:["3055","3702","39947","4081","4577"],Animals:["9606","10090","10116","9615","9913","9823","9796","9031","8364","7955","6239","7227","7460","7165","9598"],Invertebrates:["6239","7227","7460","7165"],Vertebrates:["9606","10090","10116","9615","9913","9796","9823","9031","8364","7955","9598"],Mammals:["9606","9598","9615","9796","9823","9913","10090","10116"],Birds:["9031"],Amphibians:["8364"],Fish:["7955"]},c={};Object.keys(o).forEach(function(e){c[o[e]]=e}),a["a"]={SPECIES_SHORT_NAMES:Object.freeze(n),SPECIES_FULL_NAMES_SORTED:Object.freeze(i),SPECIES_FULL_NAMES:Object.freeze(o),SPECIES_NAMES_IDS:Object.freeze(c),SPECIES_SYNONYMS:Object.freeze(r),CLADOGRAM:Object.freeze(s)}},"1b21":function(e,a,t){"use strict";var n=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("input",{staticClass:"typeahead",attrs:{type:"text",placeholder:e.placeholder}})},i=[],o=t("5440"),r=o["a"],s=(t("4d06"),t("2877")),c=Object(s["a"])(r,n,i,!1,null,null,null);a["a"]=c.exports},"1b7c":function(e,a,t){"use strict";var n=t("9d33"),i=t.n(n);i.a},2377:function(e,a,t){},"23d1":function(e,a,t){},"2a3d":function(e,a,t){"use strict";(function(e){t("6ffb"),t("6ea2"),t("ad7f"),t("baee");a["a"]={name:"DataOverview.vue",mounted:function(){var a=e(this.$refs.table);a.footable(),a.removeClass("hidden")}}}).call(this,t("9537"))},"2f39":function(e,a,t){"use strict";t.r(a);var n=t("967e"),i=t.n(n),o=(t("96cf"),t("fa84")),r=t.n(o),s=(t("e54f"),t("62f2"),t("7e6d"),t("2b0e")),c=t("b05d"),l=t("cb32"),d=t("9c40"),u=t("58ea"),b=t("07d0"),p=t("9898"),m=t("0016"),_=t("74f7"),g=t("27f9"),h=t("6ab5"),f=t("e208"),v=t("033f"),y=t("4d5a"),C=t("497d"),S=t("2ea3"),P=t("c7a0"),w=t("7867"),x=t("ddd8"),A=t("cf57"),I=t("7460"),E=t("429b"),M=t("65c6"),k=t("6ac5"),L=t("714f"),H=t("2a19"),O=t("9c64");s["a"].use(c["a"],{config:{},components:{QAvatar:l["a"],QBtn:d["a"],QCircularProgress:u["a"],QFooter:b["a"],QHeader:p["a"],QIcon:m["a"],QInnerLoading:_["a"],QInput:g["a"],QItem:h["a"],QItemLabel:f["a"],QItemSection:v["a"],QLayout:y["a"],QList:C["a"],QPage:S["a"],QPageContainer:P["a"],QRouteTab:w["a"],QSelect:x["a"],QSpinnerGears:A["a"],QTab:I["a"],QTabs:E["a"],QToolbar:M["a"],QToolbarTitle:k["a"]},directives:{Ripple:L["a"]},plugins:{Notify:H["a"],Meta:O["a"]}});var N=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("div",{attrs:{id:"q-app"}},[t("router-view")],1)},R=[],G=(t("ad09"),t("ef24")),T=t.n(G);s["a"].use(T.a,{accessToken:"e6478f65192445f39d8c27fc7d7a3518",captureUncaught:!0,captureUnhandledRejections:!0,enabled:!0,source_map_enabled:!0,environment:"production",payload:{client:{javascript:{code_version:"1.0"}}}}),window.Vue=s["a"],s["a"].config.errorHandler=function(e,a,t){console.error(e),s["a"].rollbar.error(e)};var D={name:"App"};ga("create","UA-15579211-1","auto"),ga("require","eventTracker"),ga("require","outboundLinkTracker"),ga("require","maxScrollTracker"),ga("require","pageVisibilityTracker"),ga("require","urlChangeTracker"),ga("send","pageview");var B=D,W=t("2877"),U=Object(W["a"])(B,N,R,!1,null,null,null),K=U.exports,F=t("8c4f"),q=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-layout",{attrs:{view:"hhh lpR fff"}},[t("q-header",{staticClass:"bg-grey-2 text-grey-7",attrs:{elevated:""}},[t("paxdb-header"),t("search-box")],1),t("q-page-container",[t("keep-alive",{attrs:{include:"HomePage,DownloadsPage,HelpPage,LicensePage,AboutPage"}},[t("router-view")],1)],1),t("q-footer",{staticClass:"bg-grey-2 text-grey-8",attrs:{elevated:""}},[t("div",{staticClass:"column"},[t("div",{staticClass:"col q-pa-sm text-body1 text-center text-weight-light"},[t("span",[e._v("© 2009– PaxDb team")]),t("span",{staticClass:"q-px-xs"},[e._v("·")]),t("span",[t("router-link",{attrs:{to:"/license"}},[e._v("License")])],1),t("span",{staticClass:"q-px-xs"},[e._v("·")]),t("span",[t("router-link",{attrs:{to:"/contact"}},[e._v("Contact")])],1),t("span",{staticClass:"q-px-xs"},[e._v("·")]),t("span",[t("router-link",{attrs:{to:"/about"}},[e._v("About")])],1)]),t("div",{staticClass:"row justify-center items-stretch"},[t("div",{staticClass:"q-pa-sm"},[t("a",{attrs:{href:"https://sib.swiss/",target:"_blank"}},[t("img",{attrs:{alt:"SIB logo",src:"statics/logo_sib.png"}})])]),t("div",{staticClass:"q-pa-sm"},[t("a",{attrs:{href:"https://www.uzh.ch",target:"_blank"}},[t("object",{staticStyle:{"vertical-align":"middle"},attrs:{data:"statics/uzh_logo_d_pos.svg",type:"image/svg+xml",height:"37"}},[t("img",{attrs:{alt:"Universität Zürich",height:"37",src:"statics/uzh_logo_d_pos_web_main.jpg"}})]),t("span",{staticStyle:{speak:"normal",position:"absolute","font-size":"0",visibility:"hidden"}},[e._v("Universität Zürich")])])])])])])],1)},z=[],j=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("div",{staticClass:"row bg-white q-pa-md",attrs:{id:"search_container"}},[t("div",{staticClass:"q-pb-sm col-xs-12 col-sm-6 col-md-4 col-4 justify-right "},[t("q-toolbar",{staticClass:"rounded-borders"},[t("div",{staticClass:"q-field  items-start full-width full-height q-field--standout q-field--with-bottom q-field--error",attrs:{required:"required"}},[t("div",{staticClass:"q-field__inner relative-position col self-stretch column justify-center"},[t("div",{staticClass:"q-field__control relative-position row no-wrap text-black bg-white"},[e._m(0),t("q-btn",{attrs:{name:"clear","text-color":"grey",icon:"clear"},on:{click:e.resetSpecies}})],1)])])])],1),t("div",{staticClass:"col-xs-12 col-sm-6 col-md-8 col-8 justify-left"},[t("q-toolbar",[t("q-input",{ref:"proteins",staticClass:"full-width ",attrs:{required:"",debounce:"300",autogrow:"",square:"",outlined:"",label:"protein(s) id/name",hint:"ctrl+enter to search",rules:[e.organismSelectedRule,e.maximumIdsRule]},on:{keydown:function(a){return!a.type.indexOf("key")&&e._k(a.keyCode,"enter",13,a.key,"Enter")?null:a.ctrlKey?e.doSearch(a):null}},scopedSlots:e._u([{key:"prepend",fn:function(){return[""!==e.proteins?t("q-icon",{staticClass:"cursor-pointer",attrs:{name:"clear"},on:{click:e.resetProteins}}):e._e()]},proxy:!0},{key:"append",fn:function(){return[""!==e.proteins?t("q-icon",{staticClass:"cursor-pointer",attrs:{name:"search"},on:{click:e.doSearch}}):e._e()]},proxy:!0}]),model:{value:e.proteins,callback:function(a){e.proteins=a},expression:"proteins"}})],1)],1)])},Q=[function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("organism-selectbox",{ref:"speciesList",staticClass:"q-field full-width full-height",attrs:{"bg-color":"white",color:"black",placeholder:"All Organisms",standout:""},on:{selected:e.organismSelectedHandler}})}],Y=(t("4917"),t("1b21")),J={data:function(){return{speciesId:"",proteins:""}},components:{"organism-selectbox":Y["a"]},methods:{doSearch:function(){this.$router.push({path:"/search",query:{q:this.proteins,speciesId:this.speciesId}})},resetProteins:function(){this.$refs.proteins.resetValidation(),this.proteins=""},resetSpecies:function(){this.$refs.speciesList.reset(),this.speciesId=""},organismSelectedHandler:function(e){this.speciesId=e},isMultipleProteinsInput:function(e){return e.indexOf(",")>0||e.trim().indexOf("\n")>0||e.trim().indexOf("\r")>0},organismSelectedRule:function(e){if(!this.speciesId)return this.isMultipleProteinsInput(e)?"To search for multiple proteins, please first select species":void 0},maximumIdsRule:function(e){var a=(e.trim().match(/,|\n|\r\n/g)||[]).length;if(a>100)return"max 100 ids allowed (".concat(a,")")}}},V=J,X=(t("9e35"),Object(W["a"])(V,j,Q,!1,null,null,null)),$=X.exports,Z=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-toolbar",{staticClass:"q-py-md"},[t("q-btn",{attrs:{flat:"",to:"/"}},[t("img",{attrs:{src:"statics/paxdb_logo.png"}})]),t("q-toolbar-title",[t("span",{staticClass:"gt-sm"},[e._v("\n      PAXdb: Protein Abundance Database\n    ")])]),t("q-tabs",{attrs:{align:"right"}},[t("q-route-tab",{staticClass:"desktop-only gt-xs",attrs:{to:"/download",label:"Download"}}),t("q-route-tab",{attrs:{to:"/help",label:"Help"}}),t("q-route-tab",{attrs:{to:"/about",label:"About"}})],1)],1)},ee=[],ae={name:"Header"},te=ae,ne=Object(W["a"])(te,Z,ee,!1,null,"589710f1",null),ie=ne.exports,oe={data:function(){return{}},components:{SearchBox:$,PaxdbHeader:ie}},re=oe,se=Object(W["a"])(re,q,z,!1,null,"3e7e0fe9",null),ce=se.exports,le=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-page",[t("div",{staticClass:"row q-px-lg q-mx-auto justify-center",staticStyle:{"max-width":"2000px"}},[t("div",{staticClass:"col-12"},[t("h1",[e._v("Browse species")])]),t("div",{staticClass:"col-12"},[t("div",{staticClass:"row"},[t("div",{staticClass:"q-px-sm col-xs-12 col-sm-12 col-md-3 col-lg-3"},[t("div",{attrs:{id:"cladogram"}},[t("tree-of-life-menu",{staticClass:"full-width",on:{selectedSpecies:e.cladeSelectedHandler}})],1)]),t("div",{staticClass:"col-xs-12 col-sm-12 col-md-9 col-lg-9"},[t("div",{staticClass:"row",staticStyle:{"overflow-x":"auto"}},[t("div",{staticClass:"row no-wrap items-end text-center"},e._l(e.firstRowOrganisms,function(a){return t("div",{key:a},[t("figure",[t("router-link",{attrs:{to:"/species/"+a}},[t("img",{attrs:{width:"150px",src:"statics/face/"+a+".jpg"}})])],1),t("figcaption",[t("router-link",{attrs:{to:"/species/"+a}},[t("span",[e._v(e._s(e.organismNames[a]))])])],1)])}),0),t("div",{staticClass:"row no-wrap items-end text-center"},e._l(e.secondRowOrganisms,function(a){return t("div",{key:a},[t("figure",[t("router-link",{attrs:{to:"/species/"+a}},[t("img",{attrs:{width:"150px",src:"statics/face/"+a+".jpg"}})])],1),t("figcaption",[t("router-link",{attrs:{to:"/species/"+a}},[t("span",[e._v(e._s(e.organismNames[a]))])])],1)])}),0)])])])])]),t("div",{staticClass:"row q-px-lg q-mx-auto justify-center",staticStyle:{"max-width":"2000px"}},[t("data-overview",{staticClass:"full-width"})],1)])},de=[],ue=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("ul",{ref:"drilldown",on:{click:function(a){return a.preventDefault(),e.onMenuItemSelected(a)}}},[e._m(0),e._m(1),e._m(2),e._m(3)])},be=[function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("li",[t("a",{attrs:{href:"#"}},[e._v("all organisms (56)")])])},function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("li",[t("a",{attrs:{href:"#"}},[e._v("Bacteria (26)")]),t("ul",[t("li",[t("a",{attrs:{href:"#"}},[e._v("All Bacteria (26)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Cyanobacteria (2)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Proteobacteria (13)")]),t("ul",[t("li",[t("a",{attrs:{href:"#"}},[e._v("All Proteobacteria (13)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Deproteobacteria (3)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Gammaproteobacteria (8)")])])])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Firmicutes (6)")])])])])},function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("li",[t("a",{attrs:{href:"#"}},[e._v("Archaea (2)")])])},function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("li",[t("a",{attrs:{href:"#"}},[e._v("Eukaryotes (28)")]),t("ul",[t("li",[t("a",{attrs:{href:"#"}},[e._v("All Eukaryotes (28)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Animals (15)")]),t("ul",[t("li",[t("a",{attrs:{href:"#"}},[e._v("All Animals (15)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Vertebrates (11)")]),t("ul",[t("li",[t("a",{attrs:{href:"#"}},[e._v("All Vertebrates (11)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Mammals (8)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Birds (1)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Amphibians (1)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Fish (1)")])])])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Invertebrates (4)")])])])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Fungi (4)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Plants (5)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Apicomplexa (1)")])]),t("li",[t("a",{attrs:{href:"#"}},[e._v("Kinetoplasts (2)")])])])])}],pe=t("9c5b"),me=pe["a"],_e=(t("1b7c"),Object(W["a"])(me,ue,be,!1,null,null,null)),ge=_e.exports,he=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("div",{staticClass:"row q-pd-lg q-mx-auto"},[e._m(0),t("div",{staticClass:"col-12",staticStyle:{"max-width":"1500px"}},[t("table",{ref:"table",staticClass:"footable toggle-large toggle-arrow-tiny table hidden",attrs:{"data-sorting":"true","data-paging":"true","data-paging-limit":"6","data-filtering":"true","data-filter-delay":"50","data-filter-placeholder":"species name","data-filter-position":"left"}},[e._m(1),t("tbody",[t("tr",[t("td",{attrs:{"data-sort-value":"Desulfovibrio vulgaris Hildenborough","data-filter-value":"Desulfovibrio vulgaris Hildenborough"}},[t("router-link",{attrs:{to:"/species/882"}},[e._v("Desulfovibrio vulgaris\n            str. Hildenborough\n          ")])],1),t("td",[e._v("3380")]),t("td",[e._v("5")]),t("td",{attrs:{"data-sort-value":"28"}},[e._v("28%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Synechocystis PCC 6803","data-filter-value":"Synechocystis sp. PCC 6803"}},[t("router-link",{attrs:{to:"/species/1148"}},[e._v("Synechocystis sp. PCC 6803")])],1),t("td",[e._v("3171")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"37"}},[e._v("37%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Chlamydomonas reinhardtii","data-filter-value":"Chlamydomonas reinhardtii"}},[t("router-link",{attrs:{to:"/species/3055"}},[e._v("Chlamydomonas reinhardtii")])],1),t("td",[e._v("14416")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"35"}},[e._v("35%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Arabidopsis thaliana","data-filter-value":"Arabidopsis thaliana"}},[t("router-link",{attrs:{to:"/species/3702"}},[e._v("Arabidopsis\n            thaliana\n          ")])],1),t("td",[e._v("27416")]),t("td",[e._v("46")]),t("td",{attrs:{"data-sort-value":"76"}},[e._v("76%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Solanum lycopersicum","data-filter-value":"Solanum lycopersicum"}},[t("router-link",{attrs:{to:"/species/4081"}},[e._v("Solanum\n            lycopersicum\n          ")])],1),t("td",[e._v("34675")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"6"}},[e._v("6%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Zea mays corn","data-filter-value":"Zea mays corn"}},[t("router-link",{attrs:{to:"/species/4577"}},[e._v("Zea mays")])],1),t("td",[e._v("92413")]),t("td",[e._v("7")]),t("td",{attrs:{"data-sort-value":"18"}},[e._v("18%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Schizosaccharomyces pombe","data-filter-value":"Schizosaccharomyces pombe"}},[t("router-link",{attrs:{to:"/species/4896"}},[e._v("Schizosaccharomyces pombe")])],1),t("td",[e._v("5144")]),t("td",[e._v("8")]),t("td",{attrs:{"data-sort-value":"90"}},[e._v("90%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Saccharomyces cerevisiae yeast","data-filter-value":"Saccharomyces cerevisiae yeast"}},[t("router-link",{attrs:{to:"/species/4932"}},[e._v("Saccharomyces cerevisiae")])],1),t("td",[e._v("6692")]),t("td",[e._v("17")]),t("td",{attrs:{"data-sort-value":"96"}},[e._v("96%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Aspergillus niger","data-filter-value":"Aspergillus niger"}},[t("router-link",{attrs:{to:"/species/5061"}},[e._v("Aspergillus\n            niger\n          ")])],1),t("td",[e._v("14068")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"5"}},[e._v("5%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Trypanosoma brucei","data-filter-value":"Trypanosoma brucei"}},[t("router-link",{attrs:{to:"/species/5691"}},[e._v("Trypanosoma\n            brucei\n          ")])],1),t("td",[e._v("8747")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"65"}},[e._v("65%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Plasmodium falciparum","data-filter-value":"Plasmodium falciparum"}},[t("router-link",{attrs:{to:"/species/5833"}},[e._v("Plasmodium\n            falciparum\n          ")])],1),t("td",[e._v("5429")]),t("td",[e._v("5")]),t("td",{attrs:{"data-sort-value":"94"}},[e._v("94%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Caenorhabditis elegans","data-filter-value":"Caenorhabditis elegans"}},[t("router-link",{attrs:{to:"/species/6239"}},[e._v("Caenorhabditis elegans")])],1),t("td",[e._v("20517")]),t("td",[e._v("10")]),t("td",{attrs:{"data-sort-value":"60"}},[e._v("60%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Anopheles gambiae mosquito","data-filter-value":"Anopheles gambiae mosquito"}},[t("router-link",{attrs:{to:"/species/7165"}},[e._v("Anopheles gambiae")])],1),t("td",[e._v("12810")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"33"}},[e._v("33%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Drosophila melanogaster fruitfly","data-filter-value":"Drosophila melanogaster fruitfly"}},[t("router-link",{attrs:{to:"/species/7227"}},[e._v("Drosophila melanogaster")])],1),t("td",[e._v("13937")]),t("td",[e._v("10")]),t("td",{attrs:{"data-sort-value":"95"}},[e._v("95%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Apis mellifera honeybee","data-filter-value":"Apis mellifera honeybee"}},[t("router-link",{attrs:{to:"/species/7460"}},[e._v("Apis mellifera")])],1),t("td",[e._v("10694")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"31"}},[e._v("31%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Danio rerio zebrafish","data-filter-value":"Danio rerio zebrafish"}},[t("router-link",{attrs:{to:"/species/7955"}},[e._v("Danio\n            rerio\n          ")])],1),t("td",[e._v("26163")]),t("td",[e._v("20")]),t("td",{attrs:{"data-sort-value":"59"}},[e._v("59%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Xenopus tropicalis frog","data-filter-value":"Xenopus tropicalis frog"}},[t("router-link",{attrs:{to:"/species/8364"}},[e._v("Xenopus (Silurana) tropicalis")])],1),t("td",[e._v("18429")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"10"}},[e._v("10%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Gallus gallus chicken","data-filter-value":"Gallus gallus chicken"}},[t("router-link",{attrs:{to:"/species/9031"}},[e._v("Gallus\n            gallus\n          ")])],1),t("td",[e._v("16736")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"73"}},[e._v("73%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Pan troglodytes chimpanzee","data-filter-value":"Pan troglodytes chimpanzee"}},[t("router-link",{attrs:{to:"/species/9598"}},[e._v("Pan troglodytes")])],1),t("td",[e._v("18759")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"15"}},[e._v("15%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Homo sapiens human","data-filter-value":"Homo sapiens human"}},[t("router-link",{attrs:{to:"/species/9606"}},[e._v("Homo\n            sapiens\n          ")])],1),t("td",[e._v("20457")]),t("td",[e._v("170")]),t("td",{attrs:{"data-sort-value":"98"}},[e._v("98%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Canis lupus familiaris dog","data-filter-value":"Canis lupus familiaris dog"}},[t("router-link",{attrs:{to:"/species/9615"}},[e._v("Canis lupus familiaris")])],1),t("td",[e._v("19856")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"21"}},[e._v("21%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Equus caballus horse","data-filter-value":"Equus caballus horse"}},[t("router-link",{attrs:{to:"/species/9796"}},[e._v("Equus\n            caballus\n          ")])],1),t("td",[e._v("20449")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"15"}},[e._v("15%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Sus scrofa pig","data-filter-value":"Sus scrofa pig"}},[t("router-link",{attrs:{to:"/species/9823"}},[e._v("Sus scrofa")])],1),t("td",[e._v("21630")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"14"}},[e._v("14%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Bos taurus cow","data-filter-value":"Bos taurus cow"}},[t("router-link",{attrs:{to:"/species/9913"}},[e._v("Bos taurus")])],1),t("td",[e._v("19994")]),t("td",[e._v("5")]),t("td",{attrs:{"data-sort-value":"54"}},[e._v("54%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Mus musculus mouse","data-filter-value":"Mus musculus mouse"}},[t("router-link",{attrs:{to:"/species/10090"}},[e._v("Mus\n            musculus\n          ")])],1),t("td",[e._v("22668")]),t("td",[e._v("75")]),t("td",{attrs:{"data-sort-value":"90"}},[e._v("90%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Rattus norvegicus","data-filter-value":"Rattus norvegicus"}},[t("router-link",{attrs:{to:"/species/10116"}},[e._v("Rattus\n            norvegicus\n          ")])],1),t("td",[e._v("22941")]),t("td",[e._v("5")]),t("td",{attrs:{"data-sort-value":"74"}},[e._v("74%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Oryza sativa Japonica rice","data-filter-value":"Oryza sativa Japonica rice"}},[t("router-link",{attrs:{to:"/species/39947"}},[e._v("Oryza sativa Japonica Group")])],1),t("td",[e._v("57939")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"9"}},[e._v("9%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Dictyostelium discoideum","data-filter-value":"Dictyostelium discoideum"}},[t("router-link",{attrs:{to:"/species/44689"}},[e._v("Dictyostelium discoideum")])],1),t("td",[e._v("13212")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"32"}},[e._v("32%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Halobacterium NRC-1","data-filter-value":"Halobacterium NRC-1"}},[t("router-link",{attrs:{to:"/species/64091"}},[e._v("Halobacterium\n            sp. NRC-1\n          ")])],1),t("td",[e._v("2075")]),t("td",[e._v("4")]),t("td",{attrs:{"data-sort-value":"70"}},[e._v("70%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Mycobacterium tuberculosis H37Rv","data-filter-value":"Mycobacterium tuberculosis H37Rv"}},[t("router-link",{attrs:{to:"/species/83332"}},[e._v("Mycobacterium tuberculosis H37Rv")])],1),t("td",[e._v("4003")]),t("td",[e._v("7")]),t("td",{attrs:{"data-sort-value":"84"}},[e._v("84%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Helicobacter pylori ","data-filter-value":"Helicobacter pylori "}},[t("router-link",{attrs:{to:"/species/85962"}},[e._v("Helicobacter\n            pylori 26695\n          ")])],1),t("td",[e._v("1573")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"98"}},[e._v("98%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Salmonella enterica Typhimurium LT2","data-filter-value":"Salmonella enterica Typhimurium LT2"}},[t("router-link",{attrs:{to:"/species/99287"}},[e._v("Salmonella enterica (Typhimurium str. LT2)")])],1),t("td",[e._v("4423")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"59"}},[e._v("59%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Neisseria meningitidis MC58","data-filter-value":"Neisseria meningitidis MC58"}},[t("router-link",{attrs:{to:"/species/122586"}},[e._v("Neisseria meningitidis MC58")])],1),t("td",[e._v("2063")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"19"}},[e._v("19%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Staphylococcus aureus Mu50","data-filter-value":"Staphylococcus aureus Mu50"}},[t("router-link",{attrs:{to:"/species/158878"}},[e._v("Staphylococcus aureus subsp. aureus Mu50")])],1),t("td",[e._v("2730")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"62"}},[e._v("62%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Streptococcus pyogenes M1 GAS","data-filter-value":"Streptococcus pyogenes M1 GAS"}},[t("router-link",{attrs:{to:"/species/160490"}},[e._v("Streptococcus pyogenes M1 GAS")])],1),t("td",[e._v("1696")]),t("td",[e._v("5")]),t("td",{attrs:{"data-sort-value":"75"}},[e._v("75%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Listeria monocytogenes EGD-e","data-filter-value":"Listeria monocytogenes EGD-e"}},[t("router-link",{attrs:{to:"/species/169963"}},[e._v("Listeria monocytogenes EGD-e")])],1),t("td",[e._v("2846")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"16"}},[e._v("16%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Campylobacter jejuni  NCTC ","data-filter-value":"Campylobacter jejuni  NCTC "}},[t("router-link",{attrs:{to:"/species/192222"}},[e._v("Campylobacter jejuni subsp. jejuni NCTC 11168")])],1),t("td",[e._v("1623")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"47"}},[e._v("47%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Shigella flexneri","data-filter-value":"Shigella flexneri"}},[t("router-link",{attrs:{to:"/species/198214"}},[e._v("Shigella\n            flexneri 2a str. 301\n          ")])],1),t("td",[e._v("4439")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"39"}},[e._v("39%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Pseudomonas aeruginosa PAO1","data-filter-value":"Pseudomonas aeruginosa PAO1"}},[t("router-link",{attrs:{to:"/species/208964"}},[e._v("Pseudomonas aeruginosa PAO1")])],1),t("td",[e._v("5571")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"43"}},[e._v("43%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Shewanella oneidensis MR-1","data-filter-value":"Shewanella oneidensis MR-1"}},[t("router-link",{attrs:{to:"/species/211586"}},[e._v("Shewanella oneidensis MR-1")])],1),t("td",[e._v("4318")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"30"}},[e._v("30%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Yersinia pestis CO92","data-filter-value":"Yersinia pestis CO92"}},[t("router-link",{attrs:{to:"/species/214092"}},[e._v("Yersinia\n            pestis CO92\n          ")])],1),t("td",[e._v("3885")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"30"}},[e._v("30%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Cryptococcus neoformans JEC21","data-filter-value":"Cryptococcus neoformans JEC21"}},[t("router-link",{attrs:{to:"/species/214684"}},[e._v("Cryptococcus neoformans var. neoformans JEC21")])],1),t("td",[e._v("6594")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"37"}},[e._v("37%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Bacillus subtilis","data-filter-value":"Bacillus subtilis"}},[t("router-link",{attrs:{to:"/species/224308"}},[e._v("Bacillus\n            subtilis subsp. subtilis str. 168\n          ")])],1),t("td",[e._v("4280")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"92"}},[e._v("92%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Bacteroides thetaiotaomicron VPI-5482","data-filter-value":"Bacteroides thetaiotaomicron VPI-5482"}},[t("router-link",{attrs:{to:"/species/226186"}},[e._v("Bacteroides\n            thetaiotaomicron VPI-5482\n          ")])],1),t("td",[e._v("4778")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"16"}},[e._v("16%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Acidithiobacillus ferrooxidans ATCC","data-filter-value":"Acidithiobacillus ferrooxidans ATCC"}},[t("router-link",{attrs:{to:"/species/243159"}},[e._v("Acidithiobacillus\n            ferrooxidans ATCC 23270\n          ")])],1),t("td",[e._v("3147")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"41"}},[e._v("41%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Bacillus anthracis Sterne","data-filter-value":"Bacillus anthracis Sterne"}},[t("router-link",{attrs:{to:"/species/260799"}},[e._v("Bacillus anthracis str. Sterne")])],1),t("td",[e._v("5289")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"26"}},[e._v("26%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Leptospira interrogans Copenhageni  Fiocruz","data-filter-value":"Leptospira interrogans Copenhageni  Fiocruz"}},[t("router-link",{attrs:{to:"/species/267671"}},[e._v("Leptospira\n            interrogans serovar Copenhageni str. Fiocruz L1-130\n          ")])],1),t("td",[e._v("3667")]),t("td",[e._v("5")]),t("td",{attrs:{"data-sort-value":"66"}},[e._v("66%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Lactococcus lactis Il1403","data-filter-value":"Lactococcus lactis Il1403"}},[t("router-link",{attrs:{to:"/species/272623"}},[e._v("Lactococcus lactis subsp. lactis Il1403")])],1),t("td",[e._v("2321")]),t("td",[e._v("3")]),t("td",{attrs:{"data-sort-value":"56"}},[e._v("56%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Legionella pneumophila Philadelphia","data-filter-value":"Legionella pneumophila Philadelphia"}},[t("router-link",{attrs:{to:"/species/272624"}},[e._v("Legionella pneumophila\n            subsp. pneumophila str. Philadelphia 1\n          ")])],1),t("td",[e._v("2943")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"25"}},[e._v("25%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Bartonella henselae  Houston","data-filter-value":"Bartonella henselae  Houston"}},[t("router-link",{attrs:{to:"/species/283166"}},[e._v("Bartonella henselae str. Houston-1")])],1),t("td",[e._v("1488")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"85"}},[e._v("85%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Trypanosoma cruzi  Brener","data-filter-value":"Trypanosoma cruzi  Brener"}},[t("router-link",{attrs:{to:"/species/353153"}},[e._v("Trypanosoma cruzi strain CL Brener")])],1),t("td",[e._v("19602")]),t("td",[e._v("6")]),t("td",{attrs:{"data-sort-value":"23"}},[e._v("23%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Microcystis aeruginosa NIES-843","data-filter-value":"Microcystis aeruginosa NIES-843"}},[t("router-link",{attrs:{to:"/species/449447"}},[e._v("Microcystis aeruginosa NIES-843")])],1),t("td",[e._v("6312")]),t("td",[e._v("7")]),t("td",{attrs:{"data-sort-value":"79"}},[e._v("79%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Escherichia coli  K-12  MG1655","data-filter-value":"Escherichia coli  K-12  MG1655"}},[t("router-link",{attrs:{to:"/species/511145"}},[e._v("Escherichia coli str. K-12 substr. MG1655")])],1),t("td",[e._v("4146")]),t("td",[e._v("18")]),t("td",{attrs:{"data-sort-value":"98"}},[e._v("98%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Deinococcus deserti VCD115","data-filter-value":"Deinococcus deserti VCD115"}},[t("router-link",{attrs:{to:"/species/546414"}},[e._v("Deinococcus deserti VCD115")])],1),t("td",[e._v("2597")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"51"}},[e._v("51%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Thermococcus gammatolerans EJ3","data-filter-value":"Thermococcus gammatolerans EJ3"}},[t("router-link",{attrs:{to:"/species/593117"}},[e._v("Thermococcus gammatolerans EJ3")])],1),t("td",[e._v("2156")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"62"}},[e._v("62%")])]),t("tr",[t("td",{attrs:{"data-sort-value":"Mycoplasma pneumoniae FH","data-filter-value":"Mycoplasma pneumoniae FH"}},[t("router-link",{attrs:{to:"/species/722438"}},[e._v("Mycoplasma pneumoniae FH")])],1),t("td",[e._v("629")]),t("td",[e._v("1")]),t("td",{attrs:{"data-sort-value":"60"}},[e._v("60%")])])])])])])},fe=[function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("div",{staticClass:"col-12"},[t("h1",[e._v("Data overview")])])},function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("thead",{staticClass:"text-nowrap"},[t("tr",[t("th",{staticClass:"footable-first-column",attrs:{"data-type":"html"}},[e._v("\n          Species\n        ")]),t("th",{staticClass:"text-center-important",staticStyle:{display:"none"},attrs:{"data-breakpoints":"xs","data-type":"number","data-filterable":"false","data-classes":"text-center-important"}},[e._v("\n          Predicted proteome size\n        ")]),t("th",{staticClass:"text-center-important",staticStyle:{"text-align":"center",display:"none"},attrs:{"data-breakpoints":"xs","data-sorted":"true","data-direction":"DESC","data-filterable":"false","data-classes":"text-center-important","data-type":"number"}},[e._v("\n          Datasets\n        ")]),t("th",{staticClass:"text-center-important",staticStyle:{"text-align":"center"},attrs:{"data-type":"text","data-filterable":"false","data-classes":"text-center-important"}},[e._v("\n          Proteins Covered\n        ")])])])}],ve=t("2a3d"),ye=ve["a"],Ce=(t("ef6d"),Object(W["a"])(ye,he,fe,!1,null,"3d7ab2b1",null)),Se=Ce.exports,Pe=t("16b0"),we={name:"HomePage",data:function(){return{firstRowOrganisms:[],secondRowOrganisms:[],organismNames:Pe["a"].SPECIES_SHORT_NAMES}},meta:{title:"PAXdb: Protein Abundance Database"},components:{TreeOfLifeMenu:ge,DataOverview:Se},created:function(){console.log("home page created")},beforeDestroy:function(){console.log("home page destroyed")},methods:{cladeSelectedHandler:function(e){if(e.length<5)return this.firstRowOrganisms=e,void(this.secondRowOrganisms=[]);var a=Math.ceil(e.length/2);this.firstRowOrganisms=e.slice(0,a),this.secondRowOrganisms=e.slice(a)}}},xe=we,Ae=Object(W["a"])(xe,le,de,!1,null,null,null),Ie=Ae.exports,Ee=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-page",{attrs:{padding:""}},[t("div",{staticClass:"column q-pa-sm q-mx-auto",staticStyle:{"max-width":"1500px"}},[t("div",{staticClass:"row q-mb-lg justify-around"},[t("div",{staticClass:"text-center col-sm-12 col-md-5"},[t("h1",[e._v(e._s(e.species.name))]),t("img",{staticClass:"img-responsive ",attrs:{src:"/statics/species/"+e.species.id+".jpg",alt:e.species.name}})]),t("div",{staticClass:"text-center col-sm-12 col-md-7"},[t("h2",[e._v("Protein Abundance Histogram")]),t("small",[t("router-link",{attrs:{to:"/dataset/"+e.species.id+"/"+e.defaultDataset.id}},[e._v("\n            "+e._s(e.defaultDataset.name)+"\n          ")])],1),t("img",{staticClass:"img-responsive",attrs:{src:"http://api.pax-db.org/dataset/"+e.defaultDataset.id+"/histogram",alt:e.species.name+" abundance histogram"}}),t("label",[e._v("protein abundance (ppm)")])])]),t("div",{staticClass:"row q-gutter-lg justify-around",attrs:{id:"basic_info_container"}},[t("div",[t("h2",[e._v("Basic info")]),t("table",{staticClass:"table"},[t("tbody",[t("tr",[t("td",[e._v("Genome source:")]),t("td",{staticClass:"text-center"},[t("span",{domProps:{innerHTML:e._s(e.species.genome_source)}}),e.species.genome_source_version.indexOf("not available")<0?t("p",{staticClass:"text-muted"},[e._v("\n                ("+e._s(e.species.genome_source_version)+")")]):e._e()])]),t("tr",[t("td",[e._v("Proteome size:")]),t("td",[e._v(e._s(e.species.num_proteins))])]),t("tr",[t("td",[e._v("Nr. of datasets:")]),t("td",[e._v(e._s(e.species.datasets.length))])]),t("tr",[t("td",[e._v("Integrated set coverage:")]),t("td",[e._v(e._s(Math.round(e.species.total_coverage))+"%")])]),t("tr",[t("td",[e._v("Average set coverage:")]),t("td",[e._v(e._s(Math.round(e.species.average_coverage))+"%")])])])])]),t("div",{},[t("h2",[e._v("Most Abundant Proteins")]),"string"===typeof e.species.topProteins?t("p",{staticClass:"bg-danger text-center"},[e._v(e._s(e.species.topProteins))]):t("table",{staticClass:"table text-nowrap center-block",staticStyle:{"max-width":"350px"}},[t("tbody",e._l(e.species.topProteins,function(a,n){return t("tr",{key:a.id},[t("td",[t("router-link",{attrs:{to:"/protein/"+a.id}},[e._v(e._s(n+1)+". "+e._s(a.name))])],1),t("td",{staticClass:"text-right"},[e._v(e._s(a.abundance)+" [ppm]")])])}),0)])])]),t("div",{attrs:{id:"dataset_overview"}},[t("h3",[e._v("Datasets")]),t("table",{ref:"dataset_overview_table",staticClass:"footable toggle-large toggle-arrow-tiny table hidden",attrs:{"data-sorting":"true","data-paging":e.species.datasets.length>10,"data-filtering":e.species.datasets.length>3,"data-filter-delay":"50","data-filter-placeholder":"dataset name","data-filter-position":"left"}},[t("thead",{staticClass:"text-nowrap"},[t("tr",[t("th",{staticClass:"footable-first-column",attrs:{"data-type":"html"}},[e._v("\n            Name\n          ")]),t("th",{staticClass:"text-center",attrs:{"data-type":"number","data-classes":"text-center","data-filterable":"false","data-sorted":"true","data-direction":"ASC"}},[e._v("\n            Interaction consistency score\n          ")]),t("th",{staticStyle:{display:"none"},attrs:{"data-type":"text","data-breakpoints":"xs","data-classes":"text-center","data-filterable":"false"}},[e._v("\n            Coverage\n          ")]),t("th",{attrs:{"data-type":"html","data-breakpoints":"xs sm","data-classes":"text-nowrap","data-filterable":"false","data-sortable":"false"}},[e._v("\n            Download\n          ")])])]),t("tbody",e._l(e.species.datasets,function(a){return t("tr",{key:a.id},[t("td",{attrs:{"data-sort-value":a.name,"data-filter-value":a.name}},[t("router-link",{attrs:{to:"/dataset/"+e.species.id+"/"+a.id+"/"}},[e._v(e._s(a.name))])],1),t("td",[e._v("\n            "+e._s(a.score)+"\n          ")]),t("td",[e._v("\n            "+e._s(a.coverage)+"\n          ")]),t("td",[t("a",{attrs:{href:"/data/abundances/"+e.datasetFilenames[a.id],target:a.name}},[e._v("\n              Download")])])])}),0)])]),t("div",{staticClass:"column",staticStyle:{"margin-bottom":"300px"},attrs:{id:"correlate_species"}},[t("h3",[e._v("Correlation")]),t("div",{staticClass:"row no-wrap text-no-wrap items-center"},[t("label",[e._v(" "+e._s(e.species.name)+" vs  ")]),t("span",{staticClass:"full-width",staticStyle:{"max-width":"350px",height:"38px"}},[e._m(0)],1),t("QBtn",{attrs:{icon:"clear",disable:""==e.correlationTarget,"text-color":"grey"},on:{click:e.resetCorrelation}})],1),""!==e.correlationTarget?t("div",{staticClass:"justify-center q-my-md"},[t("img",{staticClass:"q-my-lg",attrs:{src:e.correlationPlot}})]):e._e()])])])},Me=[function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("organism-selectbox",{ref:"speciesList",staticClass:"full-width full-height",attrs:{"bg-color":"white",color:"black",placeholder:"pick an organism",standout:""},on:{selected:e.organismSelectedHandler}})}],ke=t("967b"),Le=ke["a"],He=Object(W["a"])(Le,Ee,Me,!1,null,null,null),Oe=He.exports,Ne=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-page",{attrs:{padding:""}},[e.isFetching?e._e():t("div",{staticClass:"column q-mx-auto",staticStyle:{"max-width":"1500px"}},[e.isMultipleProteinsSearch?t("div",{staticClass:"q-px-sm"},[e._l(e.resource.hits,function(a){return t("div",{key:a["@query"],staticClass:"q-pb-lg"},[t("div",{staticClass:"bg-teal-2 q-px-sm q-py-sm rounded-borders"},[t("p",{staticClass:"hit-title text-weight-light"},[t("em",[e._v('"'+e._s(a["@query"])+'" in\n              '),t("router-link",{attrs:{to:"/species/"+e.speciesId}},[e._v(e._s(e.speciesNames[e.speciesId]))])],1)]),a["@totalHits"]>10?t("p",[e._v("Showing Showing 1 - 10 of "+e._s(a["@totalHits"])+" matches")]):e._e()]),t("div",{staticClass:"search_results q-py-xs"},[a.protein&&0!==a.protein.length?t("div",e._l(a.protein,function(a,n){return t("div",{key:a["@id"],staticClass:"search_hit"},[t("span",[t("input",{attrs:{type:"checkbox",id:a["@id"],tabindex:1*n},domProps:{value:0===n?"on":"off",checked:0==n}}),t("label",[e._v(e._s(n+1)+". "+e._s(a["@name"]))])]),a.highlighted?t("span",{domProps:{innerHTML:e._s(a.highlighted.replace(/search\_hit/g,"search_hit mark"))}}):e._e()])}),0):t("label",{staticClass:"text-negative text-weight-medium"},[e._v("no data found")])])])}),t("div",{staticClass:"text-center"},[t("QBtn",{staticClass:"q-py-md",staticStyle:{width:"250px"},attrs:{standout:"",disable:this.$el.querySelectorAll("input:checked").length>0,tabIndex:"1",color:"primary",label:"Continue",size:"20px"},on:{click:e.toProteinsPage}})],1)],2):t("div",{},[t("div",{staticClass:"bg-teal-2 q-px-sm q-py-xs"},[t("h3",[t("em",[e._v('"'+e._s(e.resource.hits["@query"])+'"\n            '),e.speciesId?t("span",[e._v("in "),t("router-link",{attrs:{to:"/species/"+e.speciesId}},[e._v(e._s(e.speciesNames[e.speciesId]))])],1):t("span",[e._v("in all species")])])]),e.resource.hits["@totalHits"]>0?t("p",[e._v("Showing "+e._s(e.firstHit)+" - "+e._s(e.lastHit)+"\n          "),e.lastHit<e.totalHits?t("span",[e._v("of "+e._s(e.totalHits))]):e._e(),e._v(" matches")]):e._e()]),t("div",[t("div",{staticClass:"search_results"},[0==e.totalHits?t("p",{staticClass:"text-negative text-weight-medium"},[e._v("no data found")]):e._e(),e.resource.hits.protein?t("div",e._l(e.resource.hits.protein,function(a,n){return t("div",{key:a["@id"],staticClass:"search_hit"},[t("h4",[e._v(e._s(e.firstHit+n)+".\n                "),t("router-link",{attrs:{to:"/protein/"+a["@id"]}},[e._v(e._s(a["@name"]))])],1),e.speciesId?e._e():t("small",[t("router-link",{attrs:{to:"/species/"+a["@speciesId"]}},[e._v(e._s(e.speciesNames[a["@speciesId"]]))])],1),a.highlighted?t("span",{domProps:{innerHTML:e._s(a.highlighted.replace(/search\_hit/g,"mark"))}}):e._e()])}),0):e._e()]),e.totalHits>e.pageSize?t("div",{staticClass:"row justify-between bg-teal-2",attrs:{id:"search_pager"}},[t("div",{staticClass:"q-mx-lg"},[t("nav",[t("ul",{staticClass:"pagination"},[1!=e.currentPage?t("li",[t("router-link",{attrs:{to:e.make_search_url(e.resource,e.resource.hits,-(e.resource.hits["@currentPage"]-1),e.speciesId,e.pageSize)}},[t("span",{attrs:{"aria-hidden":"true"}},[e._v("«")]),t("span",{staticClass:"sr-only"})])],1):e._e(),e.resource.hits["@currentPage"]>1?t("li",[t("router-link",{attrs:{to:e.make_search_url(e.resource,e.resource.hits,-1,e.speciesId,e.pageSize)}},[e._v("\n                    "+e._s(parseInt(e.resource.hits["@currentPage"]-1))+"\n                  ")])],1):e._e(),t("li",{staticClass:"active"},[t("a",{attrs:{href:""}},[e._v(e._s(e.resource.hits["@currentPage"]))])]),e.totalHits>e.pageSize*e.currentPage?t("li",[t("router-link",{attrs:{to:e.make_search_url(e.resource,e.resource.hits,1,e.speciesId,e.pageSize)}},[e._v("\n                    "+e._s(parseInt(e.resource.hits["@currentPage"])+1)+"\n                  ")])],1):e._e(),e.totalHits>e.pageSize*e.currentPage?t("li",[t("router-link",{attrs:{to:e.make_search_url(e.resource,e.resource.hits,+(e.last_page(e.resource.hits)-parseInt(e.resource.hits["@currentPage"])),e.speciesId,e.pageSize)}},[t("span",{attrs:{"aria-hidden":"true"}},[e._v("»")]),t("span",{staticClass:"sr-only"})])],1):e._e()])])]),t("div",{staticClass:"no-wrap q-mx-lg gt-xs"},[t("ul",{staticClass:"pagination"},[t("li",{staticClass:"info"},[t("a",[e._v("Results per page")])]),e._l([10,20,50],function(a){return t("li",{key:a,class:a==e.pageSize?"active":""},[a!=e.pageSize?t("router-link",{attrs:{to:e.make_search_url(e.resource,e.resource.hits,0,e.speciesId,a)}},[e._v("\n                  "+e._s(a)+"\n                ")]):t("span",[e._v(e._s(a))])],1)})],2)])]):e._e()])])])])},Re=[],Ge=t("f3e3"),Te=t.n(Ge),De=(t("5df3"),t("1c4c"),t("f508")),Be={name:"SearchPage",data:function(){return{speciesNames:Pe["a"].SPECIES_SHORT_NAMES,proteins:this.$route.query.q,speciesId:this.$route.query.speciesId,page:this.$route.query.page||1,pageSize:this.$route.query.pageSize||10,isFetching:!0,isMultipleProteinsSearch:!1,resource:{}}},created:function(){return this.loadData()},meta:{title:"PAXdb - Search"},watch:{$route:function(e,a){this.proteins=this.$route.query.q,this.speciesId=this.$route.query.speciesId,this.page=this.$route.query.page||1,this.pageSize=this.$route.query.pageSize||10,this.isFetching=!0},apiQuery:function(){this.loadData()}},computed:{apiQuery:function(){var e="/search?q=".concat(encodeURIComponent(this.proteins));return this.speciesId&&(e+="&species=".concat(this.speciesId)),this.page&&(e+="&page=".concat(this.page)),this.pageSize&&(e+="&pageSize=".concat(this.pageSize)),e}},mounted:function(){De["a"].show()},methods:{toProteinsPage:function(){var e=Array.from(this.$el.querySelectorAll("input:checked")).map(function(e){return e.id}).join(",");this.$router.push("/proteins?ids=".concat(e))},onApiResponse:function(e){var a=e.data;if(this.isMultipleProteinsSearch=!1,Object.prototype.hasOwnProperty.call(a.searchResponse,"hits")||(a.searchResponse.hits={"@pageSize":"10","@currentPage":"1","@totalHits":"0","@query":this.query}),Object.prototype.hasOwnProperty.call(a.searchResponse.hits,"length")){if(a.searchResponse.hits.length>1)return this.resource=a.searchResponse,De["a"].hide(),this.resource.hits.forEach(function(e){1===parseInt(e["@totalHits"],10)&&(e.protein=[e.protein])}),this.isFetching=!1,void(this.isMultipleProteinsSearch=!0);var t=Te()(a.searchResponse.hits,1);a.searchResponse.hits=t[0]}var n=a.searchResponse.hits;if("1"!==n["@totalHits"])n.protein&&!Object.prototype.hasOwnProperty.call(n.protein,"length")&&(n.protein=[a.searchResponse.hits.protein]),this.currentPage=parseInt(n["@currentPage"],10),this.totalHits=parseInt(n["@totalHits"],10),this.firstHit=(this.currentPage-1)*this.pageSize+1,this.lastHit=this.currentPage*this.pageSize>this.totalHits?this.totalHits:this.currentPage*this.pageSize,this.resource=a.searchResponse,De["a"].hide(),this.isFetching=!1;else{var i=n.protein["@id"];this.$router.push("/protein/".concat(i))}},loadData:function(){var e=this;return this.isFetching=!0,De["a"].show(),this.$axios.get("http://search-api.pax-db.org".concat(this.apiQuery)).then(this.onApiResponse).catch(function(a){e.$q.notify({color:"negative",position:"top",message:"Search failed",icon:"report_problem"}),s["a"].rollbar.error(a)})},classname_if_equal:function(e,a,t){return e==a?t:""},last_page:function(e){var a=parseFloat(e["@totalHits"])/parseFloat(e["@pageSize"]);return a+=e["@totalHits"]%e["@pageSize"]==0?0:1,a},make_search_url:function(e,a,t,n,i){t=parseInt(t,10);var o=parseInt(e.hits["@currentPage"],10)+t;if(o<1)return"";var r=i||10,s="/search?q=".concat(a["@query"]).concat(this.speciesId?"&organism=".concat(this.speciesId):"").concat(1==o?"":"&page=".concat(o)).concat(10==r?"":"&pageSize=".concat(r));return s}}},We=Be,Ue=(t("0538"),Object(W["a"])(We,Ne,Re,!1,null,null,null)),Ke=Ue.exports,Fe=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-page",{attrs:{padding:""}},[e.isFetching?e._e():t("div",{staticClass:"column q-pa-sm q-mx-auto",staticStyle:{"max-width":"1500px"}},[t("h1",[e._v(e._s(e.resource.proteins.length)+"\n      "),t("router-link",{attrs:{to:"/species/"+e.resource.speciesId}},[t("em",[e._v(e._s(e.speciesName))])]),e._v("\n      proteins selected\n    ")],1),t("div",{staticStyle:{overflow:"auto",width:"100%"}},[t("table",{ref:"data_table",staticClass:"footable table hidden",attrs:{"data-sorting":"true","data-paging-size":"20","data-paging":e.resource.datasets.length>20}},[t("thead",[t("tr",[t("th",{staticClass:"footable-first-column",attrs:{"data-type":"html"}},[e._v("\n            Protein\n          ")]),e._l(e.resource.datasets,function(a){return t("th",{key:a.id,staticClass:"text-center-important",attrs:{"data-type":"text","data-sortable":"false","data-filterable":"false","data-class":"text-center-important"}},[e._v("\n            "+e._s(a.name)+"\n          ")])})],2)]),t("tbody",e._l(e.resource.proteins,function(a){return t("tr",{key:a.extarnalId},[t("td",{attrs:{"data-sort-value":a.name,"data-filter-value":a.name+" "+a.externalId}},[t("router-link",{attrs:{to:"/protein/"+a.id}},[e._v(e._s(a.name))])],1),e._l(a.abundances,function(a){return t("td",{key:a},[e._v(" "+e._s(a))])})],2)}),0)])]),t("p",{staticClass:"q-my-lg"},[t("q-btn",{attrs:{color:"primary"},on:{click:function(a){return a.preventDefault(),e.exportData(a)}}},[e._v("Download as a TSV file")])],1)])])},qe=[],ze=t("bdea"),je=ze["a"],Qe=Object(W["a"])(je,Fe,qe,!1,null,null,null),Ye=Qe.exports,Je=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-page",{attrs:{padding:""}},[e.isFetching?e._e():t("div",{staticClass:"column q-pa-sm q-mx-auto",staticStyle:{"max-width":"1500px"}},[t("div",[t("h1",[e._v(e._s(e.protein.name))]),t("span",[t("router-link",{attrs:{to:"/species/"+e.speciesId}},[t("em",[e._v(e._s(e.species.name))])]),e._v(" protein\n      ")],1),t("blockquote",[e._v("\n        "+e._s(e.protein.externalId.substring(e.protein.externalId.indexOf(".")+1))+" :\n        "+e._s(e.protein.annotation)+":\n      ")]),t("p",[t("small",[e._v(e._s(e.protein.name)+": in:")]),e._l(e.uniprotLink,function(e){return t("a",{key:e,staticStyle:{margin:"0px 5px"},attrs:{href:e,target:"_blank"}},[t("img",{staticStyle:{"vertical-align":"middle",border:"0px"},attrs:{src:"https://string-db.org/images/uniprot_logo_small.png",width:"42",height:"19",alt:"UniProt"}})])})],2)]),t("div",[e.protein.abundances?t("div",{staticClass:"row"},[t("div",{staticClass:"q-my-xs col-sm-12 col-lg-6"},[t("h3",{staticClass:"text-center"},[e._v(" "+e._s(e.protein.name)+" abundance information")]),t("table",{ref:"abundance_table",staticClass:"footable  toggle-large  toggle-arrow-tiny center_with_auto_margins table hidden",attrs:{"data-sorting":e.protein.abundances.length>1,"data-filtering":e.protein.abundances.length>5,"data-filter-delay":"50","data-filter-placeholder":"dataset name","data-filter-position":"left","data-paging":e.protein.abundances.length>5,"data-paging-size":"5"}},[t("thead",[t("tr",[t("th",{staticClass:"footable-first-column",attrs:{"data-type":"html"}},[e._v("\n                Dataset\n              ")]),t("th",{staticClass:"text-center text-nowrap",attrs:{"data-type":"number","data-filterable":"false","data-sorted":"true","data-direction":"DESC","data-class":"text-right"}},[e._v("\n                Abundance\n              ")]),t("th",{staticClass:"text-center",attrs:{"data-breakpoints":"xs sm md","data-filterable":"false","data-sortable":"false","data-class":"text-center text-nowrap"}},[e._v("\n                Rank\n              ")])])]),t("tbody",e._l(e.protein.abundances,function(a){return t("tr",[t("td",{attrs:{"data-sort-value":a.datasetInfo.name,"data-filter-value":a.datasetInfo.name}},[t("router-link",{attrs:{to:"/dataset/"+e.speciesId+"/"+a.datasetInfo.id}},[e._v("\n                  "+e._s(a.datasetInfo.name)+"\n                ")])],1),t("td",[e._v("\n                "+e._s(a.formattedAbundance)+"\n              ")]),t("td",[e._v("\n                "+e._s(a.rank)+"\n              ")])])}),0)])]),t("div",{staticClass:"col-sm-12 col-lg-6"},[t("div",{staticClass:"text-center"},[t("h3",[e._v("Protein Abundance Histogram")]),t("small",[t("router-link",{attrs:{to:"/dataset/"+e.species.id+"/"+e.dataset.id}},[e._v("\n                "+e._s(e.dataset.name)+"\n              ")])],1),t("img",{staticClass:"img-responsive center-block",attrs:{src:"https://pax-db.org/dataset/"+e.dataset.id+"/histogram?hightlightProteinId="+e.protein.id,alt:"abundance histogram"}}),t("label",[e._v("protein abundance (ppm) "),e.protein.abundances?t("span",[e._v("["+e._s(e.protein.name)+" in "),t("span",{staticStyle:{color:"red"}},[e._v("red")]),e._v(" bin]")]):e._e()])])])]):t("div",[e._v("no abundance information available for "+e._s(e.protein.name))])]),t("div",{staticStyle:{"overflow-x":"auto",width:"100%"}},[t("protein-family",{attrs:{protein:e.protein}})],1),t("div",{staticClass:"q-my-lg"},[t("h2",[e._v("Interaction network "),t("small",[e._v("(from "),t("a",{attrs:{href:e.stringdbProteinUrl,target:"string_info-"+e.protein.id}},[e._v("string-db.org")]),e._v(")")])]),t("a",{attrs:{href:e.stringdbProteinUrl,target:"_blank"}},[t("img",{staticClass:"img-responsive",attrs:{src:e.proteinNetworkImage}})])])])])},Ve=[],Xe=t("9f7e"),$e=Xe["a"],Ze=(t("d1ac"),Object(W["a"])($e,Je,Ve,!1,null,"9938187e",null)),ea=Ze.exports,aa=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("q-page",{attrs:{padding:""}},[t("div",{staticClass:"column q-pa-sm q-mx-auto",staticStyle:{"max-width":"1500px"}},[t("div",{staticClass:"row q-gutter-lg justify-around"},[t("div",{staticClass:"col-sm-12 col-md-5"},[t("h1",[e._v(e._s(e.dataset.name))]),t("table",{staticClass:"table info_table center_with_auto_margins"},[t("tbody",[t("tr",[t("td",{attrs:{colspan:"2"}})]),t("tr",[t("td",[e._v("Interaction consistency score:")]),t("td",[e._v(e._s(e.dataset.score))])]),t("tr",[t("td",[e._v("Coverage:")]),t("td",[e._v(e._s(e.dataset.coverage)+"%")])]),t("tr",[t("td",[e._v("#Abundances:")]),t("td",[e._v(e._s(e.dataset.num_abundances))])]),e.dataset.publication_year?t("tr",[t("td",[e._v("Published:")]),t("td",[e._v(e._s(e.dataset.publication_year))])]):e._e()])])]),t("div",{staticClass:"text-center col-sm-12 col-md-6"},[t("h2",[e._v("Protein Abundance Histogram")]),t("img",{staticClass:"img-responsive center-block",attrs:{src:"http://api.pax-db.org/dataset/"+e.dataset.id+"/histogram",alt:e.dataset.name+" abundance histogram"}}),t("label",[e._v("protein abundance (ppm)")])])]),t("div",{staticClass:"row"},[t("h2",[e._v("Abundances")]),t("dataset-abundances-table",{ref:"abundancesTable",attrs:{datasetId:e.dataset.id,speciesId:e.species.id,totalAbundances:e.dataset.num_abundances}}),t("div",{staticClass:"q-my-lg desktop-only gt-xs"},[t("div",{staticClass:"row q-gutter-lg"},e._l(["tsv","csv","json"],function(a){return t("q-btn",{key:a,attrs:{color:"primary"},on:{click:function(t){return t.preventDefault(),e.exportData(a)}}},[e._v("Export to\n            "+e._s(a.toUpperCase())+"\n          ")])}),1),t("h4",[t("a",{attrs:{href:"https://pax-db.org/downloads/4.1/datasets/"+e.species.id+"/"+e.datasetFilenames[e.id],target:e.dataset.name}},[e._v("\n            Download entire dataset as TSV\n          ")])])])],1),e.species.datasets.length>1?t("div",[t("h3",[e._v("Other datasets in\n        "),t("router-link",{attrs:{to:"/species/"+e.species.id}},[e._v(e._s(e.species.name))])],1),t("div",{attrs:{id:"dataset_overview"}},[t("table",{ref:"dataset_overview_table",staticClass:"footable  toggle-large  toggle-arrow-tiny table hidden",staticStyle:{"max-width":"900px"},attrs:{"data-sorting":"true","data-paging":e.species.datasets.length>10,"data-filtering":"true","data-filter-delay":"50","data-filter-placeholder":"dataset name","data-filter-position":"left"}},[t("thead",[t("tr",[t("th",{staticClass:"footable-first-column",attrs:{"data-type":"text"}},[e._v("\n              Name\n            ")]),t("th",{staticClass:"text-center",attrs:{"data-type":"number","data-classes":"text-center","data-filterable":"false","data-sorted":"true","data-direction":"ASC"}},[e._v("\n              Interaction consistency score\n            ")]),t("th",{staticStyle:{display:"none"},attrs:{"data-type":"text","data-breakpoints":"xs","data-classes":"text-center","data-filterable":"false"}},[e._v("\n              Coverage\n            ")]),t("th",{attrs:{"data-type":"html","data-breakpoints":"xs sm","data-classes":"text-nowrap","data-filterable":"false","data-sortable":"false"}},[e._v("\n              Download\n            ")])])]),t("tbody",e._l(e.species.datasets,function(a){return t("tr",{key:a.id},[t("td",{attrs:{"data-sort-value":a.name,"data-filter-value":a.name}},[t("router-link",{attrs:{to:"/dataset/"+e.species.id+"/"+a.id+"/"}},[e._v(e._s(a.name))])],1),t("td",[e._v("\n                "+e._s(a.score)+"\n              ")]),t("td",[e._v("\n                "+e._s(a.coverage)+"\n              ")]),t("td",[t("span",{staticClass:"text-right"},[t("a",{attrs:{href:"https://pax-db.org/downloads/4.1/datasets/"+e.species.id+"/"+e.datasetFilenames[a.id],target:a.name}},[e._v(e._s(a.id)+".txt")])])])])}),0)])])]):e._e()])])},ta=[],na=t("a075"),ia=na["a"],oa=Object(W["a"])(ia,aa,ta,!1,null,null,null),ra=oa.exports,sa=[{path:"/",component:ce,children:[{path:"",component:Ie}]},{path:"/species/:id",component:ce,children:[{path:"",component:Oe}]},{path:"/dataset/:speciesId/:datasetId",component:ce,children:[{path:"",component:ra}]},{path:"/protein/:id",component:ce,children:[{path:"",component:ea}]},{path:"/proteins",component:ce,children:[{path:"",component:Ye}]},{path:"/search",component:ce,children:[{path:"",component:Ke}]},{path:"/download",component:ce,children:[{path:"",component:function(){return t.e("f213270c").then(t.bind(null,"4499"))}}]},{path:"/about",component:ce,children:[{path:"",component:function(){return t.e("2d207d33").then(t.bind(null,"a1d1"))}}]},{path:"/license",component:ce,children:[{path:"",component:function(){return t.e("2d22c18e").then(t.bind(null,"f27c"))}}]},{path:"/contact",component:ce,children:[{path:"",component:function(){return t.e("2d216da8").then(t.bind(null,"c3df"))}}]},{path:"/help",component:ce,children:[{path:"",component:function(){return t.e("1e10e808").then(t.bind(null,"9fc0"))}}]}];sa.push({path:"*",component:function(){return t.e("4b47640d").then(t.bind(null,"e51e"))}});var ca=sa;s["a"].use(F["a"]);var la=function(){var e=new F["a"]({scrollBehavior:function(){return{x:0,y:0}},routes:ca,mode:"history",base:"/"});return e},da=function(){var e="function"===typeof la?la({Vue:s["a"]}):la,a={el:"#q-app",router:e,render:function(e){return e(K)}};return{app:a,router:e}},ua=t("bc3a"),ba=t.n(ua),pa=function(){var e=r()(i.a.mark(function e(a){var t;return i.a.wrap(function(e){while(1)switch(e.prev=e.next){case 0:t=a.Vue,t.prototype.$axios=ba.a;case 2:case"end":return e.stop()}},e)}));return function(a){return e.apply(this,arguments)}}(),ma=da(),_a=ma.app,ha=ma.router;function fa(){return va.apply(this,arguments)}function va(){return va=r()(i.a.mark(function e(){var a,t;return i.a.wrap(function(e){while(1)switch(e.prev=e.next){case 0:a=[pa],t=0;case 2:if(!(t<a.length)){e.next=20;break}if("function"===typeof a[t]){e.next=5;break}return e.abrupt("continue",17);case 5:return e.prev=5,e.next=8,a[t]({app:_a,router:ha,Vue:s["a"],ssrContext:null});case 8:e.next=17;break;case 10:if(e.prev=10,e.t0=e["catch"](5),!e.t0||!e.t0.url){e.next=15;break}return window.location.href=e.t0.url,e.abrupt("return");case 15:return console.error("[Quasar] boot error:",e.t0),e.abrupt("return");case 17:t++,e.next=2;break;case 20:new s["a"](_a);case 21:case"end":return e.stop()}},e,null,[[5,10]])})),va.apply(this,arguments)}fa()},"38a5":function(e,a,t){"use strict";var n=t("399a"),i=t.n(n);i.a},"399a":function(e,a,t){},"3be7":function(e,a,t){},"3e0f":function(e,a,t){"use strict";a["a"]={882:{id:882,name:"Desulfovibrio vulgaris str. Hildenborough",compact_name:"Desulfovibrio vulgaris Hildenborough",num_proteins:3380,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid882[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:97,name:"D.vulgaris - Whole organism, Formate-Exponential (Zhang,Proteomics,2006)",score:3.19,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, Formate-Exponential, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=16819729&retmode=ref&cmd=prlinks" target="_blank">Zhang,Proteomics,2006</a><br/><b>Interaction consistency score</b>: 3.19&nbsp<b>Coverage</b>: 28',organ:"WHOLE_ORGANISM",integrated:!1,coverage:28,publication_year:2006,num_abundances:956,hasPeptideCounts:!1,filename:"882-Desulfo_Form_Exp_SC_zhang_2006.txt"},{id:263,name:"D.vulgaris - Whole organism, Formate-Stationary (Zhang,Proteomics,2006)",score:.61,weight:"20%",description:'abundance based on MAPPED_BY_AUTHORS, Formate-Stationary, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=16819729&retmode=ref&cmd=prlinks" target="_blank">Zhang,Proteomics,2006</a><br/><b>Interaction consistency score</b>: 0.61&nbsp<b>Coverage</b>: 28',organ:"WHOLE_ORGANISM",integrated:!1,coverage:28,publication_year:2006,num_abundances:956,hasPeptideCounts:!1,filename:"882-Desulfo_Form_Stat_SC_zhang_2006.txt"},{id:23,name:"D.vulgaris - Whole organism, Lactose-Exponential (Zhang,Proteomics,2006)",score:1.47,weight:"20%",description:'abundance based on MAPPED_BY_AUTHORS, Lactose-Exponential, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=16819729&retmode=ref&cmd=prlinks" target="_blank">Zhang,Proteomics,2006</a><br/><b>Interaction consistency score</b>: 1.47&nbsp<b>Coverage</b>: 28',organ:"WHOLE_ORGANISM",integrated:!1,coverage:28,publication_year:2006,num_abundances:956,hasPeptideCounts:!1,filename:"882-Desulfo_Lac_Exp_SC_zhang_2006.txt"},{id:160,name:"D.vulgaris - Whole organism, Lactose-Stationary (Zhang,Proteomics,2006)",score:1.25,weight:"10%",description:'abundance based on MAPPED_BY_AUTHORS, Lactose-Stationary, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=16819729&retmode=ref&cmd=prlinks" target="_blank">Zhang,Proteomics,2006</a><br/><b>Interaction consistency score</b>: 1.25&nbsp<b>Coverage</b>: 28',organ:"WHOLE_ORGANISM",integrated:!1,coverage:28,publication_year:2006,num_abundances:956,hasPeptideCounts:!1,filename:"882-Desulfo_Lac_Stat_SC_zhang_2006.txt"},{id:32,name:"D.vulgaris - Whole organism (Integrated)",score:5.29,weight:"",description:"integrated dataset: weighted average of all D.vulgaris WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 5.29&nbsp<b>Coverage</b>: 28",organ:"WHOLE_ORGANISM",integrated:!0,coverage:28,publication_year:2017,num_abundances:956,hasPeptideCounts:!1,filename:"882-WHOLE_ORGANISM-integrated.txt"}],total_coverage:28,average_coverage:28,topProteins:[{id:6004,abundance:54436,rank:0,name:"DVU_2405",annotation:"Iron-containing alcohol dehydrogenase"},{id:4419,abundance:38923,rank:1,name:"DVU_0799",annotation:"Hypothetical protein"},{id:4466,abundance:32804,rank:2,name:"aprA",annotation:"Adenylylsulfate reductase subunit alpha"},{id:4909,abundance:31645,rank:3,name:"sat",annotation:"Sulfate adenylyltransferase"},{id:5583,abundance:27305,rank:4,name:"groL",annotation:"Chaperonin GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:5036,abundance:26904,rank:5,name:"DVU_1422",annotation:"OmpA family protein"},{id:4028,abundance:20505,rank:6,name:"dsrA",annotation:"Dissimilatory sulfite reductase subunit alpha; Part of the complex that catalyzes the reduction of sulfite to sulfide. The alpha and beta subunits may have arisen by gene duplication. They both bind 2 iron-sulfur clusters, but the alpha subunit seems to be catalytically inactive, due to substitutions along the putative substrate access channel, and because it binds sirohydrochlorin (the dematallated form of siroheme) instead of siroheme"},{id:5711,abundance:20351,rank:7,name:"DVU_2109",annotation:"Hypothetical protein; Binds and transfers iron-sulfur (Fe-S) clusters to target apoproteins. Can hydrolyze ATP"}]},1148:{id:1148,name:"Synechocystis sp. PCC 6803",compact_name:"Synechocystis sp. PCC6803",num_proteins:3171,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid1148[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:410,name:"Synechocystis.sp. 6803 - Whole organism, SC (Wegener,MCP,2010)",score:6.57,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/20858728" target="_blank">Wegener,MCP,2010</a><br/><b>Interaction consistency score</b>: 6.57&nbsp<b>Coverage</b>: 38',organ:"WHOLE_ORGANISM",integrated:!1,coverage:38,publication_year:2010,num_abundances:1200,hasPeptideCounts:!0,filename:"1148-SynechocystisSp_strain_PRIDE.txt"}],total_coverage:37,average_coverage:38,topProteins:[{id:13907,abundance:58118,rank:0,name:"psbH",annotation:"Photosystem II PsbH protein; One of the components of the core complex of photosystem II (PSII), required for its stability and/or assembly. May regulate electron transport between the quinone binding sites of the reaction center (PubMed:7626631, Ref.6). PSII is a light- driven water:plastoquinone oxidoreductase that uses light energy to abstract electrons from H(2)O, generating O(2) and a proton gradient subsequently used for ATP formation. It consists of a core antenna complex that captures photons, and an electron transfer chain that converts photonic excitation into a charge separation"},{id:13509,abundance:48813,rank:1,name:"cpcB",annotation:"Phycocyanin b subunit; Light-harvesting photosynthetic bile pigment-protein from the phycobiliprotein complex"},{id:14150,abundance:27706,rank:2,name:"apcA",annotation:"Allophycocyanin a chain; Light-harvesting photosynthetic bile pigment-protein from the phycobiliprotein complex. Allophycocyanin has a maximum absorption at approximately 650 nanometers"},{id:13508,abundance:22837,rank:3,name:"cpcA",annotation:"Phycocyanin a subunit; Light-harvesting photosynthetic bile pigment-protein from the phycobiliprotein complex"},{id:14151,abundance:19587,rank:4,name:"apcB",annotation:"Allophycocyanin b chain; Light-harvesting photosynthetic bile pigment-protein from the phycobiliprotein complex. Allophycocyanin has a maximum absorption at approximately 650 nanometers"},{id:15077,abundance:17186,rank:5,name:"cbbL",annotation:"Ribulose-1,5-bisphosphate carboxylase/oxygenase large subunit; RuBisCO catalyzes two reactions: the carboxylation of D- ribulose 1,5-bisphosphate, the primary event in carbon dioxide fixation, as well as the oxidative fragmentation of the pentose substrate in the photorespiration process. Both reactions occur simultaneously and in competition at the same active site"},{id:14814,abundance:13752,rank:6,name:"glnB",annotation:"Nitrogen regulatory protein P-II; P-II indirectly controls the transcription of the GS gene (glnA). P-II prevents NR-II-catalyzed conversion of NR-I to NR-I-phosphate, the transcriptional activator of glnA. When P-II is phosphorylated, these events are reversed. In nitrogen-limiting conditions, when the ratio of Gln to 2-ketoglutarate decreases, P- II is phosphorylated which allows the deadenylation of glutamine synthetase (GS), thus activating the enzyme (By similarity)"},{id:14660,abundance:11584,rank:7,name:"psaE",annotation:"Photosystem I subunit IV; Stabilizes the interaction between PsaC and the PSI core, assists the docking of the ferredoxin to PSI and interacts with ferredoxin-NADP oxidoreductase"}]},3055:{id:3055,name:"Chlamydomonas reinhardtii",compact_name:"Chlamydomonas reinhardtii",num_proteins:14416,genome_source:"<a href='http://plants.ensembl.org/Chlamydomonas_reinhardtii//Info/Index?db=core'>Ensembl Plants</a>",genome_source_version:"release 17",datasets:[{id:47,name:"C.reinhardtii - Whole organism, SC (GPM,Aug,2014)",score:16.46,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 16.46&nbsp<b>Coverage</b>: 36',organ:"WHOLE_ORGANISM",integrated:!1,coverage:36,publication_year:2014,num_abundances:5163,hasPeptideCounts:!0,filename:"3055-GPM_201408.txt"}],total_coverage:35,average_coverage:36,topProteins:[{id:27432,abundance:20559,rank:0,name:"EDO95759",annotation:"Histone H3"},{id:27678,abundance:16256,rank:1,name:"EDO96005",annotation:"Histone H3"},{id:27433,abundance:11382,rank:2,name:"EDO95760",annotation:"Histone H4"},{id:27380,abundance:11382,rank:3,name:"EDO95707",annotation:"Histone H4"},{id:29291,abundance:8240,rank:4,name:"PSBQ",annotation:"Oxygen evolving enhancer protein 3"},{id:33918,abundance:8091,rank:5,name:"PSBO",annotation:"Oxygen-evolving enhancer protein 1 of photosystem II"},{id:34558,abundance:5962,rank:6,name:"LHCBM2",annotation:"Light-harvesting protein of photosystem II"},{id:34686,abundance:5962,rank:7,name:"LHCBM7",annotation:"Chlorophyll a-b binding protein of LHCII"}]},3702:{id:3702,name:"Arabidopsis thaliana",compact_name:"Arabidopsis thaliana",num_proteins:27416,genome_source:"<a href='http://plants.ensembl.org/Arabidopsis_thaliana//Info/Index?db=core'>Ensembl Plants</a>",genome_source_version:"release 17",datasets:[{id:298,name:"A.thaliana - Leaf, Short Day Optimal Water, SC (Baerenfaller,Mol_Syst_Bio,2012)",score:5.34,weight:"100%",description:'abundance based on Spectral counting SOW, Short Day Optimal Water, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3435506/" target="_blank">Baerenfaller,Mol_Syst_Bio,2012</a><br/><b>Interaction consistency score</b>: 5.34&nbsp<b>Coverage</b>: 9',organ:"LEAF",integrated:!1,coverage:9,publication_year:2012,num_abundances:2438,hasPeptideCounts:!0,filename:"3702-2.1ArabidopsisLeaf_waterDefict_SOW.txt"},{id:401,name:"A.thaliana - Leaf, Short Day Mild Water Deficit, SC (Baerenfaller,Mol_Syst_Bio,2012)",score:4.07,weight:"100%",description:'abundance based on Spectral counting SWD, Short Day Mild Water Deficit, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3435506/" target="_blank">Baerenfaller,Mol_Syst_Bio,2012</a><br/><b>Interaction consistency score</b>: 4.07&nbsp<b>Coverage</b>: 7',organ:"LEAF",integrated:!1,coverage:7,publication_year:2012,num_abundances:2031,hasPeptideCounts:!0,filename:"3702-2.2ArabidopsisLeaf_waterDefict_SWD.txt"},{id:284,name:"A.thaliana - Leaf, SC (Gfeller,Plant_Physiology,2011)",score:6.45,weight:"100%",description:'abundance based on Spectral counting control, from<a href="http://www.plantphysiol.org/content/early/2011/06/21/pp.111.181008.short?keytype=ref&ijkey=4lFzrQqi1J1v7pY" target="_blank">Gfeller,Plant_Physiology,2011</a><br/><b>Interaction consistency score</b>: 6.45&nbsp<b>Coverage</b>: 15',organ:"LEAF",integrated:!1,coverage:15,publication_year:2011,num_abundances:4190,hasPeptideCounts:!0,filename:"3702-6.1JasmonateControls_controlLeaves.txt"},{id:233,name:"A.thaliana - Leaf, SC (Gfeller,Plant_Physiology,2011)",score:6.42,weight:"100%",description:'abundance based on Spectral counting wounded, from<a href="http://www.plantphysiol.org/content/early/2011/06/21/pp.111.181008.short?keytype=ref&ijkey=4lFzrQqi1J1v7pY" target="_blank">Gfeller,Plant_Physiology,2011</a><br/><b>Interaction consistency score</b>: 6.42&nbsp<b>Coverage</b>: 16',organ:"LEAF",integrated:!1,coverage:16,publication_year:2011,num_abundances:4261,hasPeptideCounts:!0,filename:"3702-6.2JasmonateControls_woundedLeaves.txt"},{id:270,name:"A.thaliana - Rosette, End of day, SC (Reiland,Plant_Physiology,2009)",score:5.74,weight:"100%",description:'abundance based on Spectral counting ED, end of day, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2689975/" target="_blank">Reiland,Plant_Physiology,2009</a><br/><b>Interaction consistency score</b>: 5.74&nbsp<b>Coverage</b>: 9',organ:"ROSETTE",integrated:!1,coverage:9,publication_year:2009,num_abundances:2344,hasPeptideCounts:!0,filename:"3702-7.1Large-scalePhosphoproteome_rosette_ED.txt"},{id:385,name:"A.thaliana - Rosette, End of night, SC (Reiland,Plant_Physiology,2009)",score:5.63,weight:"100%",description:'abundance based on Spectral counting EN, end of night, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2689975/" target="_blank">Reiland,Plant_Physiology,2009</a><br/><b>Interaction consistency score</b>: 5.63&nbsp<b>Coverage</b>: 9',organ:"ROSETTE",integrated:!1,coverage:9,publication_year:2009,num_abundances:2512,hasPeptideCounts:!0,filename:"3702-7.2Large-scalePhosphoproteome_rosette_EN.txt"},{id:2,name:"A.thaliana - Shoot, SC (Reiland,PNAS,2011)",score:6.78,weight:"100%",description:'abundance based on Spectral counting, from<a href=" http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3150903/" target="_blank">Reiland,PNAS,2011</a><br/><b>Interaction consistency score</b>: 6.78&nbsp<b>Coverage</b>: 8',organ:"SHOOT",integrated:!1,coverage:8,publication_year:2011,num_abundances:2280,hasPeptideCounts:!0,filename:"3702-7.3Large-scalePhosphoproteome_shoot.txt"},{id:163,name:"A.thaliana - Carpel, SC, SC (Baerenfaller,IntegrBiol,2011)",score:.95,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 0.95&nbsp<b>Coverage</b>: 14',organ:"CARPEL",integrated:!1,coverage:14,publication_year:2011,num_abundances:3969,hasPeptideCounts:!0,filename:"3702-Arabidopsis_capel_Baerenfaller_2011_IntegrBiol.txt"},{id:209,name:"A.thaliana - Cotyledon, SC, SC (Baerenfaller,IntegrBiol,2011)",score:3.22,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 3.22&nbsp<b>Coverage</b>: 13',organ:"COTYLEDON",integrated:!1,coverage:13,publication_year:2011,num_abundances:3676,hasPeptideCounts:!0,filename:"3702-Arabidopsis_cotyledon_Baerenfaller_2011_IntegrBiol.txt"},{id:279,name:"A.thaliana - Flower, SC, SC (Baerenfaller,IntegrBiol,2011)",score:4.48,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 4.48&nbsp<b>Coverage</b>: 19',organ:"FLOWER",integrated:!1,coverage:19,publication_year:2011,num_abundances:5277,hasPeptideCounts:!0,filename:"3702-Arabidopsis_flower_Baerenfaller_2011_IntegrBiol.txt"},{id:413,name:"A.thaliana - Flowerbud, SC, SC (Baerenfaller,IntegrBiol,2011)",score:3.65,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 3.65&nbsp<b>Coverage</b>: 19',organ:"FLOWERBUD",integrated:!1,coverage:19,publication_year:2011,num_abundances:5149,hasPeptideCounts:!0,filename:"3702-Arabidopsis_flowerbud_Baerenfaller_2011_IntegrBiol.txt"},{id:283,name:"A.thaliana - Juvenile leaf, SC, SC (Baerenfaller,IntegrBiol,2011)",score:2.74,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 2.74&nbsp<b>Coverage</b>: 14',organ:"JUVENILE_LEAF",integrated:!1,coverage:14,publication_year:2011,num_abundances:3920,hasPeptideCounts:!0,filename:"3702-Arabidopsis_juvenileLeaf_Baerenfaller_2011_IntegrBiol.txt"},{id:124,name:"A.thaliana - Pollen, SC, SC (Grobei,Genome_Research,2009)",score:5.42,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2765272/" target="_blank">Grobei,Genome_Research,2009</a><br/><b>Interaction consistency score</b>: 5.42&nbsp<b>Coverage</b>: 13',organ:"POLLEN",integrated:!1,coverage:13,publication_year:2009,num_abundances:3519,hasPeptideCounts:!0,filename:"3702-Arabidopsis_pollen_Grobei_2009_GenomeRes.txt"},{id:135,name:"A.thaliana - Root, SC, SC (Baerenfaller,IntegrBiol,2011)",score:6.01,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 6.01&nbsp<b>Coverage</b>: 23',organ:"ROOT",integrated:!1,coverage:23,publication_year:2011,num_abundances:6185,hasPeptideCounts:!0,filename:"3702-Arabidopsis_root_Baerenfaller_2011_IntegrBiol.txt"},{id:174,name:"A.thaliana - Seed, SC, SC (Baerenfaller,IntegrBiol,2011)",score:2.4,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 2.4&nbsp<b>Coverage</b>: 14',organ:"SEED",integrated:!1,coverage:14,publication_year:2011,num_abundances:3806,hasPeptideCounts:!0,filename:"3702-Arabidopsis_seed_Baerenfaller_2011_IntegrBiol.txt"},{id:213,name:"A.thaliana - Silique, SC, SC (Baerenfaller,IntegrBiol,2011)",score:4.97,weight:"100%",description:'abundance based on Spectral counting, Spectral counting, from<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/IB/c0ib00078g" target="_blank">Baerenfaller, IntegrBiol,2011</a><br/><b>Interaction consistency score</b>: 4.97&nbsp<b>Coverage</b>: 21',organ:"SILIQUE",integrated:!1,coverage:21,publication_year:2011,num_abundances:5815,hasPeptideCounts:!0,filename:"3702-Arabidopsis_silique_Baerenfaller_2011_IntegrBiol.txt"},{id:322,name:"A.thaliana - Whole organism, SC (Motohashi,Plant_Physiol,2012)",score:4.85,weight:"40%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23027667" target="_blank">Motohashi,Plant_Physiol,2012</a><br/><b>Interaction consistency score</b>: 4.85&nbsp<b>Coverage</b>: 6',organ:"WHOLE_ORGANISM",integrated:!1,coverage:6,publication_year:2012,num_abundances:1657,hasPeptideCounts:!0,filename:"3702-Arabidopsis_thaliana_SC_biomart_27037_E__1reps.txt"},{id:7,name:"A.thaliana - Carpel (Integrated)",score:2.01,weight:"",description:"integrated dataset: weighted average of all A.thaliana CARPEL datasets<br/><b>Interaction consistency score</b>: 2.01&nbsp<b>Coverage</b>: 15",organ:"CARPEL",integrated:!0,coverage:15,publication_year:2017,num_abundances:4136,hasPeptideCounts:!1,filename:"3702-CARPEL-integrated.txt"},{id:341,name:"A.thaliana - Cotyledon (Integrated)",score:4.8,weight:"",description:"integrated dataset: weighted average of all A.thaliana COTYLEDON datasets<br/><b>Interaction consistency score</b>: 4.8&nbsp<b>Coverage</b>: 19",organ:"COTYLEDON",integrated:!0,coverage:19,publication_year:2017,num_abundances:5215,hasPeptideCounts:!1,filename:"3702-COTYLEDON-integrated.txt"},{id:355,name:"A.thaliana - Flower (Integrated)",score:4.87,weight:"",description:"integrated dataset: weighted average of all A.thaliana FLOWER datasets<br/><b>Interaction consistency score</b>: 4.87&nbsp<b>Coverage</b>: 30",organ:"FLOWER",integrated:!0,coverage:30,publication_year:2017,num_abundances:8351,hasPeptideCounts:!1,filename:"3702-FLOWER-integrated.txt"},{id:374,name:"A.thaliana - Flowerbud (Integrated)",score:5.72,weight:"",description:"integrated dataset: weighted average of all A.thaliana FLOWERBUD datasets<br/><b>Interaction consistency score</b>: 5.72&nbsp<b>Coverage</b>: 25",organ:"FLOWERBUD",integrated:!0,coverage:25,publication_year:2017,num_abundances:6726,hasPeptideCounts:!1,filename:"3702-FLOWERBUD-integrated.txt"},{id:14,name:"A.thaliana - Whole organism, SC (GPM,Aug,2014)",score:12.57,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 12.57&nbsp<b>Coverage</b>: 42',organ:"WHOLE_ORGANISM",integrated:!1,coverage:42,publication_year:2014,num_abundances:11506,hasPeptideCounts:!0,filename:"3702-GPM_201408.txt"},{id:447,name:"A.thaliana - Juvenile leaf (Integrated)",score:5.08,weight:"",description:"integrated dataset: weighted average of all A.thaliana JUVENILE_LEAF datasets<br/><b>Interaction consistency score</b>: 5.08&nbsp<b>Coverage</b>: 21",organ:"JUVENILE_LEAF",integrated:!0,coverage:21,publication_year:2017,num_abundances:5731,hasPeptideCounts:!1,filename:"3702-JUVENILE_LEAF-integrated.txt"},{id:342,name:"A.thaliana - Leaf (Integrated)",score:5.31,weight:"",description:"integrated dataset: weighted average of all A.thaliana LEAF datasets<br/><b>Interaction consistency score</b>: 5.31&nbsp<b>Coverage</b>: 32",organ:"LEAF",integrated:!0,coverage:32,publication_year:2017,num_abundances:8734,hasPeptideCounts:!1,filename:"3702-LEAF-integrated.txt"},{id:178,name:"A.thaliana - Whole organism (Motohashi,Plant_Physiol,2012)",score:5.13,weight:"10%",description:'abundance based on MAPPED_BY_AUTHORS, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23027667" target="_blank">Motohashi,Plant_Physiol,2012</a><br/><b>Interaction consistency score</b>: 5.13&nbsp<b>Coverage</b>: 13',organ:"WHOLE_ORGANISM",integrated:!1,coverage:13,publication_year:2012,num_abundances:3429,hasPeptideCounts:!1,filename:"3702-Motohashi_2012_avgWT_Prot_nSPC.txt"},{id:36,name:"A.thaliana - Pollen (Integrated)",score:8.85,weight:"",description:"integrated dataset: weighted average of all A.thaliana POLLEN datasets<br/><b>Interaction consistency score</b>: 8.85&nbsp<b>Coverage</b>: 18",organ:"POLLEN",integrated:!0,coverage:18,publication_year:2017,num_abundances:5050,hasPeptideCounts:!1,filename:"3702-POLLEN-integrated.txt"},{id:444,name:"A.thaliana - Root (Integrated)",score:13.5,weight:"",description:"integrated dataset: weighted average of all A.thaliana ROOT datasets<br/><b>Interaction consistency score</b>: 13.5&nbsp<b>Coverage</b>: 43",organ:"ROOT",integrated:!0,coverage:43,publication_year:2017,num_abundances:11653,hasPeptideCounts:!1,filename:"3702-ROOT-integrated.txt"},{id:258,name:"A.thaliana - Rosette (Integrated)",score:7.47,weight:"",description:"integrated dataset: weighted average of all A.thaliana ROSETTE datasets<br/><b>Interaction consistency score</b>: 7.47&nbsp<b>Coverage</b>: 10",organ:"ROSETTE",integrated:!0,coverage:10,publication_year:2017,num_abundances:2859,hasPeptideCounts:!1,filename:"3702-ROSETTE-integrated.txt"},{id:304,name:"A.thaliana - Seed (Integrated)",score:3.66,weight:"",description:"integrated dataset: weighted average of all A.thaliana SEED datasets<br/><b>Interaction consistency score</b>: 3.66&nbsp<b>Coverage</b>: 15",organ:"SEED",integrated:!0,coverage:15,publication_year:2017,num_abundances:4019,hasPeptideCounts:!1,filename:"3702-SEED-integrated.txt"},{id:82,name:"A.thaliana - Shoot (Integrated)",score:7.49,weight:"",description:"integrated dataset: weighted average of all A.thaliana SHOOT datasets<br/><b>Interaction consistency score</b>: 7.49&nbsp<b>Coverage</b>: 21",organ:"SHOOT",integrated:!0,coverage:21,publication_year:2017,num_abundances:5644,hasPeptideCounts:!1,filename:"3702-SHOOT-integrated.txt"},{id:324,name:"A.thaliana - Silique (Integrated)",score:3.95,weight:"",description:"integrated dataset: weighted average of all A.thaliana SILIQUE datasets<br/><b>Interaction consistency score</b>: 3.95&nbsp<b>Coverage</b>: 25",organ:"SILIQUE",integrated:!0,coverage:25,publication_year:2017,num_abundances:6973,hasPeptideCounts:!1,filename:"3702-SILIQUE-integrated.txt"},{id:323,name:"A.thaliana - Whole organism (Integrated)",score:22.27,weight:"",description:"integrated dataset: weighted average of all A.thaliana WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 22.27&nbsp<b>Coverage</b>: 74",organ:"WHOLE_ORGANISM",integrated:!0,coverage:74,publication_year:2017,num_abundances:20185,hasPeptideCounts:!1,filename:"3702-WHOLE_ORGANISM-integrated.txt"},{id:122,name:"A.thaliana - Carpel, SC (Baerenfaller,pep2pro,2012)",score:1.76,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 1.76&nbsp<b>Coverage</b>: 15',organ:"CARPEL",integrated:!1,coverage:15,publication_year:2012,num_abundances:4032,hasPeptideCounts:!0,filename:"3702-arabidopsis1a_carpel_Baerenfaller_PRIDE.txt"},{id:152,name:"A.thaliana - Cotyledon, SC (Baerenfaller,pep2pro,2012)",score:5.24,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 5.24&nbsp<b>Coverage</b>: 19',organ:"COTYLEDON",integrated:!1,coverage:19,publication_year:2012,num_abundances:5104,hasPeptideCounts:!0,filename:"3702-arabidopsis1c_cotyledon_Baerenfaller_PRIDE.txt"},{id:91,name:"A.thaliana - Flower, SC (Baerenfaller,pep2pro,2012)",score:2.47,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 2.47&nbsp<b>Coverage</b>: 30',organ:"FLOWER",integrated:!1,coverage:30,publication_year:2012,num_abundances:8247,hasPeptideCounts:!0,filename:"3702-arabidopsis1d_flower_Baerenfaller_PRIDE.txt"},{id:244,name:"A.thaliana - Flowerbud, SC (Baerenfaller,pep2pro,2012)",score:5.9,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 5.9&nbsp<b>Coverage</b>: 24',organ:"FLOWERBUD",integrated:!1,coverage:24,publication_year:2012,num_abundances:6604,hasPeptideCounts:!0,filename:"3702-arabidopsis1e_flowerBud_Baerenfaller_PRIDE.txt"},{id:42,name:"A.thaliana - Juvenile leaf, SC (Baerenfaller,pep2pro,2012)",score:5.57,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 5.57&nbsp<b>Coverage</b>: 21',organ:"JUVENILE_LEAF",integrated:!1,coverage:21,publication_year:2012,num_abundances:5626,hasPeptideCounts:!0,filename:"3702-arabidopsis1f_juvenileLeaf_Baerenfaller_PRIDE.txt"},{id:173,name:"A.thaliana - Leaf, SC (Baerenfaller,pep2pro,2012)",score:5.35,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 5.35&nbsp<b>Coverage</b>: 26',organ:"LEAF",integrated:!1,coverage:26,publication_year:2012,num_abundances:7211,hasPeptideCounts:!0,filename:"3702-arabidopsis1g_leaf_Baerenfaller_PRIDE.txt"},{id:191,name:"A.thaliana - Pollen, SC (Baerenfaller,pep2pro,2012)",score:6.35,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 6.35&nbsp<b>Coverage</b>: 18',organ:"POLLEN",integrated:!1,coverage:18,publication_year:2012,num_abundances:4974,hasPeptideCounts:!0,filename:"3702-arabidopsis1h_pollen_Baerenfaller_PRIDE.txt"},{id:153,name:"A.thaliana - Root, SC (Baerenfaller,pep2pro,2012)",score:6.81,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 6.81&nbsp<b>Coverage</b>: 42',organ:"ROOT",integrated:!1,coverage:42,publication_year:2012,num_abundances:11486,hasPeptideCounts:!0,filename:"3702-arabidopsis1i_root_Baerenfaller_PRIDE.txt"},{id:302,name:"A.thaliana - Seed, SC (Baerenfaller,pep2pro,2012)",score:2.31,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 2.31&nbsp<b>Coverage</b>: 14',organ:"SEED",integrated:!1,coverage:14,publication_year:2012,num_abundances:3785,hasPeptideCounts:!0,filename:"3702-arabidopsis1k_seed_Baerenfaller_PRIDE.txt"},{id:28,name:"A.thaliana - Shoot, SC (Baerenfaller,pep2pro,2012)",score:6.82,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 6.82&nbsp<b>Coverage</b>: 20',organ:"SHOOT",integrated:!1,coverage:20,publication_year:2012,num_abundances:5384,hasPeptideCounts:!0,filename:"3702-arabidopsis1l_shoot_Baerenfaller_PRIDE.txt"},{id:414,name:"A.thaliana - Silique, SC (Baerenfaller,pep2pro,2012)",score:3.57,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://fgcz-pep2pro.uzh.ch/" target="_blank">Baerenfaller,pep2pro,2012</a><br/><b>Interaction consistency score</b>: 3.57&nbsp<b>Coverage</b>: 25',organ:"SILIQUE",integrated:!1,coverage:25,publication_year:2012,num_abundances:6757,hasPeptideCounts:!0,filename:"3702-arabidopsis1m_silique_Baerenfaller_PRIDE.txt"},{id:89,name:"A.thaliana - Leaf, SC (Piques,Mol_Syst_Biol,2009)",score:7.09,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2779082/" target="_blank">Piques,Mol_Syst_Biol, 2009</a><br/><b>Interaction consistency score</b>: 7.09&nbsp<b>Coverage</b>: 5',organ:"LEAF",integrated:!1,coverage:5,publication_year:2009,num_abundances:1378,hasPeptideCounts:!0,filename:"3702-arabidopsis2_estimationLeaf_PRIDE.txt"},{id:116,name:"A.thaliana - Whole organism,  MS/MS data, SC (Castellana,PNAS,2008)",score:11.4,weight:"10%",description:'abundance based on Spectral counting,  MS/MS data, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19098097" target="_blank">Castellana, PNAS, 2008</a><br/><b>Interaction consistency score</b>: 11.4&nbsp<b>Coverage</b>: 66',organ:"WHOLE_ORGANISM",integrated:!1,coverage:66,publication_year:2008,num_abundances:18049,hasPeptideCounts:!0,filename:"3702-countCastellana.txt"},{id:412,name:"A.thaliana - Whole organism, SC (Baerenfaller,Science,2008)",score:9.38,weight:"50%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/18436743" target="_blank">Baerenfaller, Science,2008</a><br/><b>Interaction consistency score</b>: 9.38&nbsp<b>Coverage</b>: 47',organ:"WHOLE_ORGANISM",integrated:!1,coverage:47,publication_year:2008,num_abundances:12987,hasPeptideCounts:!0,filename:"3702-peptide.Baerenfaller_2008_new.txt"}],total_coverage:76,average_coverage:19.87878787878788,topProteins:[{id:142540,abundance:53821,rank:0,name:"RBCL",annotation:"Ribulose-bisphosphate carboxylases.[Source:TAIR;Acc:ATCG00490]; RuBisCO catalyzes two reactions: the carboxylation of D- ribulose 1,5-bisphosphate, the primary event in carbon dioxide fixation, as well as the oxidative fragmentation of the pentose substrate in the photorespiration process. Both reactions occur simultaneously and in competition at the same active site"},{id:142557,abundance:21976,rank:1,name:"PSBB",annotation:"Photosystem II reaction center protein B; One of the components of the core complex of photosystem II (PSII). It binds chlorophyll and helps catalyze the primary light-induced photochemical processes of PSII. PSII is a light- driven water:plastoquinone oxidoreductase, using light energy to abstract electrons from H(2)O, generating O(2) and a proton gradient subsequently used for ATP formation"},{id:125723,abundance:17456,rank:2,name:"RCA",annotation:"Rubisco activase; Activation of RuBisCO (ribulose-1,5-bisphosphate carboxylase/oxygenase; EC 4.1.1.39) involves the ATP-dependent carboxylation of the epsilon-amino group of lysine leading to a carbamate structure"},{id:142527,abundance:17227,rank:3,name:"PSBD",annotation:"PSII D2 protein; Photosystem II (PSII) is a light-driven water: plastoquinone oxidoreductase that uses light energy to abstract electrons from H(2)O, generating O(2) and a proton gradient subsequently used for ATP formation. It consists of a core antenna complex that captures photons, and an electron transfer chain that converts photonic excitation into a charge separation. The D1/D2 (PsbA/PsbA) reaction center heterodimer binds P680, the primary electron donor of PSII as well as several subsequent electron acceptors. D2 is needed for assembly of a stable PSII complex"},{id:142511,abundance:9734,rank:4,name:"PSBA",annotation:"Photosystem II reaction center protein A; Photosystem II (PSII) is a light-driven water: plastoquinone oxidoreductase that uses light energy to abstract electrons from H(2)O, generating O(2) and a proton gradient subsequently used for ATP formation. It consists of a core antenna complex that captures photons, and an electron transfer chain that converts photonic excitation into a charge separation. The D1/D2 (PsbA/PsbA) reaction center heterodimer binds P680, the primary electron donor of PSII as well as several subsequent electron acceptors"},{id:142528,abundance:8737,rank:5,name:"PSBC",annotation:"Photosystem II reaction center protein C; One of the components of the core complex of photosystem II (PSII). It binds chlorophyll and helps catalyze the primary light-induced photochemical processes of PSII. PSII is a light- driven water:plastoquinone oxidoreductase, using light energy to abstract electrons from H(2)O, generating O(2) and a proton gradient subsequently used for ATP formation"},{id:136068,abundance:8176,rank:6,name:"FBA2",annotation:"Fructose-bisphosphate aldolase 2"},{id:142539,abundance:7678,rank:7,name:"PB",annotation:"ATP synthase subunit beta; Produces ATP from ADP in the presence of a proton gradient across the membrane. The catalytic sites are hosted primarily by the beta subunits"}]},4081:{id:4081,name:"Solanum lycopersicum",compact_name:"Solanum lycopersicum",num_proteins:34675,genome_source:"<a href='http://plants.ensembl.org/Solanum_lycopersicum//Info/Index?db=core'>Ensembl Plants</a>",genome_source_version:"release 17",datasets:[{id:192,name:"S.lycopersicum - Whole organism, SC (Schilmiller,Plant_Physiol,2010)",score:10.79,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/20431087" target="_blank">Schilmiller, Plant_Physiol,2010</a><br/><b>Interaction consistency score</b>: 10.79&nbsp<b>Coverage</b>: 6',organ:"WHOLE_ORGANISM",integrated:!1,coverage:6,publication_year:2010,num_abundances:2158,hasPeptideCounts:!0,filename:"4081-Solanum_lycopersicum_SC_biomart_18197_E__1reps.txt"}],total_coverage:6,average_coverage:6,topProteins:[{id:249839,abundance:29073,rank:0,name:"Solyc04g005700.2.1",annotation:"Uncharacterized protein "},{id:238953,abundance:20070,rank:1,name:"loxC",annotation:"Lipoxygenase ; Plant lipoxygenase may be involved in a number of diverse aspects of plant physiology including growth and development, pest resistance, and senescence or responses to wounding"},{id:249952,abundance:17158,rank:2,name:"Solyc04g007820.2.1",annotation:"Uncharacterized protein "},{id:260283,abundance:11913,rank:3,name:"PHS1",annotation:"Terpene synthase (EC:4.2.3.51); Monoterpene synthase catalyzing the production of beta- phellandrene from neryl diphosphate. Produces also lower amounts of delta-2-carene, alpha-phellandrene and limonene. When incubated in vitro with geranyl diphosphate, catalyzes the formation of acyclic myrcene and ocimene as major products in addition to beta- phellandrene"},{id:254488,abundance:9103,rank:4,name:"Solyc05g052240.2.1",annotation:"Chalcone-flavonone isomerase family protein "},{id:263943,abundance:7789,rank:5,name:"Solyc09g059170.1.1",annotation:"Uncharacterized protein "},{id:244533,abundance:7394,rank:6,name:"Solyc02g071710.2.1",annotation:"Uncharacterized protein "},{id:258101,abundance:7029,rank:7,name:"Solyc07g008710.2.1",annotation:"Uncharacterized protein "}]},4577:{id:4577,name:"Zea mays",compact_name:"Zea mays",num_proteins:92413,genome_source:"<a href='http://plants.ensembl.org/Zea_mays//Info/Index?db=core'>Ensembl Plants</a>",genome_source_version:"release 17",datasets:[{id:65,name:"Z.mays - Leaf (Integrated)",score:3.93,weight:"",description:"integrated dataset: weighted average of all Z.mays LEAF datasets<br/><b>Interaction consistency score</b>: 3.93&nbsp<b>Coverage</b>: 18",organ:"LEAF",integrated:!0,coverage:18,publication_year:2017,num_abundances:16910,hasPeptideCounts:!1,filename:"4577-LEAF-integrated.txt"},{id:426,name:"Z.mays - Leaf, Mature, SC (Facette,PlantCell,2013)",score:1.78,weight:"100%",description:'abundance based on Spectral counting, mature, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23933881" target="_blank">Facette,Plant Cell,2013</a><br/><b>Interaction consistency score</b>: 1.78&nbsp<b>Coverage</b>: 9',organ:"LEAF",integrated:!1,coverage:9,publication_year:2013,num_abundances:8414,hasPeptideCounts:!0,filename:"4577-Mays_SC_mature_rep1234_Facette_2013.txt"},{id:261,name:"Z.mays - Leaf, Zone1, SC (Facette,PlantCell,2013)",score:2.85,weight:"100%",description:'abundance based on Spectral counting, zone1, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23933881" target="_blank">Facette,Plant Cell,2013</a><br/><b>Interaction consistency score</b>: 2.85&nbsp<b>Coverage</b>: 14',organ:"LEAF",integrated:!1,coverage:14,publication_year:2013,num_abundances:13356,hasPeptideCounts:!0,filename:"4577-Mays_SC_zone1_rep1234_Facette_2013.txt"},{id:186,name:"Z.mays - Leaf, Zone2, SC (Facette,PlantCell,2013)",score:3.12,weight:"100%",description:'abundance based on Spectral counting, zone2, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23933881" target="_blank">Facette,Plant Cell,2013</a><br/><b>Interaction consistency score</b>: 3.12&nbsp<b>Coverage</b>: 15',organ:"LEAF",integrated:!1,coverage:15,publication_year:2013,num_abundances:14310,hasPeptideCounts:!0,filename:"4577-Mays_SC_zone2_rep1234_Facette_2013.txt"},{id:449,name:"Z.mays - Leaf, Zone3, SC (Facette,PlantCell,2013)",score:.14,weight:"100%",description:'abundance based on Spectral counting, zone3, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23933881" target="_blank">Facette,Plant Cell,2013</a><br/><b>Interaction consistency score</b>: 0.14&nbsp<b>Coverage</b>: 16',organ:"LEAF",integrated:!1,coverage:16,publication_year:2013,num_abundances:14427,hasPeptideCounts:!0,filename:"4577-Mays_SC_zone3_rep1234_Facette_2013.txt"},{id:487,name:"Z.mays - Leaf, NSAF (Ponnala,Plant_J,2014)",score:9.67,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, NSAF, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=24547885&retmode=ref&cmd=prlinks" target="_blank">Ponnala,Plant_J,2014</a><br/><b>Interaction consistency score</b>: 9.67&nbsp<b>Coverage</b>: 3',organ:"LEAF",integrated:!1,coverage:3,publication_year:2014,num_abundances:2750,hasPeptideCounts:!1,filename:"4577-Mays_avgNSAF_ponnala_2014.txt"},{id:475,name:"Z.mays - Leaf, NadjSPC (Ponnala,Plant_J,2014)",score:6.87,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, NadjSPC, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=24547885&retmode=ref&cmd=prlinks" target="_blank">Ponnala,Plant_J,2014</a><br/><b>Interaction consistency score</b>: 6.87&nbsp<b>Coverage</b>: 3',organ:"LEAF",integrated:!1,coverage:3,publication_year:2014,num_abundances:2750,hasPeptideCounts:!1,filename:"4577-Mays_avgNadjSPC_ponnala_2014.txt"}],total_coverage:18,average_coverage:10,topProteins:[{id:519745,abundance:10652,rank:0,name:"Zm.158284",annotation:"Histone H1 "},{id:522071,abundance:10314,rank:1,name:"psb3",annotation:"Photosystem II3 "},{id:494438,abundance:7386,rank:2,name:"GRMZM2G047813_P01",annotation:"Histone H2A ; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:484684,abundance:7313,rank:3,name:"H1",annotation:"Histone H1 ; Histones H1 are necessary for the condensation of nucleosome chains into higher-order structures"},{id:498002,abundance:7251,rank:4,name:"glp1",annotation:"Hypothetical protein LOC100272933 "},{id:484741,abundance:7183,rank:5,name:"GRMZM2G003306_P01",annotation:"Histone H2A "},{id:510452,abundance:6543,rank:6,name:"Zm.86959",annotation:"Histone H1 "},{id:482617,abundance:5053,rank:7,name:"AC216889.3_FGP008",annotation:"Annotation not available"}]},4896:{id:4896,name:"Schizosaccharomyces pombe",compact_name:"Schizosaccharomyces pombe",num_proteins:5144,genome_source:"<a href='http://fungi.ensembl.org/Schizosaccharomyces_pombe//Info/Index?db=core'>Ensembl Fungi</a>",genome_source_version:"release 17",datasets:[{id:12,name:"S.pombe - Whole organism, SC (GPM,Aug,2014)",score:16.34,weight:"50%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 16.34&nbsp<b>Coverage</b>: 77',organ:"WHOLE_ORGANISM",integrated:!1,coverage:77,publication_year:2014,num_abundances:3979,hasPeptideCounts:!0,filename:"4896-GPM_201408.txt"},{id:48,name:"S.pombe - Whole organism, SC (Mancuso,J-Proteomics,2012)",score:14.61,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22728601" target="_blank">Mancuso,J-Proteomics,2012</a><br/><b>Interaction consistency score</b>: 14.61&nbsp<b>Coverage</b>: 20',organ:"WHOLE_ORGANISM",integrated:!1,coverage:20,publication_year:2012,num_abundances:1018,hasPeptideCounts:!0,filename:"4896-SchizosaccharomycesPombe_FissionYeast_PRIDE.txt"},{id:473,name:"S.pombe - Whole organism, SC (Marguerat,Cell,2012)",score:22.72,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.cell.com/abstract/S0092-8674%2812%2901126-9" target="_blank">Marguerat,Cell,2012</a><br/><b>Interaction consistency score</b>: 22.72&nbsp<b>Coverage</b>: 80',organ:"WHOLE_ORGANISM",integrated:!1,coverage:80,publication_year:2012,num_abundances:4104,hasPeptideCounts:!0,filename:"4896-Schizosaccharomyces_pombe_SC_biomart_29627_E_BTO0000214cellculture_1reps.txt"},{id:286,name:"S.pombe - Whole organism, EmPAI (Gunaratne,MCP,2013)",score:4.28,weight:"10%",description:'abundance based on MAPPED_BY_AUTHORS, emPAI, from<a href="http://www.mcponline.org/content/12/6/1741.abstract" target="_blank">Gunaratne ,MCP,2013</a><br/><b>Interaction consistency score</b>: 4.28&nbsp<b>Coverage</b>: 79',organ:"WHOLE_ORGANISM",integrated:!1,coverage:79,publication_year:2013,num_abundances:4040,hasPeptideCounts:!1,filename:"4896-Spombe_emPAI_Gunaratne_2013.txt"},{id:250,name:"S.pombe - Whole organism (Integrated)",score:18.49,weight:"",description:"integrated dataset: weighted average of all S.pombe WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 18.49&nbsp<b>Coverage</b>: 90",organ:"WHOLE_ORGANISM",integrated:!0,coverage:90,publication_year:2017,num_abundances:4637,hasPeptideCounts:!1,filename:"4896-WHOLE_ORGANISM-integrated.txt"},{id:442,name:"S.pombe - Whole organism, Copy per cell, cpc normalized data (Marguerat,Cell,2012)",score:8.59,weight:"20%",description:'abundance based on cpc normalized data, copy per cell, from<a href="http://www.cell.com/abstract/S0092-8674%2812%2901126-9" target="_blank">Marguerat,Cell,2012</a><br/><b>Interaction consistency score</b>: 8.59&nbsp<b>Coverage</b>: 58',organ:"WHOLE_ORGANISM",integrated:!1,coverage:58,publication_year:2012,num_abundances:2990,hasPeptideCounts:!1,filename:"4896-pombe_cell2012_protein.txt"},{id:387,name:"S.pombe - Whole organism, Proliferating (Marguerat,Cell,2012)",score:9.18,weight:"20%",description:'abundance based on MAPPED_BY_AUTHORS, proliferating, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=23101633&retmode=ref&cmd=prlinks" target="_blank">Marguerat,Cell,2012</a><br/><b>Interaction consistency score</b>: 9.18&nbsp<b>Coverage</b>: 77',organ:"WHOLE_ORGANISM",integrated:!1,coverage:77,publication_year:2012,num_abundances:3986,hasPeptideCounts:!1,filename:"4896-proliferating_protID_copynr_Marguerat_2012.txt"},{id:429,name:"S.pombe - Whole organism, Quiescent (Marguerat,Cell,2012)",score:10.34,weight:"70%",description:'abundance based on MAPPED_BY_AUTHORS, quiescent, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=23101633&retmode=ref&cmd=prlinks" target="_blank">Marguerat,Cell,2012</a><br/><b>Interaction consistency score</b>: 10.34&nbsp<b>Coverage</b>: 68',organ:"WHOLE_ORGANISM",integrated:!1,coverage:68,publication_year:2012,num_abundances:3489,hasPeptideCounts:!1,filename:"4896-quiescent_protID_copynr_Marguerat_2012.txt"}],total_coverage:90,average_coverage:65.57142857142857,topProteins:[{id:617884,abundance:16649,rank:0,name:"cid14",annotation:"poly(A) polymerase Cid14; Required for 3' polyadenylation of the 5.8S and 25S rRNAs as a prelude ot their degradation in the exosome. Involved in the nucleolar organization to ensure faithful chromosome segregation during mitosis"},{id:621808,abundance:13306,rank:1,name:"air1",annotation:"Zinc knuckle TRAMP complex subunit Air1; Component of the TRAMP (TRF4) and TRAMP5 complexes which have a poly(A) RNA polymerase activity and are involved in a post- transcriptional quality control mechanism limiting inappropriate expression of genetic information. Polyadenylation is required for the degradative activity of the exosome on several of its nuclear RNA substrates like cryptic transcripts generated by RNA polymerase II and III, or hypomethylated pre-tRNAi-Met. Both complexes polyadenylate RNA processing and degradation intermediates of snRNAs, snoRNAs and mRNAs that accumula [...] "},{id:622683,abundance:12296,rank:2,name:"cbf11",annotation:"CBF1/Su(H)/LAG-1 family transcription factor Cbf11; Transcription factor that behaves as a negative regulator of adhesion. Recognizes specifically the canonical CSL response element GTGA/GGAA. May also play a cbf12-antagonistic role in the regulation of a number of other important processes such as extracellular material production, colony morphogenesis, ploidy maintenance, or meiosis"},{id:621740,abundance:11568,rank:3,name:"SPBC947.10",annotation:"Golgi Dsc E3 ligase complex subunit Dsc1; Catalytic component of the DSC E3 ubiquitin ligase complex which is required for the sre1 transcriptional activator proteolytic cleavage to release the soluble transcription factor from the membrane in low oxygen or sterol conditions. The complex plays also an important role in the multivesicular body (MVB) pathway and functions in a post-endoplasmic reticulum pathway for protein degradation"},{id:619938,abundance:9259,rank:4,name:"paa1",annotation:"Protein phosphatase regulatory subunit Paa1; Phosphatase 2A affects a variety of biological processes in the cell such as transcription, cell cycle progression and cellular morphogenesis, and provides an initial identification of critical substrates for this phosphatase. The regulatory subunit may direct the catalytic subunit to distinct, albeit overlapping, subsets of substrates (By similarity)"},{id:620565,abundance:9183,rank:5,name:"SPBC1773.01.1",annotation:"SIP/FAR complex striatin subunit, Far8/Csc3"},{id:619095,abundance:8364,rank:6,name:"rpl3802",annotation:"60S ribosomal protein L38 (predicted)"},{id:621151,abundance:7122,rank:7,name:"tdh1",annotation:"Glyceraldehyde-3-phosphate dehydrogenase Tdh1"}]},4932:{id:4932,name:"Saccharomyces cerevisiae",compact_name:"Saccharomyces cerevisiae",num_proteins:6692,genome_source:"<a href='http://www.ensembl.org/Saccharomyces_cerevisiae//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:450,name:"S.cerevisiae - Whole organism, SC (GPM,Oct,2012)",score:15.55,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 15.55&nbsp<b>Coverage</b>: 90',organ:"WHOLE_ORGANISM",integrated:!1,coverage:90,publication_year:2012,num_abundances:5991,hasPeptideCounts:!0,filename:"4932-GPM_2012_09_Saccharomyces_cerevisiae.txt"},{id:243,name:"S.cerevisiae - Whole organism, SC (GPM,Aug,2014)",score:22.88,weight:"90%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 22.88&nbsp<b>Coverage</b>: 79',organ:"WHOLE_ORGANISM",integrated:!1,coverage:79,publication_year:2014,num_abundances:5289,hasPeptideCounts:!0,filename:"4932-GPM_201408.txt"},{id:358,name:"S.cerevisiae - Whole organism, Immunodetection of tagged proteins, normalized data (Ghaemmaghami,Nature,2003)",score:3.81,weight:"50%",description:'abundance based on normalized data, immunodetection of tagged proteins, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/14562106" target="_blank">Ghaemmaghami,Nature,2003</a><br/><b>Interaction consistency score</b>: 3.81&nbsp<b>Coverage</b>: 58',organ:"WHOLE_ORGANISM",integrated:!1,coverage:58,publication_year:2003,num_abundances:3862,hasPeptideCounts:!1,filename:"4932-Ghaemmaghami_et_al_yeast_data.txt"},{id:25,name:"S.cerevisiae - Whole organism, Intensity of GFP-tagged proteins, SD normalized data (Newman,Nature,2006)",score:5.05,weight:"80%",description:'abundance based on SD normalized data, intensity of GFP-tagged proteins, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16699522" target="_blank">Newman,Nature,2006</a><br/><b>Interaction consistency score</b>: 5.05&nbsp<b>Coverage</b>: 37',organ:"WHOLE_ORGANISM",integrated:!1,coverage:37,publication_year:2006,num_abundances:2450,hasPeptideCounts:!1,filename:"4932-Newman_et_al_yeast_data_SD.txt"},{id:300,name:"S.cerevisiae - Whole organism, Intensity of GFP-tagged proteins, YEPD normalized data (Newman,Nature,2006)",score:6.41,weight:"90%",description:'abundance based on YEPD normalized data, intensity of GFP-tagged proteins, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16699522" target="_blank">Newman,Nature,2006</a><br/><b>Interaction consistency score</b>: 6.41&nbsp<b>Coverage</b>: 38',organ:"WHOLE_ORGANISM",integrated:!1,coverage:38,publication_year:2006,num_abundances:2526,hasPeptideCounts:!1,filename:"4932-Newman_et_al_yeast_data_YEPD.txt"},{id:242,name:"S.cerevisiae - Whole organism, SC (PeptideAtlas,March,2013)",score:21.86,weight:"40%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,March,2013</a><br/><b>Interaction consistency score</b>: 21.86&nbsp<b>Coverage</b>: 72',organ:"WHOLE_ORGANISM",integrated:!1,coverage:72,publication_year:2013,num_abundances:4828,hasPeptideCounts:!0,filename:"4932-PA_201303.txt"},{id:331,name:"S.cerevisiae - Whole organism, SC (Piening,J_Proteome_Res,2006)",score:17.94,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16823959" target="_blank">Piening,J_Proteome_Res,2006</a><br/><b>Interaction consistency score</b>: 17.94&nbsp<b>Coverage</b>: 25',organ:"WHOLE_ORGANISM",integrated:!1,coverage:25,publication_year:2006,num_abundances:1691,hasPeptideCounts:!0,filename:"4932-Saccharomyces_cerevisiae_SC_biomart_17916_O__1reps.txt"},{id:110,name:"S.cerevisiae - Whole organism, SC (Serikawa,MCP,2003)",score:1.93,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/12684541" target="_blank">Serikawa,MCP,2003</a><br/><b>Interaction consistency score</b>: 1.93&nbsp<b>Coverage</b>: 20',organ:"WHOLE_ORGANISM",integrated:!1,coverage:20,publication_year:2003,num_abundances:1341,hasPeptideCounts:!0,filename:"4932-Saccharomyces_cerevisiae_SC_biomart_18097_O__1reps.txt"},{id:3,name:"S.cerevisiae - Whole organism (Integrated)",score:21.24,weight:"",description:"integrated dataset: weighted average of all S.cerevisiae WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 21.24&nbsp<b>Coverage</b>: 96",organ:"WHOLE_ORGANISM",integrated:!0,coverage:96,publication_year:2017,num_abundances:6440,hasPeptideCounts:!1,filename:"4932-WHOLE_ORGANISM-integrated.txt"},{id:16,name:"S.cerevisiae - Whole organism, SC (PeptideAtlas,May,2009)",score:20.36,weight:"100%",description:'abundance based on Spectral counting, from<a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/defaultBuildsPepsProts?_subtab=3" target="_blank">PeptideAtlas,May,2009</a><br/><b>Interaction consistency score</b>: 20.36&nbsp<b>Coverage</b>: 62',organ:"WHOLE_ORGANISM",integrated:!1,coverage:62,publication_year:2009,num_abundances:4163,hasPeptideCounts:!0,filename:"4932-Yeast_2009-0522.txt"},{id:106,name:"S.cerevisiae - Whole organism, SC (Breker,J_Cell_Biol.,2013)",score:12.99,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23509072" target="_blank">Breker,J_Cell_Biol.,2013</a><br/><b>Interaction consistency score</b>: 12.99&nbsp<b>Coverage</b>: 78',organ:"WHOLE_ORGANISM",integrated:!1,coverage:78,publication_year:2013,num_abundances:5242,hasPeptideCounts:!1,filename:"4932-Yeast_GeneName_Intensity_Breker_2103.txt"},{id:203,name:"S.cerevisiae - Whole organism, YMD normalized data (Lu,Nat_Biotechnol,2007)",score:8.04,weight:"30%",description:'abundance based on YMD normalized data, from<a href="http://www.nature.com/nbt/journal/v25/n1/full/nbt1270.html" target="_blank">Lu,Nat_Biotechnol,2007</a><br/><b>Interaction consistency score</b>: 8.04&nbsp<b>Coverage</b>: 16',organ:"WHOLE_ORGANISM",integrated:!1,coverage:16,publication_year:2007,num_abundances:1075,hasPeptideCounts:!1,filename:"4932-Yeast_Lu_2007_YMD.txt"},{id:285,name:"S.cerevisiae - Whole organism, YPD normalized data (Lu,Nat_Biotechnol,2007)",score:5.24,weight:"20%",description:'abundance based on YPD normalized data, from<a href="http://www.nature.com/nbt/journal/v25/n1/full/nbt1270.html" target="_blank">Lu,Nat_Biotechnol,2007</a><br/><b>Interaction consistency score</b>: 5.24&nbsp<b>Coverage</b>: 16',organ:"WHOLE_ORGANISM",integrated:!1,coverage:16,publication_year:2007,num_abundances:1089,hasPeptideCounts:!1,filename:"4932-Yeast_Lu_2007_YPD.txt"},{id:61,name:"S.cerevisiae - Whole organism, SC (Denervaud,PNAS,2013)",score:14.02,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.pnas.org/content/110/39/15842.abstract" target="_blank">Denervaud,PNAS,2013</a><br/><b>Interaction consistency score</b>: 14.02&nbsp<b>Coverage</b>: 61',organ:"WHOLE_ORGANISM",integrated:!1,coverage:61,publication_year:2013,num_abundances:4078,hasPeptideCounts:!1,filename:"4932-Yeast_ORF_intensity_Denervaud_2013.txt"},{id:40,name:"S.cerevisiae - Whole organism, MS/MS abundance data, SILAC normalized data (De_Godoy,Nature,2008)",score:13.62,weight:"90%",description:'abundance based on SILAC normalized data, MS/MS abundance data, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/18820680" target="_blank">de_Godoy, Nature,2008</a><br/><b>Interaction consistency score</b>: 13.62&nbsp<b>Coverage</b>: 60',organ:"WHOLE_ORGANISM",integrated:!1,coverage:60,publication_year:2008,num_abundances:4030,hasPeptideCounts:!1,filename:"4932-deGodoy_et_al_yeast_data.txt"},{id:128,name:"S.cerevisiae - Whole organism, SC (PeptideAtlas,April,2009)",score:18.13,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/yeast/index.php" target="_blank">PeptideAtlas,April,2009</a><br/><b>Interaction consistency score</b>: 18.13&nbsp<b>Coverage</b>: 60',organ:"WHOLE_ORGANISM",integrated:!1,coverage:60,publication_year:2009,num_abundances:4023,hasPeptideCounts:!0,filename:"4932-peptidesatalas_build_April_2009_yeast.new.txt"},{id:184,name:"S.cerevisiae - Whole organism, Intensity (Nagaraj,MCP,2012)",score:8.75,weight:"90%",description:'abundance based on MAPPED_BY_AUTHORS, intensity, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/22021278" target="_blank">Nagaraj,MCP,2012</a><br/><b>Interaction consistency score</b>: 8.75&nbsp<b>Coverage</b>: 59',organ:"WHOLE_ORGANISM",integrated:!1,coverage:59,publication_year:2012,num_abundances:3957,hasPeptideCounts:!1,filename:"4932-yeast_Nagaraj_2012.txt"}],total_coverage:96,average_coverage:51.9375,topProteins:[{id:632514,abundance:20543,rank:0,name:"FBA1",annotation:"Fructose 1,6-bisphosphate aldolase, required for glycolysis and gluconeogenesis; catalyzes conversion of fructose 1,6 bisphosphate to glyceraldehyde-3-P and dihydroxyacetone-P; locates to mitochondrial outer surface upon oxidative stress; Catalyzes the aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3- phosphate (G3P) to form fructose 1,6-bisphosphate (FBP) in gluconeogenesis and the reverse reaction in glycolysis"},{id:631742,abundance:15998,rank:1,name:"ENO2",annotation:"Enolase II, a phosphopyruvate hydratase that catalyzes the conversion of 2-phosphoglycerate to phosphoenolpyruvate during glycolysis and the reverse reaction during gluconeogenesis; expression is induced in response to glucose"},{id:629459,abundance:15913,rank:2,name:"PGK1",annotation:"3-phosphoglycerate kinase, catalyzes transfer of high-energy phosphoryl groups from the acyl phosphate of 1,3-bisphosphoglycerate to ADP to produce ATP; key enzyme in glycolysis and gluconeogenesis"},{id:628841,abundance:14914,rank:3,name:"CDC19",annotation:"Pyruvate kinase, functions as a homotetramer in glycolysis to convert phosphoenolpyruvate to pyruvate, the input for aerobic (TCA cycle) or anaerobic (glucose fermentation) respiration"},{id:631369,abundance:13635,rank:4,name:"TDH3",annotation:"Glyceraldehyde-3-phosphate dehydrogenase, isozyme 3, involved in glycolysis and gluconeogenesis; tetramer that catalyzes the reaction of glyceraldehyde-3-phosphate to 1,3 bis-phosphoglycerate; detected in the cytoplasm and cell wall"},{id:632913,abundance:11472,rank:5,name:"PDC1",annotation:"Major of three pyruvate decarboxylase isozymes, key enzyme in alcoholic fermentation, decarboxylates pyruvate to acetaldehyde; subject to glucose-, ethanol-, and autoregulation; involved in amino acid catabolism; Major of three pyruvate decarboxylases (PDC1, PDC5, PDC6) implicated in the nonoxidative conversion of pyruvate to acetaldehyde and carbon dioxide during alcoholic fermentation. Most of the produced acetaldehyde is subsequently reduced to ethanol, but some is required for cytosolic acetyl-CoA production for biosynthetic pathways. The enzyme is also one of five 2-oxo acid decar [...] "},{id:632822,abundance:10528,rank:6,name:"SSA2",annotation:"ATP binding protein involved in protein folding and vacuolar import of proteins; member of heat shock protein 70 (HSP70) family; associated with the chaperonin-containing T-complex; present in the cytoplasm, vacuolar membrane and cell wall; 98% iden /.../with Ssa1p, but subtle differences between the two proteins provide functional specificity with respect to propagation of yeast [URE3] prions and vacuolar-mediated degradations of gluconeogenesis enzymes; May play a role in the transport of polypeptides both across the mitochondrial membranes and into the endoplasmic reticulum. A funct [...] "},{id:630895,abundance:10216,rank:7,name:"PMA1",annotation:"Plasma membrane H+-ATPase, pumps protons out of the cell; major regulator of cytoplasmic pH and plasma membrane potential; P2-type ATPase; Hsp30p plays a role in Pma1p regulation; interactions with Std1p appear to propagate [GAR+]; The plasma membrane ATPase of plants and fungi is a hydrogen ion pump. The proton gradient it generates drives the active transport of nutrients by H(+)-symport. The resulting external acidification and/or internal alkinization may mediate growth responses"}]},5061:{id:5061,name:"Aspergillus niger",compact_name:"Aspergillus niger",num_proteins:14068,genome_source:"<a href='http://fungi.ensembl.org/Aspergillus_niger//Info/Index?db=core'>Ensembl Fungi</a>",genome_source_version:"release 17",datasets:[{id:420,name:"A.niger - Whole organism, SC (GPM,Aug,2014)",score:4.07,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 4.07&nbsp<b>Coverage</b>: 6',organ:"WHOLE_ORGANISM",integrated:!1,coverage:6,publication_year:2014,num_abundances:800,hasPeptideCounts:!0,filename:"5061-GPM_201408.txt"}],total_coverage:5,average_coverage:6,topProteins:[{id:698136,abundance:61492,rank:0,name:"pep1",annotation:"Aspartic protease pep1; Secreted aspartic endopeptidase that allows assimilation of proteinaceous substrates. Can catalyze hydrolysis of the major structural proteins of basement membrane, elastin, collagen, and laminin (By similarity)"},{id:698636,abundance:57461,rank:1,name:"abfB",annotation:"Probable alpha-N-arabinofuranosidase B; Alpha-L-arabinofuranosidase involved in the degradation of arabinoxylan, a major component of plant hemicellulose. Able to hydrolyze 1,5-, 1,3- and 1,2-alpha-linkages not only in L- arabinofuranosyl oligosaccharides, but also in polysaccharides containing terminal non-reducing L-arabinofuranoses in side chains, like L-arabinan, arabinogalactan and arabinoxylan (By similarity)"},{id:694472,abundance:45201,rank:2,name:"abnA",annotation:"Probable arabinan endo-1,5-alpha-L-arabinosidase A; Endo-1,5-alpha-L-arabinanase involved in degradation of pectin. Its preferred substrate is linear 1,5-alpha-L-arabinan (By similarity)"},{id:689125,abundance:37950,rank:3,name:"An02g07010",annotation:"Remark: a splice site was detected upstream of the START codon"},{id:693845,abundance:29959,rank:4,name:"An08g06940",annotation:"Histone H4; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:688951,abundance:26945,rank:5,name:"An02g05240",annotation:"Histone H4; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:699248,abundance:25753,rank:6,name:"An16g00670",annotation:"Similarity to hypothetical protein AAO51454.1 - Dictyostelium discoideum"},{id:687244,abundance:22870,rank:7,name:"An01g02880",annotation:"Putative uncharacterized protein An01g02880"}]},5691:{id:5691,name:"Trypanosoma brucei",compact_name:"Trypanosoma brucei",num_proteins:8747,genome_source:"<a href='http://protists.ensembl.org/Trypanosoma_brucei//Info/Index?db=core'>Ensembl Protists</a>",genome_source_version:"release 17",datasets:[{id:198,name:"T.brucei - Whole organism, SC (GPM,Aug,2014)",score:18.52,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 18.52&nbsp<b>Coverage</b>: 65',organ:"WHOLE_ORGANISM",integrated:!1,coverage:65,publication_year:2014,num_abundances:5722,hasPeptideCounts:!0,filename:"5691-GPM_201408.txt"}],total_coverage:65,average_coverage:65,topProteins:[{id:827976,abundance:28922,rank:0,name:"AAZ13290",annotation:"60S ribosomal protein L39, putative"},{id:828492,abundance:11726,rank:1,name:"CAJ16364",annotation:"Alpha tubulin"},{id:828490,abundance:11726,rank:2,name:"CAJ16362",annotation:"Alpha tubulin"},{id:828488,abundance:11726,rank:3,name:"CAJ16360",annotation:"Alpha tubulin"},{id:828494,abundance:11726,rank:4,name:"CAJ16366",annotation:"Alpha tubulin; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:828487,abundance:10986,rank:5,name:"CAJ16358",annotation:"Beta tubulin"},{id:828489,abundance:10986,rank:6,name:"CAJ16361",annotation:"Beta tubulin"},{id:828493,abundance:10986,rank:7,name:"CAJ16365",annotation:"Beta tubulin; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"}]},5833:{id:5833,name:"Plasmodium falciparum",compact_name:"Plasmodium falciparum",num_proteins:5429,genome_source:"<a href='http://protists.ensembl.org/Plasmodium_falciparum//Info/Index?db=core'>Ensembl Protists</a>",genome_source_version:"release 17",datasets:[{id:50,name:"P.falciparum - Whole organism, Dd2, SC (Tao,MCP,2014)",score:2.86,weight:"50%",description:'abundance based on Spectral counting, Dd2, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25056935" target="_blank">Tao,MCP,2014</a><br/><b>Interaction consistency score</b>: 2.86&nbsp<b>Coverage</b>: 80',organ:"WHOLE_ORGANISM",integrated:!1,coverage:80,publication_year:2014,num_abundances:4351,hasPeptideCounts:!0,filename:"5833-Dd2_Tao_2014.txt"},{id:388,name:"P.falciparum - Whole organism, SC (GPM,Aug,2014)",score:13.55,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 13.55&nbsp<b>Coverage</b>: 47',organ:"WHOLE_ORGANISM",integrated:!1,coverage:47,publication_year:2014,num_abundances:2575,hasPeptideCounts:!0,filename:"5833-GPM_201408.txt"},{id:311,name:"P.falciparum - Whole organism, GiRBC, SC (Tao,MCP,2014)",score:3.04,weight:"10%",description:'abundance based on Spectral counting, GiRBC, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25056935" target="_blank">Tao,MCP,2014</a><br/><b>Interaction consistency score</b>: 3.04&nbsp<b>Coverage</b>: 88',organ:"WHOLE_ORGANISM",integrated:!1,coverage:88,publication_year:2014,num_abundances:4782,hasPeptideCounts:!0,filename:"5833-GiRBC_Tao_2014.txt"},{id:246,name:"P.falciparum - Whole organism, SC (PeptideAtlas,Sep,2012)",score:13.9,weight:"60%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Sep,2012</a><br/><b>Interaction consistency score</b>: 13.9&nbsp<b>Coverage</b>: 34',organ:"WHOLE_ORGANISM",integrated:!1,coverage:34,publication_year:2012,num_abundances:1868,hasPeptideCounts:!0,filename:"5833-PA_201209.txt"},{id:131,name:"P.falciparum - Whole organism (Integrated)",score:7.28,weight:"",description:"integrated dataset: weighted average of all P.falciparum WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 7.28&nbsp<b>Coverage</b>: 95",organ:"WHOLE_ORGANISM",integrated:!0,coverage:95,publication_year:2017,num_abundances:5143,hasPeptideCounts:!1,filename:"5833-WHOLE_ORGANISM-integrated.txt"}],total_coverage:94,average_coverage:62.25,topProteins:[{id:933989,abundance:30385,rank:0,name:"PFB0100c",annotation:"Knob-associated histidine-rich protein"},{id:934706,abundance:21509,rank:1,name:"PfSBP1",annotation:"Skeleton-binding protein 1"},{id:933635,abundance:16194,rank:2,name:"GAPDH",annotation:"Glyceraldehyde-3-phosphate dehydrogenase"},{id:931896,abundance:13200,rank:3,name:"PF10_0084",annotation:"Tubulin beta chain; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:932262,abundance:12154,rank:4,name:"PF11_0061",annotation:"Histone H4; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:932263,abundance:10381,rank:5,name:"PF11_0062",annotation:"Histone H2B"},{id:933025,abundance:9542,rank:6,name:"PF13_0346",annotation:"60S ribosomal protein L40/UBI, putative"},{id:934507,abundance:8150,rank:7,name:"PFD0310w",annotation:"Sexual stage-specific protein"}]},6239:{id:6239,name:"Caenorhabditis elegans",compact_name:"Caenorhabditis elegans",num_proteins:20517,genome_source:"<a href='http://www.ensembl.org/Caenorhabditis_elegans//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:49,name:"C.elegans - Whole organism, SC (Merrihew,Genome_Res,2008)",score:5.65,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://genome.cshlp.org/content/18/10/1660.long" target="_blank">Merrihew,Genome_Res, 2008</a><br/><b>Interaction consistency score</b>: 5.65&nbsp<b>Coverage</b>: 32',organ:"WHOLE_ORGANISM",integrated:!1,coverage:32,publication_year:2008,num_abundances:6623,hasPeptideCounts:!0,filename:"6239-Celegans_Merrihew2008_GenomeRes_peptide.txt"},{id:425,name:"C.elegans - Whole organism, SC (GPM,Oct,2012)",score:7.2,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct, 2012</a><br/><b>Interaction consistency score</b>: 7.2&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2012,num_abundances:10015,hasPeptideCounts:!0,filename:"6239-GPM_2012_09_Caenorhabditis_elegans.txt"},{id:366,name:"C.elegans - Whole organism, SC (GPM,Aug,2014)",score:9.86,weight:"40%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 9.86&nbsp<b>Coverage</b>: 32',organ:"WHOLE_ORGANISM",integrated:!1,coverage:32,publication_year:2014,num_abundances:6648,hasPeptideCounts:!0,filename:"6239-GPM_201408.txt"},{id:58,name:"C.elegans - Whole organism, SC (PeptideAtlas,Sep,2013)",score:16.07,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Sep,2013</a><br/><b>Interaction consistency score</b>: 16.07&nbsp<b>Coverage</b>: 45',organ:"WHOLE_ORGANISM",integrated:!1,coverage:45,publication_year:2013,num_abundances:9332,hasPeptideCounts:!0,filename:"6239-PA_201309.txt"},{id:204,name:"C.elegans - Whole organism, SC (PeptideAtlas,Aug,2011)",score:13.24,weight:"90%",description:'abundance based on Spectral counting, from<a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/defaultBuildsPepsProts?_subtab=3" target="_blank">PeptideAtlas,Aug,2011</a><br/><b>Interaction consistency score</b>: 13.24&nbsp<b>Coverage</b>: 42',organ:"WHOLE_ORGANISM",integrated:!1,coverage:42,publication_year:2011,num_abundances:8718,hasPeptideCounts:!0,filename:"6239-Spectral_counting_C.elegans_PeptideAtals_Build_Aug_2011.txt"},{id:380,name:"C.elegans - Whole organism (Integrated)",score:23.77,weight:"",description:"integrated dataset: weighted average of all C.elegans WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 23.77&nbsp<b>Coverage</b>: 61",organ:"WHOLE_ORGANISM",integrated:!0,coverage:61,publication_year:2017,num_abundances:12418,hasPeptideCounts:!1,filename:"6239-WHOLE_ORGANISM-integrated.txt"},{id:114,name:"C.elegans - Whole organism, SC (PeptideAtlas,May,2008)",score:13.53,weight:"80%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/celegans/index.php" target="_blank">PeptideAtlas,May,2008</a><br/><b>Interaction consistency score</b>: 13.53&nbsp<b>Coverage</b>: 42',organ:"WHOLE_ORGANISM",integrated:!1,coverage:42,publication_year:2008,num_abundances:8718,hasPeptideCounts:!0,filename:"6239-celegans_atalas_build_247.new.txt"},{id:181,name:"C.elegans - Whole organism, Dauer, SC (Schrimpf,PLoS_Biology,2009)",score:6.11,weight:"80%",description:'abundance based on Spectral counting, Dauer, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19260763" target="_blank">Schrimpf, PLoS _Biology,2009</a><br/><b>Interaction consistency score</b>: 6.11&nbsp<b>Coverage</b>: 29',organ:"WHOLE_ORGANISM",integrated:!1,coverage:29,publication_year:2009,num_abundances:6047,hasPeptideCounts:!0,filename:"6239-new_celegans_Dauer_peptides.new.txt"},{id:282,name:"C.elegans - Whole organism, L1, SC (Schrimpf,PLoS_Biology,2009)",score:6.49,weight:"100%",description:'abundance based on Spectral counting, L1, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19260763" target="_blank">Schrimpf, PLoS _Biology,2009</a><br/><b>Interaction consistency score</b>: 6.49&nbsp<b>Coverage</b>: 30',organ:"WHOLE_ORGANISM",integrated:!1,coverage:30,publication_year:2009,num_abundances:6138,hasPeptideCounts:!0,filename:"6239-new_celegans_L1_peptides.new.txt"},{id:22,name:"C.elegans - Whole organism, Embryo, SC (Schrimpf,PLoS_Biology,2009)",score:4.72,weight:"90%",description:'abundance based on Spectral counting, embryo, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19260763" target="_blank">Schrimpf, PLoS _Biology,2009</a><br/><b>Interaction consistency score</b>: 4.72&nbsp<b>Coverage</b>: 25',organ:"WHOLE_ORGANISM",integrated:!1,coverage:25,publication_year:2009,num_abundances:5108,hasPeptideCounts:!0,filename:"6239-new_celegans_embryo_peptides.new.txt"}],total_coverage:60,average_coverage:36.22222222222222,topProteins:[{id:1038884,abundance:6418,rank:0,name:"rps-18",annotation:"Ribosomal Protein, Small subunit family member (rps-18) "},{id:1021511,abundance:6132,rank:1,name:"rpl-14",annotation:"Ribosomal Protein, Large subunit family member (rpl-14) "},{id:1035347,abundance:6038,rank:2,name:"T24B8.3",annotation:"Protein T24B8.3, isoform a "},{id:1040246,abundance:5684,rank:3,name:"ubq-2",annotation:"Ubiquitin-60S ribosomal protein L40 Ubiquitin 60S ribosomal protein L40; Ubiquitin: exists either covalently attached to another protein, or free (unanchored). When covalently bound, it is conjugated to target proteins via an isopeptide bond either as a monomer (monoubiquitin), a polymer linked via different Lys residues of the ubiquitin (polyubiquitin chains) or a linear polymer linked via the initiator Met of the ubiquitin (linear polyubiquitin chains). Polyubiquitin chains, when attached to a target protein, have different functions depending on the Lys residue of the ubiquitin that [...] "},{id:1039058,abundance:5451,rank:4,name:"rla-2",annotation:"Ribosomal protein, Large subunit, Acidic (P1) family member (rla-2) "},{id:1023111,abundance:5228,rank:5,name:"rpl-39",annotation:"60S ribosomal protein L39 "},{id:1037848,abundance:5191,rank:6,name:"rpl-18",annotation:"60S ribosomal protein L18 "},{id:1033231,abundance:5170,rank:7,name:"rpl-28",annotation:"60S ribosomal protein L28 "}]},7165:{id:7165,name:"Anopheles gambiae",compact_name:"Anopheles gambiae",num_proteins:12810,genome_source:"<a href='http://metazoa.ensembl.org/Anopheles_gambiae//Info/Index?db=core'>Ensembl Metazoa</a>",genome_source_version:"release 17",datasets:[{id:369,name:"A.gambiae - Whole organism, SC (GPM,Aug,2014)",score:14.34,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 14.34&nbsp<b>Coverage</b>: 34',organ:"WHOLE_ORGANISM",integrated:!1,coverage:34,publication_year:2014,num_abundances:4339,hasPeptideCounts:!0,filename:"7165-GPM_201408.txt"}],total_coverage:33,average_coverage:34,topProteins:[{id:1192344,abundance:20964,rank:0,name:"AGAP002858-PC",annotation:"Sodium/potassium-transporting ATPase subunit alpha "},{id:1196982,abundance:16194,rank:1,name:"AGAP007927-PA",annotation:"AGAP007927-PA "},{id:1190732,abundance:15829,rank:2,name:"AGAP001053-PA",annotation:"AGAP001053-PA "},{id:1190381,abundance:15294,rank:3,name:"Act5C",annotation:"Actin-5C ; Actins are highly conserved proteins that are involved in various types of cell motility and are ubiquitously expressed in all eukaryotic cells. Multiple isoforms are involved in various cellular functions such as cytoskeleton structure, cell mobility, chromosome movement and muscle contraction"},{id:1196848,abundance:14714,rank:4,name:"AGAP007790-PA",annotation:"AGAP007790-PA "},{id:1200626,abundance:14167,rank:5,name:"AGAP011802-PA",annotation:"AGAP011802-PA "},{id:1200356,abundance:13225,rank:6,name:"AGAP011516-PA",annotation:"AGAP011516-PA "},{id:1191878,abundance:13188,rank:7,name:"AGAP002350-PB",annotation:"AGAP002350-PB "}]},7227:{id:7227,name:"Drosophila melanogaster",compact_name:"Drosophila melanogaster",num_proteins:13937,genome_source:"<a href='http://www.ensembl.org/Drosophila_melanogaster//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:10,name:"D.melanogaster - Whole organism, SC (PeptideAtlas,Sep,2011)",score:8.82,weight:"10%",description:'abundance based on Spectral counting, from<a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/defaultBuildsPepsProts?_subtab=3" target="_blank">PeptideAtlas,Sep,2011</a><br/><b>Interaction consistency score</b>: 8.82&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2011,num_abundances:6811,hasPeptideCounts:!0,filename:"7227-Drosophila_2011-09_Ens62.txt"},{id:390,name:"D.melanogaster - Whole organism, SC (Xing,Proteomics,2014)",score:16.49,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24259522" target="_blank">Xing,Proteomics, 2014</a><br/><b>Interaction consistency score</b>: 16.49&nbsp<b>Coverage</b>: 60',organ:"WHOLE_ORGANISM",integrated:!1,coverage:60,publication_year:2014,num_abundances:8368,hasPeptideCounts:!0,filename:"7227-Drosophila_melanogaster_SC_biomart_31126_O_BTO0001489wholebody_119reps.txt"},{id:53,name:"D.melanogaster - Whole organism, SC (GPM,Aug,2014)",score:14.47,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 14.47&nbsp<b>Coverage</b>: 61',organ:"WHOLE_ORGANISM",integrated:!1,coverage:61,publication_year:2014,num_abundances:8529,hasPeptideCounts:!0,filename:"7227-GPM_201408.txt"},{id:1,name:"D.melanogaster - Whole organism, SC (PeptideAtlas,Aug,2012)",score:6.48,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2012</a><br/><b>Interaction consistency score</b>: 6.48&nbsp<b>Coverage</b>: 40',organ:"WHOLE_ORGANISM",integrated:!1,coverage:40,publication_year:2012,num_abundances:5532,hasPeptideCounts:!0,filename:"7227-PA_201208.txt"},{id:220,name:"D.melanogaster - Whole organism (Integrated)",score:5.43,weight:"",description:"integrated dataset: weighted average of all D.melanogaster WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 5.43&nbsp<b>Coverage</b>: 95",organ:"WHOLE_ORGANISM",integrated:!0,coverage:95,publication_year:2017,num_abundances:13264,hasPeptideCounts:!1,filename:"7227-WHOLE_ORGANISM-integrated.txt"},{id:252,name:"D.melanogaster - Whole organism, SC (Brunner,Nat_Biotechnol,2007)",score:10.43,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/17450130" target="_blank">Brunner, Nat_Biotechnol, 2007</a><br/><b>Interaction consistency score</b>: 10.43&nbsp<b>Coverage</b>: 62',organ:"WHOLE_ORGANISM",integrated:!1,coverage:62,publication_year:2007,num_abundances:8704,hasPeptideCounts:!0,filename:"7227-drosophila.peptides_Brunner_2007.txt"},{id:296,name:"D.melanogaster - Whole organism, SC (PeptideAtlas,July,2009)",score:8.28,weight:"60%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/drosophila//index.php" target="_blank">PeptideAtlas,July,2009</a><br/><b>Interaction consistency score</b>: 8.28&nbsp<b>Coverage</b>: 48',organ:"WHOLE_ORGANISM",integrated:!1,coverage:48,publication_year:2009,num_abundances:6741,hasPeptideCounts:!0,filename:"7227-drosophila_peptideatlas_July_2009.new.txt"},{id:105,name:"D.melanogaster - Whole organism, SC (PeptideAtlas,June,2010)",score:9.1,weight:"90%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/drosophila//index.php" target="_blank">PeptideAtlas,June,2010</a><br/><b>Interaction consistency score</b>: 9.1&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2010,num_abundances:6796,hasPeptideCounts:!0,filename:"7227-drosophila_peptideatlas_June_2010.new.txt"},{id:384,name:"D.melanogaster - Whole organism, Normal embyro, SC (PRIDE,PRD00007,2009)",score:12.96,weight:"50%",description:'abundance based on Spectral counting, normal embyro, from<a href="http://www.ebi.ac.uk/pride/" target="_blank">PRIDE,PRD00007,2009</a><br/><b>Interaction consistency score</b>: 12.96&nbsp<b>Coverage</b>: 34',organ:"WHOLE_ORGANISM",integrated:!1,coverage:34,publication_year:2009,num_abundances:4697,hasPeptideCounts:!0,filename:"7227-fruitFly1_normalEmbryo_PRIDE.txt"},{id:20,name:"D.melanogaster - Whole organism, Earlier embyro, SC (PRIDE,PRD000083,2013)",score:3.01,weight:"10%",description:'abundance based on Spectral counting, earlier embyro, from<a href="http://www.ebi.ac.uk/pride/" target="_blank">PRIDE,PRD000083,2013</a><br/><b>Interaction consistency score</b>: 3.01&nbsp<b>Coverage</b>: 92',organ:"WHOLE_ORGANISM",integrated:!1,coverage:92,publication_year:2013,num_abundances:12875,hasPeptideCounts:!0,filename:"7227-fruitFly2_earlyEmbryoCell_PRIDE.txt"}],total_coverage:95,average_coverage:55,topProteins:[{id:1252006,abundance:27905,rank:0,name:"RpL35",annotation:"Ribosomal protein L35"},{id:1256485,abundance:14179,rank:1,name:"CG17108",annotation:"CG17108 "},{id:1264959,abundance:8239,rank:2,name:"F1",annotation:"Elongation factor 1alpha48D; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:1252987,abundance:7969,rank:3,name:"RpL23A",annotation:"Ribosomal protein L23A"},{id:1260973,abundance:6941,rank:4,name:"RpLP2",annotation:"Ribosomal protein LP2; Plays an important role in the elongation step of protein synthesis"},{id:1265225,abundance:6738,rank:5,name:"l(1)G0230",annotation:"Lethal (1) G0230"},{id:1256689,abundance:6432,rank:6,name:"RpL24",annotation:"Ribosomal protein L24"},{id:1260117,abundance:6428,rank:7,name:"bic",annotation:"Bicaudal"}]},7460:{id:7460,name:"Apis mellifera",compact_name:"Apis mellifera",num_proteins:10694,genome_source:"<a href='http://metazoa.ensembl.org/Apis_mellifera//Info/Index?db=core'>Ensembl Metazoa</a>",genome_source_version:"release 17",datasets:[{id:265,name:"A.mellifera - Whole organism, SC (PeptideAtlas,Sep,2013)",score:14.61,weight:"90%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Sep,2013</a><br/><b>Interaction consistency score</b>: 14.61&nbsp<b>Coverage</b>: 30',organ:"WHOLE_ORGANISM",integrated:!1,coverage:30,publication_year:2013,num_abundances:3186,hasPeptideCounts:!0,filename:"7460-PA_201309.txt"},{id:104,name:"A.mellifera - Whole organism (Integrated)",score:15.68,weight:"",description:"integrated dataset: weighted average of all A.mellifera WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 15.68&nbsp<b>Coverage</b>: 31",organ:"WHOLE_ORGANISM",integrated:!0,coverage:31,publication_year:2017,num_abundances:3319,hasPeptideCounts:!1,filename:"7460-WHOLE_ORGANISM-integrated.txt"},{id:51,name:"A.mellifera - Whole organism, SC (PeptideAtlas,March,2010)",score:14.47,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/honeybee//index.php" target="_blank">PeptideAtlas, March,2010</a><br/><b>Interaction consistency score</b>: 14.47&nbsp<b>Coverage</b>: 27',organ:"WHOLE_ORGANISM",integrated:!1,coverage:27,publication_year:2010,num_abundances:2878,hasPeptideCounts:!0,filename:"7460-honeybee_atlas_build_282_new.txt"}],total_coverage:31,average_coverage:28.5,topProteins:[{id:1348958,abundance:37582,rank:0,name:"Hex110",annotation:"High Glx storage protein"},{id:1345473,abundance:23216,rank:1,name:"HEX70b",annotation:"HexamerinHexamerin 70b"},{id:1354989,abundance:13796,rank:2,name:"LOC552283",annotation:"Uncharacterized protein "},{id:1347199,abundance:13535,rank:3,name:"CPR2",annotation:"Cuticular protein 2"},{id:1349481,abundance:12304,rank:4,name:"Mrjp1",annotation:"Major royal jelly protein 1Jellein-1Jellein-2Jellein-4; Major royal jelly protein 1: induces the differentiation of honeybee larvae into queens through an Egfr-mediated signaling pathway. Promotes body size increase by activating p70 S6 kinase, stimulates ovary development by augmenting the titer of vitellogenin (Vg) and juvenile hormone, and reduces developmental time by increasing the activity of mitogen-activated protein kinase and inducing the 20-hydroxyecdysone protein (20E). Most abundant protein found in the royal jelly which is the food of the queen honey bee larva. The royal j [...] "},{id:1350835,abundance:12261,rank:5,name:"Mrjp2",annotation:"Major royal jelly protein 2; May play an important role in honeybee nutrition. It is found in the royal jelly which is the food of the queen honey bee larva. The royal jelly determines the development of the young larvae and is responsible for the high reproductive ability of the honeybee queen"},{id:1351046,abundance:11571,rank:6,name:"Mrjp3",annotation:"Major royal jelly protein 3; May play an important role in honeybee nutrition. It is found in the royal jelly which is the food of the queen honey bee larva. The royal jelly determines the development of the young larvae and is responsible for the high reproductive ability of the honeybee queen"},{id:1349981,abundance:11277,rank:7,name:"LOC727045",annotation:"Uncharacterized protein "}]},7955:{id:7955,name:"Danio rerio",compact_name:"Danio rerio",num_proteins:26163,genome_source:"<a href='http://www.ensembl.org/Danio_rerio//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:264,name:"D.rerio - Embryo (Integrated)",score:15.06,weight:"",description:"integrated dataset: weighted average of all D.rerio EMBRYO datasets<br/><b>Interaction consistency score</b>: 15.06&nbsp<b>Coverage</b>: 10",organ:"EMBRYO",integrated:!0,coverage:10,publication_year:2017,num_abundances:2665,hasPeptideCounts:!1,filename:"7955-EMBRYO-integrated.txt"},{id:151,name:"D.rerio - Whole organism, SC (GPM,Oct,2012)",score:14.04,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct, 2012</a><br/><b>Interaction consistency score</b>: 14.04&nbsp<b>Coverage</b>: 35',organ:"WHOLE_ORGANISM",integrated:!1,coverage:35,publication_year:2012,num_abundances:9216,hasPeptideCounts:!0,filename:"7955-GPM_2012_09_Danio_rerio.txt"},{id:165,name:"D.rerio - Whole organism, SC (GPM,Aug,2014)",score:12.36,weight:"50%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 12.36&nbsp<b>Coverage</b>: 25',organ:"WHOLE_ORGANISM",integrated:!1,coverage:25,publication_year:2014,num_abundances:6459,hasPeptideCounts:!0,filename:"7955-GPM_201408.txt"},{id:362,name:"D.rerio - Whole organism, SC (PeptideAtlas,March,2014)",score:20.61,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,March,2014</a><br/><b>Interaction consistency score</b>: 20.61&nbsp<b>Coverage</b>: 43',organ:"WHOLE_ORGANISM",integrated:!1,coverage:43,publication_year:2014,num_abundances:11374,hasPeptideCounts:!0,filename:"7955-PA_201403.txt"},{id:240,name:"D.rerio - Whole organism (Integrated)",score:27.49,weight:"",description:"integrated dataset: weighted average of all D.rerio WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 27.49&nbsp<b>Coverage</b>: 57",organ:"WHOLE_ORGANISM",integrated:!0,coverage:57,publication_year:2017,num_abundances:14901,hasPeptideCounts:!1,filename:"7955-WHOLE_ORGANISM-integrated.txt"},{id:121,name:"D.rerio - Brain, SC (Kelkar,MCP,2014)",score:15.58,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 15.58&nbsp<b>Coverage</b>: 13',organ:"BRAIN",integrated:!1,coverage:13,publication_year:2014,num_abundances:3369,hasPeptideCounts:!0,filename:"7955-ZF_Brain_Kelkar_2014_SEQUEST.txt"},{id:38,name:"D.rerio - Embryo, 120hr, SC (Kelkar,MCP,2014)",score:11.32,weight:"100%",description:'abundance based on Spectral counting, 120hr, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 11.32&nbsp<b>Coverage</b>: 9',organ:"EMBRYO",integrated:!1,coverage:9,publication_year:2014,num_abundances:2336,hasPeptideCounts:!0,filename:"7955-ZF_Embryo120hr_Kelkar_2014_SEQUEST.txt"},{id:397,name:"D.rerio - Embryo, 48hr, SC (Kelkar,MCP,2014)",score:12.72,weight:"100%",description:'abundance based on Spectral counting, 48hr, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 12.72&nbsp<b>Coverage</b>: 7',organ:"EMBRYO",integrated:!1,coverage:7,publication_year:2014,num_abundances:1744,hasPeptideCounts:!0,filename:"7955-ZF_Embryo48hrs_Kelkar_2014_SEQUEST.txt"},{id:281,name:"D.rerio - Eye, SC (Kelkar,MCP,2014)",score:10.77,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 10.77&nbsp<b>Coverage</b>: 7',organ:"EYE",integrated:!1,coverage:7,publication_year:2014,num_abundances:1723,hasPeptideCounts:!0,filename:"7955-ZF_Eye_Kelkar_2014_SEQUEST.txt"},{id:469,name:"D.rerio - Whole organism, Pepxml, SC (Kelkar,MCP,2014)",score:10.96,weight:"30%",description:'abundance based on Spectral counting, pepxml, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 10.96&nbsp<b>Coverage</b>: 5',organ:"WHOLE_ORGANISM",integrated:!1,coverage:5,publication_year:2014,num_abundances:1197,hasPeptideCounts:!0,filename:"7955-ZF_GrossBody_Kelkar_2014_SEQUEST.txt"},{id:145,name:"D.rerio - Head, SC (Kelkar,MCP,2014)",score:15.5,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 15.5&nbsp<b>Coverage</b>: 9',organ:"HEAD",integrated:!1,coverage:9,publication_year:2014,num_abundances:2414,hasPeptideCounts:!0,filename:"7955-ZF_Head_Kelkar_2014_SEQUEST.txt"},{id:288,name:"D.rerio - Heart, SC (Kelkar,MCP,2014)",score:12.9,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 12.9&nbsp<b>Coverage</b>: 6',organ:"HEART",integrated:!1,coverage:6,publication_year:2014,num_abundances:1461,hasPeptideCounts:!0,filename:"7955-ZF_Heart_Kelkar_2014_SEQUEST.txt"},{id:295,name:"D.rerio - Pancreas, SC (Kelkar,MCP,2014)",score:10.87,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 10.87&nbsp<b>Coverage</b>: 8',organ:"PANCREAS",integrated:!1,coverage:8,publication_year:2014,num_abundances:1992,hasPeptideCounts:!0,filename:"7955-ZF_IntPanc_Kelkar_2014_SEQUEST.txt"},{id:308,name:"D.rerio - Kidney, SC (Kelkar,MCP,2014)",score:8.44,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 8.44&nbsp<b>Coverage</b>: 9',organ:"KIDNEY",integrated:!1,coverage:9,publication_year:2014,num_abundances:2433,hasPeptideCounts:!0,filename:"7955-ZF_Kidney_Kelkar_2014_SEQUEST.txt"},{id:332,name:"D.rerio - Liver, SC (Kelkar,MCP,2014)",score:12.55,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 12.55&nbsp<b>Coverage</b>: 9',organ:"LIVER",integrated:!1,coverage:9,publication_year:2014,num_abundances:2243,hasPeptideCounts:!0,filename:"7955-ZF_Liver_Kelkar_2014_SEQUEST.txt"},{id:290,name:"D.rerio - Muscle, SC (Kelkar,MCP,2014)",score:16.85,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 16.85&nbsp<b>Coverage</b>: 6',organ:"MUSCLE",integrated:!1,coverage:6,publication_year:2014,num_abundances:1450,hasPeptideCounts:!0,filename:"7955-ZF_Muscle_Kelkar_2014_SEQUEST.txt"},{id:54,name:"D.rerio - Female gonad, SC (Kelkar,MCP,2014)",score:13.36,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 13.36&nbsp<b>Coverage</b>: 8',organ:"FEMALE_GONAD",integrated:!1,coverage:8,publication_year:2014,num_abundances:2100,hasPeptideCounts:!0,filename:"7955-ZF_Ovary_Kelkar_2014_SEQUEST.txt"},{id:445,name:"D.rerio - Spleen, SC (Kelkar,MCP,2014)",score:13.7,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 13.7&nbsp<b>Coverage</b>: 9',organ:"SPLEEN",integrated:!1,coverage:9,publication_year:2014,num_abundances:2313,hasPeptideCounts:!0,filename:"7955-ZF_Spleen_Kelkar_2014_SEQUEST.txt"},{id:100,name:"D.rerio - Testis, SC (Kelkar,MCP,2014)",score:14.05,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 14.05&nbsp<b>Coverage</b>: 16',organ:"TESTIS",integrated:!1,coverage:16,publication_year:2014,num_abundances:4259,hasPeptideCounts:!0,filename:"7955-ZF_Testis_Kelkar_2014_SEQUEST.txt"},{id:255,name:"D.rerio - Whole organism, SC (Kelkar,MCP,2014)",score:12.12,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060758" target="_blank">Kelkar,MCP,2014</a><br/><b>Interaction consistency score</b>: 12.12&nbsp<b>Coverage</b>: 21',organ:"WHOLE_ORGANISM",integrated:!1,coverage:21,publication_year:2014,num_abundances:5551,hasPeptideCounts:!0,filename:"7955-zf_velos_Kelkar_2014_SEQUEST.txt"}],total_coverage:59,average_coverage:13.61111111111111,topProteins:[{id:1493245,abundance:29932,rank:0,name:"ba1",annotation:"Ba1 globin; Involved in oxygen transport from gills to the various peripheral tissues"},{id:1496158,abundance:21184,rank:1,name:"hbaa1",annotation:"Hemoglobin alpha adult-1; Involved in oxygen transport from gills to the various peripheral tissues"},{id:1490075,abundance:7376,rank:2,name:"nme2b.1",annotation:"Non-metastatic cells 2b.2, protein (NM23B) expressed in"},{id:1488083,abundance:7212,rank:3,name:"gapdh",annotation:"Glyceraldehyde-3-phosphate dehydrogenase; Has both glyceraldehyde-3-phosphate dehydrogenase and nitrosylase activities, thereby playing a role in glycolysis and nuclear functions, respectively. Glyceraldehyde-3-phosphate dehydrogenase is a key enzyme in glycolysis that catalyzes the first step of the pathway by converting D-glyceraldehyde 3- phosphate (G3P) into 3-phospho-D-glyceroyl phosphate. Modulates the organization and assembly of the cytoskeleton. Also participates in nuclear events including transcription, RNA transport, DNA replication and apoptosis. Nuclear functions are prob [...] "},{id:1485929,abundance:6814,rank:4,name:"vtg1",annotation:"Vitellogenin 1"},{id:1501849,abundance:6179,rank:5,name:"apoa1b",annotation:"Apolipoprotein A-I"},{id:1497962,abundance:6135,rank:6,name:"tnnt3b",annotation:"Troponin T3b, skeletal, fast"},{id:1484819,abundance:5815,rank:7,name:"ckma",annotation:"Creatine kinase, muscle a"}]},8364:{id:8364,name:"Xenopus (Silurana) tropicalis",compact_name:"Xenopus Silurana",num_proteins:18429,genome_source:"<a href='http://www.ensembl.org/Xenopus_(Silurana)_tropicalis//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:92,name:"X.tropicalis - Whole organism, SC (GPM,Oct,2012)",score:4.47,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 4.47&nbsp<b>Coverage</b>: 11',organ:"WHOLE_ORGANISM",integrated:!1,coverage:11,publication_year:2012,num_abundances:2013,hasPeptideCounts:!0,filename:"8364-GPM_2012_09_Xenopus_tropicalis.txt"}],total_coverage:10,average_coverage:11,topProteins:[{id:1602965,abundance:33335,rank:0,name:"actg1",annotation:"Actin, gamma 1; Actins are highly conserved proteins that are involved in various types of cell motility and are ubiquitously expressed in all eukaryotic cells"},{id:1589953,abundance:26754,rank:1,name:"ps1tp5bp1",annotation:"Actin, beta; Actins are highly conserved proteins that are involved in various types of cell motility and are ubiquitously expressed in all eukaryotic cells"},{id:1599594,abundance:17149,rank:2,name:"rps18",annotation:"Ribosomal protein S18"},{id:1602449,abundance:14978,rank:3,name:"gapdh",annotation:"Glyceraldehyde-3-phosphate dehydrogenase "},{id:1598827,abundance:14588,rank:4,name:"rps16",annotation:"Ribosomal protein S16"},{id:1597868,abundance:14409,rank:5,name:"rpl35",annotation:"Ribosomal protein L35"},{id:1601560,abundance:13849,rank:6,name:"lmna",annotation:"Lamin A/C"},{id:1594920,abundance:11866,rank:7,name:"nme2",annotation:"Non-metastatic cells 2, protein (NM23B) expressed in"}]},9031:{id:9031,name:"Gallus gallus",compact_name:"Gallus gallus",num_proteins:16736,genome_source:"<a href='http://www.ensembl.org/Gallus_gallus//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:234,name:"G.gallus - Whole organism, SC (GPM,Oct,2012)",score:3.35,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 3.35&nbsp<b>Coverage</b>: 72',organ:"WHOLE_ORGANISM",integrated:!1,coverage:72,publication_year:2012,num_abundances:11978,hasPeptideCounts:!0,filename:"9031-GPM_2012_09_Gallus_gallus.txt"},{id:461,name:"G.gallus - Whole organism, SC (GPM,Aug,2014)",score:9.43,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 9.43&nbsp<b>Coverage</b>: 27',organ:"WHOLE_ORGANISM",integrated:!1,coverage:27,publication_year:2014,num_abundances:4480,hasPeptideCounts:!0,filename:"9031-GPM_201408.txt"},{id:274,name:"G.gallus - Whole organism (Integrated)",score:2.75,weight:"",description:"integrated dataset: weighted average of all G.gallus WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 2.75&nbsp<b>Coverage</b>: 74",organ:"WHOLE_ORGANISM",integrated:!0,coverage:74,publication_year:2017,num_abundances:12318,hasPeptideCounts:!1,filename:"9031-WHOLE_ORGANISM-integrated.txt"}],total_coverage:73,average_coverage:49.5,topProteins:[{id:1611494,abundance:85596,rank:0,name:"ENSGALG00000005921",annotation:"12K serum protein, beta-2-m cross-reactive; Uncharacterized protein "},{id:1620686,abundance:41119,rank:1,name:"SPINK5",annotation:"Ovoinhibitor precursor ; Seems to have at least five active inhibitory domains; two for trypsin, two for chymotrypsin and one for elastase"},{id:1621655,abundance:18036,rank:2,name:"HBE1",annotation:"Hemoglobin subunit beta ; Involved in oxygen transport from the lung to the various peripheral tissues. The beta chain is a component of adult hemoglobin A and D"},{id:1620050,abundance:14453,rank:3,name:"LOC417953",annotation:"Histone H3.2 ; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:1622443,abundance:14453,rank:4,name:"ENSGALG00000023228",annotation:"Histone H3.2 ; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:1622436,abundance:14453,rank:5,name:"ENSGALG00000023220",annotation:"Histone H3.2 "},{id:1615581,abundance:14001,rank:6,name:"ENSGALG00000023219",annotation:"Histone H3.2 "},{id:1615514,abundance:14e3,rank:7,name:"ALB",annotation:"Serum albumin precursor ; Serum albumin, the main protein of plasma, has a good binding capacity for water, Ca(2+), Na(+), K(+), fatty acids, hormones, bilirubin and drugs. Its main function is the regulation of the colloidal osmotic pressure of blood"}]},9598:{id:9598,name:"Pan troglodytes",compact_name:"Pan troglodytes",num_proteins:18759,genome_source:"<a href='http://www.ensembl.org/Pan_troglodytes//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:241,name:"P.troglodytes - Whole organism, IBAQ (Khan,Science,2013)",score:8.05,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, iBAQ, from<a href="http://www.sciencemag.org/content/342/6162/1100.abstract" target="_blank">Khan,Science,2013</a><br/><b>Interaction consistency score</b>: 8.05&nbsp<b>Coverage</b>: 15',organ:"WHOLE_ORGANISM",integrated:!1,coverage:15,publication_year:2013,num_abundances:2893,hasPeptideCounts:!1,filename:"9598-Chimp_iBAQ_Khan_2013.txt"}],total_coverage:15,average_coverage:15,topProteins:[{id:1805524,abundance:40177,rank:0,name:"GAPDH",annotation:"PREDICTED: Pan troglodytes glyceraldehyde-3-phosphate dehydrogenase-like (LOC739038), miscRNA"},{id:1807811,abundance:29565,rank:1,name:"LOC748477",annotation:"Profilin 1"},{id:1805848,abundance:16106,rank:2,name:"ATP5B",annotation:"ATP synthase, H+ transporting, mitochondrial F1 complex, beta polypeptide; Produces ATP from ADP in the presence of a proton gradient across the membrane"},{id:1814544,abundance:10342,rank:3,name:"FABP5",annotation:"Fatty acid binding protein 5 (psoriasis-associated)"},{id:1803813,abundance:10264,rank:4,name:"TAGLN2",annotation:"Transgelin 2"},{id:1807042,abundance:9309,rank:5,name:"PKM2",annotation:"Pyruvate kinase, muscle"},{id:1818548,abundance:8811,rank:6,name:"SNRPE",annotation:"PREDICTED: Pan troglodytes small nuclear ribonucleoprotein E-like (LOC739189), miscRNA"},{id:1803313,abundance:7305,rank:7,name:"PRDX1",annotation:"Peroxiredoxin 1"}]},9606:{id:9606,name:"Homo sapiens",compact_name:"Homo sapiens",num_proteins:20457,genome_source:"<a href='http://www.ensembl.org/Homo_sapiens//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:99,name:"H.sapiens - Cell line, A431 (Lundberg,MSB,2010)",score:7.86,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, A431, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21179022" target="_blank">lundberg,MSB,2010</a><br/><b>Interaction consistency score</b>: 7.86&nbsp<b>Coverage</b>: 24',organ:"CELL_LINE",integrated:!1,coverage:24,publication_year:2010,num_abundances:5388,hasPeptideCounts:!1,filename:"9606-A431_cellline_Lundberg_2010.txt"},{id:172,name:"H.sapiens - Cell line, A549, SC (Geiger,MCP,2012)",score:5.57,weight:"100%",description:'abundance based on Spectral counting, A549, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 5.57&nbsp<b>Coverage</b>: 17',organ:"CELL_LINE",integrated:!1,coverage:17,publication_year:2012,num_abundances:3921,hasPeptideCounts:!0,filename:"9606-A549_Geiger_2012.txt"},{id:299,name:"H.sapiens - Adrenal gland, SC (Kim,Nature,2014)",score:18.02,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 18.02&nbsp<b>Coverage</b>: 36',organ:"ADRENAL_GLAND",integrated:!1,coverage:36,publication_year:2014,num_abundances:8303,hasPeptideCounts:!0,filename:"9606-Adult_Adrenalgland_Kim_2014_SEQUEST.txt"},{id:293,name:"H.sapiens - B cell, SC (Kim,Nature,2014)",score:21.46,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 21.46&nbsp<b>Coverage</b>: 41',organ:"B_CELL",integrated:!1,coverage:41,publication_year:2014,num_abundances:9356,hasPeptideCounts:!0,filename:"9606-Adult_Bcells_Kim_2014_SEQUEST.txt"},{id:71,name:"H.sapiens - Cell line, CD4, SC (Kim,Nature,2014)",score:17.91,weight:"100%",description:'abundance based on Spectral counting, CD4, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 17.91&nbsp<b>Coverage</b>: 34',organ:"CELL_LINE",integrated:!1,coverage:34,publication_year:2014,num_abundances:7705,hasPeptideCounts:!0,filename:"9606-Adult_CD4Tcells_Kim_2014_SEQUEST.txt"},{id:350,name:"H.sapiens - Cell line, CD8, SC (Kim,Nature,2014)",score:25.23,weight:"100%",description:'abundance based on Spectral counting, CD8, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 25.23&nbsp<b>Coverage</b>: 48',organ:"CELL_LINE",integrated:!1,coverage:48,publication_year:2014,num_abundances:11072,hasPeptideCounts:!0,filename:"9606-Adult_CD8Tcells_Kim_2014_SEQUEST.txt"},{id:96,name:"H.sapiens - Colon, SC (Kim,Nature,2014)",score:14.42,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 14.42&nbsp<b>Coverage</b>: 35',organ:"COLON",integrated:!1,coverage:35,publication_year:2014,num_abundances:8045,hasPeptideCounts:!0,filename:"9606-Adult_Colon_Kim_2014_SEQUEST.txt"},{id:108,name:"H.sapiens - Esophagus, SC (Kim,Nature,2014)",score:12.63,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 12.63&nbsp<b>Coverage</b>: 22',organ:"ESOPHAGUS",integrated:!1,coverage:22,publication_year:2014,num_abundances:5067,hasPeptideCounts:!0,filename:"9606-Adult_Esophagus_Kim_2014_SEQUEST.txt"},{id:480,name:"H.sapiens - Frontal cortex, SC (Kim,Nature,2014)",score:18.77,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 18.77&nbsp<b>Coverage</b>: 41',organ:"FRONTAL_CORTEX",integrated:!1,coverage:41,publication_year:2014,num_abundances:9437,hasPeptideCounts:!0,filename:"9606-Adult_Frontalcortex_Kim_2014_SEQUEST.txt"},{id:45,name:"H.sapiens - Gall bladder, SC (Kim,Nature,2014)",score:8.58,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 8.58&nbsp<b>Coverage</b>: 32',organ:"GALL_BLADDER",integrated:!1,coverage:32,publication_year:2014,num_abundances:7279,hasPeptideCounts:!0,filename:"9606-Adult_Gallbladder_Kim_2014_SEQUEST.txt"},{id:44,name:"H.sapiens - Heart, SC (Kim,Nature,2014)",score:19.1,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 19.1&nbsp<b>Coverage</b>: 29',organ:"HEART",integrated:!1,coverage:29,publication_year:2014,num_abundances:6537,hasPeptideCounts:!0,filename:"9606-Adult_Heart_Kim_2014_SEQUEST.txt"},{id:170,name:"H.sapiens - Kidney, SC (Kim,Nature,2014)",score:18.02,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 18.02&nbsp<b>Coverage</b>: 30',organ:"KIDNEY",integrated:!1,coverage:30,publication_year:2014,num_abundances:6813,hasPeptideCounts:!0,filename:"9606-Adult_Kidney_Kim_2014_SEQUEST.txt"},{id:212,name:"H.sapiens - Liver, SC (Kim,Nature,2014)",score:19.07,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 19.07&nbsp<b>Coverage</b>: 38',organ:"LIVER",integrated:!1,coverage:38,publication_year:2014,num_abundances:8757,hasPeptideCounts:!0,filename:"9606-Adult_Liver_Kim_2014_SEQUEST.txt"},{id:267,name:"H.sapiens - Lung, SC (Kim,Nature,2014)",score:14.13,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 14.13&nbsp<b>Coverage</b>: 29',organ:"LUNG",integrated:!1,coverage:29,publication_year:2014,num_abundances:6652,hasPeptideCounts:!0,filename:"9606-Adult_Lung_Kim_2014_SEQUEST.txt"},{id:4,name:"H.sapiens - Monocyte, SC (Kim,Nature,2014)",score:13.17,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 13.17&nbsp<b>Coverage</b>: 23',organ:"MONOCYTE",integrated:!1,coverage:23,publication_year:2014,num_abundances:5242,hasPeptideCounts:!0,filename:"9606-Adult_Monocytes_Kim_2014_SEQUEST.txt"},{id:466,name:"H.sapiens - Cell line, NK, SC (Kim,Nature,2014)",score:21.97,weight:"100%",description:'abundance based on Spectral counting, NK, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 21.97&nbsp<b>Coverage</b>: 39',organ:"CELL_LINE",integrated:!1,coverage:39,publication_year:2014,num_abundances:8909,hasPeptideCounts:!0,filename:"9606-Adult_NKcells_Kim_2014_SEQUEST.txt"},{id:434,name:"H.sapiens - Female gonad, SC (Kim,Nature,2014)",score:22.29,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 22.29&nbsp<b>Coverage</b>: 49',organ:"FEMALE_GONAD",integrated:!1,coverage:49,publication_year:2014,num_abundances:11195,hasPeptideCounts:!0,filename:"9606-Adult_Ovary_Kim_2014_SEQUEST.txt"},{id:123,name:"H.sapiens - Pancreas, SC (Kim,Nature,2014)",score:16.03,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 16.03&nbsp<b>Coverage</b>: 43',organ:"PANCREAS",integrated:!1,coverage:43,publication_year:2014,num_abundances:9848,hasPeptideCounts:!0,filename:"9606-Adult_Pancreas_Kim_2014_SEQUEST.txt"},{id:383,name:"H.sapiens - Platelet, SC (Kim,Nature,2014)",score:14.82,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 14.82&nbsp<b>Coverage</b>: 28',organ:"PLATELET",integrated:!1,coverage:28,publication_year:2014,num_abundances:6429,hasPeptideCounts:!0,filename:"9606-Adult_Platelets_Kim_2014_SEQUEST.txt"},{id:386,name:"H.sapiens - Prostate gland, SC (Kim,Nature,2014)",score:15.68,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 15.68&nbsp<b>Coverage</b>: 41',organ:"PROSTATE_GLAND",integrated:!1,coverage:41,publication_year:2014,num_abundances:9293,hasPeptideCounts:!0,filename:"9606-Adult_Prostate_Kim_2014_SEQUEST.txt"},{id:435,name:"H.sapiens - Rectum, SC (Kim,Nature,2014)",score:16.58,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 16.58&nbsp<b>Coverage</b>: 37',organ:"RECTUM",integrated:!1,coverage:37,publication_year:2014,num_abundances:8384,hasPeptideCounts:!0,filename:"9606-Adult_Rectum_Kim_2014_SEQUEST.txt"},{id:363,name:"H.sapiens - Retina, SC (Kim,Nature,2014)",score:16.58,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 16.58&nbsp<b>Coverage</b>: 46',organ:"RETINA",integrated:!1,coverage:46,publication_year:2014,num_abundances:10511,hasPeptideCounts:!0,filename:"9606-Adult_Retina_Kim_2014_SEQUEST.txt"},{id:229,name:"H.sapiens - Spinal cord, SC (Kim,Nature,2014)",score:14.27,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 14.27&nbsp<b>Coverage</b>: 35',organ:"SPINAL_CORD",integrated:!1,coverage:35,publication_year:2014,num_abundances:7978,hasPeptideCounts:!0,filename:"9606-Adult_Spinalcord_Kim_2014_SEQUEST.txt"},{id:8,name:"H.sapiens - Testis, SC (Kim,Nature,2014)",score:21.88,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 21.88&nbsp<b>Coverage</b>: 54',organ:"TESTIS",integrated:!1,coverage:54,publication_year:2014,num_abundances:12281,hasPeptideCounts:!0,filename:"9606-Adult_Testis_Kim_2014_SEQUEST.txt"},{id:17,name:"H.sapiens - Urinary bladder, SC (Kim,Nature,2014)",score:15.11,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 15.11&nbsp<b>Coverage</b>: 36',organ:"URINARY_BLADDER",integrated:!1,coverage:36,publication_year:2014,num_abundances:8202,hasPeptideCounts:!0,filename:"9606-Adult_Urinarybladder_Kim_2014_SEQUEST.txt"},{id:180,name:"H.sapiens - Brain (Integrated)",score:14.76,weight:"",description:"integrated dataset: weighted average of all H.sapiens BRAIN datasets<br/><b>Interaction consistency score</b>: 14.76&nbsp<b>Coverage</b>: 43",organ:"BRAIN",integrated:!0,coverage:43,publication_year:2017,num_abundances:9821,hasPeptideCounts:!1,filename:"9606-BRAIN-integrated.txt"},{id:260,name:"H.sapiens - Cell line (Integrated)",score:2.62,weight:"",description:"integrated dataset: weighted average of all H.sapiens CELL_LINE datasets<br/><b>Interaction consistency score</b>: 2.62&nbsp<b>Coverage</b>: 81",organ:"CELL_LINE",integrated:!0,coverage:81,publication_year:2017,num_abundances:18473,hasPeptideCounts:!1,filename:"9606-CELL_LINE-integrated.txt"},{id:463,name:"H.sapiens - Colon (Integrated)",score:18.16,weight:"",description:"integrated dataset: weighted average of all H.sapiens COLON datasets<br/><b>Interaction consistency score</b>: 18.16&nbsp<b>Coverage</b>: 37",organ:"COLON",integrated:!0,coverage:37,publication_year:2017,num_abundances:8362,hasPeptideCounts:!1,filename:"9606-COLON-integrated.txt"},{id:407,name:"H.sapiens - Cell line, U2OS (Beck,MSB,2011)",score:1.88,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, U2OS, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=22068332&retmode=ref&cmd=prlinks" target="_blank">Beck,MSB,2011</a><br/><b>Interaction consistency score</b>: 1.88&nbsp<b>Coverage</b>: 32',organ:"CELL_LINE",integrated:!1,coverage:32,publication_year:2011,num_abundances:7202,hasPeptideCounts:!1,filename:"9606-Cellline_U2OS_Beck_2011.txt"},{id:357,name:"H.sapiens - Esophagus (Integrated)",score:12.66,weight:"",description:"integrated dataset: weighted average of all H.sapiens ESOPHAGUS datasets<br/><b>Interaction consistency score</b>: 12.66&nbsp<b>Coverage</b>: 29",organ:"ESOPHAGUS",integrated:!0,coverage:29,publication_year:2017,num_abundances:6523,hasPeptideCounts:!1,filename:"9606-ESOPHAGUS-integrated.txt"},{id:141,name:"H.sapiens - Cell line, ES H1 (Phanstiel,Nat_Methods,2011)",score:3.93,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, ES_H1, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 3.93&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6549,hasPeptideCounts:!1,filename:"9606-ES_(H1)_iTRAQ-117_Phanstiel_2011_gene.txt"},{id:117,name:"H.sapiens - Cell line, ES 14 (Phanstiel,Nat_Methods,2011)",score:4.15,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, ES_14, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.15&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6536,hasPeptideCounts:!1,filename:"9606-ES_(H14)_iTRAQ-119_Phanstiel_2011_gene.txt"},{id:95,name:"H.sapiens - Cell line, ES H7 (Phanstiel,Nat_Methods,2011)",score:4.32,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, ES_H7, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.32&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6550,hasPeptideCounts:!1,filename:"9606-ES_(H7)_iTRAQ-115_Phanstiel_2011_gene.txt"},{id:126,name:"H.sapiens - Cell line, ES H9 (Phanstiel,Nat_Methods,2011)",score:4.24,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, ES_H9, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.24&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6548,hasPeptideCounts:!1,filename:"9606-ES_(H9)_iTRAQ-113_Phanstiel_2011_gene.txt"},{id:235,name:"H.sapiens - Female gonad (Integrated)",score:25.86,weight:"",description:"integrated dataset: weighted average of all H.sapiens FEMALE_GONAD datasets<br/><b>Interaction consistency score</b>: 25.86&nbsp<b>Coverage</b>: 55",organ:"FEMALE_GONAD",integrated:!0,coverage:55,publication_year:2017,num_abundances:12551,hasPeptideCounts:!1,filename:"9606-FEMALE_GONAD-integrated.txt"},{id:200,name:"H.sapiens - Brain, Fetal, SC (Kim,Nature,2014)",score:13.97,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 13.97&nbsp<b>Coverage</b>: 38',organ:"BRAIN",integrated:!1,coverage:38,publication_year:2014,num_abundances:8654,hasPeptideCounts:!0,filename:"9606-Fetal_Brain_Kim_2014_SEQUEST.txt"},{id:371,name:"H.sapiens - Gut, Fetal, SC (Kim,Nature,2014)",score:16.22,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 16.22&nbsp<b>Coverage</b>: 37',organ:"GUT",integrated:!1,coverage:37,publication_year:2014,num_abundances:8410,hasPeptideCounts:!0,filename:"9606-Fetal_Gut_Kim_2014_SEQUEST.txt"},{id:431,name:"H.sapiens - Heart, Fetal, SC (Kim,Nature,2014)",score:19.08,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 19.08&nbsp<b>Coverage</b>: 44',organ:"HEART",integrated:!1,coverage:44,publication_year:2014,num_abundances:10050,hasPeptideCounts:!0,filename:"9606-Fetal_Heart_Kim_2014_SEQUEST.txt"},{id:27,name:"H.sapiens - Liver, Fetal, SC (Kim,Nature,2014)",score:15.94,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 15.94&nbsp<b>Coverage</b>: 39',organ:"LIVER",integrated:!1,coverage:39,publication_year:2014,num_abundances:8877,hasPeptideCounts:!0,filename:"9606-Fetal_Liver_Kim_2014_SEQUEST.txt"},{id:404,name:"H.sapiens - Female gonad, Fetal, SC (Kim,Nature,2014)",score:19.71,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 19.71&nbsp<b>Coverage</b>: 42',organ:"FEMALE_GONAD",integrated:!1,coverage:42,publication_year:2014,num_abundances:9493,hasPeptideCounts:!0,filename:"9606-Fetal_Ovary_Kim_2014_SEQUEST.txt"},{id:314,name:"H.sapiens - Placenta, Fetal, SC (Kim,Nature,2014)",score:9.7,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 9.7&nbsp<b>Coverage</b>: 28',organ:"PLACENTA",integrated:!1,coverage:28,publication_year:2014,num_abundances:6376,hasPeptideCounts:!0,filename:"9606-Fetal_Placenta_Kim_2014_SEQUEST.txt"},{id:318,name:"H.sapiens - Testis, Fetal, SC (Kim,Nature,2014)",score:13.03,weight:"100%",description:'abundance based on Spectral counting, Fetal, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13302.html" target="_blank">Kim,Nature,2014</a><br/><b>Interaction consistency score</b>: 13.03&nbsp<b>Coverage</b>: 37',organ:"TESTIS",integrated:!1,coverage:37,publication_year:2014,num_abundances:8382,hasPeptideCounts:!0,filename:"9606-Fetal_Testis_Kim_2014_SEQUEST.txt"},{id:276,name:"H.sapiens - Gall bladder (Integrated)",score:12.69,weight:"",description:"integrated dataset: weighted average of all H.sapiens GALL_BLADDER datasets<br/><b>Interaction consistency score</b>: 12.69&nbsp<b>Coverage</b>: 34",organ:"GALL_BLADDER",integrated:!0,coverage:34,publication_year:2017,num_abundances:7774,hasPeptideCounts:!1,filename:"9606-GALL_BLADDER-integrated.txt"},{id:140,name:"H.sapiens - Cell line, GAMG, SC (Geiger,MCP,2012)",score:4.63,weight:"100%",description:'abundance based on Spectral counting, GAMG, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 4.63&nbsp<b>Coverage</b>: 18',organ:"CELL_LINE",integrated:!1,coverage:18,publication_year:2012,num_abundances:4080,hasPeptideCounts:!0,filename:"9606-GAMG_Geiger_2012.txt"},{id:197,name:"H.sapiens - Whole organism, SC (GPM,Oct,2012)",score:2.97,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 2.97&nbsp<b>Coverage</b>: 80',organ:"WHOLE_ORGANISM",integrated:!1,coverage:80,publication_year:2012,num_abundances:18172,hasPeptideCounts:!0,filename:"9606-GPM_2012_09_Homo_sapiens_-_ensembl.txt"},{id:162,name:"H.sapiens - Whole organism, SC (GPM,Aug,2014)",score:23.9,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 23.9&nbsp<b>Coverage</b>: 85',organ:"WHOLE_ORGANISM",integrated:!1,coverage:85,publication_year:2014,num_abundances:19471,hasPeptideCounts:!0,filename:"9606-GPM_201408.txt"},{id:150,name:"H.sapiens - Cell line, HCT116 (Koch,Proteomics,2014)",score:3.18,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, HCT116, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25156699" target="_blank">Koch,Proteomics,2014</a><br/><b>Interaction consistency score</b>: 3.18&nbsp<b>Coverage</b>: 4',organ:"CELL_LINE",integrated:!1,coverage:4,publication_year:2014,num_abundances:922,hasPeptideCounts:!1,filename:"9606-HCT116_Koch_2014_emPAI.txt"},{id:143,name:"H.sapiens - Heart (Integrated)",score:21.84,weight:"",description:"integrated dataset: weighted average of all H.sapiens HEART datasets<br/><b>Interaction consistency score</b>: 21.84&nbsp<b>Coverage</b>: 51",organ:"HEART",integrated:!0,coverage:51,publication_year:2017,num_abundances:11758,hasPeptideCounts:!1,filename:"9606-HEART-integrated.txt"},{id:485,name:"H.sapiens - Cell line, HEK293, SC (Geiger,MCP,2012)",score:8.03,weight:"100%",description:'abundance based on Spectral counting, HEK293, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 8.03&nbsp<b>Coverage</b>: 19',organ:"CELL_LINE",integrated:!1,coverage:19,publication_year:2012,num_abundances:4278,hasPeptideCounts:!0,filename:"9606-HEK293_Geiger_2012.txt"},{id:249,name:"H.sapiens - Cell line, Hela, SC (Geiger,MCP,2012)",score:7.83,weight:"100%",description:'abundance based on Spectral counting, hela, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 7.83&nbsp<b>Coverage</b>: 17',organ:"CELL_LINE",integrated:!1,coverage:17,publication_year:2012,num_abundances:3916,hasPeptideCounts:!0,filename:"9606-HeLa_Geiger_2012.txt"},{id:315,name:"H.sapiens - Cell line, HepG2, SC (Geiger,MCP,2012)",score:5.94,weight:"100%",description:'abundance based on Spectral counting, HepG2, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 5.94&nbsp<b>Coverage</b>: 15',organ:"CELL_LINE",integrated:!1,coverage:15,publication_year:2012,num_abundances:3515,hasPeptideCounts:!0,filename:"9606-HepG2_Geiger_2012.txt"},{id:317,name:"H.sapiens - Whole organism, SC (PeptideAtlas,May,2010)",score:4.91,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/human/" target="_blank">PeptideAtlas,May,2010</a><br/><b>Interaction consistency score</b>: 4.91&nbsp<b>Coverage</b>: 36',organ:"WHOLE_ORGANISM",integrated:!1,coverage:36,publication_year:2010,num_abundances:8264,hasPeptideCounts:!0,filename:"9606-HsapiensBuild242_May2010.peptides.new.txt"},{id:381,name:"H.sapiens - Cell line, Huh7, SC (Pando-Robles,J_Proteomics,2014)",score:9.87,weight:"100%",description:'abundance based on Spectral counting, Huh7, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25009145" target="_blank">Pando-Robles,J_Proteomics,2014</a><br/><b>Interaction consistency score</b>: 9.87&nbsp<b>Coverage</b>: 7',organ:"CELL_LINE",integrated:!1,coverage:7,publication_year:2014,num_abundances:1605,hasPeptideCounts:!0,filename:"9606-Huh-7_Pando-Robles_2014.txt"},{id:330,name:"H.sapiens - Liver, Chinese liver, SC (DLPEP,2006)",score:2.82,weight:"100%",description:'abundance based on Spectral counting, Chinese liver, from<a href="http://hlpic.hupo.org.cn/dblep/main.jsf" target="_blank">DLPEP,2006</a><br/><b>Interaction consistency score</b>: 2.82&nbsp<b>Coverage</b>: 45',organ:"LIVER",integrated:!1,coverage:45,publication_year:2006,num_abundances:10273,hasPeptideCounts:!0,filename:"9606-Human_Chinese_liver_dblep.txt"},{id:118,name:"H.sapiens - Liver, French liver, SC (DLPEP,2007)",score:3,weight:"100%",description:'abundance based on Spectral counting, French liver, from<a href="http://hlpic.hupo.org.cn/dblep/main.jsf" target="_blank">DLPEP,2007</a><br/><b>Interaction consistency score</b>: 3.0&nbsp<b>Coverage</b>: 28',organ:"LIVER",integrated:!1,coverage:28,publication_year:2007,num_abundances:6493,hasPeptideCounts:!0,filename:"9606-Human_French_liver_dblep.txt"},{id:15,name:"H.sapiens - Heart, SC (Kline,J.Proteome_Res,2008)",score:3.49,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://pubs.acs.org/doi/full/10.1021/pr800239e" target="_blank">Kline,J.Proteome_Res,2008</a><br/><b>Interaction consistency score</b>: 3.49&nbsp<b>Coverage</b>: 14',organ:"HEART",integrated:!1,coverage:14,publication_year:2008,num_abundances:3228,hasPeptideCounts:!0,filename:"9606-Human_Heart_Kline2008_JProteomeRes_peptide.txt"},{id:453,name:"H.sapiens - Cell line, Hela, SC (Kocher,Proteomics,2014)",score:13.04,weight:"100%",description:'abundance based on Spectral counting, hela, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24920484" target="_blank">Kocher,Proteomics,2014</a><br/><b>Interaction consistency score</b>: 13.04&nbsp<b>Coverage</b>: 17',organ:"CELL_LINE",integrated:!1,coverage:17,publication_year:2014,num_abundances:3857,hasPeptideCounts:!0,filename:"9606-Human_PXD000396_hela_Kocher_2014.txt"},{id:66,name:"H.sapiens - Cell line, A460, SC (Balbin,Nat_Commun,2013)",score:6.77,weight:"100%",description:'abundance based on Spectral counting, A460, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 6.77&nbsp<b>Coverage</b>: 42',organ:"CELL_LINE",integrated:!1,coverage:42,publication_year:2013,num_abundances:9689,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_A460.txt"},{id:199,name:"H.sapiens - Cell line, A549, SC (Balbin,Nat_Commun,2013)",score:6.41,weight:"100%",description:'abundance based on Spectral counting, A549, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 6.41&nbsp<b>Coverage</b>: 40',organ:"CELL_LINE",integrated:!1,coverage:40,publication_year:2013,num_abundances:9198,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_A549.txt"},{id:289,name:"H.sapiens - Cell line, H1155, SC (Balbin,Nat_Commun,2013)",score:9.61,weight:"100%",description:'abundance based on Spectral counting, H1155, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 9.61&nbsp<b>Coverage</b>: 49',organ:"CELL_LINE",integrated:!1,coverage:49,publication_year:2013,num_abundances:11132,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H1155.txt"},{id:391,name:"H.sapiens - Cell line, H1734, SC (Balbin,Nat_Commun,2013)",score:3.72,weight:"100%",description:'abundance based on Spectral counting, H1734, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 3.72&nbsp<b>Coverage</b>: 25',organ:"CELL_LINE",integrated:!1,coverage:25,publication_year:2013,num_abundances:5602,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H1734.txt"},{id:287,name:"H.sapiens - Cell line, H1792, SC (Balbin,Nat_Commun,2013)",score:8.6,weight:"100%",description:'abundance based on Spectral counting, H1792, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 8.6&nbsp<b>Coverage</b>: 40',organ:"CELL_LINE",integrated:!1,coverage:40,publication_year:2013,num_abundances:9130,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H1792.txt"},{id:31,name:"H.sapiens - Cell line, H2009, SC (Balbin,Nat_Commun,2013)",score:2.36,weight:"100%",description:'abundance based on Spectral counting, H2009, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 2.36&nbsp<b>Coverage</b>: 23',organ:"CELL_LINE",integrated:!1,coverage:23,publication_year:2013,num_abundances:5249,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H2009.txt"},{id:448,name:"H.sapiens - Cell line, H2122, SC (Balbin,Nat_Commun,2013)",score:5.78,weight:"100%",description:'abundance based on Spectral counting, H2122, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 5.78&nbsp<b>Coverage</b>: 32',organ:"CELL_LINE",integrated:!1,coverage:32,publication_year:2013,num_abundances:7199,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H2122.txt"},{id:316,name:"H.sapiens - Cell line, H23, SC (Balbin,Nat_Commun,2013)",score:6.89,weight:"100%",description:'abundance based on Spectral counting, H23, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 6.89&nbsp<b>Coverage</b>: 40',organ:"CELL_LINE",integrated:!1,coverage:40,publication_year:2013,num_abundances:9045,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H23.txt"},{id:416,name:"H.sapiens - Cell line, H358, SC (Balbin,Nat_Commun,2013)",score:5.71,weight:"100%",description:'abundance based on Spectral counting, H358, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 5.71&nbsp<b>Coverage</b>: 42',organ:"CELL_LINE",integrated:!1,coverage:42,publication_year:2013,num_abundances:9695,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H358.txt"},{id:344,name:"H.sapiens - Cell line, H441, SC (Balbin,Nat_Commun,2013)",score:6.75,weight:"100%",description:'abundance based on Spectral counting, H441, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 6.75&nbsp<b>Coverage</b>: 44',organ:"CELL_LINE",integrated:!1,coverage:44,publication_year:2013,num_abundances:9997,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H441.txt"},{id:245,name:"H.sapiens - Cell line, H727, SC (Balbin,Nat_Commun,2013)",score:5.43,weight:"100%",description:'abundance based on Spectral counting, H727, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 5.43&nbsp<b>Coverage</b>: 32',organ:"CELL_LINE",integrated:!1,coverage:32,publication_year:2013,num_abundances:7208,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_H727.txt"},{id:227,name:"H.sapiens - Cell line, SKLU1, SC (Balbin,Nat_Commun,2013)",score:6.69,weight:"100%",description:'abundance based on Spectral counting, SKLU1, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24135919" target="_blank">Balbin,Nat_Commun,2013</a><br/><b>Interaction consistency score</b>: 6.69&nbsp<b>Coverage</b>: 31',organ:"CELL_LINE",integrated:!1,coverage:31,publication_year:2013,num_abundances:6977,hasPeptideCounts:!0,filename:"9606-Human_PXD000439_Balbin_2013_SKLU1.txt"},{id:193,name:"H.sapiens - Cell line, Vitreous humor, SC (Murthy,Clin_Proteomics,2014)",score:6.21,weight:"100%",description:'abundance based on Spectral counting, vitreous_humor, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25097467" target="_blank">Murthy,Clin_Proteomics,2014</a><br/><b>Interaction consistency score</b>: 6.21&nbsp<b>Coverage</b>: 4',organ:"CELL_LINE",integrated:!1,coverage:4,publication_year:2014,num_abundances:825,hasPeptideCounts:!0,filename:"9606-Human_PXD000957_vitreous_humor_Murthy_2014_MASCOT.txt"},{id:155,name:"H.sapiens - Whole organism, SC (PeptideAtlas,Aug,2011)",score:10.97,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/human/" target="_blank">PeptideAtlas,Aug,2011</a><br/><b>Interaction consistency score</b>: 10.97&nbsp<b>Coverage</b>: 42',organ:"WHOLE_ORGANISM",integrated:!1,coverage:42,publication_year:2011,num_abundances:9697,hasPeptideCounts:!0,filename:"9606-Human_PeptideAtlas_2011-08_Ens62.new.txt"},{id:24,name:"H.sapiens - Liver, Fetal liver, SC (DLPEP,2005)",score:7.79,weight:"100%",description:'abundance based on Spectral counting, Fetal liver, from<a href="http://hlpic.hupo.org.cn/dblep/main.jsf" target="_blank">DLPEP,2005</a><br/><b>Interaction consistency score</b>: 7.79&nbsp<b>Coverage</b>: 10',organ:"LIVER",integrated:!1,coverage:10,publication_year:2005,num_abundances:2371,hasPeptideCounts:!0,filename:"9606-Human_fetal_liver_dblep..txt"},{id:55,name:"H.sapiens - Heart, normalized data APEX (Aye,Mol_Bio_Syst,2010)",score:10.05,weight:"100%",description:'abundance based on normalized data APEX, from<a href="http://pubs.rsc.org/en/content/articlehtml/2010/mb/c004495d#fn1" target="_blank">Aye,Mol_Bio_Syst, 2010</a><br/><b>Interaction consistency score</b>: 10.05&nbsp<b>Coverage</b>: 10',organ:"HEART",integrated:!1,coverage:10,publication_year:2010,num_abundances:2200,hasPeptideCounts:!1,filename:"9606-Human_heart_ThinThinAye_MolecularBioSystems2010_APEX.txt"},{id:356,name:"H.sapiens - Heart, SC (Aye,Mol_Bio_Syst,2010)",score:11.15,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://pubs.rsc.org/en/content/articlehtml/2010/mb/c004495d#fn1" target="_blank">Aye,Mol_Bio_Syst, 2010</a><br/><b>Interaction consistency score</b>: 11.15&nbsp<b>Coverage</b>: 14',organ:"HEART",integrated:!1,coverage:14,publication_year:2010,num_abundances:3261,hasPeptideCounts:!0,filename:"9606-Human_heart_ThinThinAye_MolecularBioSystems2010_combine_MethodABC_I.txt"},{id:206,name:"H.sapiens - Whole organism, IBAQ (Khan,Science,2013)",score:5.95,weight:"30%",description:'abundance based on MAPPED_BY_AUTHORS, iBAQ, from<a href="http://www.sciencemag.org/content/342/6162/1100.abstract" target="_blank">Khan,Science,2013</a><br/><b>Interaction consistency score</b>: 5.95&nbsp<b>Coverage</b>: 15',organ:"WHOLE_ORGANISM",integrated:!1,coverage:15,publication_year:2013,num_abundances:3378,hasPeptideCounts:!1,filename:"9606-Human_iBAQ_Khan_2013.txt"},{id:87,name:"H.sapiens - Liver, normalized data SCIN (CHLPPC,J.Proteome_Res,2010)",score:4.82,weight:"100%",description:'abundance based on normalized data SCIN, from<a href="http://pubs.acs.org/doi/full/10.1021/pr900532r" target="_blank">CHLPPC,J.Proteome_Res,2010</a><br/><b>Interaction consistency score</b>: 4.82&nbsp<b>Coverage</b>: 24',organ:"LIVER",integrated:!1,coverage:24,publication_year:2010,num_abundances:5491,hasPeptideCounts:!1,filename:"9606-Human_liver_Chinese_2010_J._Proteome_Res.txt"},{id:393,name:"H.sapiens - Cell line, Jurkat, SC (Geiger,MCP,2012)",score:3.77,weight:"100%",description:'abundance based on Spectral counting, Jurkat, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 3.77&nbsp<b>Coverage</b>: 17',organ:"CELL_LINE",integrated:!1,coverage:17,publication_year:2012,num_abundances:3950,hasPeptideCounts:!0,filename:"9606-Jurkat_Geiger_2012.txt"},{id:214,name:"H.sapiens - Cell line, K562, SC (Geiger,MCP,2012)",score:4.57,weight:"100%",description:'abundance based on Spectral counting, K562, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 4.57&nbsp<b>Coverage</b>: 15',organ:"CELL_LINE",integrated:!1,coverage:15,publication_year:2012,num_abundances:3326,hasPeptideCounts:!0,filename:"9606-K562_Geiger_2012.txt"},{id:336,name:"H.sapiens - Kidney (Integrated)",score:16.94,weight:"",description:"integrated dataset: weighted average of all H.sapiens KIDNEY datasets<br/><b>Interaction consistency score</b>: 16.94&nbsp<b>Coverage</b>: 35",organ:"KIDNEY",integrated:!0,coverage:35,publication_year:2017,num_abundances:7955,hasPeptideCounts:!1,filename:"9606-KIDNEY-integrated.txt"},{id:11,name:"H.sapiens - Liver (Integrated)",score:5.95,weight:"",description:"integrated dataset: weighted average of all H.sapiens LIVER datasets<br/><b>Interaction consistency score</b>: 5.95&nbsp<b>Coverage</b>: 65",organ:"LIVER",integrated:!0,coverage:65,publication_year:2017,num_abundances:14774,hasPeptideCounts:!1,filename:"9606-LIVER-integrated.txt"},{id:158,name:"H.sapiens - Lung (Integrated)",score:12.36,weight:"",description:"integrated dataset: weighted average of all H.sapiens LUNG datasets<br/><b>Interaction consistency score</b>: 12.36&nbsp<b>Coverage</b>: 32",organ:"LUNG",integrated:!0,coverage:32,publication_year:2017,num_abundances:7214,hasPeptideCounts:!1,filename:"9606-LUNG-integrated.txt"},{id:183,name:"H.sapiens - Cell line, LnCap, SC (Geiger,MCP,2012)",score:9.24,weight:"100%",description:'abundance based on Spectral counting, LnCap, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 9.24&nbsp<b>Coverage</b>: 17',organ:"CELL_LINE",integrated:!1,coverage:17,publication_year:2012,num_abundances:3939,hasPeptideCounts:!0,filename:"9606-LnCap_Geiger_2012.txt"},{id:382,name:"H.sapiens - Cell line, MCF7, SC (Geiger,MCP,2012)",score:8,weight:"100%",description:'abundance based on Spectral counting, MCF7, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 8.0&nbsp<b>Coverage</b>: 18',organ:"CELL_LINE",integrated:!1,coverage:18,publication_year:2012,num_abundances:3996,hasPeptideCounts:!0,filename:"9606-MCF7_Geiger_2012.txt"},{id:361,name:"H.sapiens - Pancreas (Integrated)",score:16.85,weight:"",description:"integrated dataset: weighted average of all H.sapiens PANCREAS datasets<br/><b>Interaction consistency score</b>: 16.85&nbsp<b>Coverage</b>: 44",organ:"PANCREAS",integrated:!0,coverage:44,publication_year:2017,num_abundances:10094,hasPeptideCounts:!1,filename:"9606-PANCREAS-integrated.txt"},{id:389,name:"H.sapiens - Whole organism, SC (PeptideAtlas,Aug,2014)",score:26.05,weight:"70%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2014</a><br/><b>Interaction consistency score</b>: 26.05&nbsp<b>Coverage</b>: 72',organ:"WHOLE_ORGANISM",integrated:!1,coverage:72,publication_year:2014,num_abundances:16437,hasPeptideCounts:!0,filename:"9606-PA_201408.txt"},{id:377,name:"H.sapiens - Brain, SC (PeptideAtlas,Aug,2013)",score:8.48,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 8.48&nbsp<b>Coverage</b>: 18',organ:"BRAIN",integrated:!1,coverage:18,publication_year:2013,num_abundances:4138,hasPeptideCounts:!0,filename:"9606-PA_brain_201308.txt"},{id:83,name:"H.sapiens - Heart, SC (PeptideAtlas,Aug,2014)",score:14.58,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2014</a><br/><b>Interaction consistency score</b>: 14.58&nbsp<b>Coverage</b>: 34',organ:"HEART",integrated:!1,coverage:34,publication_year:2014,num_abundances:7761,hasPeptideCounts:!0,filename:"9606-PA_heart_201408.txt"},{id:139,name:"H.sapiens - Kidney, SC (PeptideAtlas,Aug,2013)",score:8.35,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 8.35&nbsp<b>Coverage</b>: 21',organ:"KIDNEY",integrated:!1,coverage:21,publication_year:2013,num_abundances:4841,hasPeptideCounts:!0,filename:"9606-PA_kidney_201308.txt"},{id:325,name:"H.sapiens - Liver, SC (PeptideAtlas,Aug,2013)",score:8.6,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 8.6&nbsp<b>Coverage</b>: 23',organ:"LIVER",integrated:!1,coverage:23,publication_year:2013,num_abundances:5153,hasPeptideCounts:!0,filename:"9606-PA_liver_201308.txt"},{id:210,name:"H.sapiens - Plasma, SC (PeptideAtlas,Aug,2013)",score:10.96,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 10.96&nbsp<b>Coverage</b>: 17',organ:"PLASMA",integrated:!1,coverage:17,publication_year:2013,num_abundances:3874,hasPeptideCounts:!0,filename:"9606-PA_plasma.txt"},{id:247,name:"H.sapiens - Urine, SC (PeptideAtlas,Aug,2013)",score:7.76,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 7.76&nbsp<b>Coverage</b>: 13',organ:"URINE",integrated:!1,coverage:13,publication_year:2013,num_abundances:2954,hasPeptideCounts:!0,filename:"9606-PA_urine_201308.txt"},{id:310,name:"H.sapiens - Placenta (Integrated)",score:12.99,weight:"",description:"integrated dataset: weighted average of all H.sapiens PLACENTA datasets<br/><b>Interaction consistency score</b>: 12.99&nbsp<b>Coverage</b>: 30",organ:"PLACENTA",integrated:!0,coverage:30,publication_year:2017,num_abundances:6862,hasPeptideCounts:!1,filename:"9606-PLACENTA-integrated.txt"},{id:171,name:"H.sapiens - Plasma (Integrated)",score:8.85,weight:"",description:"integrated dataset: weighted average of all H.sapiens PLASMA datasets<br/><b>Interaction consistency score</b>: 8.85&nbsp<b>Coverage</b>: 20",organ:"PLASMA",integrated:!0,coverage:20,publication_year:2017,num_abundances:4492,hasPeptideCounts:!1,filename:"9606-PLASMA-integrated.txt"},{id:94,name:"H.sapiens - Platelet (Integrated)",score:14.18,weight:"",description:"integrated dataset: weighted average of all H.sapiens PLATELET datasets<br/><b>Interaction consistency score</b>: 14.18&nbsp<b>Coverage</b>: 28",organ:"PLATELET",integrated:!0,coverage:28,publication_year:2017,num_abundances:6486,hasPeptideCounts:!1,filename:"9606-PLATELET-integrated.txt"},{id:68,name:"H.sapiens - Prostate gland (Integrated)",score:17.38,weight:"",description:"integrated dataset: weighted average of all H.sapiens PROSTATE_GLAND datasets<br/><b>Interaction consistency score</b>: 17.38&nbsp<b>Coverage</b>: 42",organ:"PROSTATE_GLAND",integrated:!0,coverage:42,publication_year:2017,num_abundances:9617,hasPeptideCounts:!1,filename:"9606-PROSTATE_GLAND-integrated.txt"},{id:479,name:"H.sapiens - Cell line, Pancreatic metastatic, SC (Kim,MCP,2014)",score:14.76,weight:"100%",description:'abundance based on Spectral counting, Pancreatic_metastatic, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24895378" target="_blank">Kim,MCP,2014</a><br/><b>Interaction consistency score</b>: 14.76&nbsp<b>Coverage</b>: 27',organ:"CELL_LINE",integrated:!1,coverage:27,publication_year:2014,num_abundances:6204,hasPeptideCounts:!0,filename:"9606-Panc_cellline_Kim_2014.txt"},{id:419,name:"H.sapiens - Rectum (Integrated)",score:19.57,weight:"",description:"integrated dataset: weighted average of all H.sapiens RECTUM datasets<br/><b>Interaction consistency score</b>: 19.57&nbsp<b>Coverage</b>: 38",organ:"RECTUM",integrated:!0,coverage:38,publication_year:2017,num_abundances:8663,hasPeptideCounts:!1,filename:"9606-RECTUM-integrated.txt"},{id:334,name:"H.sapiens - Cell line, RKO, SC (Geiger,MCP,2012)",score:5.94,weight:"100%",description:'abundance based on Spectral counting, RKO, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 5.94&nbsp<b>Coverage</b>: 15',organ:"CELL_LINE",integrated:!1,coverage:15,publication_year:2012,num_abundances:3400,hasPeptideCounts:!0,filename:"9606-RKO_Geiger_2012.txt"},{id:142,name:"H.sapiens - Saliva (Integrated)",score:8.31,weight:"",description:"integrated dataset: weighted average of all H.sapiens SALIVA datasets<br/><b>Interaction consistency score</b>: 8.31&nbsp<b>Coverage</b>: 9",organ:"SALIVA",integrated:!0,coverage:9,publication_year:2017,num_abundances:2008,hasPeptideCounts:!1,filename:"9606-SALIVA-integrated.txt"},{id:367,name:"H.sapiens - Skin (Integrated)",score:5.99,weight:"",description:"integrated dataset: weighted average of all H.sapiens SKIN datasets<br/><b>Interaction consistency score</b>: 5.99&nbsp<b>Coverage</b>: 20",organ:"SKIN",integrated:!0,coverage:20,publication_year:2017,num_abundances:4550,hasPeptideCounts:!1,filename:"9606-SKIN-integrated.txt"},{id:333,name:"H.sapiens - Testis (Integrated)",score:22.54,weight:"",description:"integrated dataset: weighted average of all H.sapiens TESTIS datasets<br/><b>Interaction consistency score</b>: 22.54&nbsp<b>Coverage</b>: 58",organ:"TESTIS",integrated:!0,coverage:58,publication_year:2017,num_abundances:13253,hasPeptideCounts:!1,filename:"9606-TESTIS-integrated.txt"},{id:221,name:"H.sapiens - Cell line, U251MG (Lundberg,MSB,2010)",score:8.21,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, U251MG, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21179022" target="_blank">lundberg,MSB,2010</a><br/><b>Interaction consistency score</b>: 8.21&nbsp<b>Coverage</b>: 24',organ:"CELL_LINE",integrated:!1,coverage:24,publication_year:2010,num_abundances:5379,hasPeptideCounts:!1,filename:"9606-U251MG_cellline_Lundberg_2010.txt"},{id:353,name:"H.sapiens - Cell line, U2OS, SC (Geiger,MCP,2012)",score:8.21,weight:"100%",description:'abundance based on Spectral counting, U2OS, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 8.21&nbsp<b>Coverage</b>: 18',organ:"CELL_LINE",integrated:!1,coverage:18,publication_year:2012,num_abundances:4013,hasPeptideCounts:!0,filename:"9606-U2OS_Geiger_2012.txt"},{id:81,name:"H.sapiens - Cell line, U2OS (Lundberg,MSB,2010)",score:8.58,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, U2OS, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21179022" target="_blank">lundberg,MSB,2010</a><br/><b>Interaction consistency score</b>: 8.58&nbsp<b>Coverage</b>: 24',organ:"CELL_LINE",integrated:!1,coverage:24,publication_year:2010,num_abundances:5387,hasPeptideCounts:!1,filename:"9606-U2OS_cellline_Lundberg_2010.txt"},{id:224,name:"H.sapiens - Urine (Integrated)",score:8.02,weight:"",description:"integrated dataset: weighted average of all H.sapiens URINE datasets<br/><b>Interaction consistency score</b>: 8.02&nbsp<b>Coverage</b>: 14",organ:"URINE",integrated:!0,coverage:14,publication_year:2017,num_abundances:3114,hasPeptideCounts:!1,filename:"9606-URINE-integrated.txt"},{id:477,name:"H.sapiens - Uterus (Integrated)",score:7.19,weight:"",description:"integrated dataset: weighted average of all H.sapiens UTERUS datasets<br/><b>Interaction consistency score</b>: 7.19&nbsp<b>Coverage</b>: 21",organ:"UTERUS",integrated:!0,coverage:21,publication_year:2017,num_abundances:4847,hasPeptideCounts:!1,filename:"9606-UTERUS-integrated.txt"},{id:29,name:"H.sapiens - Whole organism (Integrated)",score:15.5,weight:"",description:"integrated dataset: weighted average of all H.sapiens WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 15.5&nbsp<b>Coverage</b>: 87",organ:"WHOLE_ORGANISM",integrated:!0,coverage:87,publication_year:2017,num_abundances:19949,hasPeptideCounts:!1,filename:"9606-WHOLE_ORGANISM-integrated.txt"},{id:64,name:"H.sapiens - Ascitic fluid, SC (Wilhelm,Nature,2014)",score:7.14,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.14&nbsp<b>Coverage</b>: 10',organ:"ASCITIC_FLUID",integrated:!1,coverage:10,publication_year:2014,num_abundances:2222,hasPeptideCounts:!0,filename:"9606-ascites_Wilhelm_2014_Maxquant.txt"},{id:189,name:"H.sapiens - Cardia of stomach, SC (Wilhelm,Nature,2014)",score:5.4,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 5.4&nbsp<b>Coverage</b>: 16',organ:"CARDIA_OF_STOMACH",integrated:!1,coverage:16,publication_year:2014,num_abundances:3597,hasPeptideCounts:!0,filename:"9606-cardia_Wilhelm_2014_Maxquant.txt"},{id:217,name:"H.sapiens - Cerebral cortex, SC (Wilhelm,Nature,2014)",score:2.13,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 2.13&nbsp<b>Coverage</b>: 11',organ:"CEREBRAL_CORTEX",integrated:!1,coverage:11,publication_year:2014,num_abundances:2607,hasPeptideCounts:!0,filename:"9606-cerebral_cortex_Wilhelm_2014_Maxquant.txt"},{id:77,name:"H.sapiens - Uterine cervix, SC (Wilhelm,Nature,2014)",score:6.75,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 6.75&nbsp<b>Coverage</b>: 17',organ:"UTERINE_CERVIX",integrated:!1,coverage:17,publication_year:2014,num_abundances:3812,hasPeptideCounts:!0,filename:"9606-cervix_uteri_Wilhelm_2014_Maxquant.txt"},{id:467,name:"H.sapiens - Colon, SC (Wilhelm,Nature,2014)",score:10.26,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 10.26&nbsp<b>Coverage</b>: 21',organ:"COLON",integrated:!1,coverage:21,publication_year:2014,num_abundances:4879,hasPeptideCounts:!0,filename:"9606-colon_Wilhelm_2014_Maxquant.txt"},{id:216,name:"H.sapiens - Earwax, SC (Wilhelm,Nature,2014)",score:5.04,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 5.04&nbsp<b>Coverage</b>: 6',organ:"EARWAX",integrated:!1,coverage:6,publication_year:2014,num_abundances:1426,hasPeptideCounts:!0,filename:"9606-earwax_Wilhelm_2014_Maxquant.txt"},{id:472,name:"H.sapiens - Esophagus, SC (Wilhelm,Nature,2014)",score:7.82,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.82&nbsp<b>Coverage</b>: 22',organ:"ESOPHAGUS",integrated:!1,coverage:22,publication_year:2014,num_abundances:4919,hasPeptideCounts:!0,filename:"9606-esophagus_Wilhelm_2014_Maxquant.txt"},{id:340,name:"H.sapiens - Gall bladder, SC (Wilhelm,Nature,2014)",score:10.61,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 10.61&nbsp<b>Coverage</b>: 16',organ:"GALL_BLADDER",integrated:!1,coverage:16,publication_year:2014,num_abundances:3613,hasPeptideCounts:!0,filename:"9606-gall_bladder_Wilhelm_2014_Maxquant.txt"},{id:208,name:"H.sapiens - Hair follicle, SC (Wilhelm,Nature,2014)",score:5.01,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 5.01&nbsp<b>Coverage</b>: 4',organ:"HAIR_FOLLICLE",integrated:!1,coverage:4,publication_year:2014,num_abundances:911,hasPeptideCounts:!0,filename:"9606-hair_follicle_Chymotrypsin_Wilhelm_2014_Maxquant.txt"},{id:41,name:"H.sapiens - Cell line, Hela, SC (Nagaraj,MSB,2011)",score:13.43,weight:"100%",description:'abundance based on Spectral counting, hela, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=22068331&retmode=ref&cmd=prlinks" target="_blank">Nagaraj,MSB,2011</a><br/><b>Interaction consistency score</b>: 13.43&nbsp<b>Coverage</b>: 39',organ:"CELL_LINE",integrated:!1,coverage:39,publication_year:2011,num_abundances:8817,hasPeptideCounts:!0,filename:"9606-hela_Nagaraj_2011.txt"},{id:327,name:"H.sapiens - Cell line, Hela, CPC normalized (Nagaraj,MSB,2011)",score:5.67,weight:"100%",description:'abundance based on CPC_normalized, hela, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=22068331&retmode=ref&cmd=prlinks" target="_blank">Nagaraj,MSB,2011</a><br/><b>Interaction consistency score</b>: 5.67&nbsp<b>Coverage</b>: 36',organ:"CELL_LINE",integrated:!1,coverage:36,publication_year:2011,num_abundances:8116,hasPeptideCounts:!1,filename:"9606-hela_Nagaraj_2011_cpc.txt"},{id:455,name:"H.sapiens - Cell line, Hela, IBAQ normalized (Nagaraj,MSB,2011)",score:6.15,weight:"100%",description:'abundance based on IBAQ_normalized, hela, from<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=22068331&retmode=ref&cmd=prlinks" target="_blank">Nagaraj,MSB,2011</a><br/><b>Interaction consistency score</b>: 6.15&nbsp<b>Coverage</b>: 36',organ:"CELL_LINE",integrated:!1,coverage:36,publication_year:2011,num_abundances:8116,hasPeptideCounts:!1,filename:"9606-hela_Nagaraj_2011_iBAQ.txt"},{id:430,name:"H.sapiens - Plasma, Non alkylated, SC (Qian,MCP,2005)",score:.04,weight:"100%",description:'abundance based on Spectral counting, non alkylated, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1829297/" target="_blank">Qian,MCP,2005</a><br/><b>Interaction consistency score</b>: 0.04&nbsp<b>Coverage</b>: 3',organ:"PLASMA",integrated:!1,coverage:3,publication_year:2005,num_abundances:788,hasPeptideCounts:!0,filename:"9606-human3_non-alkylatedPlasma_PRIDE.txt"},{id:185,name:"H.sapiens - Skin, SC (L.Amon,FHCRC,ID18251-18264)",score:3.38,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/" target="_blank">L.Amon,FHCRC,ID18251-18264</a><br/><b>Interaction consistency score</b>: 3.38&nbsp<b>Coverage</b>: 9',organ:"SKIN",integrated:!1,coverage:9,publication_year:null,num_abundances:1983,hasPeptideCounts:!0,filename:"9606-human4_normal_PRIDE.txt"},{id:56,name:"H.sapiens - Saliva, SC (EMBL,UK,LLC,ID22142-22144)",score:6.22,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/" target="_blank">EMBL,UK,LLC,ID22142-22144</a><br/><b>Interaction consistency score</b>: 6.22&nbsp<b>Coverage</b>: 3',organ:"SALIVA",integrated:!1,coverage:3,publication_year:null,num_abundances:735,hasPeptideCounts:!0,filename:"9606-human5_salivaCollected_PRIDE.txt"},{id:90,name:"H.sapiens - Brain, SC (Martens,proteomics,2006)",score:16.19,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.200600291/abstract" target="_blank">Martens,proteomics,2006</a><br/><b>Interaction consistency score</b>: 16.19&nbsp<b>Coverage</b>: 15',organ:"BRAIN",integrated:!1,coverage:15,publication_year:2006,num_abundances:3425,hasPeptideCounts:!0,filename:"9606-human_brain_Martens_2006_proteomics.txt"},{id:471,name:"H.sapiens - Brain, SC (Peptideatlas,Jun,2011)",score:5.88,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/human/brain/index.php" target="_blank">Peptideatlas,Jun,2011</a><br/><b>Interaction consistency score</b>: 5.88&nbsp<b>Coverage</b>: 13',organ:"BRAIN",integrated:!1,coverage:13,publication_year:2011,num_abundances:2919,hasPeptideCounts:!0,filename:"9606-human_brain_atlas_201106.txt"},{id:468,name:"H.sapiens - Lung, Peptide Ion Intensity Values, SC (Abdul-Salam,Circulation,2010)",score:3.47,weight:"100%",description:'abundance based on Spectral counting, Peptide Ion Intensity Values, from<a href=" http://circ.ahajournals.org/content/122/20/2058.long" target="_blank">Abdul-Salam,Circulation,2010</a><br/><b>Interaction consistency score</b>: 3.47&nbsp<b>Coverage</b>: 2',organ:"LUNG",integrated:!1,coverage:2,publication_year:2010,num_abundances:507,hasPeptideCounts:!0,filename:"9606-human_lungPeptides_Abdul-Salam_2010_Circulation_PeptideIonIntensityValues.txt"},{id:338,name:"H.sapiens - Lung, Peptide Ion Intensity Values, nomalized data (Abdul-Salam,Circulation,2010)",score:.8,weight:"100%",description:'abundance based on nomalized data, Peptide Ion Intensity Values, from<a href=" http://circ.ahajournals.org/content/122/20/2058.long" target="_blank">Abdul-Salam,Circulation,2010</a><br/><b>Interaction consistency score</b>: 0.8&nbsp<b>Coverage</b>: 1',organ:"LUNG",integrated:!1,coverage:1,publication_year:2010,num_abundances:332,hasPeptideCounts:!1,filename:"9606-human_lungProteins_Abdul-Salam_2010_Circulation_PeptideIonIntensityValues.txt"},{id:492,name:"H.sapiens - Plasma, SC (Peptideatlas,May,2010)",score:5.85,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/human/plasma/index.php" target="_blank">Peptideatlas,May,2010</a><br/><b>Interaction consistency score</b>: 5.85&nbsp<b>Coverage</b>: 10',organ:"PLASMA",integrated:!1,coverage:10,publication_year:2010,num_abundances:2216,hasPeptideCounts:!0,filename:"9606-human_plasma_atlas_201005.txt"},{id:433,name:"H.sapiens - Platelet, SC (Martens,Proteomics,2006)",score:.21,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.200401142/abstract;jsessionid=C3DD079FC6E27896523F256C594477A6.d01t04" target="_blank">Martens,Proteomics,2006</a><br/><b>Interaction consistency score</b>: 0.21&nbsp<b>Coverage</b>: 3',organ:"PLATELET",integrated:!1,coverage:3,publication_year:2006,num_abundances:716,hasPeptideCounts:!0,filename:"9606-human_platelet_Martens_2005_Proteomics.txt"},{id:57,name:"H.sapiens - Urine, SC (Peptideatlas,Jun,2010)",score:7.49,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/human/urine/index.php" target="_blank">Peptideatlas,Jun,2010</a><br/><b>Interaction consistency score</b>: 7.49&nbsp<b>Coverage</b>: 6',organ:"URINE",integrated:!1,coverage:6,publication_year:2010,num_abundances:1464,hasPeptideCounts:!0,filename:"9606-human_urine_atalas_201006.txt"},{id:292,name:"H.sapiens - Cell line, A549 (Geiger,MCP,2012)",score:15.94,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, A549, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 15.94&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_A549_Geiger_2012_uniprot.txt"},{id:337,name:"H.sapiens - Cell line, GAMG (Geiger,MCP,2012)",score:12.15,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, GAMG, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 12.15&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_GAMG_Geiger_2012_uniprot.txt"},{id:329,name:"H.sapiens - Cell line, HEK293 (Geiger,MCP,2012)",score:12.31,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, HEK293, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 12.31&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_HEK293_Geiger_2012_uniprot.txt"},{id:182,name:"H.sapiens - Cell line, Hela (Geiger,MCP,2012)",score:15.68,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, hela, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 15.68&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_HeLa_Geiger_2012_uniprot.txt"},{id:257,name:"H.sapiens - Cell line, HepG2 (Geiger,MCP,2012)",score:16.19,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, HepG2, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 16.19&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_HepG2_Geiger_2012_uniprot.txt"},{id:175,name:"H.sapiens - Cell line, Jurkat (Geiger,MCP,2012)",score:18.03,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, Jurkat, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 18.03&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_Jurkat_Geiger_2012_uniprot.txt"},{id:294,name:"H.sapiens - Cell line, K562 (Geiger,MCP,2012)",score:21.14,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, K562, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 21.14&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_K562_Geiger_2012_uniprot.txt"},{id:451,name:"H.sapiens - Cell line, LnCap (Geiger,MCP,2012)",score:12.82,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, LnCap, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 12.82&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_LnCap_Geiger_2012_uniprot.txt"},{id:441,name:"H.sapiens - Cell line, MCF7 (Geiger,MCP,2012)",score:11.77,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, MCF7, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 11.77&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_MCF7_Geiger_2012_uniprot.txt"},{id:395,name:"H.sapiens - Cell line, RKO (Geiger,MCP,2012)",score:17.55,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, RKO, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 17.55&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_RKO_Geiger_2012_uniprot.txt"},{id:359,name:"H.sapiens - Cell line, U2OS (Geiger,MCP,2012)",score:13.41,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, U2OS, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M111.014050" target="_blank">Geiger,MCP,2012</a><br/><b>Interaction consistency score</b>: 13.41&nbsp<b>Coverage</b>: 45',organ:"CELL_LINE",integrated:!1,coverage:45,publication_year:2012,num_abundances:10225,hasPeptideCounts:!1,filename:"9606-iBAQ_U2OS_Geiger_2012_uniprot.txt"},{id:232,name:"H.sapiens - Cell line, DF19.11 (Phanstiel,Nat_Methods,2011)",score:4.16,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, DF19.11, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.16&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6548,hasPeptideCounts:!1,filename:"9606-iPS_(DF19.11)_iTRAQ-114_Phanstiel_2011_gene.txt"},{id:225,name:"H.sapiens - Cell line, DF19.7 (Phanstiel,Nat_Methods,2011)",score:4.31,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, DF19.7, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.31&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6547,hasPeptideCounts:!1,filename:"9606-iPS_(DF19.7)_iTRAQ-116_Phanstiel_2011_gene.txt"},{id:418,name:"H.sapiens - Cell line, DF4.7 (Phanstiel,Nat_Methods,2011)",score:4.03,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, DF4.7, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.03&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6547,hasPeptideCounts:!1,filename:"9606-iPS_(DF4.7)_iTRAQ-121_Phanstiel_2011_gene.txt"},{id:398,name:"H.sapiens - Cell line, DF6.9 (Phanstiel,Nat_Methods,2011)",score:4.21,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, DF6.9, from<a href="http://www.nature.com/nmeth/journal/v8/n10/full/nmeth.1699.html" target="_blank">Phanstiel,Nat_Methods,2011</a><br/><b>Interaction consistency score</b>: 4.21&nbsp<b>Coverage</b>: 29',organ:"CELL_LINE",integrated:!1,coverage:29,publication_year:2011,num_abundances:6548,hasPeptideCounts:!1,filename:"9606-iPS_(DF6.9)_iTRAQ-118_Phanstiel_2011_gene.txt"},{id:376,name:"H.sapiens - Kidney, SC (Wilhelm,Nature,2014)",score:9.19,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 9.19&nbsp<b>Coverage</b>: 19',organ:"KIDNEY",integrated:!1,coverage:19,publication_year:2014,num_abundances:4227,hasPeptideCounts:!0,filename:"9606-kidney_Wilhelm_2014_Maxquant.txt"},{id:470,name:"H.sapiens - Liver, SC (Wilhelm,Nature,2014)",score:7.35,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.35&nbsp<b>Coverage</b>: 21',organ:"LIVER",integrated:!1,coverage:21,publication_year:2014,num_abundances:4822,hasPeptideCounts:!0,filename:"9606-liver_Wilhelm_2014_Maxquant.txt"},{id:168,name:"H.sapiens - Lung, SC (Wilhelm,Nature,2014)",score:9.95,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 9.95&nbsp<b>Coverage</b>: 17',organ:"LUNG",integrated:!1,coverage:17,publication_year:2014,num_abundances:3888,hasPeptideCounts:!0,filename:"9606-lung_Wilhelm_2014_Maxquant.txt"},{id:119,name:"H.sapiens - Lymph node, SC (Wilhelm,Nature,2014)",score:10.17,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 10.17&nbsp<b>Coverage</b>: 26',organ:"LYMPH_NODE",integrated:!1,coverage:26,publication_year:2014,num_abundances:5877,hasPeptideCounts:!0,filename:"9606-lymph_node_Wilhelm_2014_Maxquant.txt"},{id:335,name:"H.sapiens - Milk, SC (Wilhelm,Nature,2014)",score:3.96,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 3.96&nbsp<b>Coverage</b>: 8',organ:"MILK",integrated:!1,coverage:8,publication_year:2014,num_abundances:1841,hasPeptideCounts:!0,filename:"9606-milk_Wilhelm_2014_Maxquant.txt"},{id:406,name:"H.sapiens - Nasopharynx, SC (Wilhelm,Nature,2014)",score:6.44,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 6.44&nbsp<b>Coverage</b>: 8',organ:"NASOPHARYNX",integrated:!1,coverage:8,publication_year:2014,num_abundances:1879,hasPeptideCounts:!0,filename:"9606-nasopharynx_Wilhelm_2014_Maxquant.txt"},{id:205,name:"H.sapiens - Oral cavity, SC (Wilhelm,Nature,2014)",score:4.79,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 4.79&nbsp<b>Coverage</b>: 17',organ:"ORAL_CAVITY",integrated:!1,coverage:17,publication_year:2014,num_abundances:3926,hasPeptideCounts:!0,filename:"9606-oral_cavity_Wilhelm_2014_Maxquant.txt"},{id:275,name:"H.sapiens - Female gonad, SC (Wilhelm,Nature,2014)",score:8.56,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 8.56&nbsp<b>Coverage</b>: 23',organ:"FEMALE_GONAD",integrated:!1,coverage:23,publication_year:2014,num_abundances:5165,hasPeptideCounts:!0,filename:"9606-ovary_Wilhelm_2014_Maxquant.txt"},{id:187,name:"H.sapiens - Pancreas, SC (Wilhelm,Nature,2014)",score:9.42,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 9.42&nbsp<b>Coverage</b>: 19',organ:"PANCREAS",integrated:!1,coverage:19,publication_year:2014,num_abundances:4417,hasPeptideCounts:!0,filename:"9606-pancreas_Wilhelm_2014_Maxquant.txt"},{id:33,name:"H.sapiens - Whole organism, SC (PeptideAtlas,March,2009)",score:4.25,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/human/" target="_blank">PeptideAtlas,March,2009</a><br/><b>Interaction consistency score</b>: 4.25&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2009,num_abundances:11232,hasPeptideCounts:!0,filename:"9606-peptideatlaBuild_March2009.peptides.new.txt"},{id:313,name:"H.sapiens - Placenta, SC (Wilhelm,Nature,2014)",score:8.09,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 8.09&nbsp<b>Coverage</b>: 18',organ:"PLACENTA",integrated:!1,coverage:18,publication_year:2014,num_abundances:4149,hasPeptideCounts:!0,filename:"9606-placenta_Wilhelm_2014_Maxquant.txt"},{id:443,name:"H.sapiens - Prostate gland, SC (Wilhelm,Nature,2014)",score:7.45,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.45&nbsp<b>Coverage</b>: 22',organ:"PROSTATE_GLAND",integrated:!1,coverage:22,publication_year:2014,num_abundances:4924,hasPeptideCounts:!0,filename:"9606-prostate_Wilhelm_2014_Maxquant.txt"},{id:465,name:"H.sapiens - Rectum, SC (Wilhelm,Nature,2014)",score:9.52,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 9.52&nbsp<b>Coverage</b>: 20',organ:"RECTUM",integrated:!1,coverage:20,publication_year:2014,num_abundances:4510,hasPeptideCounts:!0,filename:"9606-rectum_Wilhelm_2014_Maxquant.txt"},{id:423,name:"H.sapiens - Saliva, SC (Wilhelm,Nature,2014)",score:9.86,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 9.86&nbsp<b>Coverage</b>: 8',organ:"SALIVA",integrated:!1,coverage:8,publication_year:2014,num_abundances:1830,hasPeptideCounts:!0,filename:"9606-saliva_Wilhelm_2014_Maxquant.txt"},{id:39,name:"H.sapiens - Saliva secreting gland, SC (Wilhelm,Nature,2014)",score:10.56,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 10.56&nbsp<b>Coverage</b>: 25',organ:"SALIVA_SECRETING_GLAND",integrated:!1,coverage:25,publication_year:2014,num_abundances:5648,hasPeptideCounts:!0,filename:"9606-salivary_gland_Wilhelm_2014_Maxquant.txt"},{id:59,name:"H.sapiens - Seminal vesicle, SC (Wilhelm,Nature,2014)",score:6.05,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 6.05&nbsp<b>Coverage</b>: 16',organ:"SEMINAL_VESICLE",integrated:!1,coverage:16,publication_year:2014,num_abundances:3720,hasPeptideCounts:!0,filename:"9606-seminal_vesicle_Wilhelm_2014_Maxquant.txt"},{id:112,name:"H.sapiens - Skin, SC (Wilhelm,Nature,2014)",score:6.94,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 6.94&nbsp<b>Coverage</b>: 18',organ:"SKIN",integrated:!1,coverage:18,publication_year:2014,num_abundances:4162,hasPeptideCounts:!0,filename:"9606-skin_Wilhelm_2014_Maxquant.txt"},{id:136,name:"H.sapiens - Spleen, SC (Wilhelm,Nature,2014)",score:7.2,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.2&nbsp<b>Coverage</b>: 20',organ:"SPLEEN",integrated:!1,coverage:20,publication_year:2014,num_abundances:4647,hasPeptideCounts:!0,filename:"9606-spleen_Wilhelm_2014_Maxquant.txt"},{id:457,name:"H.sapiens - Stomach, SC (Wilhelm,Nature,2014)",score:7.29,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.29&nbsp<b>Coverage</b>: 23',organ:"STOMACH",integrated:!1,coverage:23,publication_year:2014,num_abundances:5354,hasPeptideCounts:!0,filename:"9606-stomach_Wilhelm_2014_Maxquant.txt"},{id:239,name:"H.sapiens - Testis, SC (Wilhelm,Nature,2014)",score:12,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 12.0&nbsp<b>Coverage</b>: 25',organ:"TESTIS",integrated:!1,coverage:25,publication_year:2014,num_abundances:5662,hasPeptideCounts:!0,filename:"9606-testis_Wilhelm_2014_Maxquant.txt"},{id:491,name:"H.sapiens - Thyroid gland, SC (Wilhelm,Nature,2014)",score:8.85,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 8.85&nbsp<b>Coverage</b>: 17',organ:"THYROID_GLAND",integrated:!1,coverage:17,publication_year:2014,num_abundances:3874,hasPeptideCounts:!0,filename:"9606-thyroid_gland_Wilhelm_2014_Maxquant.txt"},{id:269,name:"H.sapiens - Tonsil, SC (Wilhelm,Nature,2014)",score:8.34,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 8.34&nbsp<b>Coverage</b>: 21',organ:"TONSIL",integrated:!1,coverage:21,publication_year:2014,num_abundances:4743,hasPeptideCounts:!0,filename:"9606-tonsil_Wilhelm_2014_Maxquant.txt"},{id:394,name:"H.sapiens - Fallopian tube, SC (Wilhelm,Nature,2014)",score:7.92,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.92&nbsp<b>Coverage</b>: 20',organ:"FALLOPIAN_TUBE",integrated:!1,coverage:20,publication_year:2014,num_abundances:4493,hasPeptideCounts:!0,filename:"9606-tube_Wilhelm_2014_Maxquant.txt"},{id:230,name:"H.sapiens - Uterus, Post-menopause, SC (Wilhelm,Nature,2014)",score:6.07,weight:"100%",description:'abundance based on Spectral counting, post-menopause, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 6.07&nbsp<b>Coverage</b>: 16',organ:"UTERUS",integrated:!1,coverage:16,publication_year:2014,num_abundances:3720,hasPeptideCounts:!0,filename:"9606-uterus_post-menopause_Wilhelm_2014_Maxquant.txt"},{id:98,name:"H.sapiens - Uterus, Pre-menopause, SC (Wilhelm,Nature,2014)",score:5.65,weight:"100%",description:'abundance based on Spectral counting, pre-menopause, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 5.65&nbsp<b>Coverage</b>: 20',organ:"UTERUS",integrated:!1,coverage:20,publication_year:2014,num_abundances:4550,hasPeptideCounts:!0,filename:"9606-uterus_pre-menopause_Wilhelm_2014_Maxquant.txt"},{id:303,name:"H.sapiens - Vulva, SC (Wilhelm,Nature,2014)",score:7.81,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.nature.com/nature/journal/v509/n7502/full/nature13319.html" target="_blank">Wilhelm,Nature,2014</a><br/><b>Interaction consistency score</b>: 7.81&nbsp<b>Coverage</b>: 11',organ:"VULVA",integrated:!1,coverage:11,publication_year:2014,num_abundances:2539,hasPeptideCounts:!0,filename:"9606-vulva_Wilhelm_2014_Maxquant.txt"}],total_coverage:98,average_coverage:27.256756756756758,topProteins:[{id:1857259,abundance:39179,rank:0,name:"TMSB4X",annotation:"Thymosin beta 4, X-linked; Plays an important role in the organization of the cytoskeleton (By similarity). Binds to and sequesters actin monomers (G actin) and therefore inhibits actin polymerization"},{id:1854701,abundance:30334,rank:1,name:"APOA2",annotation:"Apolipoprotein A-II; May stabilize HDL (high density lipoprotein) structure by its association with lipids, and affect the HDL metabolism"},{id:1855488,abundance:24741,rank:2,name:"RBP4",annotation:"Retinol binding protein 4, plasma; Delivers retinol from the liver stores to the peripheral tissues. In plasma, the RBP-retinol complex interacts with transthyretin, this prevents its loss by filtration through the kidney glomeruli"},{id:1847637,abundance:17631,rank:3,name:"ALB",annotation:"Albumin; Serum albumin, the main protein of plasma, has a good binding capacity for water, Ca(2+), Na(+), K(+), fatty acids, hormones, bilirubin and drugs. Its main function is the regulation of the colloidal osmotic pressure of blood. Major zinc transporter in plasma, typically binds about 80% of all plasma zinc"},{id:1844690,abundance:16714,rank:4,name:"ORM1",annotation:"Orosomucoid 1; Functions as transport protein in the blood stream. Binds various ligands in the interior of its beta-barrel domain. Also binds synthetic drugs and influences their distribution and availability in the body. Appears to function in modulating the activity of the immune system during the acute-phase reaction"},{id:1843185,abundance:13168,rank:5,name:"GAPDH",annotation:"Glyceraldehyde-3-phosphate dehydrogenase; Has both glyceraldehyde-3-phosphate dehydrogenase and nitrosylase activities, thereby playing a role in glycolysis and nuclear functions, respectively. Participates in nuclear events including transcription, RNA transport, DNA replication and apoptosis. Nuclear functions are probably due to the nitrosylase activity that mediates cysteine S-nitrosylation of nuclear target proteins such as SIRT1, HDAC2 and PRKDC. Modulates the organization and assembly of the cytoskeleton. Facilitates the CHP1-dependent microtubule and membrane associations throu [...] "},{id:1843442,abundance:12178,rank:6,name:"APOA1",annotation:"Apolipoprotein A-I; Participates in the reverse transport of cholesterol from tissues to the liver for excretion by promoting cholesterol efflux from tissues and by acting as a cofactor for the lecithin cholesterol acyltransferase (LCAT). As part of the SPAP complex, activates spermatozoa motility"},{id:1863965,abundance:12162,rank:7,name:"APOC2",annotation:"Apolipoprotein C-II; Component of chylomicrons, very low-density lipoproteins (VLDL), low-density lipoproteins (LDL), and high-density lipoproteins (HDL) in plasma. Plays an important role in lipoprotein metabolism as an activator of lipoprotein lipase. Both proapolipoprotein C-II and apolipoprotein C-II can activate lipoprotein lipase. In normolipidemic individuals, it is mainly distributed in the HDL, whereas in hypertriglyceridemic individuals, predominantly found in the VLDL and LDL"}]},9615:{id:9615,name:"Canis lupus familiaris",compact_name:"Canis lupus",num_proteins:19856,genome_source:"<a href='http://www.ensembl.org/Canis_lupus_familiaris//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:403,name:"C.familiaris - Whole organism, SC (GPM,Oct,2012)",score:6,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 6.0&nbsp<b>Coverage</b>: 22',organ:"WHOLE_ORGANISM",integrated:!1,coverage:22,publication_year:2012,num_abundances:4311,hasPeptideCounts:!0,filename:"9615-GPM_2012_09_Canis_familiaris.txt"}],total_coverage:21,average_coverage:22,topProteins:[{id:1866788,abundance:35952,rank:0,name:"CYCS",annotation:"Cytochrome c ; Electron carrier protein. The oxidized form of the cytochrome c heme group can accept an electron from the heme group of the cytochrome c1 subunit of cytochrome reductase. Cytochrome c then transfers this electron to the cytochrome oxidase complex, the final protein carrier in the mitochondrial electron-transport chain"},{id:1866902,abundance:15898,rank:1,name:"ALB",annotation:"Serum albumin precursor ; Serum albumin, the main protein of plasma, has a good binding capacity for water, Ca(2+), Na(+), K(+), fatty acids, hormones, bilirubin and drugs. Its main function is the regulation of the colloidal osmotic pressure of blood. Major zinc transporter in plasma, typically binds about 80% of all plasma zinc"},{id:1867852,abundance:14776,rank:2,name:"VIM",annotation:"Uncharacterized protein "},{id:1867376,abundance:11175,rank:3,name:"PRSS1",annotation:"Uncharacterized protein "},{id:1875350,abundance:10622,rank:4,name:"KRT10",annotation:"Keratin, type I cytoskeletal 10 "},{id:1869587,abundance:9757,rank:5,name:"KRT1",annotation:"Keratin, type II cytoskeletal 1 ; May regulate the activity of kinases such as PKC and SRC via binding to integrin beta-1 (ITB1) and the receptor of activated protein C kinase 1 (RACK1). In complex with C1QBP is a high affinity receptor for kininogen-1/HMWK (By similarity)"},{id:1875357,abundance:9482,rank:6,name:"ACTB",annotation:"Actin, cytoplasmic 1 Actin, cytoplasmic 1, N-terminally processed; Actins are highly conserved proteins that are involved in various types of cell motility and are ubiquitously expressed in all eukaryotic cells"},{id:1874777,abundance:8192,rank:7,name:"ACTG1",annotation:"Uncharacterized protein "}]},9796:{id:9796,name:"Equus caballus",compact_name:"Equus caballus",num_proteins:20449,genome_source:"<a href='http://www.ensembl.org/Equus_caballus//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:120,name:"E.caballus - Whole organism, SC (PeptideAtlas,April,2013)",score:11.63,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,April,2013</a><br/><b>Interaction consistency score</b>: 11.63&nbsp<b>Coverage</b>: 15',organ:"WHOLE_ORGANISM",integrated:!1,coverage:15,publication_year:2013,num_abundances:3155,hasPeptideCounts:!0,filename:"9796-PA_201304.txt"}],total_coverage:15,average_coverage:15,topProteins:[{id:1987490,abundance:98317,rank:0,name:"HBB",annotation:"Hemoglobin subunit beta ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:1988488,abundance:77955,rank:1,name:"ALB",annotation:"Serum albumin precursor ; Serum albumin, the main protein of plasma, has a good binding capacity for water, Ca(2+), Na(+), K(+), fatty acids, hormones, bilirubin and drugs. Its main function is the regulation of the colloidal osmotic pressure of blood. Major zinc transporter in plasma, typically binds about 80% of all plasma zinc"},{id:1986667,abundance:73079,rank:2,name:"TMSB4X",annotation:"Thymosin beta-4 Hematopoietic system regulatory peptide; Plays an important role in the organization of the cytoskeleton. Binds to and sequesters actin monomers (G actin) and therefore inhibits actin polymerization (By similarity)"},{id:1986969,abundance:24131,rank:3,name:"ENSECAG00000009556",annotation:"Uncharacterized protein "},{id:1983290,abundance:22714,rank:4,name:"IGLC1",annotation:"Uncharacterized protein "},{id:1994012,abundance:21430,rank:5,name:"MB",annotation:"Myoglobin ; Serves as a reserve supply of oxygen and facilitates the movement of oxygen within muscles"},{id:1987136,abundance:15173,rank:6,name:"APOA1",annotation:"Apolipoprotein A-I"},{id:1981307,abundance:13527,rank:7,name:"IGKC",annotation:"Immunoglobulin kappa constant"}]},9823:{id:9823,name:"Sus scrofa",compact_name:"Sus scrofa",num_proteins:21630,genome_source:"<a href='http://www.ensembl.org/Sus_scrofa//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:30,name:"S.scrofa - Whole organism, SC (PeptideAtlas,Aug,2011)",score:8.76,weight:"100%",description:'abundance based on Spectral counting, from<a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/defaultBuildsPepsProts?_subtab=3" target="_blank">PeptideAtlas,Aug,2011</a><br/><b>Interaction consistency score</b>: 8.76&nbsp<b>Coverage</b>: 14',organ:"WHOLE_ORGANISM",integrated:!1,coverage:14,publication_year:2011,num_abundances:2952,hasPeptideCounts:!0,filename:"9823-Pig_PeptideAtlas_2011-08_Ens62.txt"}],total_coverage:13,average_coverage:14,topProteins:[{id:2022183,abundance:120368,rank:0,name:"HBA",annotation:"Hemoglobin subunit alpha ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:2031604,abundance:103665,rank:1,name:"CKM",annotation:"Creatine kinase M-type "},{id:2027100,abundance:79641,rank:2,name:"HBB",annotation:"Hemoglobin subunit beta ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:2017098,abundance:39982,rank:3,name:"GAPDH",annotation:"Glyceraldehyde-3-phosphate dehydrogenase ; Has both glyceraldehyde-3-phosphate dehydrogenase and nitrosylase activities, thereby playing a role in glycolysis and nuclear functions, respectively. Glyceraldehyde-3-phosphate dehydrogenase is a key enzyme in glycolysis that catalyzes the first step of the pathway by converting D-glyceraldehyde 3- phosphate (G3P) into 3-phospho-D-glyceroyl phosphate. Modulates the organization and assembly of the cytoskeleton. Facilitates the CHP1-dependent microtubule and membrane associations through its ability to stimulate the binding of CHP1 to microtu [...] "},{id:2020612,abundance:34526,rank:4,name:"LOC396596",annotation:"Beta-lactoglobulin-1A/1C ; Lactoglobulin is the primary component of whey, it binds retinol and is probably involved in the transport of that molecule"},{id:2029455,abundance:34079,rank:5,name:"ENO3",annotation:"Beta-enolase ; Appears to have a function in striated muscle development and regeneration"},{id:2017092,abundance:27032,rank:6,name:"TPI1",annotation:"Triosephosphate isomerase "},{id:2027102,abundance:25515,rank:7,name:"ENSSSCG00000014727",annotation:"Uncharacterized protein "}]},9913:{id:9913,name:"Bos taurus",compact_name:"Bos taurus",num_proteins:19994,genome_source:"<a href='http://www.ensembl.org/Bos_taurus//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:21,name:"B.taurus - Whole organism, SC (GPM,Oct,2012)",score:5.9,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct, 2012</a><br/><b>Interaction consistency score</b>: 5.9&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2012,num_abundances:9823,hasPeptideCounts:!0,filename:"9913-GPM_2012_09_Bos_taurus.txt"},{id:231,name:"B.taurus - Whole organism, SC (GPM,Aug,2014)",score:15.81,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 15.81&nbsp<b>Coverage</b>: 22',organ:"WHOLE_ORGANISM",integrated:!1,coverage:22,publication_year:2014,num_abundances:4482,hasPeptideCounts:!0,filename:"9913-GPM_201408.txt"},{id:365,name:"B.taurus - Whole organism, SC (PeptideAtlas,Dec,2011)",score:5.22,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Dec,2011</a><br/><b>Interaction consistency score</b>: 5.22&nbsp<b>Coverage</b>: 12',organ:"WHOLE_ORGANISM",integrated:!1,coverage:12,publication_year:2011,num_abundances:2408,hasPeptideCounts:!0,filename:"9913-PA_201112.txt"},{id:456,name:"B.taurus - Whole organism (Integrated)",score:15.47,weight:"",description:"integrated dataset: weighted average of all B.taurus WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 15.47&nbsp<b>Coverage</b>: 54",organ:"WHOLE_ORGANISM",integrated:!0,coverage:54,publication_year:2017,num_abundances:10879,hasPeptideCounts:!1,filename:"9913-WHOLE_ORGANISM-integrated.txt"},{id:202,name:"B.taurus - Whole organism, SC (PeptideAtlas,Dec,2011)",score:4.89,weight:"40%",description:'abundance based on Spectral counting, from<a href="www.peptideatlas.org/builds/cow/201112/atlas_build_320.mysql.gz" target="_blank">PeptideAtlas, Dec,2011</a><br/><b>Interaction consistency score</b>: 4.89&nbsp<b>Coverage</b>: 12',organ:"WHOLE_ORGANISM",integrated:!1,coverage:12,publication_year:2011,num_abundances:2408,hasPeptideCounts:!0,filename:"9913-cow_atlas_build_320.txt"}],total_coverage:54,average_coverage:23.75,topProteins:[{id:2046343,abundance:120389,rank:0,name:"PAEP",annotation:"Beta-lactoglobulin precursor ; Primary component of whey, it binds retinol and is probably involved in the transport of that molecule"},{id:2039716,abundance:58752,rank:1,name:"CSN2",annotation:"Beta-casein Casoparan Antioxidant peptide Casohypotensin; Important role in determination of the surface properties of the casein micelles"},{id:2042531,abundance:47651,rank:2,name:"CSN1S1",annotation:"Alpha-S1-casein Antioxidant peptide; Important role in the capacity of milk to transport calcium phosphate"},{id:2047723,abundance:39836,rank:3,name:"ALB",annotation:"Serum albumin ; Serum albumin, the main protein of plasma, has a good binding capacity for water, Ca(2+), Na(+), K(+), fatty acids, hormones, bilirubin and drugs. Its main function is the regulation of the colloidal osmotic pressure of blood. Major zinc transporter in plasma, typically binds about 80% of all plasma zinc"},{id:2040308,abundance:25230,rank:4,name:"LOC615026",annotation:"Cationic trypsin Alpha-trypsin chain 1 Alpha-trypsin chain 2"},{id:2051080,abundance:25230,rank:5,name:"ENSBTAG00000024245",annotation:"Cationic trypsin Alpha-trypsin chain 1 Alpha-trypsin chain 2"},{id:2052946,abundance:21522,rank:6,name:"HBB",annotation:"Hemoglobin subunit beta ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:2050172,abundance:18121,rank:7,name:"CSN3",annotation:"Kappa-casein Casoxin-C Casoxin-6 Casoxin-A Casoxin-B Casoplatelin; Kappa-casein stabilizes micelle formation, preventing casein precipitation in milk"}]},10090:{id:10090,name:"Mus musculus",compact_name:"Mus musculus",num_proteins:22668,genome_source:"<a href='http://www.ensembl.org/Mus_musculus//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:278,name:"M.musculus - Adrenal gland,  Protein Intensity (Geiger,MCP,2013)",score:9.88,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.88&nbsp<b>Coverage</b>: 33',organ:"ADRENAL_GLAND",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Adrenal_gland_geiger_2013.txt"},{id:154,name:"M.musculus - Brain (Integrated)",score:10.19,weight:"",description:"integrated dataset: weighted average of all M.musculus BRAIN datasets<br/><b>Interaction consistency score</b>: 10.19&nbsp<b>Coverage</b>: 44",organ:"BRAIN",integrated:!0,coverage:44,publication_year:2017,num_abundances:10025,hasPeptideCounts:!1,filename:"10090-BRAIN-integrated.txt"},{id:343,name:"M.musculus - Brown adipose tissue (Integrated)",score:15.32,weight:"",description:"integrated dataset: weighted average of all M.musculus BROWN_ADIPOSE_TISSUE datasets<br/><b>Interaction consistency score</b>: 15.32&nbsp<b>Coverage</b>: 36",organ:"BROWN_ADIPOSE_TISSUE",integrated:!0,coverage:36,publication_year:2017,num_abundances:8120,hasPeptideCounts:!1,filename:"10090-BROWN_ADIPOSE_TISSUE-integrated.txt"},{id:291,name:"M.musculus - Cerebral cortex,  Protein Intensity (Geiger,MCP,2013)",score:8.17,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 8.17&nbsp<b>Coverage</b>: 33',organ:"CEREBRAL_CORTEX",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Brain_cortex_geiger_2013.txt"},{id:409,name:"M.musculus - Medulla oblongata,  Protein Intensity (Geiger,MCP,2013)",score:9.73,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.73&nbsp<b>Coverage</b>: 33',organ:"MEDULLA_OBLONGATA",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Brain_medulla_geiger_2013.txt"},{id:238,name:"M.musculus - Brown adipose tissue,  Protein Intensity (Geiger,MCP,2013)",score:12.43,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 12.43&nbsp<b>Coverage</b>: 33',organ:"BROWN_ADIPOSE_TISSUE",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Brown_fat_geiger_2013.txt"},{id:251,name:"M.musculus - Cell line (Integrated)",score:3.29,weight:"",description:"integrated dataset: weighted average of all M.musculus CELL_LINE datasets<br/><b>Interaction consistency score</b>: 3.29&nbsp<b>Coverage</b>: 42",organ:"CELL_LINE",integrated:!0,coverage:42,publication_year:2017,num_abundances:9652,hasPeptideCounts:!1,filename:"10090-CELL_LINE-integrated.txt"},{id:218,name:"M.musculus - Cerebellum,  Protein Intensity (Geiger,MCP,2013)",score:9.05,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.05&nbsp<b>Coverage</b>: 33',organ:"CEREBELLUM",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Cerebellum_geiger_2013.txt"},{id:378,name:"M.musculus - Colon,  Protein Intensity (Geiger,MCP,2013)",score:10.88,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 10.88&nbsp<b>Coverage</b>: 33',organ:"COLON",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Colon_geiger_2013.txt"},{id:439,name:"M.musculus - Diaphragm,  Protein Intensity (Geiger,MCP,2013)",score:9.42,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.42&nbsp<b>Coverage</b>: 33',organ:"DIAPHRAGM",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Diaphragm_geiger_2013.txt"},{id:6,name:"M.musculus - Duodenum,  Protein Intensity (Geiger,MCP,2013)",score:9.27,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.27&nbsp<b>Coverage</b>: 33',organ:"DUODENUM",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Duodenum_geiger_2013.txt"},{id:78,name:"M.musculus - Embryonic tissue,  Protein Intensity (Geiger,MCP,2013)",score:8.76,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 8.76&nbsp<b>Coverage</b>: 33',organ:"EMBRYONIC_TISSUE",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Embryonic_tissue_geiger_2013.txt"},{id:115,name:"M.musculus - Eye,  Protein Intensity (Geiger,MCP,2013)",score:9.14,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.14&nbsp<b>Coverage</b>: 33',organ:"EYE",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Eye_geiger_2013.txt"},{id:149,name:"M.musculus - Whole organism, SC (GPM,Oct,2012)",score:12.69,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 12.69&nbsp<b>Coverage</b>: 78',organ:"WHOLE_ORGANISM",integrated:!1,coverage:78,publication_year:2012,num_abundances:17698,hasPeptideCounts:!0,filename:"10090-GPM_2012_09_Mus_musculus.txt"},{id:481,name:"M.musculus - Whole organism, SC (GPM,Aug,2014)",score:24.72,weight:"90%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 24.72&nbsp<b>Coverage</b>: 80',organ:"WHOLE_ORGANISM",integrated:!1,coverage:80,publication_year:2014,num_abundances:18117,hasPeptideCounts:!0,filename:"10090-GPM_201408.txt"},{id:73,name:"M.musculus - Heart (Integrated)",score:9.83,weight:"",description:"integrated dataset: weighted average of all M.musculus HEART datasets<br/><b>Interaction consistency score</b>: 9.83&nbsp<b>Coverage</b>: 37",organ:"HEART",integrated:!0,coverage:37,publication_year:2017,num_abundances:8310,hasPeptideCounts:!1,filename:"10090-HEART-integrated.txt"},{id:72,name:"M.musculus - Heart,  Protein Intensity (Geiger,MCP,2013)",score:8.85,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 8.85&nbsp<b>Coverage</b>: 33',organ:"HEART",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Heart_geiger_2013.txt"},{id:222,name:"M.musculus - Ileum,  Protein Intensity (Geiger,MCP,2013)",score:12,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 12.0&nbsp<b>Coverage</b>: 33',organ:"ILEUM",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Ileum_geiger_2013.txt"},{id:9,name:"M.musculus - Jejunum,  Protein Intensity (Geiger,MCP,2013)",score:11.89,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 11.89&nbsp<b>Coverage</b>: 33',organ:"JEJUNUM",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Jejunum_geiger_2013.txt"},{id:440,name:"M.musculus - Kidney (Integrated)",score:13.06,weight:"",description:"integrated dataset: weighted average of all M.musculus KIDNEY datasets<br/><b>Interaction consistency score</b>: 13.06&nbsp<b>Coverage</b>: 25",organ:"KIDNEY",integrated:!0,coverage:25,publication_year:2017,num_abundances:5709,hasPeptideCounts:!1,filename:"10090-KIDNEY-integrated.txt"},{id:427,name:"M.musculus - Cortex of kidney,  Protein Intensity (Geiger,MCP,2013)",score:13.25,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 13.25&nbsp<b>Coverage</b>: 33',organ:"CORTEX_OF_KIDNEY",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Kidney_cortex_geiger_2013.txt"},{id:132,name:"M.musculus - Renal medulla,  Protein Intensity (Geiger,MCP,2013)",score:9.07,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 9.07&nbsp<b>Coverage</b>: 33',organ:"RENAL_MEDULLA",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Kidney_medulla_geiger_2013.txt"},{id:254,name:"M.musculus - Liver (Integrated)",score:13.84,weight:"",description:"integrated dataset: weighted average of all M.musculus LIVER datasets<br/><b>Interaction consistency score</b>: 13.84&nbsp<b>Coverage</b>: 38",organ:"LIVER",integrated:!0,coverage:38,publication_year:2017,num_abundances:8649,hasPeptideCounts:!1,filename:"10090-LIVER-integrated.txt"},{id:74,name:"M.musculus - Lung (Integrated)",score:10.85,weight:"",description:"integrated dataset: weighted average of all M.musculus LUNG datasets<br/><b>Interaction consistency score</b>: 10.85&nbsp<b>Coverage</b>: 38",organ:"LUNG",integrated:!0,coverage:38,publication_year:2017,num_abundances:8681,hasPeptideCounts:!1,filename:"10090-LUNG-integrated.txt"},{id:488,name:"M.musculus - Liver,  Protein Intensity (Geiger,MCP,2013)",score:13.17,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 13.17&nbsp<b>Coverage</b>: 33',organ:"LIVER",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Liver_geiger_2013.txt"},{id:339,name:"M.musculus - Lung,  Protein Intensity (Geiger,MCP,2013)",score:6.08,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 6.08&nbsp<b>Coverage</b>: 33',organ:"LUNG",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Lung_geiger_2013.txt"},{id:421,name:"M.musculus - Whole organism, NSAF (Menschaert,MCP,2013)",score:2.7,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, NSAF, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23429522" target="_blank">Menschaert,MCP,2013</a><br/><b>Interaction consistency score</b>: 2.7&nbsp<b>Coverage</b>: 13',organ:"WHOLE_ORGANISM",integrated:!1,coverage:13,publication_year:2013,num_abundances:2879,hasPeptideCounts:!1,filename:"10090-Menschaert_protID_NSAF.txt"},{id:161,name:"M.musculus - Whole organism, EmPAI (Menschaert,MCP,2013)",score:2.61,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, emPAI, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23429522" target="_blank">Menschaert,MCP,2013</a><br/><b>Interaction consistency score</b>: 2.61&nbsp<b>Coverage</b>: 13',organ:"WHOLE_ORGANISM",integrated:!1,coverage:13,publication_year:2013,num_abundances:2879,hasPeptideCounts:!1,filename:"10090-Menschaert_protID_emPAI.txt"},{id:400,name:"M.musculus - Midbrain,  Protein Intensity (Geiger,MCP,2013)",score:6.98,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 6.98&nbsp<b>Coverage</b>: 33',organ:"MIDBRAIN",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Midbrain_geiger_2013.txt"},{id:346,name:"M.musculus - Whole organism, SC (PeptideAtlas,June,2010)",score:13.3,weight:"70%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/mouse/index.php" target="_blank">PeptideAtlas,June,2010</a><br/><b>Interaction consistency score</b>: 13.3&nbsp<b>Coverage</b>: 33',organ:"WHOLE_ORGANISM",integrated:!1,coverage:33,publication_year:2010,num_abundances:7410,hasPeptideCounts:!0,filename:"10090-MouseBuild243_Jun2010.peptides.new.txt"},{id:26,name:"M.musculus - Whole organism, SC (PeptideAtlas,June,2011)",score:12.36,weight:"10%",description:'abundance based on Spectral counting, from<a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/defaultBuildsPepsProts?_subtab=3" target="_blank">PeptideAtlas,June,2011</a><br/><b>Interaction consistency score</b>: 12.36&nbsp<b>Coverage</b>: 26',organ:"WHOLE_ORGANISM",integrated:!1,coverage:26,publication_year:2011,num_abundances:5910,hasPeptideCounts:!0,filename:"10090-Mouse_PeptideAtlas_2011-06_Ens62.txt"},{id:148,name:"M.musculus - Brain, SC (Huttlin,cell,2010)",score:15.29,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 15.29&nbsp<b>Coverage</b>: 25',organ:"BRAIN",integrated:!1,coverage:25,publication_year:2010,num_abundances:5575,hasPeptideCounts:!0,filename:"10090-Mouse_brain_Huttlin_cell2010.csv.txt"},{id:18,name:"M.musculus - Brown adipose tissue, SC (Huttlin,cell,2010)",score:14.65,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 14.65&nbsp<b>Coverage</b>: 19',organ:"BROWN_ADIPOSE_TISSUE",integrated:!1,coverage:19,publication_year:2010,num_abundances:4335,hasPeptideCounts:!0,filename:"10090-Mouse_brownfat_Huttlin_cell2010.csv.txt"},{id:459,name:"M.musculus - Erythrocyte, SC (Kruger,Cell,2008)",score:2.68,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">Kruger,Cell,2008</a><br/><b>Interaction consistency score</b>: 2.68&nbsp<b>Coverage</b>: 1',organ:"ERYTHROCYTE",integrated:!1,coverage:1,publication_year:2008,num_abundances:292,hasPeptideCounts:!0,filename:"10090-Mouse_erythrocytes_Kruger2008_cell.txt"},{id:76,name:"M.musculus - Gut epithelium, SC (Kruger,Cell,2008)",score:3.06,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">Kruger,Cell,2008</a><br/><b>Interaction consistency score</b>: 3.06&nbsp<b>Coverage</b>: 3',organ:"GUT_EPITHELIUM",integrated:!1,coverage:3,publication_year:2008,num_abundances:731,hasPeptideCounts:!0,filename:"10090-Mouse_gut_epithelium_Kruger2008_cell.txt"},{id:402,name:"M.musculus - Heart, SC (Huttlin,cell,2010)",score:16.78,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 16.78&nbsp<b>Coverage</b>: 21',organ:"HEART",integrated:!1,coverage:21,publication_year:2010,num_abundances:4748,hasPeptideCounts:!0,filename:"10090-Mouse_heart_Huttlin_cell2010.csv.txt"},{id:268,name:"M.musculus - Heart, SC (Kruger,Cell,2008)",score:1.97,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">Kruger,Cell,2008</a><br/><b>Interaction consistency score</b>: 1.97&nbsp<b>Coverage</b>: 1',organ:"HEART",integrated:!1,coverage:1,publication_year:2008,num_abundances:272,hasPeptideCounts:!0,filename:"10090-Mouse_heart_Kruger2008_cell.txt"},{id:482,name:"M.musculus - Kidney, SC (Huttlin,cell,2010)",score:15.47,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 15.47&nbsp<b>Coverage</b>: 24',organ:"KIDNEY",integrated:!1,coverage:24,publication_year:2010,num_abundances:5490,hasPeptideCounts:!0,filename:"10090-Mouse_kidney_Huttlin_cell2010.csv.txt"},{id:125,name:"M.musculus - Liver, SC (Huttlin,cell,2010)",score:15.49,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 15.49&nbsp<b>Coverage</b>: 22',organ:"LIVER",integrated:!1,coverage:22,publication_year:2010,num_abundances:5056,hasPeptideCounts:!0,filename:"10090-Mouse_liver_Huttlin_cell2010.csv.txt"},{id:446,name:"M.musculus - Liver, SC (Kruger,Cell,2008)",score:1.55,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">Kruger,Cell,2008</a><br/><b>Interaction consistency score</b>: 1.55&nbsp<b>Coverage</b>: 2',organ:"LIVER",integrated:!1,coverage:2,publication_year:2008,num_abundances:460,hasPeptideCounts:!0,filename:"10090-Mouse_liver_Kruger2008_cell.txt"},{id:375,name:"M.musculus - Liver, SC (Wang,ProteomicsC.A.,2009)",score:8.02,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/prca.200800145/abstract" target="_blank">Wang,ProteomicsC.A.,2009</a><br/><b>Interaction consistency score</b>: 8.02&nbsp<b>Coverage</b>: 3',organ:"LIVER",integrated:!1,coverage:3,publication_year:2009,num_abundances:681,hasPeptideCounts:!0,filename:"10090-Mouse_liver_Wang2009_Proteomics_ClinApp.txt"},{id:67,name:"M.musculus - Lung, SC (Huttlin,cell,2010)",score:11.67,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 11.67&nbsp<b>Coverage</b>: 25',organ:"LUNG",integrated:!1,coverage:25,publication_year:2010,num_abundances:5736,hasPeptideCounts:!0,filename:"10090-Mouse_lung_Huttlin_cell2010.csv.txt"},{id:405,name:"M.musculus - Pancreas, SC (Huttlin,cell,2010)",score:13.67,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 13.67&nbsp<b>Coverage</b>: 20',organ:"PANCREAS",integrated:!1,coverage:20,publication_year:2010,num_abundances:4464,hasPeptideCounts:!0,filename:"10090-Mouse_pancreas_Huttlin_cell2010.csv.txt"},{id:277,name:"M.musculus - Platelet, SC (Kruger,Cell,2008)",score:1.43,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">Kruger,Cell,2008</a><br/><b>Interaction consistency score</b>: 1.43&nbsp<b>Coverage</b>: 1',organ:"PLATELET",integrated:!1,coverage:1,publication_year:2008,num_abundances:292,hasPeptideCounts:!0,filename:"10090-Mouse_platelets_Kruger2008_cell.txt"},{id:452,name:"M.musculus - Spleen, SC (Kruger,Cell,2008)",score:1.5,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">Kruger,Cell,2008</a><br/><b>Interaction consistency score</b>: 1.5&nbsp<b>Coverage</b>: 1',organ:"SPLEEN",integrated:!1,coverage:1,publication_year:2008,num_abundances:177,hasPeptideCounts:!0,filename:"10090-Mouse_spleen_B_cells_Kruger2008_cell.txt"},{id:259,name:"M.musculus - Spleen, SC (Huttlin,cell,2010)",score:11.66,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 11.66&nbsp<b>Coverage</b>: 27',organ:"SPLEEN",integrated:!1,coverage:27,publication_year:2010,num_abundances:6131,hasPeptideCounts:!0,filename:"10090-Mouse_spleen_Huttlin_cell2010.csv.txt"},{id:411,name:"M.musculus - Testis, SC (Huttlin,cell,2010)",score:14.53,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867410013796" target="_blank">Huttlin,cell, 2010</a><br/><b>Interaction consistency score</b>: 14.53&nbsp<b>Coverage</b>: 28',organ:"TESTIS",integrated:!1,coverage:28,publication_year:2010,num_abundances:6422,hasPeptideCounts:!0,filename:"10090-Mouse_testis_Huttlin_cell2010_peptide.csv.txt"},{id:436,name:"M.musculus - Saliva, SC (Gundry,Proteomics,2010)",score:9.23,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/20512790" target="_blank">Gundry,Proteomics,2010</a><br/><b>Interaction consistency score</b>: 9.23&nbsp<b>Coverage</b>: 7',organ:"SALIVA",integrated:!1,coverage:7,publication_year:2010,num_abundances:1564,hasPeptideCounts:!0,filename:"10090-Mus_musculus_SC_biomart_11364_E__2reps.txt"},{id:424,name:"M.musculus - Cell line, Cell line, SC (Menschaert,MCP,2013)",score:.26,weight:"100%",description:'abundance based on Spectral counting, cell_line, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23429522" target="_blank">Menschaert,MCP, 2013</a><br/><b>Interaction consistency score</b>: 0.26&nbsp<b>Coverage</b>: 42',organ:"CELL_LINE",integrated:!1,coverage:42,publication_year:2013,num_abundances:9638,hasPeptideCounts:!0,filename:"10090-Mus_musculus_SC_biomart_28162_O_BTO0005136ESE14cell_1reps.txt"},{id:69,name:"M.musculus - Muscle,  Protein Intensity (Geiger,MCP,2013)",score:7.17,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 7.17&nbsp<b>Coverage</b>: 33',organ:"MUSCLE",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Muscle_geiger_2013.txt"},{id:345,name:"M.musculus - Olfactory bulb,  Protein Intensity (Geiger,MCP,2013)",score:7.85,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 7.85&nbsp<b>Coverage</b>: 33',organ:"OLFACTORY_BULB",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Olfactory_bulb_geiger_2013.txt"},{id:328,name:"M.musculus - Female gonad,  Protein Intensity (Geiger,MCP,2013)",score:4.65,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 4.65&nbsp<b>Coverage</b>: 33',organ:"FEMALE_GONAD",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Ovary_geiger_2013.txt"},{id:372,name:"M.musculus - Pancreas (Integrated)",score:10.88,weight:"",description:"integrated dataset: weighted average of all M.musculus PANCREAS datasets<br/><b>Interaction consistency score</b>: 10.88&nbsp<b>Coverage</b>: 37",organ:"PANCREAS",integrated:!0,coverage:37,publication_year:2017,num_abundances:8494,hasPeptideCounts:!1,filename:"10090-PANCREAS-integrated.txt"},{id:266,name:"M.musculus - Plasma, SC (PeptideAtlas,Sep,2012)",score:2.79,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/" target="_blank">PeptideAtlas,Sep,2012</a><br/><b>Interaction consistency score</b>: 2.79&nbsp<b>Coverage</b>: 4',organ:"PLASMA",integrated:!1,coverage:4,publication_year:2012,num_abundances:969,hasPeptideCounts:!0,filename:"10090-PA_plasma.txt"},{id:319,name:"M.musculus - Pancreas,  Protein Intensity (Geiger,MCP,2013)",score:7.51,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 7.51&nbsp<b>Coverage</b>: 33',organ:"PANCREAS",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Pancreas_geiger_2013.txt"},{id:79,name:"M.musculus - Spleen (Integrated)",score:11.35,weight:"",description:"integrated dataset: weighted average of all M.musculus SPLEEN datasets<br/><b>Interaction consistency score</b>: 11.35&nbsp<b>Coverage</b>: 39",organ:"SPLEEN",integrated:!0,coverage:39,publication_year:2017,num_abundances:8789,hasPeptideCounts:!1,filename:"10090-SPLEEN-integrated.txt"},{id:102,name:"M.musculus - Saliva secreting gland,  Protein Intensity (Geiger,MCP,2013)",score:6.77,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 6.77&nbsp<b>Coverage</b>: 33',organ:"SALIVA_SECRETING_GLAND",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Salivary_gland_geiger_2013.txt"},{id:364,name:"M.musculus - Spleen,  Protein Intensity (Geiger,MCP,2013)",score:4.6,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 4.6&nbsp<b>Coverage</b>: 33',organ:"SPLEEN",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Spleeen_geiger_2013.txt"},{id:80,name:"M.musculus - Stomach,  Protein Intensity (Geiger,MCP,2013)",score:5.79,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 5.79&nbsp<b>Coverage</b>: 33',organ:"STOMACH",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Stomach_geiger_2013.txt"},{id:373,name:"M.musculus - Thymus,  Protein Intensity (Geiger,MCP,2013)",score:5.04,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 5.04&nbsp<b>Coverage</b>: 33',organ:"THYMUS",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Thymus_geiger_2013.txt"},{id:320,name:"M.musculus - Uterus,  Protein Intensity (Geiger,MCP,2013)",score:5.13,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 5.13&nbsp<b>Coverage</b>: 33',organ:"UTERUS",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-Uterus_geiger_2013.txt"},{id:196,name:"M.musculus - Whole organism (Integrated)",score:14.15,weight:"",description:"integrated dataset: weighted average of all M.musculus WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 14.15&nbsp<b>Coverage</b>: 89",organ:"WHOLE_ORGANISM",integrated:!0,coverage:89,publication_year:2017,num_abundances:20219,hasPeptideCounts:!1,filename:"10090-WHOLE_ORGANISM-integrated.txt"},{id:219,name:"M.musculus - White adipose tissue,  Protein Intensity (Geiger,MCP,2013)",score:8.26,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS,  Protein_Intensity, from<a href="http://www.mcponline.org/cgi/doi/10.1074/mcp.M112.024919" target="_blank">Geiger,MCP,2013</a><br/><b>Interaction consistency score</b>: 8.26&nbsp<b>Coverage</b>: 33',organ:"WHITE_ADIPOSE_TISSUE",integrated:!1,coverage:33,publication_year:2013,num_abundances:7495,hasPeptideCounts:!1,filename:"10090-White_fat_geiger_2013.txt"},{id:305,name:"M.musculus - Brain, SC (Martens,Proteomics,2006)",score:2.35,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.200600291/abstract" target="_blank">Martens,Proteomics,2006</a><br/><b>Interaction consistency score</b>: 2.35&nbsp<b>Coverage</b>: 17',organ:"BRAIN",integrated:!1,coverage:17,publication_year:2006,num_abundances:3889,hasPeptideCounts:!0,filename:"10090-mouse_brain_Martens_2006_proteomics.txt"},{id:490,name:"M.musculus - Brain, SC (Wang,JProteomeRes,2006)",score:5.29,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16457602" target="_blank">Wang,JProteomeRes,2006</a><br/><b>Interaction consistency score</b>: 5.29&nbsp<b>Coverage</b>: 34',organ:"BRAIN",integrated:!1,coverage:34,publication_year:2006,num_abundances:7760,hasPeptideCounts:!0,filename:"10090-mouse_brain_Wang_2007_J_proteome_Res.txt"},{id:489,name:"M.musculus - Brain, Protein peptide count (Kislinger,cell,2006)",score:.6,weight:"100%",description:'abundance based on Protein peptide count, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898" target="_blank">Kislinger,cell,2006</a><br/><b>Interaction consistency score</b>: 0.6&nbsp<b>Coverage</b>: 8',organ:"BRAIN",integrated:!1,coverage:8,publication_year:2006,num_abundances:1749,hasPeptideCounts:!1,filename:"10090-mouse_brain_kislinger2006_cell.txt"},{id:273,name:"M.musculus - Kidney, MaxQuant intensity (F.Waanders,PNAS,2009)",score:3.49,weight:"100%",description:'abundance based on MaxQuant intensity, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">F. Waanders, PNAS,2009</a><br/><b>Interaction consistency score</b>: 3.49&nbsp<b>Coverage</b>: 4',organ:"KIDNEY",integrated:!1,coverage:4,publication_year:2009,num_abundances:968,hasPeptideCounts:!1,filename:"10090-mouse_glomeruli_kidney_F.Waanders2009_PNAs.txt"},{id:309,name:"M.musculus - Sperm, SC (Guo,J.Proteome,2010)",score:3.85,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://pubs.acs.org/doi/abs/10.1021/pr900735k" target="_blank">Guo, J.Proteome,2010</a><br/><b>Interaction consistency score</b>: 3.85&nbsp<b>Coverage</b>: 11',organ:"SPERM",integrated:!1,coverage:11,publication_year:2010,num_abundances:2587,hasPeptideCounts:!0,filename:"10090-mouse_haploidGermCells_Guo_2010_JProteome.txt"},{id:109,name:"M.musculus - Heart, Protein peptide count (Kislinger,cell,2006)",score:1.66,weight:"100%",description:'abundance based on Protein peptide count, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898" target="_blank">Kislinger,cell,2006</a><br/><b>Interaction consistency score</b>: 1.66&nbsp<b>Coverage</b>: 6',organ:"HEART",integrated:!1,coverage:6,publication_year:2006,num_abundances:1301,hasPeptideCounts:!1,filename:"10090-mouse_heart_kislinger2006_cell.txt"},{id:144,name:"M.musculus - Kidney, Protein peptide count (Kislinger,cell,2006)",score:.49,weight:"100%",description:'abundance based on Protein peptide count, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898" target="_blank">Kislinger,cell,2006</a><br/><b>Interaction consistency score</b>: 0.49&nbsp<b>Coverage</b>: 6',organ:"KIDNEY",integrated:!1,coverage:6,publication_year:2006,num_abundances:1359,hasPeptideCounts:!1,filename:"10090-mouse_kidney_kislinger2006_cell.txt"},{id:417,name:"M.musculus - Liver, SC (Shi,Jofproteome,2007)",score:8.48,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://pubs.acs.org/doi/full/10.1021/pr0605668" target="_blank">Shi,Jofproteome,2007</a><br/><b>Interaction consistency score</b>: 8.48&nbsp<b>Coverage</b>: 12',organ:"LIVER",integrated:!1,coverage:12,publication_year:2007,num_abundances:2804,hasPeptideCounts:!0,filename:"10090-mouse_liver_Shi2007_JourmalofProteome.txt"},{id:34,name:"M.musculus - Liver, Protein peptide count (Kislinger,cell,2006)",score:2.69,weight:"100%",description:'abundance based on Protein peptide count, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898" target="_blank">Kislinger,cell,2006</a><br/><b>Interaction consistency score</b>: 2.69&nbsp<b>Coverage</b>: 6',organ:"LIVER",integrated:!1,coverage:6,publication_year:2006,num_abundances:1395,hasPeptideCounts:!1,filename:"10090-mouse_liver_kislinger2006_cell.txt"},{id:177,name:"M.musculus - Lung, Protein peptide count (Kislinger,cell,2006)",score:.7,weight:"100%",description:'abundance based on Protein peptide count, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898" target="_blank">Kislinger,cell,2006</a><br/><b>Interaction consistency score</b>: 0.7&nbsp<b>Coverage</b>: 9',organ:"LUNG",integrated:!1,coverage:9,publication_year:2006,num_abundances:2119,hasPeptideCounts:!1,filename:"10090-mouse_lung_kislinger2006_cell.txt"},{id:228,name:"M.musculus - Pancreas, MaxQuant intensity (F.Waanders,PNAS,2009)",score:8.08,weight:"100%",description:'abundance based on MaxQuant intensity, from<a href="http://www.sciencedirect.com/science/article/pii/S0092867408006958" target="_blank">F. Waanders, PNAS,2009</a><br/><b>Interaction consistency score</b>: 8.08&nbsp<b>Coverage</b>: 9',organ:"PANCREAS",integrated:!1,coverage:9,publication_year:2009,num_abundances:2038,hasPeptideCounts:!1,filename:"10090-mouse_pancreas_F.Waanders2009_PNAs_combine.txt"},{id:262,name:"M.musculus - Placenta, Protein peptide count (Kislinger,cell,2006)",score:1.17,weight:"100%",description:'abundance based on Protein peptide count, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898" target="_blank">Kislinger,cell,2006</a><br/><b>Interaction consistency score</b>: 1.17&nbsp<b>Coverage</b>: 9',organ:"PLACENTA",integrated:!1,coverage:9,publication_year:2006,num_abundances:1954,hasPeptideCounts:!1,filename:"10090-mouse_placenta_kislinger2006_cell.txt"}],total_coverage:90,average_coverage:24.215384615384615,topProteins:[{id:2093255,abundance:51108,rank:0,name:"Tubb5",annotation:"Tubulin, beta 5 class I; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:2101050,abundance:22097,rank:1,name:"Ptma",annotation:"Prothymosin alpha; Prothymosin alpha may mediate immune function by conferring resistance to certain opportunistic infections"},{id:2100734,abundance:18398,rank:2,name:"Tubb4b",annotation:"Tubulin, beta 4B class IVB; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:2093220,abundance:18163,rank:3,name:"Ckb",annotation:"Creatine kinase, brain; Reversibly catalyzes the transfer of phosphate between ATP and various phosphogens (e.g. creatine phosphate). Creatine kinase isoenzymes play a central role in energy transduction in tissues with large, fluctuating energy demands, such as skeletal muscle, heart, brain and spermatozoa"},{id:2111622,abundance:17321,rank:4,name:"Tmsb4x",annotation:"Thymosin, beta 4, X chromosome; Plays an important role in the organization of the cytoskeleton. Binds to and sequesters actin monomers (G actin) and therefore inhibits actin polymerization"},{id:2107800,abundance:15552,rank:5,name:"Tuba3a",annotation:"Tubulin, alpha 3A; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:2107312,abundance:13261,rank:6,name:"Rplp2",annotation:"Ribosomal protein, large P2; Plays an important role in the elongation step of protein synthesis"},{id:2111359,abundance:11044,rank:7,name:"Calm1",annotation:"Calmodulin 1; Calmodulin mediates the control of a large number of enzymes, ion channels, aquaporins and other proteins by Ca(2+). Among the enzymes to be stimulated by the calmodulin-Ca(2+) complex are a number of protein kinases and phosphatases. Together with CCP110 and centrin, is involved in a genetic pathway that regulates the centrosome cycle and progression through cytokinesis (By similarity)"}]},10116:{id:10116,name:"Rattus norvegicus",compact_name:"Rattus norvegicus",num_proteins:22941,genome_source:"<a href='http://www.ensembl.org/Rattus_norvegicus//Info/Index?db=core'>Ensembl</a>",genome_source_version:"release 70",datasets:[{id:476,name:"R.norvegicus - Whole organism, SC (GPM,Oct,2012)",score:4.84,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 4.84&nbsp<b>Coverage</b>: 55',organ:"WHOLE_ORGANISM",integrated:!1,coverage:55,publication_year:2012,num_abundances:12506,hasPeptideCounts:!0,filename:"10116-GPM_2012_09_Rattus_norvegicus.txt"},{id:408,name:"R.norvegicus - Whole organism, SC (GPM,Aug,2014)",score:14.91,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 14.91&nbsp<b>Coverage</b>: 55',organ:"WHOLE_ORGANISM",integrated:!1,coverage:55,publication_year:2014,num_abundances:12617,hasPeptideCounts:!0,filename:"10116-GPM_201408.txt"},{id:462,name:"R.norvegicus - Whole organism, SC (PeptideAtlas,Nov,2013)",score:13.43,weight:"70%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Nov,2013</a><br/><b>Interaction consistency score</b>: 13.43&nbsp<b>Coverage</b>: 50',organ:"WHOLE_ORGANISM",integrated:!1,coverage:50,publication_year:2013,num_abundances:11435,hasPeptideCounts:!0,filename:"10116-PA_201311.txt"},{id:464,name:"R.norvegicus - Microglial cell, SC (Bell-Temin,Proteomics,2012)",score:17.14,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/22121004" target="_blank">Bell-Temin,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 17.14&nbsp<b>Coverage</b>: 25',organ:"MICROGLIAL_CELL",integrated:!1,coverage:25,publication_year:2012,num_abundances:5673,hasPeptideCounts:!0,filename:"10116-Rattus_norvegicus_SC_biomart_19302_E__6reps.txt"},{id:93,name:"R.norvegicus - Whole organism (Integrated)",score:14.59,weight:"",description:"integrated dataset: weighted average of all R.norvegicus WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 14.59&nbsp<b>Coverage</b>: 73",organ:"WHOLE_ORGANISM",integrated:!0,coverage:73,publication_year:2017,num_abundances:16858,hasPeptideCounts:!1,filename:"10116-WHOLE_ORGANISM-integrated.txt"}],total_coverage:74,average_coverage:46.25,topProteins:[{id:2137184,abundance:21179,rank:0,name:"Tmsb4x",annotation:"Thymosin beta-4 Hematopoietic system regulatory peptide; Plays an important role in the organization of the cytoskeleton. Binds to and sequesters actin monomers (G actin) and therefore inhibits actin polymerization (By similarity)"},{id:2130030,abundance:9667,rank:1,name:"Hba-a2",annotation:"Hemoglobin subunit alpha-1/2 ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:2129986,abundance:9553,rank:2,name:"Hba2",annotation:"Hemoglobin subunit alpha-1/2 ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:2118717,abundance:8178,rank:3,name:"Fabp1",annotation:"Fatty acid-binding protein, liver ; Plays a role in lipoprotein-mediated cholesterol uptake in hepatocytes (By similarity). Binds cholesterol (PubMed:25732850). Binds free fatty acids and their coenzyme A derivatives, bilirubin, and some other small molecules in the cytoplasm. May be involved in intracellular lipid transport (By similarity)"},{id:2116719,abundance:5482,rank:4,name:"Sod1",annotation:"Superoxide dismutase [Cu-Zn] ; Destroys radicals which are normally produced within the cells and which are toxic to biological systems"},{id:2131091,abundance:5090,rank:5,name:"Hbb",annotation:"Hemoglobin subunit beta-1 ; Involved in oxygen transport from the lung to the various peripheral tissues"},{id:2122430,abundance:4694,rank:6,name:"Cps1",annotation:"Carbamoyl-phosphate synthase [ammonia], mitochondrial ; Involved in the urea cycle of ureotelic animals where the enzyme plays an important role in removing excess ammonia from the cell"},{id:2130045,abundance:4625,rank:7,name:"Actb",annotation:"Actin, cytoplasmic 1 Actin, cytoplasmic 1, N-terminally processed; Actins are highly conserved proteins that are involved in various types of cell motility and are ubiquitously expressed in all eukaryotic cells"}]},39947:{id:39947,name:"Oryza sativa Japonica Group",compact_name:"Oryza sativa Japonica",num_proteins:57939,genome_source:"<a href='http://plants.ensembl.org/Oryza_sativa_Japonica_Group//Info/Index?db=core'>Ensembl Plants</a>",genome_source_version:"release 17",datasets:[{id:428,name:"O.sativa - Whole organism, SC (GPM,Oct,2012)",score:16.18,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM, Oct,2012</a><br/><b>Interaction consistency score</b>: 16.18&nbsp<b>Coverage</b>: 10',organ:"WHOLE_ORGANISM",integrated:!1,coverage:10,publication_year:2012,num_abundances:5656,hasPeptideCounts:!0,filename:"39947-GPM_2012_09_Oryza_sativa.txt"}],total_coverage:9,average_coverage:10,topProteins:[{id:2591744,abundance:89176,rank:0,name:"4338862",annotation:"Ribulose bisphosphate carboxylase large chain precursor, putative"},{id:2570269,abundance:77418,rank:1,name:"4324237",annotation:"Ribulose bisphosphate carboxylase large chain precursor, putative"},{id:2612261,abundance:63759,rank:2,name:"4126887",annotation:"Ribulose bisphosphate carboxylase large chain precursor, putative, expressed"},{id:2622771,abundance:56620,rank:3,name:"3131463",annotation:"Ribulose 1,5-bisphosphate carboxylase/oxygenase large chain; RuBisCO catalyzes two reactions: the carboxylation of D- ribulose 1,5-bisphosphate, the primary event in carbon dioxide fixation, as well as the oxidative fragmentation of the pentose substrate in the photorespiration process. Both reactions occur simultaneously and in competition at the same active site"},{id:2596957,abundance:45781,rank:4,name:"LOC_Os06g39754.1",annotation:"Expressed protein"},{id:2617092,abundance:42597,rank:5,name:"LOC_Os11g32770.1",annotation:"Ribulose bisphosphate carboxylase large chain precursor, putative"},{id:2619467,abundance:37923,rank:6,name:"LOC_Os12g10580.1",annotation:"Ribulose bisphosphate carboxylase large chain precursor, putative"},{id:2593303,abundance:22958,rank:7,name:"4339833",annotation:"Plastocyanin, chloroplast precursor, putative, expressed; Participates in electron transfer between P700 and the cytochrome b6-f complex in photosystem I"}]},44689:{id:44689,name:"Dictyostelium discoideum",compact_name:"Dictyostelium discoideum",num_proteins:13212,genome_source:"<a href='http://protists.ensembl.org/Dictyostelium_discoideum//Info/Index?db=core'>Ensembl Protists</a>",genome_source_version:"release 17",datasets:[{id:321,name:"D.discoideum - Whole organism, SC (GPM,Aug,2014)",score:10.51,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 10.51&nbsp<b>Coverage</b>: 32',organ:"WHOLE_ORGANISM",integrated:!1,coverage:32,publication_year:2014,num_abundances:4244,hasPeptideCounts:!0,filename:"44689-GPM_201408.txt"}],total_coverage:32,average_coverage:32,topProteins:[{id:2684250,abundance:21107,rank:0,name:"ubqC",annotation:"Ubiquitin-40S ribosomal protein S27aUbiquitin40S ribosomal protein S27a; Ubiquitin exists either covalently attached to another protein, or free (unanchored). When covalently bound, it is conjugated to target proteins via an isopeptide bond either as a monomer (monoubiquitin), a polymer linked via different Lys residues of the ubiquitin (polyubiquitin chains) or a linear polymer linked via the initiator Met of the ubiquitin (linear polyubiquitin chains). Polyubiquitin chains, when attached to a target protein, have different functions depending on the Lys residue of the ubiquitin that  [...] "},{id:2686499,abundance:20337,rank:1,name:"ubqB",annotation:"Ubiquitin-60S ribosomal protein L40Ubiquitin60S ribosomal protein L40; Ubiquitin: exists either covalently attached to another protein, or free (unanchored). When covalently bound, it is conjugated to target proteins via an isopeptide bond either as a monomer (monoubiquitin), a polymer linked via different Lys residues of the ubiquitin (polyubiquitin chains) or a linear polymer linked via the initiator Met of the ubiquitin (linear polyubiquitin chains). Polyubiquitin chains, when attached to a target protein, have different functions depending on the Lys residue of the ubiquitin that i [...] "},{id:2684079,abundance:13007,rank:2,name:"H3a",annotation:"Histone H3.3 type a; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:2687026,abundance:10251,rank:3,name:"H3b",annotation:"Histone H3.3 type b; Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling"},{id:2681847,abundance:9931,rank:4,name:"DDB0185126",annotation:"Major actin"},{id:2681845,abundance:9931,rank:5,name:"DDB0185124",annotation:"Major actin"},{id:2689063,abundance:9931,rank:6,name:"DDB0220450",annotation:"Major actin"},{id:2681760,abundance:9931,rank:7,name:"DDB0185015",annotation:"Major actin"}]},64091:{id:64091,name:"Halobacterium sp. NRC-1",compact_name:"Halobacterium sp. NRC1",num_proteins:2075,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid64091[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:478,name:"H.salinarum(NRC-1) - Whole organism, SC (GPM,Aug,2014)",score:7.68,weight:"30%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 7.68&nbsp<b>Coverage</b>: 59',organ:"WHOLE_ORGANISM",integrated:!1,coverage:59,publication_year:2014,num_abundances:1229,hasPeptideCounts:!0,filename:"64091-GPM_201408.txt"},{id:271,name:"H.salinarum(NRC-1) - Whole organism, SC (PeptideAtlas,March,2008)",score:9.8,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/halo/index.php" target="_blank">PeptideAtlas,March,2008</a><br/><b>Interaction consistency score</b>: 9.8&nbsp<b>Coverage</b>: 66',organ:"WHOLE_ORGANISM",integrated:!1,coverage:66,publication_year:2008,num_abundances:1366,hasPeptideCounts:!0,filename:"64091-HalobacteriumBuild_Mar2008.peptides.new.txt"},{id:438,name:"H.salinarum(NRC-1) - Whole organism, SC (PeptideAtlas,Aug,2013)",score:9.88,weight:"90%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 9.88&nbsp<b>Coverage</b>: 67',organ:"WHOLE_ORGANISM",integrated:!1,coverage:67,publication_year:2013,num_abundances:1396,hasPeptideCounts:!0,filename:"64091-PA_201308.txt"},{id:138,name:"H.salinarum(NRC-1) - Whole organism (Integrated)",score:10.71,weight:"",description:"integrated dataset: weighted average of all H.salinarum(NRC-1) WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 10.71&nbsp<b>Coverage</b>: 70",organ:"WHOLE_ORGANISM",integrated:!0,coverage:70,publication_year:2017,num_abundances:1459,hasPeptideCounts:!1,filename:"64091-WHOLE_ORGANISM-integrated.txt"}],total_coverage:70,average_coverage:64,topProteins:[{id:2904899,abundance:34227,rank:0,name:"atpK",annotation:"H+-transporting ATP synthase subunit K"},{id:2905323,abundance:32641,rank:1,name:"csg",annotation:"Cell surface glycoprotein; The S-layer is a paracrystalline mono-layered assembly of proteins which coat the surface of the cell"},{id:2904983,abundance:17350,rank:2,name:"tbpE",annotation:"Transcription factor; General factor that plays a role in the activation of archaeal genes transcribed by RNA polymerase. Binds specifically to the TATA box promoter element which lies close to the position of transcription initiation (By similarity)"},{id:2905007,abundance:16202,rank:3,name:"VNG_2273H",annotation:"Hypothetical protein"},{id:2904569,abundance:12544,rank:4,name:"rpl18",annotation:"50S ribosomal protein L18; This is one of the proteins that binds and probably mediates the attachment of the 5S RNA into the large ribosomal subunit, where it forms part of the central protuberance"},{id:2905143,abundance:10448,rank:5,name:"dps",annotation:"Starvation induced DNA-binding protein; Protects DNA from oxidative damage by sequestering intracellular Fe(2+) ion and storing it in the form of Fe(3+) oxyhydroxide mineral. One hydrogen peroxide oxidizes two Fe(2+) ions, which prevents hydroxyl radical production by the Fenton reaction (By similarity)"},{id:2905303,abundance:9774,rank:6,name:"tuf",annotation:"Elongation factor 1-alpha; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:2904969,abundance:9565,rank:7,name:"thsA",annotation:"Thermosome subunit alpha; Molecular chaperone; binds unfolded polypeptides in vitro, and has a weak ATPase activity"}]},83332:{id:83332,name:"Mycobacterium tuberculosis H37Rv",compact_name:"Mycobacterium tuberculosis H37Rv",num_proteins:4003,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid83332[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:62,name:"M.tuberculosis - Whole organism, SC (GPM,Aug,2014)",score:11.48,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 11.48&nbsp<b>Coverage</b>: 75',organ:"WHOLE_ORGANISM",integrated:!1,coverage:75,publication_year:2014,num_abundances:2999,hasPeptideCounts:!0,filename:"83332-GPM_201408.txt"},{id:147,name:"M.tuberculosis - Whole organism, SC (PeptideAtlas,Feb,2012)",score:11.08,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Feb,2012</a><br/><b>Interaction consistency score</b>: 11.08&nbsp<b>Coverage</b>: 81',organ:"WHOLE_ORGANISM",integrated:!1,coverage:81,publication_year:2012,num_abundances:3235,hasPeptideCounts:!0,filename:"83332-M.tuberculosis_atlas_build_330.txt"},{id:157,name:"M.tuberculosis - Whole organism, SC (Albrethsen,MCP,2013)",score:5.25,weight:"30%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23345537" target="_blank">Albrethsen,MCP,2013</a><br/><b>Interaction consistency score</b>: 5.25&nbsp<b>Coverage</b>: 30',organ:"WHOLE_ORGANISM",integrated:!1,coverage:30,publication_year:2013,num_abundances:1184,hasPeptideCounts:!0,filename:"83332-Mtuberculosis_Albrethsen_2013.txt"},{id:133,name:"M.tuberculosis - Whole organism, Pepxml, SC (Cortes,Cell_Rep,2013)",score:4.36,weight:"90%",description:'abundance based on Spectral counting, pepxml, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24268774" target="_blank">Cortes,Cell_Rep,2013</a><br/><b>Interaction consistency score</b>: 4.36&nbsp<b>Coverage</b>: 36',organ:"WHOLE_ORGANISM",integrated:!1,coverage:36,publication_year:2013,num_abundances:1459,hasPeptideCounts:!0,filename:"83332-My_PXD000483_Cortes_2013.txt"},{id:392,name:"M.tuberculosis - Whole organism, SC (Yun.Sang.Cho,Proteomics,2012)",score:.71,weight:"70%",description:'abundance based on Spectral counting, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201100544/abstract;jsessionid=385452BF191D88EA2C09A08DA32D95B2.d03t01" target="_blank">Yun.Sang.Cho,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 0.71&nbsp<b>Coverage</b>: 6',organ:"WHOLE_ORGANISM",integrated:!1,coverage:6,publication_year:2012,num_abundances:242,hasPeptideCounts:!0,filename:"83332-MycobacteriumTuberculosis.txt"},{id:156,name:"M.tuberculosis - Whole organism, SC (PeptideAtlas,July,2013)",score:10.54,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,July,2013</a><br/><b>Interaction consistency score</b>: 10.54&nbsp<b>Coverage</b>: 80',organ:"WHOLE_ORGANISM",integrated:!1,coverage:80,publication_year:2013,num_abundances:3220,hasPeptideCounts:!0,filename:"83332-PA_201307.txt"},{id:253,name:"M.tuberculosis - Whole organism (Integrated)",score:12.33,weight:"",description:"integrated dataset: weighted average of all M.tuberculosis WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 12.33&nbsp<b>Coverage</b>: 85",organ:"WHOLE_ORGANISM",integrated:!0,coverage:85,publication_year:2017,num_abundances:3389,hasPeptideCounts:!1,filename:"83332-WHOLE_ORGANISM-integrated.txt"}],total_coverage:84,average_coverage:51.333333333333336,topProteins:[{id:3011988,abundance:145971,rank:0,name:"hspX",annotation:"Alpha-crystallin; Acts as a chaperone, as it has a significant ability to suppress the thermal denaturation of alcohol dehydrogenase. Cells overexpressing this gene grow more slowly than wild-type cells, and are less susceptible to autolysis following saturation of the culture in vitro, suggesting this protein may slow down the growth rate of M.tuberculosis in culture and by extension during macrophage infection"},{id:3013401,abundance:112912,rank:1,name:"groS",annotation:"Chaperonin GroES; Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter"},{id:3010368,abundance:37589,rank:2,name:"groL2",annotation:"Molecular chaperone GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3013863,abundance:17611,rank:3,name:"lhp",annotation:"Protein EsxB"},{id:3013903,abundance:14898,rank:4,name:"trxC",annotation:"Thioredoxin; Participates in various redox reactions through the reversible oxidation of its active center dithiol to a disulfide and catalyzes dithiol-disulfide exchange reactions"},{id:3010589,abundance:11327,rank:5,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:3013336,abundance:11180,rank:6,name:"Rv3354",annotation:"Hypothetical protein"},{id:3012201,abundance:9440,rank:7,name:"acpM",annotation:"Acyl carrier protein; Acyl carrier protein involved in meromycolate extension"}]},85962:{id:85962,name:"Helicobacter pylori 26695",compact_name:"Helicobacter pylori 26695",num_proteins:1573,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid85962[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:207,name:"H.pylori 26695 - Whole organism, SC (GPM,Aug,2014)",score:9.31,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 9.31&nbsp<b>Coverage</b>: 61',organ:"WHOLE_ORGANISM",integrated:!1,coverage:61,publication_year:2014,num_abundances:965,hasPeptideCounts:!0,filename:"85962-GPM_201408.txt"},{id:107,name:"H.pylori - Whole organism, SC (Müller,J_Proteomics,2013)",score:10.83,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23665149" target="_blank">Müller, J_Proteomics,2013</a><br/><b>Interaction consistency score</b>: 10.83&nbsp<b>Coverage</b>: 99',organ:"WHOLE_ORGANISM",integrated:!1,coverage:99,publication_year:2013,num_abundances:1556,hasPeptideCounts:!0,filename:"85962-Helicobacter_pylori_SC_biomart_27551_E__5reps.txt"},{id:134,name:"H.pylori 26695 - Whole organism (Integrated)",score:16.13,weight:"",description:"integrated dataset: weighted average of all H.pylori 26695 WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 16.13&nbsp<b>Coverage</b>: 99",organ:"WHOLE_ORGANISM",integrated:!0,coverage:99,publication_year:2017,num_abundances:1556,hasPeptideCounts:!1,filename:"85962-WHOLE_ORGANISM-integrated.txt"}],total_coverage:98,average_coverage:80,topProteins:[{id:3024123,abundance:45182,rank:0,name:"groS",annotation:"Co-chaperonin GroES; Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter"},{id:3024122,abundance:36209,rank:1,name:"groL",annotation:"Chaperonin GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3025659,abundance:34293,rank:2,name:"tsaA",annotation:"Alkyl hydroperoxide reductase"},{id:3024182,abundance:22471,rank:3,name:"ureB",annotation:"Urease subunit beta; Ammonia produced by ureolysis increases the gastric pH thereby providing an environment permissive for colonization of the stomach"},{id:3025406,abundance:14772,rank:4,name:"rplX",annotation:"50S ribosomal protein L24; One of two assembly initiator proteins, it binds directly to the 5'-end of the 23S rRNA, where it nucleates assembly of the 50S subunit"},{id:3025258,abundance:14325,rank:5,name:"fldA",annotation:"Flavodoxin FldA; Low-potential electron donor to a number of redox enzymes"},{id:3025342,abundance:14275,rank:6,name:"rpsR",annotation:"30S ribosomal protein S18; Binds as a heterodimer with protein S6 to the central domain of the 16S rRNA, where it helps stabilize the platform of the 30S subunit"},{id:3024183,abundance:13492,rank:7,name:"ureA",annotation:"Urease subunit alpha; Ammonia produced by ureolysis increases the gastric pH thereby providing an environment permissive for colonization of the stomach"}]},99287:{id:99287,name:"Salmonella enterica subsp. enterica serovar Typhimurium str. LT2",compact_name:"Salmonella enterica LT2",num_proteins:4423,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid99287[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:46,name:"S.enterica - Whole organism, SC (GPM,Aug,2014)",score:10.83,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 10.83&nbsp<b>Coverage</b>: 38',organ:"WHOLE_ORGANISM",integrated:!1,coverage:38,publication_year:2014,num_abundances:1700,hasPeptideCounts:!0,filename:"99287-GPM_201408.txt"},{id:211,name:"S.typhimurium - Whole organism, SC (Adkins,MCP,2006)",score:11.98,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/16684765" target="_blank">Adkins, MCP, 2006</a><br/><b>Interaction consistency score</b>: 11.98&nbsp<b>Coverage</b>: 56',organ:"WHOLE_ORGANISM",integrated:!1,coverage:56,publication_year:2006,num_abundances:2493,hasPeptideCounts:!0,filename:"99287-SalmonellaTyphimurium_PRIDE.txt"},{id:248,name:"S.typhimurium - Whole organism (Integrated)",score:15.32,weight:"",description:"integrated dataset: weighted average of all S.typhimurium WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 15.32&nbsp<b>Coverage</b>: 59",organ:"WHOLE_ORGANISM",integrated:!0,coverage:59,publication_year:2017,num_abundances:2614,hasPeptideCounts:!1,filename:"99287-WHOLE_ORGANISM-integrated.txt"}],total_coverage:59,average_coverage:47,topProteins:[{id:3096338,abundance:26618,rank:0,name:"hupA",annotation:"DNA-binding protein HU-alpha; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:3092764,abundance:19592,rank:1,name:"hupB",annotation:"Transcriptional regulator HU subunit beta; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:3093659,abundance:19210,rank:2,name:"lpp1",annotation:"Murein lipoprotein; Interacts with the peptidoglycan both covalently and noncovalently (By similarity). Plays an important role in virulence"},{id:3095652,abundance:18696,rank:3,name:"STM3445",annotation:"Elongation factor Tu"},{id:3096314,abundance:18696,rank:4,name:"tufA",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3096712,abundance:17781,rank:5,name:"STM4561",annotation:"Hypothetical protein"},{id:3096493,abundance:17139,rank:6,name:"groL",annotation:"Chaperonin GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3093363,abundance:16392,rank:7,name:"ompA",annotation:"Outer membrane protein A; Required for the action of colicins K and L and for the stabilization of mating aggregates in conjugation. Serves as a receptor for a number of T-even like phages. Also acts as a porin with low permeability that allows slow penetration of small solutes (By similarity)"}]},122586:{id:122586,name:"Neisseria meningitidis MC58",compact_name:"Neisseria meningitidis",num_proteins:2063,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid122586[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:176,name:"N.meningitidis MC58 - Whole organism, SC (GPM,Aug,2014)",score:2.34,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 2.34&nbsp<b>Coverage</b>: 20',organ:"WHOLE_ORGANISM",integrated:!1,coverage:20,publication_year:2014,num_abundances:412,hasPeptideCounts:!0,filename:"122586-GPM_201408.txt"}],total_coverage:19,average_coverage:20,topProteins:[{id:3189282,abundance:100067,rank:0,name:"opc",annotation:"Class 5 outer membrane protein"},{id:3190218,abundance:83778,rank:1,name:"porB",annotation:"Major outer membrane protein PIB; Serves as a slightly cation selective porin"},{id:3188629,abundance:79677,rank:2,name:"rmpM",annotation:"Outer membrane protein class 4"},{id:3189634,abundance:47205,rank:3,name:"porA",annotation:"Outer membrane protein PorA; Serves as a slightly cation selective porin. Major antigen on the gonococcal cell surface and it may have pathogenic properties in addition to its porin activity"},{id:3188902,abundance:22301,rank:4,name:"nspA",annotation:"Outer membrane protein"},{id:3188293,abundance:20767,rank:5,name:"NMB0026",annotation:"pilS cassette"},{id:3188285,abundance:20521,rank:6,name:"pilE",annotation:"Pilin PilE; This protein is the predominant Neisseria surface antigen, which allows adhesion of the bacterium to various host cells"},{id:3188382,abundance:17939,rank:7,name:"tufA",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"}]},158878:{id:158878,name:"Staphylococcus aureus subsp. aureus Mu50",compact_name:"Staphylococcus aureus Mu50",num_proteins:2730,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid158878[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:70,name:"S.aureus Mu50 - Whole organism, SC (GPM,Aug,2014)",score:12.84,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 12.84&nbsp<b>Coverage</b>: 62',organ:"WHOLE_ORGANISM",integrated:!1,coverage:62,publication_year:2014,num_abundances:1705,hasPeptideCounts:!0,filename:"158878-GPM_201408.txt"}],total_coverage:62,average_coverage:62,topProteins:[{id:3324735,abundance:63578,rank:0,name:"isaA",annotation:"Hypothetical protein; Is able to cleave peptidoglycan"},{id:3324470,abundance:46932,rank:1,name:"SAV2304",annotation:"Hypothetical protein"},{id:3324348,abundance:18610,rank:2,name:"asp23",annotation:"Alkaline shock protein 23; May play a key role in alkaline pH tolerance"},{id:3322729,abundance:12938,rank:3,name:"tuf",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3324291,abundance:12373,rank:4,name:"fba",annotation:"Fructose-bisphosphate aldolase; Catalyzes the aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3- phosphate (G3P) to form fructose 1,6-bisphosphate (FBP) in gluconeogenesis and the reverse reaction in glycolysis"},{id:3323645,abundance:11676,rank:5,name:"hup",annotation:"DNA-binding protein II; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:3324414,abundance:10918,rank:6,name:"rplW",annotation:"50S ribosomal protein L23; One of the early assembly proteins it binds 23S rRNA. One of the proteins that surrounds the polypeptide exit tunnel on the outside of the ribosome. Forms the main docking site for trigger factor binding to the ribosome"},{id:3324403,abundance:9107,rank:7,name:"rpsZ",annotation:"30S ribosomal protein S14; Binds 16S rRNA, required for the assembly of 30S particles and may also be responsible for determining the conformation of the 16S rRNA at the A site"}]},160490:{id:160490,name:"Streptococcus pyogenes M1 GAS",compact_name:"Streptococcus pyogenes",num_proteins:1696,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid160490[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:415,name:"S.pyogenes M1GAS - Whole organism, SC (GPM,Aug,2014)",score:11.83,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 11.83&nbsp<b>Coverage</b>: 46',organ:"WHOLE_ORGANISM",integrated:!1,coverage:46,publication_year:2014,num_abundances:788,hasPeptideCounts:!0,filename:"160490-GPM_201408.txt"},{id:360,name:"S.pyogenes M1GAS - Whole organism, SC (PeptideAtlas,Feb,2007)",score:8.7,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Feb,2007</a><br/><b>Interaction consistency score</b>: 8.7&nbsp<b>Coverage</b>: 57',organ:"WHOLE_ORGANISM",integrated:!1,coverage:57,publication_year:2007,num_abundances:967,hasPeptideCounts:!0,filename:"160490-PA_200702.txt"},{id:307,name:"S.pyogenes M1GAS - Whole organism, N , SC (Okamoto,BMC_Microbiol,2011)",score:13.65,weight:"100%",description:'abundance based on Spectral counting, N , from<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22070424" target="_blank">Okamoto,BMC_Microbiol,2011</a><br/><b>Interaction consistency score</b>: 13.65&nbsp<b>Coverage</b>: 66',organ:"WHOLE_ORGANISM",integrated:!1,coverage:66,publication_year:2011,num_abundances:1111,hasPeptideCounts:!0,filename:"160490-StreptococcusPyogenes_M1GAS_PRIDE.txt"},{id:159,name:"S.pyogenes M1GAS - Whole organism (Integrated)",score:12.45,weight:"",description:"integrated dataset: weighted average of all S.pyogenes M1GAS WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 12.45&nbsp<b>Coverage</b>: 76",organ:"WHOLE_ORGANISM",integrated:!0,coverage:76,publication_year:2017,num_abundances:1287,hasPeptideCounts:!1,filename:"160490-WHOLE_ORGANISM-integrated.txt"},{id:280,name:"S.pyogenes M1GAS - Whole organism, SC (PeptideAtlas,Feb,2007)",score:8.61,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Feb,2007</a><br/><b>Interaction consistency score</b>: 8.61&nbsp<b>Coverage</b>: 57',organ:"WHOLE_ORGANISM",integrated:!1,coverage:57,publication_year:2007,num_abundances:967,hasPeptideCounts:!0,filename:"160490-atlas_build_101.txt"}],total_coverage:75,average_coverage:56.5,topProteins:[{id:3335251,abundance:32917,rank:0,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:3335996,abundance:22803,rank:1,name:"speB",annotation:"Pyrogenic exotoxin B; Important streptococcal virulence factor which cleaves human fibronectin and degrades vitronectin. Also cleaves human IL1B precursor to form biologically active IL1B. Can induce apoptosis in human monocytes and epithelial cells in vitro, and reduces phagocytic activity in monocytic cells. Thus, may play a role in bacterial colonization, invasion, and inhibition of wound healing"},{id:3335984,abundance:19746,rank:2,name:"emm1",annotation:"M protein type 1"},{id:3334488,abundance:17851,rank:3,name:"rplR",annotation:"50S ribosomal protein L18; This is one of the proteins that binds and probably mediates the attachment of the 5S RNA into the large ribosomal subunit, where it forms part of the central protuberance"},{id:3335580,abundance:17838,rank:4,name:"hup",annotation:"Histone-like DNA-binding protein; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions. Seems also to act as a fortuitous virulence factor in delayed sequelae by binding to heparan sulfate-proteoglycans in the extracellular matrix of target organs and acting as a nidus for in situ immune complex formation (By similarity)"},{id:3335491,abundance:16270,rank:5,name:"ptsH",annotation:"Phosphocarrier protein HPr"},{id:3334639,abundance:14947,rank:6,name:"gap",annotation:"Glyceraldehyde-3-phosphate dehydrogenase; Catalyzes the oxidative phosphorylation of glyceraldehyde 3-phosphate (G3P) to 1,3-bisphosphoglycerate (BPG) using the cofactor NAD. The first reaction step involves the formation of a hemiacetal intermediate between G3P and a cysteine residue, and this hemiacetal intermediate is then oxidized to a thioester, with concomitant reduction of NAD to NADH. The reduced NADH is then exchanged with the second NAD, and the thioester is attacked by a nucleophilic inorganic phosphate to produce BPG"},{id:3334495,abundance:14792,rank:7,name:"rpmJ",annotation:"50S ribosomal protein L36"}]},169963:{id:169963,name:"Listeria monocytogenes EGD-e",compact_name:"Listeria monocytogenes EGDe",num_proteins:2846,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid169963[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:396,name:"L. monocytogenes EGD-e - Whole organism, SC (GPM,Aug,2014)",score:6.01,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 6.01&nbsp<b>Coverage</b>: 16',organ:"WHOLE_ORGANISM",integrated:!1,coverage:16,publication_year:2014,num_abundances:469,hasPeptideCounts:!0,filename:"169963-GPM_201408.txt"}],total_coverage:16,average_coverage:16,topProteins:[{id:3392210,abundance:127330,rank:0,name:"hup",annotation:"Hypothetical protein; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:3392292,abundance:47782,rank:1,name:"cspLB",annotation:"Hypothetical protein"},{id:3391641,abundance:46791,rank:2,name:"cspL",annotation:"Hypothetical protein"},{id:3390329,abundance:30527,rank:3,name:"rpsF",annotation:"30S ribosomal protein S6; Binds together with S18 to 16S ribosomal RNA"},{id:3392898,abundance:28131,rank:4,name:"rpsQ",annotation:"30S ribosomal protein S17; One of the primary rRNA binding proteins, it binds specifically to the 5'-end of 16S ribosomal RNA"},{id:3390969,abundance:22555,rank:5,name:"flaA",annotation:"Flagellin; Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella"},{id:3391819,abundance:21368,rank:6,name:"rplU",annotation:"50S ribosomal protein L21; This protein binds to 23S rRNA in the presence of protein L20"},{id:3391279,abundance:19948,rank:7,name:"ptsH",annotation:"Phosphocarrier protein HPr; General (non sugar-specific) component of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The phosphoryl group from phosphoenolpyruvate (PEP) is transferred to the phosphoryl carrier protein HPr by enzyme I. Phospho-HPr then transfers it to the permease (enzymes II/III)"}]},192222:{id:192222,name:"Campylobacter jejuni subsp. jejuni NCTC 11168",compact_name:"Campylobacter jejuni NCTC11168",num_proteins:1623,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid192222[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:127,name:"C.jejuni - Whole organism, SC (PRIDE,PRD000441,2011)",score:7.75,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/archive/projects/PRD000441" target="_blank">PRIDE,PRD000441,2011</a><br/><b>Interaction consistency score</b>: 7.75&nbsp<b>Coverage</b>: 47',organ:"WHOLE_ORGANISM",integrated:!1,coverage:47,publication_year:2011,num_abundances:770,hasPeptideCounts:!0,filename:"192222-Campylobacter_jejuni_SC_biomart_17684_E__1reps.txt"},{id:19,name:"C.jejuni - Whole organism, SC (PRIDE,PRD000441,2011)",score:7.3,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/archive/projects/PRD000441" target="_blank">PRIDE,PRD000441,2011</a><br/><b>Interaction consistency score</b>: 7.3&nbsp<b>Coverage</b>: 47',organ:"WHOLE_ORGANISM",integrated:!1,coverage:47,publication_year:2011,num_abundances:770,hasPeptideCounts:!0,filename:"192222-Campylobacter_jejuni_SC_biomart_17684_O__1reps.txt"},{id:484,name:"C.jejuni - Whole organism (Integrated)",score:7.74,weight:"",description:"integrated dataset: weighted average of all C.jejuni WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 7.74&nbsp<b>Coverage</b>: 47",organ:"WHOLE_ORGANISM",integrated:!0,coverage:47,publication_year:2017,num_abundances:770,hasPeptideCounts:!1,filename:"192222-WHOLE_ORGANISM-integrated.txt"}],total_coverage:47,average_coverage:47,topProteins:[{id:3470665,abundance:60446,rank:0,name:"porA",annotation:"Major outer membrane protein; Assembles to form a functional porin. May be one of the structures responsible for adhesion to intestinal cells"},{id:3470627,abundance:24141,rank:1,name:"groL",annotation:"60 kDa chaperonin; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3469811,abundance:20649,rank:2,name:"Cj0358",annotation:"Cytochrome C551 peroxidase"},{id:3469919,abundance:18522,rank:3,name:"tuf",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3470742,abundance:17498,rank:4,name:"flaA",annotation:"Flagellin; Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella"},{id:3469499,abundance:17030,rank:5,name:"rrc",annotation:"Non-haem iron protein"},{id:3470785,abundance:14890,rank:6,name:"fldA",annotation:"Flavodoxin-2; Low-potential electron donor to a number of redox enzymes"},{id:3470626,abundance:14254,rank:7,name:"groS",annotation:"10 kDa chaperonin; Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter"}]},198214:{id:198214,name:"Shigella flexneri 2a str. 301",compact_name:"Shigella flexneri",num_proteins:4439,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid198214[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:458,name:"S.flexneri 2457T - Whole organism, SC (PRIDE,PRD000545,2011)",score:8.91,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/" target="_blank">PRIDE,PRD000545,2011</a><br/><b>Interaction consistency score</b>: 8.91&nbsp<b>Coverage</b>: 39',organ:"WHOLE_ORGANISM",integrated:!1,coverage:39,publication_year:2011,num_abundances:1749,hasPeptideCounts:!0,filename:"198214-ShigellaFlexneri_2a_PRIDE.txt"}],total_coverage:39,average_coverage:39,topProteins:[{id:3505204,abundance:38900,rank:0,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:3503001,abundance:31578,rank:1,name:"lpp_2",annotation:"Murein lipoprotein"},{id:3505431,abundance:21666,rank:2,name:"groL",annotation:"Molecular chaperone GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3501541,abundance:20855,rank:3,name:"tsf",annotation:"Elongation factor Ts; Associates with the EF-Tu.GDP complex and induces the exchange of GDP to GTP. It remains bound to the aminoacyl-tRNA.EF- Tu.GTP complex up to the GTP hydrolysis stage on the ribosome"},{id:3502256,abundance:17283,rank:4,name:"rpsA",annotation:"30S ribosomal protein S1; Binds mRNA; thus facilitating recognition of the initiation point. It is needed to translate mRNA with a short Shine-Dalgarno (SD) purine-rich sequence"},{id:3502012,abundance:13844,rank:5,name:"cspE",annotation:"Cold shock protein CspE"},{id:3503928,abundance:13791,rank:6,name:"csrA",annotation:"Carbon storage regulator; Affects glycogen biosynthesis, gluconeogenesis, cell size and surface properties. Regulates glycogen synthesis under both aerobic and anaerobic conditions. Seems to accelerate the degradation of glg gene transcripts, potentially through selective RNA binding. Acts to inhibit interaction between the LetD protein and the A subunit of DNA gyrase. Also required for motility and flagellum biosynthesis through the post-transcriptional activation of flhDC expression. This involves binding to and stabilization of the flhDC message by CsrA"},{id:3502760,abundance:13374,rank:7,name:"gapA",annotation:"Glyceraldehyde-3-phosphate dehydrogenase"}]},208964:{id:208964,name:"Pseudomonas aeruginosa PAO1",compact_name:"Pseudomonas aeruginosa",num_proteins:5571,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid208964[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:88,name:"P.aeruginosa - Whole organism, SC (GPM,Aug,2014)",score:10.7,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 10.7&nbsp<b>Coverage</b>: 41',organ:"WHOLE_ORGANISM",integrated:!1,coverage:41,publication_year:2014,num_abundances:2264,hasPeptideCounts:!0,filename:"208964-GPM_201408.txt"},{id:103,name:"P.aeruginosa - Whole organism, Biofilm, SC (Toyofuku,J_Proteome_Res,2012)",score:7.63,weight:"100%",description:'abundance based on Spectral counting, biofilm, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/22909304" target="_blank">Toyofuku, J_Proteome_Res, 2012</a><br/><b>Interaction consistency score</b>: 7.63&nbsp<b>Coverage</b>: 31',organ:"WHOLE_ORGANISM",integrated:!1,coverage:31,publication_year:2012,num_abundances:1740,hasPeptideCounts:!0,filename:"208964-Pseudomonas_aeruginosa_SC_biomart_22024_E__1reps.txt"},{id:130,name:"P.aeruginosa - Whole organism (Integrated)",score:14.2,weight:"",description:"integrated dataset: weighted average of all P.aeruginosa WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 14.2&nbsp<b>Coverage</b>: 44",organ:"WHOLE_ORGANISM",integrated:!0,coverage:44,publication_year:2017,num_abundances:2427,hasPeptideCounts:!1,filename:"208964-WHOLE_ORGANISM-integrated.txt"}],total_coverage:43,average_coverage:36,topProteins:[{id:3586353,abundance:18889,rank:0,name:"fliC",annotation:"Flagellin type B; Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella"},{id:3589646,abundance:11987,rank:1,name:"groL",annotation:"Molecular chaperone GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3587040,abundance:9163,rank:2,name:"oprF",annotation:"Porin; Has porin activity, forming small water-filled channels. Also has a structural role in determining cell shape and ability to grow in low-osmolarity medium"},{id:3590106,abundance:8169,rank:3,name:"accB",annotation:"acetyl-CoA carboxylase biotin carboxyl carrier protein subunit; This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA"},{id:3586439,abundance:8056,rank:4,name:"oprH",annotation:"PhoP/Q and low Mg2+ inducible outer membrane protein H1"},{id:3589526,abundance:7908,rank:5,name:"PA4265",annotation:"Elongation factor Tu"},{id:3589538,abundance:7908,rank:6,name:"tufA",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3588003,abundance:6352,rank:7,name:"rplT",annotation:"50S ribosomal protein L20; Binds directly to 23S ribosomal RNA and is necessary for the in vitro assembly process of the 50S ribosomal subunit. It is not involved in the protein synthesizing functions of that subunit"}]},211586:{id:211586,name:"Shewanella oneidensis MR-1",compact_name:"Shewanella oneidensis",num_proteins:4318,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid211586[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:166,name:"S.oneidensis MR-1 - Whole organism, SC (GPM,Aug,2014)",score:7.5,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 7.5&nbsp<b>Coverage</b>: 31',organ:"WHOLE_ORGANISM",integrated:!1,coverage:31,publication_year:2014,num_abundances:1321,hasPeptideCounts:!0,filename:"211586-GPM_201408.txt"}],total_coverage:30,average_coverage:31,topProteins:[{id:3596523,abundance:56116,rank:0,name:"hupB",annotation:"HU family DNA-binding protein; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:3596970,abundance:27371,rank:1,name:"rplT",annotation:"50S ribosomal protein L20; Binds directly to 23S ribosomal RNA and is necessary for the in vitro assembly process of the 50S ribosomal subunit. It is not involved in the protein synthesizing functions of that subunit"},{id:3595089,abundance:19397,rank:2,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:3597414,abundance:17807,rank:3,name:"cspA",annotation:"Cold shock protein CspA"},{id:3595083,abundance:16580,rank:4,name:"tuf1",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3595095,abundance:16365,rank:5,name:"tuf2",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3595105,abundance:15707,rank:6,name:"rpmC",annotation:"50S ribosomal protein L29"},{id:3596969,abundance:15570,rank:7,name:"rpmI",annotation:"50S ribosomal protein L35"}]},214092:{id:214092,name:"Yersinia pestis CO92",compact_name:"Yersinia pestis CO92",num_proteins:3885,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid214092[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:454,name:"Y.pestis - Whole organism, SC (PRIDE,PRD000484,2012)",score:6.55,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/archive/projects/PRD000484" target="_blank">PRIDE,PRD000484,2012</a><br/><b>Interaction consistency score</b>: 6.55&nbsp<b>Coverage</b>: 31',organ:"WHOLE_ORGANISM",integrated:!1,coverage:31,publication_year:2012,num_abundances:1202,hasPeptideCounts:!0,filename:"214092-Yersinia_pestis_SC_biomart_18099_O__3reps.txt"}],total_coverage:30,average_coverage:31,topProteins:[{id:3607225,abundance:36659,rank:0,name:"lpp",annotation:"Major outer membrane lipoprotein; Interacts with the peptidoglycan both covalently and noncovalently"},{id:3606997,abundance:14524,rank:1,name:"gapA",annotation:"Glyceraldehyde 3-phosphate dehydrogenase A"},{id:3606321,abundance:13011,rank:2,name:"ompA",annotation:"Outer membrane protein A; Required for the action of colicins K and L and for the stabilization of mating aggregates in conjugation. Serves as a receptor for a number of T-even like phages. Also acts as a porin with low permeability that allows slow penetration of small solutes (By similarity)"},{id:3605294,abundance:12148,rank:3,name:"groL",annotation:"Chaperonin GroL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3608132,abundance:11038,rank:4,name:"eno",annotation:"Phosphopyruvate hydratase; Catalyzes the reversible conversion of 2- phosphoglycerate into phosphoenolpyruvate. It is essential for the degradation of carbohydrates via glycolysis"},{id:3605142,abundance:10970,rank:5,name:"fkpA",annotation:"FKBP-type peptidylprolyl isomerase"},{id:3605293,abundance:10520,rank:6,name:"groS",annotation:"10 kDa chaperonin; Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter"},{id:3607780,abundance:9919,rank:7,name:"cysK",annotation:"Cysteine synthase A"}]},214684:{id:214684,name:"Cryptococcus neoformans var. neoformans JEC21",compact_name:"Cryptococcus neoformans JEC21",num_proteins:6594,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid214684[Organism:noexp]'>Refseq</a>",genome_source_version:"fungi",datasets:[{id:352,name:"C.neoformans - Whole organism, Biofilm, SC (Santi,J_Proteome_Res,2014)",score:3.41,weight:"100%",description:'abundance based on Spectral counting, biofilm, from<a href="http://pubs.acs.org/doi/abs/10.1021/pr401075f" target="_blank">Santi,J_Proteome_Res,2014</a><br/><b>Interaction consistency score</b>: 3.41&nbsp<b>Coverage</b>: 25',organ:"WHOLE_ORGANISM",integrated:!1,coverage:25,publication_year:2014,num_abundances:1677,hasPeptideCounts:!0,filename:"214684-Cryptococcus_biofilm_Santi_2014.txt"},{id:111,name:"C.neoformans - Whole organism, Planktonic, SC (Santi,J_Proteome_Res,2014)",score:3.52,weight:"100%",description:'abundance based on Spectral counting, planktonic, from<a href="http://pubs.acs.org/doi/abs/10.1021/pr401075f" target="_blank">Santi,J_Proteome_Res,2014</a><br/><b>Interaction consistency score</b>: 3.52&nbsp<b>Coverage</b>: 34',organ:"WHOLE_ORGANISM",integrated:!1,coverage:34,publication_year:2014,num_abundances:2216,hasPeptideCounts:!0,filename:"214684-Cryptococcus_planktonic_Santi_2014.txt"},{id:37,name:"C.neoformans - Whole organism (Integrated)",score:4.95,weight:"",description:"integrated dataset: weighted average of all C.neoformans WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 4.95&nbsp<b>Coverage</b>: 37",organ:"WHOLE_ORGANISM",integrated:!0,coverage:37,publication_year:2017,num_abundances:2464,hasPeptideCounts:!1,filename:"214684-WHOLE_ORGANISM-integrated.txt"}],total_coverage:37,average_coverage:29.5,topProteins:[{id:3612388,abundance:99390,rank:0,name:"CNC06180",annotation:"Hypothetical protein"},{id:3609071,abundance:93373,rank:1,name:"UBI1",annotation:"Ubiquitin-carboxy extension protein fusion; Ubiquitin: exists either covalently attached to another protein, or free (unanchored). When covalently bound, it is conjugated to target proteins via an isopeptide bond either as a monomer (monoubiquitin), a polymer linked via different Lys residues of the ubiquitin (polyubiquitin chains) or a linear polymer linked via the initiator Met of the ubiquitin (linear polyubiquitin chains). Polyubiquitin chains, when attached to a target protein, have different functions depending on the Lys residue of the ubiquitin that is linked: Lys-6-linked may  [...] "},{id:3609423,abundance:46364,rank:2,name:"CNA06950",annotation:"Electron carrier; Electron carrier protein. The oxidized form of the cytochrome c heme group can accept an electron from the heme group of the cytochrome c1 subunit of cytochrome reductase. Cytochrome c then transfers this electron to the cytochrome oxidase complex, the final protein carrier in the mitochondrial electron-transport chain"},{id:3615083,abundance:39601,rank:3,name:"CNI01590",annotation:"Manganese superoxide dismutase; Destroys radicals which are normally produced within the cells and which are toxic to biological systems"},{id:3610873,abundance:27286,rank:4,name:"TEF1",annotation:"Translation elongation factor EF1-alpha; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3614501,abundance:21894,rank:5,name:"CNG04230",annotation:"Hypothetical protein"},{id:3610321,abundance:18176,rank:6,name:"CNK03170",annotation:"Transaldolase; Transaldolase is important for the balance of metabolites in the pentose-phosphate pathway"},{id:3613430,abundance:14809,rank:7,name:"CNE03920",annotation:"60s ribosomal protein l1-a (l10a)"}]},224308:{id:224308,name:"Bacillus subtilis subsp. subtilis str. 168",compact_name:"Bacillus subtilis 168",num_proteins:4280,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid224308[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:312,name:"B.suptilis - Whole organism, SC (Chi,MCP,2011)",score:5.8,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21749987" target="_blank">Chi, MCP,2011</a><br/><b>Interaction consistency score</b>: 5.8&nbsp<b>Coverage</b>: 92',organ:"WHOLE_ORGANISM",integrated:!1,coverage:92,publication_year:2011,num_abundances:3941,hasPeptideCounts:!0,filename:"224308-BacillusSubtilisSubsp_PRIDE.txt"},{id:493,name:"B.suptilis - Whole organism, SC (GPM,Aug,2014)",score:12.62,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 12.62&nbsp<b>Coverage</b>: 59',organ:"WHOLE_ORGANISM",integrated:!1,coverage:59,publication_year:2014,num_abundances:2539,hasPeptideCounts:!0,filename:"224308-GPM_201408.txt"},{id:256,name:"B.suptilis - Whole organism (Integrated)",score:13.45,weight:"",description:"integrated dataset: weighted average of all B.suptilis WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 13.45&nbsp<b>Coverage</b>: 93",organ:"WHOLE_ORGANISM",integrated:!0,coverage:93,publication_year:2017,num_abundances:3963,hasPeptideCounts:!1,filename:"224308-WHOLE_ORGANISM-integrated.txt"}],total_coverage:92,average_coverage:75.5,topProteins:[{id:3707180,abundance:27185,rank:0,name:"cspB",annotation:"Cold shock protein CspB; Binds to the pentamer sequences ATTGG and CCAAT with highest affinity in single-stranded DNA, and also to other sequences. Has greater affinity for ATTGG than CCAAT. Can act as transcriptional activator of cold shock genes by recognizing putative ATTGG-box elements present in promoter regions of genes induced under cold shock conditions"},{id:3708538,abundance:26545,rank:1,name:"cspD",annotation:"Cold shock protein CspD"},{id:3706348,abundance:23622,rank:2,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:3709167,abundance:19173,rank:3,name:"rplU",annotation:"50S ribosomal protein L21; This protein binds to 23S rRNA in the presence of protein L20"},{id:3709918,abundance:18102,rank:4,name:"hag",annotation:"Flagellin; Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella"},{id:3706372,abundance:15333,rank:5,name:"rpsN",annotation:"30S ribosomal protein S14; Binds 16S rRNA, required for the assembly of 30S particles and may also be responsible for determining the conformation of the 16S rRNA at the A site (By similarity). The major S14 protein in the ribosome. Required for binding of S2 and S3 to the 30S subunit and for association of the 30S with the 50S subunit"},{id:3706360,abundance:14481,rank:6,name:"rplD",annotation:"50S ribosomal protein L4; One of the primary rRNA binding proteins, this protein initially binds near the 5'-end of the 23S rRNA. It is important during the early stages of 50S assembly. It makes multiple contacts with different domains of the 23S rRNA in the assembled 50S subunit and ribosome"},{id:3710175,abundance:12297,rank:7,name:"yweA",annotation:"Hypothetical protein"}]},226186:{id:226186,name:"Bacteroides thetaiotaomicron VPI-5482",compact_name:"Bacteroides thetaiotaomicron",num_proteins:4778,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid226186[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:347,name:"B.thetaiotaomicron VPI-5482 - Whole organism, SC (GPM,Aug,2014)",score:6.63,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 6.63&nbsp<b>Coverage</b>: 16',organ:"WHOLE_ORGANISM",integrated:!1,coverage:16,publication_year:2014,num_abundances:768,hasPeptideCounts:!0,filename:"226186-GPM_201408.txt"}],total_coverage:16,average_coverage:16,topProteins:[{id:3741743,abundance:158559,rank:0,name:"BT_2414",annotation:"Ferredoxin"},{id:3742063,abundance:42898,rank:1,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:3743160,abundance:34326,rank:2,name:"rpmF",annotation:"50S ribosomal protein L32"},{id:3742047,abundance:29588,rank:3,name:"rpmC",annotation:"50S ribosomal protein L29"},{id:3743590,abundance:28715,rank:4,name:"BT_4263",annotation:"Glyceraldehyde 3-phosphate dehydrogenase"},{id:3742068,abundance:26220,rank:5,name:"tuf",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3741020,abundance:15928,rank:6,name:"BT_1691",annotation:"Fructose-bisphosphate aldolase"},{id:3739502,abundance:14113,rank:7,name:"BT_0173",annotation:"Hypothetical protein"}]},243159:{id:243159,name:"Acidithiobacillus ferrooxidans ATCC 23270",compact_name:"Acidithiobacillus ferrooxidans ATCC23270",num_proteins:3147,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid243159[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:422,name:"A.ferrooxidans - Whole organism, Biofilm (Vera,Proteomics,2013)",score:5.26,weight:"70%",description:'abundance based on MAPPED_BY_AUTHORS, biofilm, from<a href="http://doi.wiley.com/10.1002/pmic.201200386" target="_blank">Vera,Proteomics,2013</a><br/><b>Interaction consistency score</b>: 5.26&nbsp<b>Coverage</b>: 42',organ:"WHOLE_ORGANISM",integrated:!1,coverage:42,publication_year:2013,num_abundances:1319,hasPeptideCounts:!1,filename:"243159-Acidithiobac_PSM_biofilm_Vera_2013.txt"},{id:223,name:"A.ferrooxidans - Whole organism, Planktonic (Vera,Proteomics,2013)",score:6.33,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, planktonic, from<a href="http://doi.wiley.com/10.1002/pmic.201200386" target="_blank">Vera,Proteomics,2013</a><br/><b>Interaction consistency score</b>: 6.33&nbsp<b>Coverage</b>: 42',organ:"WHOLE_ORGANISM",integrated:!1,coverage:42,publication_year:2013,num_abundances:1319,hasPeptideCounts:!1,filename:"243159-Acidithiobac_PSM_planktonic_Vera_2013.txt"},{id:113,name:"A.ferrooxidans - Whole organism (Integrated)",score:8.48,weight:"",description:"integrated dataset: weighted average of all A.ferrooxidans WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 8.48&nbsp<b>Coverage</b>: 42",organ:"WHOLE_ORGANISM",integrated:!0,coverage:42,publication_year:2017,num_abundances:1319,hasPeptideCounts:!1,filename:"243159-WHOLE_ORGANISM-integrated.txt"}],total_coverage:41,average_coverage:42,topProteins:[{id:3862233,abundance:39772,rank:0,name:"groL",annotation:"Chaperonin GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:3864315,abundance:21584,rank:1,name:"omp40",annotation:"Major outer membrane protein 40"},{id:3862027,abundance:20024,rank:2,name:"tuf",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:3864707,abundance:17174,rank:3,name:"rus",annotation:"Rusticyanin; Electron carrier from cytochrome c552 to the A-type oxidase"},{id:3864241,abundance:13429,rank:4,name:"dnaK",annotation:"Chaperone protein Dnak; Acts as a chaperone"},{id:3863304,abundance:9412,rank:5,name:"cscE",annotation:"Carboxysome shell peptide"},{id:3864800,abundance:8724,rank:6,name:"fba",annotation:"Fructose-bisphosphate aldolase, class II"},{id:3864758,abundance:8324,rank:7,name:"atpD",annotation:"ATP synthase F1, beta subunit; Produces ATP from ADP in the presence of a proton gradient across the membrane. The catalytic sites are hosted primarily by the beta subunits"}]},260799:{id:260799,name:"Bacillus anthracis str. Sterne",compact_name:"Bacillus anthracis Sterne",num_proteins:5289,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid260799[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:272,name:"B.anthracis - Whole organism, SC (GPM,Aug,2014)",score:5.99,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 5.99&nbsp<b>Coverage</b>: 26',organ:"WHOLE_ORGANISM",integrated:!1,coverage:26,publication_year:2014,num_abundances:1387,hasPeptideCounts:!0,filename:"260799-GPM_201408.txt"}],total_coverage:26,average_coverage:26,topProteins:[{id:4013585,abundance:41176,rank:0,name:"cspB2",annotation:"Cold shock protein CspB"},{id:4014786,abundance:36520,rank:1,name:"sspB",annotation:"Small, acid-soluble spore protein B"},{id:4015223,abundance:33796,rank:2,name:"gap2",annotation:"Glyceraldehyde-3-phosphate dehydrogenase"},{id:4011087,abundance:27004,rank:3,name:"sasP1",annotation:"Small, acid-soluble spore protein"},{id:4013905,abundance:24714,rank:4,name:"rpsO",annotation:"30S ribosomal protein S15; One of the primary rRNA binding proteins, it binds directly to 16S rRNA where it helps nucleate assembly of the platform of the 30S subunit by binding and bridging several RNA helices of the 16S rRNA"},{id:4010405,abundance:20365,rank:5,name:"rpsN",annotation:"30S ribosomal protein S14; Binds 16S rRNA, required for the assembly of 30S particles and may also be responsible for determining the conformation of the 16S rRNA at the A site"},{id:4010343,abundance:20308,rank:6,name:"yabR",annotation:"Hypothetical protein"},{id:4010418,abundance:19905,rank:7,name:"rpsK",annotation:"30S ribosomal protein S11; Located on the platform of the 30S subunit, it bridges several disparate RNA helices of the 16S rRNA. Forms part of the Shine-Dalgarno cleft in the 70S ribosome"}]},267671:{id:267671,name:"Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130",compact_name:"Leptospira interrogans Copenhageni",num_proteins:3667,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid267671[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:460,name:"L.interrogans - Whole organism, SC (Malmström,Nature,2009)",score:10.04,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19606093" target="_blank">Malmström,Nature,2009</a><br/><b>Interaction consistency score</b>: 10.04&nbsp<b>Coverage</b>: 58',organ:"WHOLE_ORGANISM",integrated:!1,coverage:58,publication_year:2009,num_abundances:2113,hasPeptideCounts:!0,filename:"267671-Leptospira_build_121.peptides.cut.txt"},{id:195,name:"L.interrogans - Whole organism, SC (PeptideAtlas,Aug,2013)",score:13.37,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.peptideatlas.org/builds/" target="_blank">PeptideAtlas,Aug,2013</a><br/><b>Interaction consistency score</b>: 13.37&nbsp<b>Coverage</b>: 66',organ:"WHOLE_ORGANISM",integrated:!1,coverage:66,publication_year:2013,num_abundances:2408,hasPeptideCounts:!0,filename:"267671-PA_201308.txt"},{id:84,name:"L.interrogans - Whole organism (Integrated)",score:12.3,weight:"",description:"integrated dataset: weighted average of all L.interrogans WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 12.3&nbsp<b>Coverage</b>: 66",organ:"WHOLE_ORGANISM",integrated:!0,coverage:66,publication_year:2017,num_abundances:2428,hasPeptideCounts:!1,filename:"267671-WHOLE_ORGANISM-integrated.txt"},{id:349,name:"L.interrogans - Whole organism, MS1 quantification data, MS1 normalized data (Malmström,Nature,2009)",score:2.51,weight:"10%",description:'abundance based on MS1 normalized data, MS1 quantification data, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19606093" target="_blank">Malmström,Nature,2009</a><br/><b>Interaction consistency score</b>: 2.51&nbsp<b>Coverage</b>: 19',organ:"WHOLE_ORGANISM",integrated:!1,coverage:19,publication_year:2009,num_abundances:683,hasPeptideCounts:!1,filename:"267671-controlMS1_Leptospira_Malmstroem_2009.txt"},{id:354,name:"L.interrogans - Whole organism, Original spectral counting data, control normalized data (Malmström,Nature,2009)",score:3.3,weight:"10%",description:'abundance based on control normalized data, Original spectral counting data, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/19606093" target="_blank">Malmström,Nature,2009</a><br/><b>Interaction consistency score</b>: 3.3&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2009,num_abundances:1802,hasPeptideCounts:!1,filename:"267671-controlSpectral_Leptospira_Malmstroem_2009.txt"}],total_coverage:66,average_coverage:48,topProteins:[{id:4109270,abundance:17053,rank:0,name:"acpP",annotation:"Acyl carrier protein; Carrier of the growing fatty acid chain in fatty acid biosynthesis"},{id:4107106,abundance:15140,rank:1,name:"groL",annotation:"GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:4107123,abundance:15023,rank:2,name:"lipL32",annotation:"LipL32"},{id:4105820,abundance:11737,rank:3,name:"lipL21",annotation:"LipL21"},{id:4108596,abundance:10895,rank:4,name:"tuf",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:4107381,abundance:10544,rank:5,name:"LIC_11617",annotation:"ArsR family transcriptional regulator"},{id:4107645,abundance:10432,rank:6,name:"LIC_11890",annotation:"Flagellin protein; Component of the core of the flagella"},{id:4108684,abundance:9193,rank:7,name:"lipL41",annotation:"LipL41"}]},272623:{id:272623,name:"Lactococcus lactis subsp. lactis Il1403",compact_name:"Lactococcus lactis lactis",num_proteins:2321,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid272623[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:35,name:"L.lactis - Whole organism, SC (GPM,Aug,2014)",score:11.91,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 11.91&nbsp<b>Coverage</b>: 54',organ:"WHOLE_ORGANISM",integrated:!1,coverage:54,publication_year:2014,num_abundances:1258,hasPeptideCounts:!0,filename:"272623-GPM_201408.txt"},{id:348,name:"L.lactis - Whole organism, SC (Lahtvee,Microbiology,2014)",score:12.59,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/24739216" target="_blank">Lahtvee,Microbiology,2014</a><br/><b>Interaction consistency score</b>: 12.59&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2014,num_abundances:1126,hasPeptideCounts:!0,filename:"272623-Lactococcus_lactis_Lahtvee_2014.txt"},{id:194,name:"L.lactis - Whole organism (Integrated)",score:12.1,weight:"",description:"integrated dataset: weighted average of all L.lactis WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 12.1&nbsp<b>Coverage</b>: 56",organ:"WHOLE_ORGANISM",integrated:!0,coverage:56,publication_year:2017,num_abundances:1303,hasPeptideCounts:!1,filename:"272623-WHOLE_ORGANISM-integrated.txt"}],total_coverage:56,average_coverage:51.5,topProteins:[{id:4186753,abundance:21774,rank:0,name:"hup",annotation:"HU like DNA-binding protein; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:4186946,abundance:19924,rank:1,name:"ptsH",annotation:"Phosphocarrier protein HPr; General (non sugar-specific) component of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The phosphoryl group from phosphoenolpyruvate (PEP) is transferred to the phosphoryl carrier protein HPr by enzyme I. Phospho-HPr then transfers it to the permease (enzymes II/III) (By similarity)"},{id:4188426,abundance:11581,rank:2,name:"dpsA",annotation:"Non-heme iron-binding ferritin"},{id:4186618,abundance:11107,rank:3,name:"tuf",annotation:"Elongation factor Tu; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:4186254,abundance:9613,rank:4,name:"gapB",annotation:"Glyceraldehyde 3-phosphate dehydrogenase"},{id:4186667,abundance:9406,rank:5,name:"rplX",annotation:"50S ribosomal protein L24; One of two assembly initiator proteins, it binds directly to the 5'-end of the 23S rRNA, where it nucleates assembly of the 50S subunit"},{id:4187033,abundance:9381,rank:6,name:"malE",annotation:"Maltose ABC transporter substrate binding protein"},{id:4187633,abundance:9346,rank:7,name:"yhjA",annotation:"Hypothetical protein"}]},272624:{id:272624,name:"Legionella pneumophila subsp. pneumophila str. Philadelphia 1",compact_name:"Legionella pneumophila Philadelphia",num_proteins:2943,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid272624[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:75,name:"L.pneumophila - Whole organism, SC (GPM,Aug,2014)",score:7.51,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 7.51&nbsp<b>Coverage</b>: 25',organ:"WHOLE_ORGANISM",integrated:!1,coverage:25,publication_year:2014,num_abundances:743,hasPeptideCounts:!0,filename:"272624-GPM_201408.txt"}],total_coverage:25,average_coverage:25,topProteins:[{id:4189239,abundance:50247,rank:0,name:"groL",annotation:"Molecular chaperone GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:4191469,abundance:37327,rank:1,name:"lpg2961",annotation:"Major outer membrane protein"},{id:4190640,abundance:31247,rank:2,name:"lpg2112",annotation:"24 kDa macrophage-induced major protein; Antioxidant protein with alkyl hydroperoxidase activity. Required for the reduction of the AhpC active site cysteine residues and for the regeneration of the AhpC enzyme activity"},{id:4189238,abundance:23442,rank:3,name:"groS",annotation:"Co-chaperonin GroES; Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter"},{id:4190397,abundance:21295,rank:4,name:"hupB",annotation:"Hypothetical protein; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions"},{id:4188880,abundance:21254,rank:5,name:"rplL",annotation:"50S ribosomal protein L7/L12; Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation"},{id:4191033,abundance:19257,rank:6,name:"sidC",annotation:"Protein SidC, interaptin"},{id:4188874,abundance:18077,rank:7,name:"tuf2",annotation:"Elongation factor Tu"}]},283166:{id:283166,name:"Bartonella henselae str. Houston-1",compact_name:"Bartonella henselae",num_proteins:1488,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid283166[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:437,name:"B.henselae - Whole organism, SC (Omasits,Genome_Res,2013)",score:9.2,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23878158" target="_blank">Omasits,Genome_Res,2013</a><br/><b>Interaction consistency score</b>: 9.2&nbsp<b>Coverage</b>: 86',organ:"WHOLE_ORGANISM",integrated:!1,coverage:86,publication_year:2013,num_abundances:1275,hasPeptideCounts:!0,filename:"283166-Bhenselae_Albrethsen_2013.txt"}],total_coverage:85,average_coverage:86,topProteins:[{id:4303872,abundance:49298,rank:0,name:"hbpA",annotation:"Hemin binding protein A"},{id:4304757,abundance:26273,rank:1,name:"rpmH",annotation:"50S ribosomal protein L34"},{id:4304081,abundance:22083,rank:2,name:"hbpD",annotation:"Hemin binding protein D"},{id:4304591,abundance:16666,rank:3,name:"rpmC",annotation:"50S ribosomal protein L29"},{id:4303755,abundance:14151,rank:4,name:"rpsU",annotation:"30S ribosomal protein S21"},{id:4304839,abundance:12563,rank:5,name:"groL",annotation:"Chaperonin GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:4304586,abundance:12462,rank:6,name:"rpsN",annotation:"30S ribosomal protein S14; Binds 16S rRNA, required for the assembly of 30S particles and may also be responsible for determining the conformation of the 16S rRNA at the A site"},{id:4304985,abundance:12118,rank:7,name:"rpmF",annotation:"50S ribosomal protein L32"}]},353153:{id:353153,name:"Trypanosoma cruzi strain CL Brener",compact_name:"Trypanosoma cruzi",num_proteins:19602,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid353153[Organism:noexp]'>Refseq</a>",genome_source_version:"protozoa",datasets:[{id:52,name:"T.cruzi - Whole organism, Ade (DeGodoy,Proteomics,2012)",score:10.73,weight:"30%",description:'abundance based on MAPPED_BY_AUTHORS, Ade, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200078/abstract" target="_blank">deGodoy,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 10.73&nbsp<b>Coverage</b>: 23',organ:"WHOLE_ORGANISM",integrated:!1,coverage:23,publication_year:2012,num_abundances:4596,hasPeptideCounts:!1,filename:"353153-LFQ_Intensity_Ade_deGodoy_2012.txt"},{id:370,name:"T.cruzi - Whole organism, Epi (DeGodoy,Proteomics,2012)",score:11.37,weight:"60%",description:'abundance based on MAPPED_BY_AUTHORS, Epi, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200078/abstract" target="_blank">deGodoy,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 11.37&nbsp<b>Coverage</b>: 23',organ:"WHOLE_ORGANISM",integrated:!1,coverage:23,publication_year:2012,num_abundances:4596,hasPeptideCounts:!1,filename:"353153-LFQ_Intensity_Epi_deGodoy_2012.txt"},{id:215,name:"T.cruzi - Whole organism, Meta (DeGodoy,Proteomics,2012)",score:8.43,weight:"50%",description:'abundance based on MAPPED_BY_AUTHORS, Meta, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200078/abstract" target="_blank">deGodoy,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 8.43&nbsp<b>Coverage</b>: 23',organ:"WHOLE_ORGANISM",integrated:!1,coverage:23,publication_year:2012,num_abundances:4596,hasPeptideCounts:!1,filename:"353153-LFQ_Intensity_Meta_deGodoy_2012.txt"},{id:190,name:"T.cruzi - Whole organism, Stress2h (DeGodoy,Proteomics,2012)",score:11.25,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, Stress2h, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200078/abstract" target="_blank">deGodoy,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 11.25&nbsp<b>Coverage</b>: 23',organ:"WHOLE_ORGANISM",integrated:!1,coverage:23,publication_year:2012,num_abundances:4596,hasPeptideCounts:!1,filename:"353153-LFQ_Intensity_Stress2h_deGodoy_2012.txt"},{id:5,name:"T.cruzi - Whole organism, Stress30 (DeGodoy,Proteomics,2012)",score:10.34,weight:"20%",description:'abundance based on MAPPED_BY_AUTHORS, stress30, from<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200078/abstract" target="_blank">deGodoy,Proteomics,2012</a><br/><b>Interaction consistency score</b>: 10.34&nbsp<b>Coverage</b>: 23',organ:"WHOLE_ORGANISM",integrated:!1,coverage:23,publication_year:2012,num_abundances:4596,hasPeptideCounts:!1,filename:"353153-LFQ_Intensity_Stress30_deGodoy_2012.txt"},{id:129,name:"T.cruzi - Whole organism (Integrated)",score:12.44,weight:"",description:"integrated dataset: weighted average of all T.cruzi WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 12.44&nbsp<b>Coverage</b>: 23",organ:"WHOLE_ORGANISM",integrated:!0,coverage:23,publication_year:2017,num_abundances:4596,hasPeptideCounts:!1,filename:"353153-WHOLE_ORGANISM-integrated.txt"}],total_coverage:23,average_coverage:23,topProteins:[{id:5128525,abundance:81572,rank:0,name:"XP_816690.1",annotation:"Beta tubulin; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:5114743,abundance:66900,rank:1,name:"XP_802499.1",annotation:"Alpha tubulin; Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain"},{id:5129573,abundance:21057,rank:2,name:"XP_817738.1",annotation:"Heat shock protein 70 (HSP70)"},{id:5131535,abundance:9162,rank:3,name:"XP_819700.1",annotation:"Enolase"},{id:5121413,abundance:7378,rank:4,name:"XP_809575.1",annotation:"Glutamate dehydrogenase"},{id:5130690,abundance:7378,rank:5,name:"XP_818855.1",annotation:"Glutamate dehydrogenase"},{id:5133300,abundance:7139,rank:6,name:"XP_821465.1",annotation:"Chaperonin HSP60 mitochondrial precursor"},{id:5131343,abundance:7139,rank:7,name:"XP_819508.1",annotation:"Chaperonin HSP60, mitochondrial precursor"}]},449447:{id:449447,name:"Microcystis aeruginosa NIES-843",compact_name:"Microcystis aeruginosa",num_proteins:6312,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid449447[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:169,name:"M.aeruginosa - Whole organism, HUB5-3, SC (Alexova,MCP,2011)",score:3.44,weight:"10%",description:'abundance based on Spectral counting HUB5-3, HUB5-3, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21610102" target="_blank">Alexova, MCP,2011</a><br/><b>Interaction consistency score</b>: 3.44&nbsp<b>Coverage</b>: 46',organ:"WHOLE_ORGANISM",integrated:!1,coverage:46,publication_year:2011,num_abundances:2892,hasPeptideCounts:!0,filename:"449447-MicrocystisAeruginosa_strainNIES-843_HUB5-3_PRIDE.txt"},{id:167,name:"M.aeruginosa - Whole organism, PCC7055, SC (Alexova,MCP,2011)",score:3.75,weight:"100%",description:'abundance based on Spectral counting PCC7055, PCC7055, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21610102" target="_blank">Alexova, MCP,2011</a><br/><b>Interaction consistency score</b>: 3.75&nbsp<b>Coverage</b>: 52',organ:"WHOLE_ORGANISM",integrated:!1,coverage:52,publication_year:2011,num_abundances:3307,hasPeptideCounts:!0,filename:"449447-MicrocystisAeruginosa_strainNIES-843_PCC7005_PRIDE.txt"},{id:63,name:"M.aeruginosa - Whole organism, PCC7806, SC (Alexova,MCP,2011)",score:3.16,weight:"10%",description:'abundance based on Spectral counting PCC7806, PCC7806, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21610102" target="_blank">Alexova, MCP,2011</a><br/><b>Interaction consistency score</b>: 3.16&nbsp<b>Coverage</b>: 44',organ:"WHOLE_ORGANISM",integrated:!1,coverage:44,publication_year:2011,num_abundances:2802,hasPeptideCounts:!0,filename:"449447-MicrocystisAeruginosa_strainNIES-843_PCC7806_PRIDE.txt"},{id:326,name:"M.aeruginosa - Whole organism, UWOCCCBS, SC (Alexova,MCP,2011)",score:2.65,weight:"90%",description:'abundance based on Spectral counting UWOCCCBS, UWOCCCBS, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21610102" target="_blank">Alexova, MCP,2011</a><br/><b>Interaction consistency score</b>: 2.65&nbsp<b>Coverage</b>: 39',organ:"WHOLE_ORGANISM",integrated:!1,coverage:39,publication_year:2011,num_abundances:2467,hasPeptideCounts:!0,filename:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCCBS_PRIDE.txt"},{id:237,name:"M.aeruginosa - Whole organism, UWOCCMRC, SC (Alexova,MCP,2011)",score:2.29,weight:"100%",description:'abundance based on Spectral counting UWOCCMRC, UWOCCMRC, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21610102" target="_blank">Alexova, MCP,2011</a><br/><b>Interaction consistency score</b>: 2.29&nbsp<b>Coverage</b>: 35',organ:"WHOLE_ORGANISM",integrated:!1,coverage:35,publication_year:2011,num_abundances:2226,hasPeptideCounts:!0,filename:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCMRC_PRIDE.txt"},{id:368,name:"M.aeruginosa - Whole organism, UWOCCMRD, SC (Alexova,MCP,2011)",score:2.98,weight:"100%",description:'abundance based on Spectral counting UWOCCMRD, UWOCCMRD, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/21610102" target="_blank">Alexova, MCP,2011</a><br/><b>Interaction consistency score</b>: 2.98&nbsp<b>Coverage</b>: 39',organ:"WHOLE_ORGANISM",integrated:!1,coverage:39,publication_year:2011,num_abundances:2476,hasPeptideCounts:!0,filename:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCMRD_PRIDE.txt"},{id:188,name:"M.aeruginosa - Whole organism (Integrated)",score:5.07,weight:"",description:"integrated dataset: weighted average of all M.aeruginosa WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 5.07&nbsp<b>Coverage</b>: 79",organ:"WHOLE_ORGANISM",integrated:!0,coverage:79,publication_year:2017,num_abundances:5012,hasPeptideCounts:!1,filename:"449447-WHOLE_ORGANISM-integrated.txt"}],total_coverage:79,average_coverage:42.5,topProteins:[{id:6324260,abundance:128205,rank:0,name:"cpcB1",annotation:"Phycocyanin beta subunit"},{id:6324261,abundance:78859,rank:1,name:"cpcA1",annotation:"Phycocyanin alpha subunit"},{id:6322841,abundance:36764,rank:2,name:"apcB",annotation:"Allophycocyanin subunit beta"},{id:6322842,abundance:35705,rank:3,name:"apcA",annotation:"Allophycocyanin subunit alpha"},{id:6327215,abundance:27928,rank:4,name:"psbH",annotation:"Photosystem II reaction center protein H; One of the components of the core complex of photosystem II (PSII), required for its stability and/or assembly. PSII is a light-driven water:plastoquinone oxidoreductase that uses light energy to abstract electrons from H(2)O, generating O(2) and a proton gradient subsequently used for ATP formation. It consists of a core antenna complex that captures photons, and an electron transfer chain that converts photonic excitation into a charge separation"},{id:6322424,abundance:21086,rank:5,name:"MAE_06090",annotation:"Porin type major outer membrane protein"},{id:6324263,abundance:17834,rank:6,name:"cpcC1",annotation:"Phycobilisome rod linker polypeptide"},{id:6326544,abundance:16031,rank:7,name:"psaF",annotation:"Photosystem I subunit III"}]},511145:{id:511145,name:"Escherichia coli str. K-12 substr. MG1655",compact_name:"Escherichia coli K12 MG1655",num_proteins:4146,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid511145[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:179,name:"E.coli - Whole organism, Cpc (Wisniewski,J_Proteomics,2014)",score:12.96,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, cpc, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/25063446" target="_blank">Wisniewski,J_Proteomics,2014</a><br/><b>Interaction consistency score</b>: 12.96&nbsp<b>Coverage</b>: 55',organ:"WHOLE_ORGANISM",integrated:!1,coverage:55,publication_year:2014,num_abundances:2263,hasPeptideCounts:!1,filename:"511145-E_coli_Wisniewski_2014_cpc.txt"},{id:226,name:"E.coli - Whole organism, APEX (Arike,J_Proteomics,2012)",score:8.37,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, APEX, from<a href="http://www.mcponline.org/content/11/10/933.abstract" target="_blank">Arike, J_Proteomics,2012</a><br/><b>Interaction consistency score</b>: 8.37&nbsp<b>Coverage</b>: 98',organ:"WHOLE_ORGANISM",integrated:!1,coverage:98,publication_year:2012,num_abundances:4080,hasPeptideCounts:!1,filename:"511145-Ecoli_APEX_arike_2012.txt"},{id:236,name:"E.coli - Whole organism, SC (Krug,MCP,2013)",score:14.33,weight:"40%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23908556" target="_blank">Krug,MCP,2013</a><br/><b>Interaction consistency score</b>: 14.33&nbsp<b>Coverage</b>: 64',organ:"WHOLE_ORGANISM",integrated:!1,coverage:64,publication_year:2013,num_abundances:2670,hasPeptideCounts:!0,filename:"511145-Ecoli_Krug_2013.txt"},{id:43,name:"E.coli - Whole organism, EmPAI (Krug,MCP,2013)",score:5.37,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, emPAI, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/23908556" target="_blank">Krug,MCP,2013</a><br/><b>Interaction consistency score</b>: 5.37&nbsp<b>Coverage</b>: 63',organ:"WHOLE_ORGANISM",integrated:!1,coverage:63,publication_year:2013,num_abundances:2620,hasPeptideCounts:!1,filename:"511145-Ecoli_Krug_2013_emPAI.txt"},{id:474,name:"E.coli - Whole organism, EmPAI (Arike,J_Proteomics,2012)",score:4.43,weight:"80%",description:'abundance based on MAPPED_BY_AUTHORS, emPAI, from<a href="http://www.mcponline.org/content/11/10/933.abstract" target="_blank">Arike, J_Proteomics,2012</a><br/><b>Interaction consistency score</b>: 4.43&nbsp<b>Coverage</b>: 98',organ:"WHOLE_ORGANISM",integrated:!1,coverage:98,publication_year:2012,num_abundances:4080,hasPeptideCounts:!1,filename:"511145-Ecoli_emPAI_arike_2012.txt"},{id:306,name:"E.coli - Whole organism, IBAQ (Arike,J_Proteomics,2012)",score:11.49,weight:"100%",description:'abundance based on MAPPED_BY_AUTHORS, iBAQ, from<a href="http://www.mcponline.org/content/11/10/933.abstract" target="_blank">Arike, J_Proteomics,2012</a><br/><b>Interaction consistency score</b>: 11.49&nbsp<b>Coverage</b>: 98',organ:"WHOLE_ORGANISM",integrated:!1,coverage:98,publication_year:2012,num_abundances:4080,hasPeptideCounts:!1,filename:"511145-Ecoli_iBAQ_arike_2012.txt"},{id:164,name:"E.coli - Whole organism, SC (Wright,MCP,2012)",score:10.55,weight:"10%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3412976" target="_blank">Wright, MCP,2012</a><br/><b>Interaction consistency score</b>: 10.55&nbsp<b>Coverage</b>: 58',organ:"WHOLE_ORGANISM",integrated:!1,coverage:58,publication_year:2012,num_abundances:2418,hasPeptideCounts:!0,filename:"511145-Ecoli_strainK12_PRIDE.txt"},{id:432,name:"E.coli - Whole organism, SC (Valgepea,BMC_Syst_Biol,2010)",score:15.45,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.biomedcentral.com/1752-0509/4/166" target="_blank">Valgepea,BMC_Syst_Biol,2010</a><br/><b>Interaction consistency score</b>: 15.45&nbsp<b>Coverage</b>: 68',organ:"WHOLE_ORGANISM",integrated:!1,coverage:68,publication_year:2010,num_abundances:2839,hasPeptideCounts:!0,filename:"511145-Escherichia_coli_SC_biomart_12189_E__2reps.txt"},{id:86,name:"E.coli - Whole organism, SC (PRIDE,PRD000418,2011)",score:8.77,weight:"20%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/archive/projects/PRD000418" target="_blank">PRIDE,PRD000418,2011</a><br/><b>Interaction consistency score</b>: 8.77&nbsp<b>Coverage</b>: 33',organ:"WHOLE_ORGANISM",integrated:!1,coverage:33,publication_year:2011,num_abundances:1382,hasPeptideCounts:!0,filename:"511145-Escherichia_coli_SC_biomart_17112_E__2reps.txt"},{id:146,name:"E.coli - Whole organism, SC (PRIDE,PRD000485,2012)",score:12.58,weight:"90%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/archive/projects/PRD000485" target="_blank">PRIDE,PRD000485,2012</a><br/><b>Interaction consistency score</b>: 12.58&nbsp<b>Coverage</b>: 48',organ:"WHOLE_ORGANISM",integrated:!1,coverage:48,publication_year:2012,num_abundances:2e3,hasPeptideCounts:!0,filename:"511145-Escherichia_coli_SC_biomart_18119_E__5reps.txt"},{id:379,name:"E.coli - Whole organism, SC (Arike,J_Proteomics,2012)",score:17.68,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/22771841" target="_blank">Arike, J_Proteomics,2012</a><br/><b>Interaction consistency score</b>: 17.68&nbsp<b>Coverage</b>: 49',organ:"WHOLE_ORGANISM",integrated:!1,coverage:49,publication_year:2012,num_abundances:2044,hasPeptideCounts:!0,filename:"511145-Escherichia_coli_SC_biomart_29780_E_BTO0000214cellculture_2reps.txt"},{id:486,name:"E.coli - Whole organism, SC (GPM,Aug,2014)",score:18.06,weight:"40%",description:'abundance based on Spectral counting, from<a href="http://www.thegpm.org/" target="_blank">GPM,Aug,2014</a><br/><b>Interaction consistency score</b>: 18.06&nbsp<b>Coverage</b>: 69',organ:"WHOLE_ORGANISM",integrated:!1,coverage:69,publication_year:2014,num_abundances:2871,hasPeptideCounts:!0,filename:"511145-GPM_201408.txt"},{id:60,name:"E.coli - Whole organism, APEX (Lu,Nat_Biotechnol,2007)",score:9.49,weight:"10%",description:'abundance based on APEX, from<a href="http://www.nature.com/nbt/journal/v25/n1/full/nbt1270.html" target="_blank">Lu, Nat_Biotechnol,2007</a><br/><b>Interaction consistency score</b>: 9.49&nbsp<b>Coverage</b>: 11',organ:"WHOLE_ORGANISM",integrated:!1,coverage:11,publication_year:2007,num_abundances:449,hasPeptideCounts:!1,filename:"511145-Lu_ecoli_2007.txt"},{id:351,name:"E.coli - Whole organism, YFP fusion library, YFP normalized data (Taniguchi,Science,2010)",score:6.17,weight:"10%",description:'abundance based on YFP normalized data, YFP fusion library, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/20671182" target="_blank">Taniguchi,Science,2010</a><br/><b>Interaction consistency score</b>: 6.17&nbsp<b>Coverage</b>: 24',organ:"WHOLE_ORGANISM",integrated:!1,coverage:24,publication_year:2010,num_abundances:1009,hasPeptideCounts:!1,filename:"511145-Taniguchi_2010_Table_S6.txt"},{id:137,name:"E.coli - Whole organism (Integrated)",score:23.08,weight:"",description:"integrated dataset: weighted average of all E.coli WHOLE_ORGANISM datasets<br/><b>Interaction consistency score</b>: 23.08&nbsp<b>Coverage</b>: 99",organ:"WHOLE_ORGANISM",integrated:!0,coverage:99,publication_year:2017,num_abundances:4096,hasPeptideCounts:!1,filename:"511145-WHOLE_ORGANISM-integrated.txt"},{id:301,name:"E.coli - Whole organism, Gly and Lac substrate, SC (Lewis,Mol_Syst_Biol,2010)",score:8.33,weight:"40%",description:'abundance based on Spectral counting, Gly and Lac substrate, from<a href="http://www.ncbi.nlm.nih.gov/pubmed/20664636" target="_blank">Lewis,Mol_Syst_Biol,2010</a><br/><b>Interaction consistency score</b>: 8.33&nbsp<b>Coverage</b>: 21',organ:"WHOLE_ORGANISM",integrated:!1,coverage:21,publication_year:2010,num_abundances:852,hasPeptideCounts:!0,filename:"511145-WT-lewis-peptide_count.txt"},{id:101,name:"E.coli - Whole organism, HCD,CID, SC (Mancuso,J-Proteomics,2012)",score:10.37,weight:"100%",description:'abundance based on Spectral counting, HCD,CID, from<a href="http://www.sciencedirect.com/science/article/pii/S187439191200437X" target="_blank">Mancuso,J-Proteomics,2012</a><br/><b>Interaction consistency score</b>: 10.37&nbsp<b>Coverage</b>: 25',organ:"WHOLE_ORGANISM",integrated:!1,coverage:25,publication_year:2012,num_abundances:1048,hasPeptideCounts:!0,filename:"511145-ecoli1_HCD_CID_PRIDE.txt"},{id:85,name:"E.coli - Whole organism, SC (PRIDE,PRD000485,2012)",score:15.89,weight:"50%",description:'abundance based on Spectral counting, from<a href="http://www.ebi.ac.uk/pride/" target="_blank">PRIDE,PRD000485,2012</a><br/><b>Interaction consistency score</b>: 15.89&nbsp<b>Coverage</b>: 55',organ:"WHOLE_ORGANISM",integrated:!1,coverage:55,publication_year:2012,num_abundances:2281,hasPeptideCounts:!0,filename:"511145-ecoli2_resolutionCombination_PRIDE.txt"}],total_coverage:98,average_coverage:55.11764705882353,topProteins:[{id:6884078,abundance:439795,rank:0,name:"uspA",annotation:"Universal stress global response regulator; Required for resistance to DNA-damaging agents"},{id:6883932,abundance:9789,rank:1,name:"tufA",annotation:"Translation elongation factor EF-Tu 1; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:6883651,abundance:6192,rank:2,name:"yqiA",annotation:"Acyl CoA esterase; Displays esterase activity toward palmitoyl-CoA and pNP- butyrate"},{id:6883901,abundance:5826,rank:3,name:"rplE",annotation:"50S ribosomal subunit protein L5; This is 1 of the proteins that binds and probably mediates the attachment of the 5S RNA into the large ribosomal subunit, where it forms part of the central protuberance. Its 5S rRNA binding is significantly enhanced in the presence of L18"},{id:6882420,abundance:5694,rank:4,name:"lpp",annotation:"Murein lipoprotein; Interacts with the peptidoglycan both covalently and noncovalently. This interaction contributes to the maintenance of the structural and functional integrity of the cell envelope"},{id:6882890,abundance:5495,rank:5,name:"rplY",annotation:"50S ribosomal subunit protein L25; This is one of the proteins that binds to the 5S RNA in the ribosome where it forms part of the central protuberance. Binds to the 5S rRNA independently of L5 and L18. Not required for binding of the 5S rRNA/L5/L18 subcomplex to 23S rRNA"},{id:6882804,abundance:4687,rank:6,name:"gatB",annotation:"Galactitol-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in galactitol transport"},{id:6884517,abundance:4683,rank:7,name:"rplL",annotation:"50S ribosomal subunit protein L7/L12; The binding site for several of the GTPase factors involved in protein synthesis (IF-2, EF-Tu, EF-G and RF3). Is thus essential for accurate translation. Deletion of 1 of the L12 dimers from the ribosome (by deleting the binding site on L10) leads to decreased IF-2 association with the 70S ribosome and decreased stimulation of the GTPase activity of EF-G"}]},546414:{id:546414,name:"Deinococcus deserti VCD115",compact_name:"Deinococcus deserti",num_proteins:2597,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid546414[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:13,name:"D.deserti - Whole organism, SC (Baudet,MCP,2012)",score:6.13,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2830850/" target="_blank">Baudet, MCP, 2012</a><br/><b>Interaction consistency score</b>: 6.13&nbsp<b>Coverage</b>: 51',organ:"WHOLE_ORGANISM",integrated:!1,coverage:51,publication_year:2012,num_abundances:1329,hasPeptideCounts:!0,filename:"546414-DeinococcusDeserti_PRIDE.txt"}],total_coverage:51,average_coverage:51,topProteins:[{id:7232366,abundance:17865,rank:0,name:"groL",annotation:"Chaperonin GroEL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions"},{id:7231155,abundance:13130,rank:1,name:"Deide_11730",annotation:"Hypothetical protein"},{id:7232221,abundance:9831,rank:2,name:"dps",annotation:"DNA protection during starvation protein Dps"},{id:7231079,abundance:9797,rank:3,name:"Deide_11060",annotation:"ABC transporter substrate-binding protein"},{id:7230708,abundance:9209,rank:4,name:"Deide_07720",annotation:"TRAP transporter periplasmic protein"},{id:7230141,abundance:9179,rank:5,name:"Deide_02590",annotation:"Peptidyl-prolyl isomerase; PPIases accelerate the folding of proteins. It catalyzes the cis-trans isomerization of proline imidic peptide bonds in oligopeptides"},{id:7231953,abundance:8851,rank:6,name:"rpsC",annotation:"30S ribosomal protein S3; Binds the lower part of the 30S subunit head. Binds mRNA in the 70S ribosome, positioning it for translation"},{id:7231978,abundance:8511,rank:7,name:"Deide_19110",annotation:"Peptide ABC transporter substrate-binding protein"}]},593117:{id:593117,name:"Thermococcus gammatolerans EJ3",compact_name:"Thermococcus gammatolerans",num_proteins:2156,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid593117[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:297,name:"T.gammatolerans EJ3 - Whole organism, SC (Zivanovic,Genome_Biol,2009)",score:6.99,weight:"100%",description:'abundance based on Spectral counting, from<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19558674" target="_blank">Zivanovic,Genome_Biol, 2009</a><br/><b>Interaction consistency score</b>: 6.99&nbsp<b>Coverage</b>: 62',organ:"WHOLE_ORGANISM",integrated:!1,coverage:62,publication_year:2009,num_abundances:1341,hasPeptideCounts:!0,filename:"593117-ThermococcusGammatolerans_strain_PRIDE.txt"}],total_coverage:62,average_coverage:62,topProteins:[{id:7727266,abundance:55527,rank:0,name:"ghd-2",annotation:"Glutamate dehydrogenase C-terminal domain (ghd)"},{id:7727265,abundance:43097,rank:1,name:"ghd-1",annotation:"Glutamate dehydrogenase N-terminal domain, containing dimerization domain (ghd)"},{id:7725774,abundance:28667,rank:2,name:"TGAM_0331",annotation:"Solute binding protein"},{id:7726497,abundance:25425,rank:3,name:"tuf",annotation:"Elongation factor 1-alpha; This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis"},{id:7726595,abundance:23211,rank:4,name:"arcB",annotation:"Ornithine carbamoyltransferase"},{id:7727121,abundance:20760,rank:5,name:"trmB-3",annotation:"TrmB-like transcriptional regulator"},{id:7726977,abundance:20033,rank:6,name:"flaB3",annotation:"Flagellin B3 FlaB3; Flagellin is the subunit protein which polymerizes to form the filaments of archaeal flagella"},{id:7725623,abundance:17406,rank:7,name:"TGAM_0179",annotation:"4-aminobutyrate aminotransferase"}]},722438:{id:722438,name:"Mycoplasma pneumoniae FH",compact_name:"Mycoplasma pneumoniae",num_proteins:629,genome_source:"<a href='http://www.ncbi.nlm.nih.gov/genome/?term=txid722438[Organism:noexp]'>Refseq</a>",genome_source_version:"not available",datasets:[{id:201,name:"M.pneumoniae - Whole organism, Copy per cell, cpc normalized data (Kuhner,Science,2009)",score:2.52,weight:"100%",description:'abundance based on cpc normalized data, copy per cell, from<a href="http://www.sciencemag.org/content/326/5957/1235.full" target="_blank">Kuhner, Science,2009</a><br/><b>Interaction consistency score</b>: 2.52&nbsp<b>Coverage</b>: 61',organ:"WHOLE_ORGANISM",integrated:!1,coverage:61,publication_year:2009,num_abundances:383,hasPeptideCounts:!1,filename:"722438-Mycoplasma_pneumoniae_M129_Kuhner_et_al_Science2009.txt"}],total_coverage:60,average_coverage:61,topProteins:[{id:8582172,abundance:38909,rank:0,name:"tuf",annotation:"Translation elongation factor Tu"},{id:8581961,abundance:37856,rank:1,name:"gap",annotation:"Glyceraldehyde-3-phosphate dehydrogenase, type I"},{id:8582092,abundance:37075,rank:2,name:"groL",annotation:"Chaperonin GroL"},{id:8581928,abundance:29942,rank:3,name:"MPNE_0456",annotation:"Putative NADH oxidase"},{id:8581966,abundance:26137,rank:4,name:"dnaK",annotation:"Chaperone protein DnaK"},{id:8582116,abundance:22588,rank:5,name:"eno",annotation:"Phosphopyruvate hydratase"},{id:8582091,abundance:21926,rank:6,name:"MPNE_0674",annotation:"Cytosol aminopeptidase family, catalytic domain protein"},{id:8582093,abundance:21875,rank:7,name:"groS",annotation:"Chaperonin GroS"}]}}},"48d5":function(e,a,t){"use strict";(function(e){t("28a5");(function(e){e.fn.drilldown=function(a){var t,n,i,o={wrapper_class:"drilldown",menu_class:"drilldown-menu",submenu_class:"nav",parent_class:"dd-parent",parent_class_link:"dd-parent-a",active_class:"active",header_class_list:"breadcrumb",header_class:"breadcrumbwrapper",class_selected:"dd-selected",event_type:"click",hover_delay:300,speed:"slow",save_state:!0,show_count:!1,count_class:"dd-count",icon_class:"glyphicon glyphicon-chevron-right pull-right",link_type:"backlink",reset_text:'<span class="glyphicon glyphicon-folder-open"></span> All',default_text:"Select Option",show_end_nodes:!0,hide_empty:!0,menu_height:!1,show_header:!1,header_tag:"div",header_tag_class:"list-group-item active"};a=e.extend(o,a);if(a.justRehydrate){var r=this,s=e(r).parent();return e(window).resize(function(){y(r,s)}),void p(s,r)}return this.each(function(a){var t=this;u(t);var n=e(t).parent();if(c(t,n),!h(t,n)&&1==o.save_state){var i=o.wrapper_class+"-0123";f(i,t)}g(t,n),e(window).resize(function(){y(t,n)}),p(n,t)});function c(a,t){var n=b(t),r='<div id="'+n+'" class="'+o.header_class+'"></div>';e("ul",a).each(function(){e(this).addClass(o.submenu_class)}),e(a).before(r),i='<span class="'+o.icon_class+'"></span>';var s=e(a).outerWidth(!0);s+="px";var c=e("li",a).outerHeight(!0),l=e("ul",a),d=C(l),u=e('[rel="'+d+'"]',l),p=S(u);g="link"==o.link_type?c*(d+p):c*d;var m=e("> ul",t),_=e(">li",m).length,g=c*_,h=c*_+e(r).height();o.menu_height?e(t).css({height:o.menu_height,width:s,"overflow-y":"auto","overflow-x":"hidden"}):e(t).css({height:h+"px",width:s}),e(a).css({height:g+"px",width:s}),e("li",a).each(function(){if(e(this).css({width:s}),e("ul",this).css({width:s,marginRight:"-"+s,marginTop:"0"}),e("> ul",this).length&&(e(this).addClass(o.parent_class),e("> a",this).addClass(o.parent_class_link).append(i),1==o.show_count)){var a=e("a:not(."+o.parent_class_link+")",this),t=parseInt(e(a).length);e("> a",this).append(' <span class="'+o.count_class+'">('+t+")</span>")}}),e("ul",t).each(function(){e("li:last",this).addClass("last")}),e("> ul > li:last",t).addClass("last"),"link"==o.link_type&&e(l).css("top",c+"px")}function l(e){var a="1234567";return a}function d(e){var a=l(e),t=o.wrapper_class+"-"+a;return t}function u(a){e(a).addClass(o.menu_class),e(a).addClass(o.submenu_class),e(a).removeClass("hidden");var t='<div class="'+o.wrapper_class+'" />';e(a).wrap(t);var n=e(a).parent(),i=d(n);e(n).attr("id",i)}function b(e){var a=l(e),t="header-"+o.header_class+"-"+a;return t}function p(a,i){d(a);e("li a",i).click(function(r){t=this,n=e(this).parent("li").stop(),e(n).siblings(),(e("> ul",n).length||o.show_end_nodes)&&(e(t).hasClass(o.active_class)?(e("ul a",n).removeClass(o.active_class),g(i,a)):m(n,a,i)),e(a,i).scrollTop(0),e(this).next("ul").length>0?(r.preventDefault(),r.stopPropagation(),e(t).trigger("drilldown.parentclick")):e(t).addClass(o.active_class),e(t).trigger("drilldown.linklclick")});var r=e("."+o.header_class,a);r.on("click","a",function(t){var n;e(this).hasClass("link-back")?e("a."+o.active_class+":last",i).removeClass(o.active_class):(n=parseInt(e(this).index("#"+b(a)+" a")),0==n?e("a",i).removeClass(o.active_class):(n-=1,e("a."+o.active_class+":gt("+n+")",i).removeClass(o.active_class))),g(i,a),t.preventDefault(),e(i).trigger("drilldown.linklclick")})}function m(a,t,n){var i=e("."+o.header_class,t),r=e(o.header_tag,i).text(),s=e("> a",a).text();"breadcrumb"==o.link_type&&(e("ul",i).length||e(i).prepend('<ul class="'+o.header_class_list+'"></ul>'),r==o.default_text?(e("ul li:last-child",i).remove(),e("ul",i).append('<li class="first"><a href="#">'+o.reset_text+"</a></li>")):(e("ul li:last-child",i).remove(),e("ul",i).append('<li><a href="#">'+r+"</a></li>")),e("ul",i).append('<li class="'+o.active_class+'">'+s+"</li>")),"backlink"==o.link_type&&(e("a",i).length?e(".link-back",i).html(r):e(i).prepend('<a href="#" class="link-back">'+r+"</a>")),"link"==o.link_type&&(e("a",i).length||e(i).prepend('<ul><li class="first"><a href="#">'+o.reset_text+"</a></li></ul>")),_(i,s);var c=e("> a",a);e(c).addClass(o.active_class);var l=e("> ul",a),d=e(a).outerHeight(!0),u=e(">li",l).length,b=d*u;e(n).css({height:b+"px"});var p=e(i).outerHeight(!0),m=b+p;o.menu_height?o.hide_empty&&!u?e(t).css({height:p,"overflow-y":"hidden","overflow-x":"hidden"}):e(t).css({height:o.menu_height,"overflow-y":"auto","overflow-x":"hidden"}):e(t).css({height:m+"px"}),e("> ul li",a).show(),e("> ul",a).animate({"margin-right":0},o.speed);var g=e(a).siblings();if(e(g).hide(),"link"!=o.link_type&&e(c).hide(),1==o.save_state){var h=e(t).attr("id");v(h,n)}}function _(a,t){e(o.header_tag,a).length||(1==o.show_header?e(a).append("<"+o.header_tag+' class="'+o.header_tag_class+'"</'+o.header_tag+">"):e(a).append("<"+o.header_tag+' class="hidden '+o.header_tag_class+'"</'+o.header_tag+">")),e(o.header_tag,a).html(t)}function g(a,t){var n=e("#"+b(s));e("ul",n).remove(),e("a",n).remove(),e("li",a).show(),e("a",a).show();var i=e(a).outerWidth(!0);if("link"==o.link_type)if(e("a."+o.active_class+":last",a).parent("li").length){var r=e("a."+o.active_class+":last",a).parent("li");e("ul",r).css("margin-right",-i+"px")}else e("ul",a).css("margin-right",-i+"px");else e("ul",a).css("margin-right",-i+"px");if(_(n,o.default_text),"breadcrumb"==o.link_type){var c=e(o.header_tag,n).text();e("ul",n).length||e(n).prepend('<ul class="'+o.header_class_list+'"></ul>'),c==o.default_text&&e("ul",n).append('<li class="'+o.active_class+'">'+o.default_text+"</li>")}if(1==o.save_state){var l=e(t).attr("id");v(l,a)}var d=a;if(e("a."+o.active_class,a).each(function(n){var i=e(this).parent("li").stop();m(i,t,a),d=e(this).parent("li")}),e("> ul",d).length)u=e("> ul",d);else var u=d;var p=e(">li",u).length,g=e("li",u).outerHeight(!0),h=g*p,f=e(n).outerHeight(!0),y=g*p+f;o.menu_height?(o.hide_empty&&!p?e(t).css({height:f,"overflow-y":"hidden","overflow-x":"hidden"}):e(t).css({height:o.menu_height,"overflow-y":"auto","overflow-x":"hidden"}),e(a).css({height:h+"px"})):(e(t).css({height:y+"px"}),e(a).css({height:h+"px"}))}function h(a,t){var n=e("."+o.class_selected,a).parents("li");return e.each(n,function(a,t){e("> a",t).addClass(o.active_class)}),n.length}function f(a,t){var n=e.cookie(a);if(null!=n){var i=n.split(",");e.each(i,function(a,n){n=parseInt(n)+0;var i=e("li:eq("+n+")",t);e("> a",i).addClass(o.active_class)})}}function v(a,t){var n=[];e("a."+o.active_class,t).each(function(a){var i=e(this).parent("li"),o=e("li",t).index(i);n.push(o)}),e.cookie(a,n,{path:"/"})}function y(a,t){e(t).css({width:"auto"}),e(a).css({width:"auto"});var n=e(t).outerWidth(!0);n+="px",e(t).css({width:n}),e(a).css({width:n}),e("li",a).each(function(){e(this).css({width:n}),e("ul",this).css({width:n,marginRight:"-"+n,marginTop:"0"})}),g(a,t)}function C(a){var t=void 0;return e(a).each(function(){var a=parseInt(e("> li",this).length);e(this).attr("rel",a),(void 0===t||t<a)&&(t=a)}),t}function S(a){var t=void 0;return e(a).each(function(){var a=parseInt(e(this).parents("li").length);(void 0===t||t<a)&&(t=a)}),t}}})(e)}).call(this,t("9537"))},4991:function(e,a,t){},"4d06":function(e,a,t){"use strict";var n=t("c106"),i=t.n(n);i.a},5440:function(e,a,t){"use strict";(function(e){t("386d"),t("7f7f"),t("ac6a"),t("cadf"),t("06db"),t("456d");var n=t("267a"),i=t.n(n),o=t("16b0"),r=o["a"].SPECIES_FULL_NAMES_SORTED,s=o["a"].SPECIES_FULL_NAMES,c={},l=[];Object.keys(o["a"].SPECIES_SYNONYMS).forEach(function(e){var a=o["a"].SPECIES_SYNONYMS[e];l.push({name:e,official_name:s[a]}),c[e]=a}),a["a"]={props:["placeholder"],data:function(){return{}},methods:{reset:function(){e(this.$el).typeahead("val","")}},mounted:function(){var a=this,t=new i.a({datumTokenizer:i.a.tokenizers.obj.whitespace("name"),queryTokenizer:i.a.tokenizers.nonword,local:l});function n(e,a){""===e?a(r):t.search(e,a)}e(this.$el).typeahead({hint:!0,highlight:!0,minLength:0},{name:"organism",source:n,display:"official_name",limit:r.length,templates:{suggestion:function(e){return Object.hasOwnProperty.call(e,"name")?"<div><strong>".concat(e.name,"</strong><br/>").concat(e.official_name,"</div>"):"<div><strong>".concat(e.official_name,"</strong></div>")},empty:'<div class="text-warning">no organism matches</div>'}}),e(this.$el).bind("typeahead:select",function(e,t){a.$emit("selected",t.id)}),e(this.$el).bind("typeahead:change",function(e,t){t||a.$emit("selected","")})},beforeDestroy:function(){e(this.$el).typeahead("destroy")}}}).call(this,t("9537"))},"7e6d":function(e,a,t){},"82b3":function(e,a,t){"use strict";(function(e){t("7f7f");var n=t("f3e3"),i=t.n(n),o=(t("7514"),t("a481"),t("5df3"),t("4f7f"),t("4db1")),r=t.n(o),s=(t("ac6a"),t("cadf"),t("06db"),t("456d"),t("2b0e")),c=t("6e58"),l=t("16b0");function d(e){return"string"!==typeof e?"":e.charAt(0).toUpperCase()+e.slice(1).toLowerCase()}function u(e,a,t){var n=c["layout"].tree(),i=n.nodes(e),o={},s=0;i.forEach(function(e){e.depth>s&&(s=e.depth),o[e.depth]||(o[e.depth]=0),o[e.depth]+=1});var l=Object.keys(o).map(function(e){return o[e]}),d=Math.max.apply(Math,r()(l)),u=140,b=s*u<a?a:(s+1)*u,p=70,m=d*p<t?t:(d+1)*p;return{width:b,height:m}}function b(e,a){if("orthgroup"===e.type&&e.children.forEach(function(e){return b(e,a)}),e.proteins){var t=r()(new Set(e.proteins.map(function(e){return e.id})));e.children=t.map(function(e){return a[e]})}}function p(e){return d(e)}function m(e){return e.toUpperCase()}function _(e){return d(e.replace("_"," "))}function g(e){return e.replace(" ","_").toUpperCase()}var h="not available";a["a"]={name:"ProteinFamily",props:["protein"],data:function(){return{dataLoaded:!1,selectedLevel:h,selectedTissue:h,groups:{},level:{},tissue:{}}},created:function(){return this.loadData()},computed:{taxonomicLevels:function(){return this.selectedLevel!==h&&this.groups.members?this.groups.members.map(function(e){return p(e.taxonomicLevel)}):[h]},tissues:function(){return this.selectedTissue!==h&&this.level.tissues?this.level.tissues.map(function(e){return _(Object.keys(e)[0])}):[h]}},methods:{loadData:function(){if(console.log("%s loading family %s",this.protein.externalId,new Date),!this.dataLoaded){var e=this,a="http://orthologs-api.pax-db.org/protein/".concat(this.protein.externalId,"/ortholog_groups");this.$axios.get(a).then(function(a){e.groups=a.data;var t=e.groups.members[e.groups.members.length-1].taxonomicLevel;e.selectedLevel=p(t)}).catch(function(a){e.$q.notify({color:"negative",position:"top",message:"Loading protein family failed",icon:"report_problem"}),s["a"].rollbar.error(a),e.dataLoaded=!0})}},loadLevel:function(e){var a=this;if(e!==h){var t=m(a.selectedLevel);console.log("%s loading LEVEL %s",this.protein.externalId,t);var n=this.groups.members.find(function(e){return e.taxonomicLevel===t});if(!n)return a.selectedTissue=h,void(a.dataLoaded=!0);var o=n["@id"],r="http://orthologs-api.pax-db.org".concat(o);this.$axios.get(r).then(function(e){if(!e.data.tissues.length)return a.selectedTissue=h,void(a.dataLoaded=!0);a.level=e.data;var t=a.level.tissues.find(function(e){return Object.prototype.hasOwnProperty.call(e,"WHOLE_ORGANISM")});if(!t){var n=i()(a.level.tissues,1);t=n[0]}a.selectedTissue=_(Object.keys(t)[0]),a.dataLoaded=!0}).catch(function(e){a.$q.notify({color:"negative",position:"top",message:"Loading protein family failed",icon:"report_problem"}),s["a"].rollbar.error(e),a.dataLoaded=!0})}},loadTissue:function(e){var a=this;if(e!==h){var t=g(a.selectedTissue);console.log("%s loading TISSUE %s",this.protein.externalId,t);var n=a.level.tissues.find(function(e){return Object.prototype.hasOwnProperty.call(e,t)}),i="http://orthologs-api.pax-db.org".concat(n[t]);this.$axios.get(i).then(function(e){a.tissue=e.data}).catch(function(e){a.$q.notify({color:"negative",position:"top",message:"Loading protein family failed",icon:"report_problem"}),s["a"].rollbar.error(e)})}else a.dataLoaded=!0}},watch:{selectedLevel:function(e,a){console.log("selected LEVEL changed",e,a),this.selectedTissue=h,this.loadLevel(e,a)},selectedTissue:function(e,a){console.log("selected TISSUE changed",e,a),this.loadTissue(e,a)},tissue:function(a,t){var n=this,i=a.familyTree,o={};a.members.forEach(function(e){o[e.id]=e,e.type="protein"}),b(i,o);var r=u(i),s={top:20,right:150,left:100,bottom:20},p=c["layout"].cluster().size([r.height-s.top-s.bottom,r.width-s.left-s.right]);e("#family_tree").empty();var m=c["select"]("#family_tree").append("svg").attr("xmlns","http://www.w3.org/2000/svg").attr("id","family_tree_svg").attr("xmlns:xlink","http://www.w3.org/1999/xlink").attr("width",r.width).attr("height",r.height).append("g").attr("transform","translate(".concat(s.left,",").concat(s.top,")")),_=p.nodes(i),g=function(e,a){return"M".concat(e.source.y,",").concat(e.source.x,"V").concat(e.target.x,"H").concat(e.target.y)},h=(c["svg"].diagonal().projection(function(e){return[e.y,e.x]}),m.selectAll(".link").data(p.links(_)).enter().append("path").attr("class","link").attr("d",g),m.selectAll(".node").data(_).enter().append("g").attr("class","node").attr("transform",function(e){return"translate(".concat(e.y,",").concat(e.x,")")}).on("click",function(e,a){var t;c["event"]?(c["event"].stopPropagation(),c["event"].preventDefault(),t=c["event"].target.getAttribute("xlink:href")):event&&(event.stopPropagation(),event.preventDefault(),t=event.target.getAttribute("xlink:href")),t&&t!==n.$route.path&&n.$router.push(t)}));h.append("circle").attr("r",4.5);var f=h.append("text").attr("dx",function(e){return e.children?-8:8}).attr("dy",3).style("text-anchor",function(e){return e.children?"end":"start"});f.html(function(e){if("orthgroup"===e.type)return d(e.name);if("species"===e.type){var a='<a fill="#337ab7" class="species" xlink:href="/species/'.concat(e.id,'">').concat(l["a"].SPECIES_SHORT_NAMES[e.id],"</a>");return a}if("protein"===e.type){var t='<a fill="#337ab7" class="protein" xlink:href="/protein/'.concat(e.iid,'">').concat(e.name,"</a>");return"".concat(t,"&nbsp;").concat(e.abundance.value," ppm")}return console.log("error, don't know how to handle ".concat(e.id)),e.name});var v=e("#family_tree_svg")[0].getBBox();m.attr("width","".concat(v.x+v.width+30,"px"))}}}}).call(this,t("9537"))},"8f3e":function(e,a,t){},"967b":function(e,a,t){"use strict";(function(e){var n=t("2b0e"),i=t("1b21"),o=t("3e0f"),r=t("16b0"),s=t("0196");t("2c5f"),t("6ffb"),t("6ea2"),t("ad7f"),t("baee");a["a"]={name:"SpeciesPage",data:function(){return{id:this.$route.params.id,species:o["a"][this.$route.params.id],defaultDataset:Object(s["a"])(o["a"][this.$route.params.id]),datasetFilenames:{278:"10090-Adrenal_gland_geiger_2013.txt",154:"10090-BRAIN-integrated.txt",343:"10090-BROWN_ADIPOSE_TISSUE-integrated.txt",291:"10090-Brain_cortex_geiger_2013.txt",409:"10090-Brain_medulla_geiger_2013.txt",238:"10090-Brown_fat_geiger_2013.txt",251:"10090-CELL_LINE-integrated.txt",218:"10090-Cerebellum_geiger_2013.txt",378:"10090-Colon_geiger_2013.txt",439:"10090-Diaphragm_geiger_2013.txt",6:"10090-Duodenum_geiger_2013.txt",78:"10090-Embryonic_tissue_geiger_2013.txt",115:"10090-Eye_geiger_2013.txt",149:"10090-GPM_2012_09_Mus_musculus.txt",481:"10090-GPM_201408.txt",73:"10090-HEART-integrated.txt",72:"10090-Heart_geiger_2013.txt",222:"10090-Ileum_geiger_2013.txt",9:"10090-Jejunum_geiger_2013.txt",440:"10090-KIDNEY-integrated.txt",427:"10090-Kidney_cortex_geiger_2013.txt",132:"10090-Kidney_medulla_geiger_2013.txt",254:"10090-LIVER-integrated.txt",74:"10090-LUNG-integrated.txt",488:"10090-Liver_geiger_2013.txt",339:"10090-Lung_geiger_2013.txt",421:"10090-Menschaert_protID_NSAF.txt",161:"10090-Menschaert_protID_emPAI.txt",400:"10090-Midbrain_geiger_2013.txt",346:"10090-MouseBuild243_Jun2010.peptides.new.txt",26:"10090-Mouse_PeptideAtlas_2011-06_Ens62.txt",148:"10090-Mouse_brain_Huttlin_cell2010.csv.txt",18:"10090-Mouse_brownfat_Huttlin_cell2010.csv.txt",459:"10090-Mouse_erythrocytes_Kruger2008_cell.txt",76:"10090-Mouse_gut_epithelium_Kruger2008_cell.txt",402:"10090-Mouse_heart_Huttlin_cell2010.csv.txt",268:"10090-Mouse_heart_Kruger2008_cell.txt",482:"10090-Mouse_kidney_Huttlin_cell2010.csv.txt",125:"10090-Mouse_liver_Huttlin_cell2010.csv.txt",446:"10090-Mouse_liver_Kruger2008_cell.txt",375:"10090-Mouse_liver_Wang2009_Proteomics_ClinApp.txt",67:"10090-Mouse_lung_Huttlin_cell2010.csv.txt",405:"10090-Mouse_pancreas_Huttlin_cell2010.csv.txt",277:"10090-Mouse_platelets_Kruger2008_cell.txt",452:"10090-Mouse_spleen_B_cells_Kruger2008_cell.txt",259:"10090-Mouse_spleen_Huttlin_cell2010.csv.txt",411:"10090-Mouse_testis_Huttlin_cell2010_peptide.csv.txt",436:"10090-Mus_musculus_SC_biomart_11364_E__2reps.txt",424:"10090-Mus_musculus_SC_biomart_28162_O_BTO0005136ESE14cell_1reps.txt",69:"10090-Muscle_geiger_2013.txt",345:"10090-Olfactory_bulb_geiger_2013.txt",328:"10090-Ovary_geiger_2013.txt",372:"10090-PANCREAS-integrated.txt",266:"10090-PA_plasma.txt",319:"10090-Pancreas_geiger_2013.txt",79:"10090-SPLEEN-integrated.txt",102:"10090-Salivary_gland_geiger_2013.txt",364:"10090-Spleeen_geiger_2013.txt",80:"10090-Stomach_geiger_2013.txt",373:"10090-Thymus_geiger_2013.txt",320:"10090-Uterus_geiger_2013.txt",196:"10090-WHOLE_ORGANISM-integrated.txt",219:"10090-White_fat_geiger_2013.txt",305:"10090-mouse_brain_Martens_2006_proteomics.txt",490:"10090-mouse_brain_Wang_2007_J_proteome_Res.txt",489:"10090-mouse_brain_kislinger2006_cell.txt",273:"10090-mouse_glomeruli_kidney_F.Waanders2009_PNAs.txt",309:"10090-mouse_haploidGermCells_Guo_2010_JProteome.txt",109:"10090-mouse_heart_kislinger2006_cell.txt",144:"10090-mouse_kidney_kislinger2006_cell.txt",417:"10090-mouse_liver_Shi2007_JourmalofProteome.txt",34:"10090-mouse_liver_kislinger2006_cell.txt",177:"10090-mouse_lung_kislinger2006_cell.txt",228:"10090-mouse_pancreas_F.Waanders2009_PNAs_combine.txt",262:"10090-mouse_placenta_kislinger2006_cell.txt",476:"10116-GPM_2012_09_Rattus_norvegicus.txt",408:"10116-GPM_201408.txt",462:"10116-PA_201311.txt",464:"10116-Rattus_norvegicus_SC_biomart_19302_E__6reps.txt",93:"10116-WHOLE_ORGANISM-integrated.txt",410:"1148-SynechocystisSp_strain_PRIDE.txt",176:"122586-GPM_201408.txt",70:"158878-GPM_201408.txt",415:"160490-GPM_201408.txt",360:"160490-PA_200702.txt",307:"160490-StreptococcusPyogenes_M1GAS_PRIDE.txt",159:"160490-WHOLE_ORGANISM-integrated.txt",280:"160490-atlas_build_101.txt",396:"169963-GPM_201408.txt",127:"192222-Campylobacter_jejuni_SC_biomart_17684_E__1reps.txt",19:"192222-Campylobacter_jejuni_SC_biomart_17684_O__1reps.txt",484:"192222-WHOLE_ORGANISM-integrated.txt",458:"198214-ShigellaFlexneri_2a_PRIDE.txt",88:"208964-GPM_201408.txt",103:"208964-Pseudomonas_aeruginosa_SC_biomart_22024_E__1reps.txt",130:"208964-WHOLE_ORGANISM-integrated.txt",166:"211586-GPM_201408.txt",454:"214092-Yersinia_pestis_SC_biomart_18099_O__3reps.txt",352:"214684-Cryptococcus_biofilm_Santi_2014.txt",111:"214684-Cryptococcus_planktonic_Santi_2014.txt",37:"214684-WHOLE_ORGANISM-integrated.txt",312:"224308-BacillusSubtilisSubsp_PRIDE.txt",493:"224308-GPM_201408.txt",256:"224308-WHOLE_ORGANISM-integrated.txt",347:"226186-GPM_201408.txt",422:"243159-Acidithiobac_PSM_biofilm_Vera_2013.txt",223:"243159-Acidithiobac_PSM_planktonic_Vera_2013.txt",113:"243159-WHOLE_ORGANISM-integrated.txt",272:"260799-GPM_201408.txt",460:"267671-Leptospira_build_121.peptides.cut.txt",195:"267671-PA_201308.txt",84:"267671-WHOLE_ORGANISM-integrated.txt",349:"267671-controlMS1_Leptospira_Malmstroem_2009.txt",354:"267671-controlSpectral_Leptospira_Malmstroem_2009.txt",35:"272623-GPM_201408.txt",348:"272623-Lactococcus_lactis_Lahtvee_2014.txt",194:"272623-WHOLE_ORGANISM-integrated.txt",75:"272624-GPM_201408.txt",437:"283166-Bhenselae_Albrethsen_2013.txt",47:"3055-GPM_201408.txt",52:"353153-LFQ_Intensity_Ade_deGodoy_2012.txt",370:"353153-LFQ_Intensity_Epi_deGodoy_2012.txt",215:"353153-LFQ_Intensity_Meta_deGodoy_2012.txt",190:"353153-LFQ_Intensity_Stress2h_deGodoy_2012.txt",5:"353153-LFQ_Intensity_Stress30_deGodoy_2012.txt",129:"353153-WHOLE_ORGANISM-integrated.txt",298:"3702-2.1ArabidopsisLeaf_waterDefict_SOW.txt",401:"3702-2.2ArabidopsisLeaf_waterDefict_SWD.txt",284:"3702-6.1JasmonateControls_controlLeaves.txt",233:"3702-6.2JasmonateControls_woundedLeaves.txt",270:"3702-7.1Large-scalePhosphoproteome_rosette_ED.txt",385:"3702-7.2Large-scalePhosphoproteome_rosette_EN.txt",2:"3702-7.3Large-scalePhosphoproteome_shoot.txt",163:"3702-Arabidopsis_capel_Baerenfaller_2011_IntegrBiol.txt",209:"3702-Arabidopsis_cotyledon_Baerenfaller_2011_IntegrBiol.txt",279:"3702-Arabidopsis_flower_Baerenfaller_2011_IntegrBiol.txt",413:"3702-Arabidopsis_flowerbud_Baerenfaller_2011_IntegrBiol.txt",283:"3702-Arabidopsis_juvenileLeaf_Baerenfaller_2011_IntegrBiol.txt",124:"3702-Arabidopsis_pollen_Grobei_2009_GenomeRes.txt",135:"3702-Arabidopsis_root_Baerenfaller_2011_IntegrBiol.txt",174:"3702-Arabidopsis_seed_Baerenfaller_2011_IntegrBiol.txt",213:"3702-Arabidopsis_silique_Baerenfaller_2011_IntegrBiol.txt",322:"3702-Arabidopsis_thaliana_SC_biomart_27037_E__1reps.txt",7:"3702-CARPEL-integrated.txt",341:"3702-COTYLEDON-integrated.txt",355:"3702-FLOWER-integrated.txt",374:"3702-FLOWERBUD-integrated.txt",14:"3702-GPM_201408.txt",447:"3702-JUVENILE_LEAF-integrated.txt",342:"3702-LEAF-integrated.txt",178:"3702-Motohashi_2012_avgWT_Prot_nSPC.txt",36:"3702-POLLEN-integrated.txt",444:"3702-ROOT-integrated.txt",258:"3702-ROSETTE-integrated.txt",304:"3702-SEED-integrated.txt",82:"3702-SHOOT-integrated.txt",324:"3702-SILIQUE-integrated.txt",323:"3702-WHOLE_ORGANISM-integrated.txt",122:"3702-arabidopsis1a_carpel_Baerenfaller_PRIDE.txt",152:"3702-arabidopsis1c_cotyledon_Baerenfaller_PRIDE.txt",91:"3702-arabidopsis1d_flower_Baerenfaller_PRIDE.txt",244:"3702-arabidopsis1e_flowerBud_Baerenfaller_PRIDE.txt",42:"3702-arabidopsis1f_juvenileLeaf_Baerenfaller_PRIDE.txt",173:"3702-arabidopsis1g_leaf_Baerenfaller_PRIDE.txt",191:"3702-arabidopsis1h_pollen_Baerenfaller_PRIDE.txt",153:"3702-arabidopsis1i_root_Baerenfaller_PRIDE.txt",302:"3702-arabidopsis1k_seed_Baerenfaller_PRIDE.txt",28:"3702-arabidopsis1l_shoot_Baerenfaller_PRIDE.txt",414:"3702-arabidopsis1m_silique_Baerenfaller_PRIDE.txt",89:"3702-arabidopsis2_estimationLeaf_PRIDE.txt",116:"3702-countCastellana.txt",412:"3702-peptide.Baerenfaller_2008_new.txt",428:"39947-GPM_2012_09_Oryza_sativa.txt",192:"4081-Solanum_lycopersicum_SC_biomart_18197_E__1reps.txt",321:"44689-GPM_201408.txt",169:"449447-MicrocystisAeruginosa_strainNIES-843_HUB5-3_PRIDE.txt",167:"449447-MicrocystisAeruginosa_strainNIES-843_PCC7005_PRIDE.txt",63:"449447-MicrocystisAeruginosa_strainNIES-843_PCC7806_PRIDE.txt",326:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCCBS_PRIDE.txt",237:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCMRC_PRIDE.txt",368:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCMRD_PRIDE.txt",188:"449447-WHOLE_ORGANISM-integrated.txt",65:"4577-LEAF-integrated.txt",426:"4577-Mays_SC_mature_rep1234_Facette_2013.txt",261:"4577-Mays_SC_zone1_rep1234_Facette_2013.txt",186:"4577-Mays_SC_zone2_rep1234_Facette_2013.txt",449:"4577-Mays_SC_zone3_rep1234_Facette_2013.txt",487:"4577-Mays_avgNSAF_ponnala_2014.txt",475:"4577-Mays_avgNadjSPC_ponnala_2014.txt",12:"4896-GPM_201408.txt",48:"4896-SchizosaccharomycesPombe_FissionYeast_PRIDE.txt",473:"4896-Schizosaccharomyces_pombe_SC_biomart_29627_E_BTO0000214cellculture_1reps.txt",286:"4896-Spombe_emPAI_Gunaratne_2013.txt",250:"4896-WHOLE_ORGANISM-integrated.txt",442:"4896-pombe_cell2012_protein.txt",387:"4896-proliferating_protID_copynr_Marguerat_2012.txt",429:"4896-quiescent_protID_copynr_Marguerat_2012.txt",450:"4932-GPM_2012_09_Saccharomyces_cerevisiae.txt",243:"4932-GPM_201408.txt",358:"4932-Ghaemmaghami_et_al_yeast_data.txt",25:"4932-Newman_et_al_yeast_data_SD.txt",300:"4932-Newman_et_al_yeast_data_YEPD.txt",242:"4932-PA_201303.txt",331:"4932-Saccharomyces_cerevisiae_SC_biomart_17916_O__1reps.txt",110:"4932-Saccharomyces_cerevisiae_SC_biomart_18097_O__1reps.txt",3:"4932-WHOLE_ORGANISM-integrated.txt",16:"4932-Yeast_2009-0522.txt",106:"4932-Yeast_GeneName_Intensity_Breker_2103.txt",203:"4932-Yeast_Lu_2007_YMD.txt",285:"4932-Yeast_Lu_2007_YPD.txt",61:"4932-Yeast_ORF_intensity_Denervaud_2013.txt",40:"4932-deGodoy_et_al_yeast_data.txt",128:"4932-peptidesatalas_build_April_2009_yeast.new.txt",184:"4932-yeast_Nagaraj_2012.txt",420:"5061-GPM_201408.txt",179:"511145-E_coli_Wisniewski_2014_cpc.txt",226:"511145-Ecoli_APEX_arike_2012.txt",236:"511145-Ecoli_Krug_2013.txt",43:"511145-Ecoli_Krug_2013_emPAI.txt",474:"511145-Ecoli_emPAI_arike_2012.txt",306:"511145-Ecoli_iBAQ_arike_2012.txt",164:"511145-Ecoli_strainK12_PRIDE.txt",432:"511145-Escherichia_coli_SC_biomart_12189_E__2reps.txt",86:"511145-Escherichia_coli_SC_biomart_17112_E__2reps.txt",146:"511145-Escherichia_coli_SC_biomart_18119_E__5reps.txt",379:"511145-Escherichia_coli_SC_biomart_29780_E_BTO0000214cellculture_2reps.txt",486:"511145-GPM_201408.txt",60:"511145-Lu_ecoli_2007.txt",351:"511145-Taniguchi_2010_Table_S6.txt",137:"511145-WHOLE_ORGANISM-integrated.txt",301:"511145-WT-lewis-peptide_count.txt",101:"511145-ecoli1_HCD_CID_PRIDE.txt",85:"511145-ecoli2_resolutionCombination_PRIDE.txt",13:"546414-DeinococcusDeserti_PRIDE.txt",198:"5691-GPM_201408.txt",50:"5833-Dd2_Tao_2014.txt",388:"5833-GPM_201408.txt",311:"5833-GiRBC_Tao_2014.txt",246:"5833-PA_201209.txt",131:"5833-WHOLE_ORGANISM-integrated.txt",297:"593117-ThermococcusGammatolerans_strain_PRIDE.txt",49:"6239-Celegans_Merrihew2008_GenomeRes_peptide.txt",425:"6239-GPM_2012_09_Caenorhabditis_elegans.txt",366:"6239-GPM_201408.txt",58:"6239-PA_201309.txt",204:"6239-Spectral_counting_C.elegans_PeptideAtals_Build_Aug_2011.txt",380:"6239-WHOLE_ORGANISM-integrated.txt",114:"6239-celegans_atalas_build_247.new.txt",181:"6239-new_celegans_Dauer_peptides.new.txt",282:"6239-new_celegans_L1_peptides.new.txt",22:"6239-new_celegans_embryo_peptides.new.txt",478:"64091-GPM_201408.txt",271:"64091-HalobacteriumBuild_Mar2008.peptides.new.txt",438:"64091-PA_201308.txt",138:"64091-WHOLE_ORGANISM-integrated.txt",369:"7165-GPM_201408.txt",201:"722438-Mycoplasma_pneumoniae_M129_Kuhner_et_al_Science2009.txt",10:"7227-Drosophila_2011-09_Ens62.txt",390:"7227-Drosophila_melanogaster_SC_biomart_31126_O_BTO0001489wholebody_119reps.txt",53:"7227-GPM_201408.txt",1:"7227-PA_201208.txt",220:"7227-WHOLE_ORGANISM-integrated.txt",252:"7227-drosophila.peptides_Brunner_2007.txt",296:"7227-drosophila_peptideatlas_July_2009.new.txt",105:"7227-drosophila_peptideatlas_June_2010.new.txt",384:"7227-fruitFly1_normalEmbryo_PRIDE.txt",20:"7227-fruitFly2_earlyEmbryoCell_PRIDE.txt",265:"7460-PA_201309.txt",104:"7460-WHOLE_ORGANISM-integrated.txt",51:"7460-honeybee_atlas_build_282_new.txt",264:"7955-EMBRYO-integrated.txt",151:"7955-GPM_2012_09_Danio_rerio.txt",165:"7955-GPM_201408.txt",362:"7955-PA_201403.txt",240:"7955-WHOLE_ORGANISM-integrated.txt",121:"7955-ZF_Brain_Kelkar_2014_SEQUEST.txt",38:"7955-ZF_Embryo120hr_Kelkar_2014_SEQUEST.txt",397:"7955-ZF_Embryo48hrs_Kelkar_2014_SEQUEST.txt",281:"7955-ZF_Eye_Kelkar_2014_SEQUEST.txt",469:"7955-ZF_GrossBody_Kelkar_2014_SEQUEST.txt",145:"7955-ZF_Head_Kelkar_2014_SEQUEST.txt",288:"7955-ZF_Heart_Kelkar_2014_SEQUEST.txt",295:"7955-ZF_IntPanc_Kelkar_2014_SEQUEST.txt",308:"7955-ZF_Kidney_Kelkar_2014_SEQUEST.txt",332:"7955-ZF_Liver_Kelkar_2014_SEQUEST.txt",290:"7955-ZF_Muscle_Kelkar_2014_SEQUEST.txt",54:"7955-ZF_Ovary_Kelkar_2014_SEQUEST.txt",445:"7955-ZF_Spleen_Kelkar_2014_SEQUEST.txt",100:"7955-ZF_Testis_Kelkar_2014_SEQUEST.txt",255:"7955-zf_velos_Kelkar_2014_SEQUEST.txt",62:"83332-GPM_201408.txt",147:"83332-M.tuberculosis_atlas_build_330.txt",157:"83332-Mtuberculosis_Albrethsen_2013.txt",133:"83332-My_PXD000483_Cortes_2013.txt",392:"83332-MycobacteriumTuberculosis.txt",156:"83332-PA_201307.txt",253:"83332-WHOLE_ORGANISM-integrated.txt",92:"8364-GPM_2012_09_Xenopus_tropicalis.txt",207:"85962-GPM_201408.txt",107:"85962-Helicobacter_pylori_SC_biomart_27551_E__5reps.txt",134:"85962-WHOLE_ORGANISM-integrated.txt",97:"882-Desulfo_Form_Exp_SC_zhang_2006.txt",263:"882-Desulfo_Form_Stat_SC_zhang_2006.txt",23:"882-Desulfo_Lac_Exp_SC_zhang_2006.txt",160:"882-Desulfo_Lac_Stat_SC_zhang_2006.txt",32:"882-WHOLE_ORGANISM-integrated.txt",234:"9031-GPM_2012_09_Gallus_gallus.txt",461:"9031-GPM_201408.txt",274:"9031-WHOLE_ORGANISM-integrated.txt",241:"9598-Chimp_iBAQ_Khan_2013.txt",99:"9606-A431_cellline_Lundberg_2010.txt",172:"9606-A549_Geiger_2012.txt",299:"9606-Adult_Adrenalgland_Kim_2014_SEQUEST.txt",293:"9606-Adult_Bcells_Kim_2014_SEQUEST.txt",71:"9606-Adult_CD4Tcells_Kim_2014_SEQUEST.txt",350:"9606-Adult_CD8Tcells_Kim_2014_SEQUEST.txt",96:"9606-Adult_Colon_Kim_2014_SEQUEST.txt",108:"9606-Adult_Esophagus_Kim_2014_SEQUEST.txt",480:"9606-Adult_Frontalcortex_Kim_2014_SEQUEST.txt",45:"9606-Adult_Gallbladder_Kim_2014_SEQUEST.txt",44:"9606-Adult_Heart_Kim_2014_SEQUEST.txt",170:"9606-Adult_Kidney_Kim_2014_SEQUEST.txt",212:"9606-Adult_Liver_Kim_2014_SEQUEST.txt",267:"9606-Adult_Lung_Kim_2014_SEQUEST.txt",4:"9606-Adult_Monocytes_Kim_2014_SEQUEST.txt",466:"9606-Adult_NKcells_Kim_2014_SEQUEST.txt",434:"9606-Adult_Ovary_Kim_2014_SEQUEST.txt",123:"9606-Adult_Pancreas_Kim_2014_SEQUEST.txt",383:"9606-Adult_Platelets_Kim_2014_SEQUEST.txt",386:"9606-Adult_Prostate_Kim_2014_SEQUEST.txt",435:"9606-Adult_Rectum_Kim_2014_SEQUEST.txt",363:"9606-Adult_Retina_Kim_2014_SEQUEST.txt",229:"9606-Adult_Spinalcord_Kim_2014_SEQUEST.txt",8:"9606-Adult_Testis_Kim_2014_SEQUEST.txt",17:"9606-Adult_Urinarybladder_Kim_2014_SEQUEST.txt",180:"9606-BRAIN-integrated.txt",260:"9606-CELL_LINE-integrated.txt",463:"9606-COLON-integrated.txt",407:"9606-Cellline_U2OS_Beck_2011.txt",357:"9606-ESOPHAGUS-integrated.txt",141:"9606-ES_(H1)_iTRAQ-117_Phanstiel_2011_gene.txt",117:"9606-ES_(H14)_iTRAQ-119_Phanstiel_2011_gene.txt",95:"9606-ES_(H7)_iTRAQ-115_Phanstiel_2011_gene.txt",126:"9606-ES_(H9)_iTRAQ-113_Phanstiel_2011_gene.txt",235:"9606-FEMALE_GONAD-integrated.txt",200:"9606-Fetal_Brain_Kim_2014_SEQUEST.txt",371:"9606-Fetal_Gut_Kim_2014_SEQUEST.txt",431:"9606-Fetal_Heart_Kim_2014_SEQUEST.txt",27:"9606-Fetal_Liver_Kim_2014_SEQUEST.txt",404:"9606-Fetal_Ovary_Kim_2014_SEQUEST.txt",314:"9606-Fetal_Placenta_Kim_2014_SEQUEST.txt",318:"9606-Fetal_Testis_Kim_2014_SEQUEST.txt",276:"9606-GALL_BLADDER-integrated.txt",140:"9606-GAMG_Geiger_2012.txt",197:"9606-GPM_2012_09_Homo_sapiens_-_ensembl.txt",162:"9606-GPM_201408.txt",150:"9606-HCT116_Koch_2014_emPAI.txt",143:"9606-HEART-integrated.txt",485:"9606-HEK293_Geiger_2012.txt",249:"9606-HeLa_Geiger_2012.txt",315:"9606-HepG2_Geiger_2012.txt",317:"9606-HsapiensBuild242_May2010.peptides.new.txt",381:"9606-Huh-7_Pando-Robles_2014.txt",330:"9606-Human_Chinese_liver_dblep.txt",118:"9606-Human_French_liver_dblep.txt",15:"9606-Human_Heart_Kline2008_JProteomeRes_peptide.txt",453:"9606-Human_PXD000396_hela_Kocher_2014.txt",66:"9606-Human_PXD000439_Balbin_2013_A460.txt",199:"9606-Human_PXD000439_Balbin_2013_A549.txt",289:"9606-Human_PXD000439_Balbin_2013_H1155.txt",391:"9606-Human_PXD000439_Balbin_2013_H1734.txt",287:"9606-Human_PXD000439_Balbin_2013_H1792.txt",31:"9606-Human_PXD000439_Balbin_2013_H2009.txt",448:"9606-Human_PXD000439_Balbin_2013_H2122.txt",316:"9606-Human_PXD000439_Balbin_2013_H23.txt",416:"9606-Human_PXD000439_Balbin_2013_H358.txt",344:"9606-Human_PXD000439_Balbin_2013_H441.txt",245:"9606-Human_PXD000439_Balbin_2013_H727.txt",227:"9606-Human_PXD000439_Balbin_2013_SKLU1.txt",193:"9606-Human_PXD000957_vitreous_humor_Murthy_2014_MASCOT.txt",155:"9606-Human_PeptideAtlas_2011-08_Ens62.new.txt",24:"9606-Human_fetal_liver_dblep..txt",55:"9606-Human_heart_ThinThinAye_MolecularBioSystems2010_APEX.txt",356:"9606-Human_heart_ThinThinAye_MolecularBioSystems2010_combine_MethodABC_I.txt",206:"9606-Human_iBAQ_Khan_2013.txt",87:"9606-Human_liver_Chinese_2010_J._Proteome_Res.txt",393:"9606-Jurkat_Geiger_2012.txt",214:"9606-K562_Geiger_2012.txt",336:"9606-KIDNEY-integrated.txt",11:"9606-LIVER-integrated.txt",158:"9606-LUNG-integrated.txt",183:"9606-LnCap_Geiger_2012.txt",382:"9606-MCF7_Geiger_2012.txt",361:"9606-PANCREAS-integrated.txt",389:"9606-PA_201408.txt",377:"9606-PA_brain_201308.txt",83:"9606-PA_heart_201408.txt",139:"9606-PA_kidney_201308.txt",325:"9606-PA_liver_201308.txt",210:"9606-PA_plasma.txt",247:"9606-PA_urine_201308.txt",310:"9606-PLACENTA-integrated.txt",171:"9606-PLASMA-integrated.txt",94:"9606-PLATELET-integrated.txt",68:"9606-PROSTATE_GLAND-integrated.txt",479:"9606-Panc_cellline_Kim_2014.txt",419:"9606-RECTUM-integrated.txt",334:"9606-RKO_Geiger_2012.txt",142:"9606-SALIVA-integrated.txt",367:"9606-SKIN-integrated.txt",333:"9606-TESTIS-integrated.txt",221:"9606-U251MG_cellline_Lundberg_2010.txt",353:"9606-U2OS_Geiger_2012.txt",81:"9606-U2OS_cellline_Lundberg_2010.txt",224:"9606-URINE-integrated.txt",477:"9606-UTERUS-integrated.txt",29:"9606-WHOLE_ORGANISM-integrated.txt",64:"9606-ascites_Wilhelm_2014_Maxquant.txt",189:"9606-cardia_Wilhelm_2014_Maxquant.txt",217:"9606-cerebral_cortex_Wilhelm_2014_Maxquant.txt",77:"9606-cervix_uteri_Wilhelm_2014_Maxquant.txt",467:"9606-colon_Wilhelm_2014_Maxquant.txt",216:"9606-earwax_Wilhelm_2014_Maxquant.txt",472:"9606-esophagus_Wilhelm_2014_Maxquant.txt",340:"9606-gall_bladder_Wilhelm_2014_Maxquant.txt",208:"9606-hair_follicle_Chymotrypsin_Wilhelm_2014_Maxquant.txt",41:"9606-hela_Nagaraj_2011.txt",327:"9606-hela_Nagaraj_2011_cpc.txt",455:"9606-hela_Nagaraj_2011_iBAQ.txt",430:"9606-human3_non-alkylatedPlasma_PRIDE.txt",185:"9606-human4_normal_PRIDE.txt",56:"9606-human5_salivaCollected_PRIDE.txt",90:"9606-human_brain_Martens_2006_proteomics.txt",471:"9606-human_brain_atlas_201106.txt",468:"9606-human_lungPeptides_Abdul-Salam_2010_Circulation_PeptideIonIntensityValues.txt",338:"9606-human_lungProteins_Abdul-Salam_2010_Circulation_PeptideIonIntensityValues.txt",492:"9606-human_plasma_atlas_201005.txt",433:"9606-human_platelet_Martens_2005_Proteomics.txt",57:"9606-human_urine_atalas_201006.txt",292:"9606-iBAQ_A549_Geiger_2012_uniprot.txt",337:"9606-iBAQ_GAMG_Geiger_2012_uniprot.txt",329:"9606-iBAQ_HEK293_Geiger_2012_uniprot.txt",182:"9606-iBAQ_HeLa_Geiger_2012_uniprot.txt",257:"9606-iBAQ_HepG2_Geiger_2012_uniprot.txt",175:"9606-iBAQ_Jurkat_Geiger_2012_uniprot.txt",294:"9606-iBAQ_K562_Geiger_2012_uniprot.txt",451:"9606-iBAQ_LnCap_Geiger_2012_uniprot.txt",441:"9606-iBAQ_MCF7_Geiger_2012_uniprot.txt",395:"9606-iBAQ_RKO_Geiger_2012_uniprot.txt",359:"9606-iBAQ_U2OS_Geiger_2012_uniprot.txt",232:"9606-iPS_(DF19.11)_iTRAQ-114_Phanstiel_2011_gene.txt",225:"9606-iPS_(DF19.7)_iTRAQ-116_Phanstiel_2011_gene.txt",418:"9606-iPS_(DF4.7)_iTRAQ-121_Phanstiel_2011_gene.txt",398:"9606-iPS_(DF6.9)_iTRAQ-118_Phanstiel_2011_gene.txt",376:"9606-kidney_Wilhelm_2014_Maxquant.txt",470:"9606-liver_Wilhelm_2014_Maxquant.txt",168:"9606-lung_Wilhelm_2014_Maxquant.txt",119:"9606-lymph_node_Wilhelm_2014_Maxquant.txt",335:"9606-milk_Wilhelm_2014_Maxquant.txt",406:"9606-nasopharynx_Wilhelm_2014_Maxquant.txt",205:"9606-oral_cavity_Wilhelm_2014_Maxquant.txt",275:"9606-ovary_Wilhelm_2014_Maxquant.txt",187:"9606-pancreas_Wilhelm_2014_Maxquant.txt",33:"9606-peptideatlaBuild_March2009.peptides.new.txt",313:"9606-placenta_Wilhelm_2014_Maxquant.txt",443:"9606-prostate_Wilhelm_2014_Maxquant.txt",465:"9606-rectum_Wilhelm_2014_Maxquant.txt",423:"9606-saliva_Wilhelm_2014_Maxquant.txt",39:"9606-salivary_gland_Wilhelm_2014_Maxquant.txt",59:"9606-seminal_vesicle_Wilhelm_2014_Maxquant.txt",112:"9606-skin_Wilhelm_2014_Maxquant.txt",136:"9606-spleen_Wilhelm_2014_Maxquant.txt",457:"9606-stomach_Wilhelm_2014_Maxquant.txt",239:"9606-testis_Wilhelm_2014_Maxquant.txt",491:"9606-thyroid_gland_Wilhelm_2014_Maxquant.txt",269:"9606-tonsil_Wilhelm_2014_Maxquant.txt",394:"9606-tube_Wilhelm_2014_Maxquant.txt",230:"9606-uterus_post-menopause_Wilhelm_2014_Maxquant.txt",98:"9606-uterus_pre-menopause_Wilhelm_2014_Maxquant.txt",303:"9606-vulva_Wilhelm_2014_Maxquant.txt",403:"9615-GPM_2012_09_Canis_familiaris.txt",120:"9796-PA_201304.txt",30:"9823-Pig_PeptideAtlas_2011-08_Ens62.txt",21:"9913-GPM_2012_09_Bos_taurus.txt",231:"9913-GPM_201408.txt",365:"9913-PA_201112.txt",456:"9913-WHOLE_ORGANISM-integrated.txt",202:"9913-cow_atlas_build_320.txt",46:"99287-GPM_201408.txt",211:"99287-SalmonellaTyphimurium_PRIDE.txt",248:"99287-WHOLE_ORGANISM-integrated.txt"},correlationTarget:""}},meta:function(){return{title:"".concat(this.species.compact_name," in PAXdb")}},computed:{correlationPlot:function(){return this.correlationTarget?"http://api.pax-db.org/species/".concat(this.id,"/correlate/").concat(this.correlationTarget):""}},watch:{$route:function(e,a){this.id=e.params.id,this.species=o["a"][e.params.id],this.defaultDataset=Object(s["a"])(o["a"][e.params.id])}},methods:{organismSelectedHandler:function(e){if(e.id)this.correlationTarget=e.id;else if(Object.hasOwnProperty.call(r["a"].SPECIES_NAMES_IDS,e))this.correlationTarget=r["a"].SPECIES_NAMES_IDS[e];else if(Object.hasOwnProperty.call(r["a"].SPECIES_FULL_NAMES,e))this.correlationTarget=e;else{var a="Bug, failed to select correlation target, organism: ".concat(e);this.$q.notify({color:"negative",position:"top",message:a,icon:"report_problem"}),n["a"].rollbar.error(a)}},resetCorrelation:function(){this.correlationTarget="",this.$refs.speciesList.reset()}},mounted:function(){var a=e(this.$refs.dataset_overview_table);a.footable(),a.removeClass("hidden")},components:{"organism-selectbox":i["a"]}}}).call(this,t("9537"))},"9c5b":function(e,a,t){"use strict";(function(e){t("f751"),t("7514"),t("48d5");var n=t("16b0"),i={save_state:!1,wrapper_class:"drilldown panel panel-success",menu_class:"drilldown-menu",submenu_class:"nav ",parent_class:"dd-parent",parent_class_link:"dd-parent-a",active_class:"active",header_class_list:"breadcrumb",header_class:"breadcrumbwrapper",class_selected:"selected",event_type:"click",hover_delay:300,speed:"fast",show_count:!1,count_class:"dd-count",icon_class:"fa fa-chevron-right pull-right dd-icon",link_type:"breadcrumb",reset_text:"All",default_text:"Select taxa",show_end_nodes:!0,hide_empty:!1,menu_height:!1,show_header:!1,header_tag:"div",header_tag_class:"list-group-item active"};a["a"]={name:"TreeOfLifeMenu",data:function(){return{initialized:!1,selectedSpecies:n["a"].CLADOGRAM.Vertebrates}},mounted:function(){var a=e(this.$refs.drilldown),t=a.find('a:contains("All Vertebrates (11)")');if(e(t).addClass("selected"),this.initialized){var n=Object.assign({justRehydrate:!0},i);a.drilldown(n)}else a.drilldown(i),this.initialized=!0;this.$emit("selectedSpecies",this.selectedSpecies)},methods:{onMenuItemSelected:function(e,a){var t=e.target.text;if(0===t.indexOf("all organisms"))this.selectedSpecies=n["a"].CLADOGRAM["cellular organisms"];else{var i=/(All )?(\w+)\s.*/.exec(t);if(null==i||!Object.hasOwnProperty.call(n["a"].CLADOGRAM,i[2]))return!1;this.selectedSpecies=n["a"].CLADOGRAM[i[2]]}return this.$emit("selectedSpecies",this.selectedSpecies),!1}}}}).call(this,t("9537"))},"9d33":function(e,a,t){},"9e35":function(e,a,t){"use strict";var n=t("2377"),i=t.n(n);i.a},"9f7e":function(e,a,t){"use strict";(function(e){t("28a5");var n=t("f3e3"),i=t.n(n),o=(t("7f7f"),t("2b0e")),r=t("14b0"),s=t("3e0f"),c=t("16b0"),l=t("0196");t("2c5f"),t("6ffb"),t("6ea2"),t("ad7f"),t("baee");a["a"]={name:"ProteinPage",data:function(){return{id:this.$route.params.id,isFetching:!0,isLoadingFamily:!0,protein:"",species:"",dataset:"",stringdbProteinUrl:"",proteinNetworkImage:"",uniprotLink:[]}},created:function(){return this.loadData()},meta:function(){return{title:"".concat(this.protein.name," protein abundance - PAXdb")}},watch:{$route:function(e,a){this.id=this.$route.params.id},id:function(){this.loadData()}},mounted:function(){if(console.log("%s mounted %s",this.id,new Date),!this.isFetching){var a=e(this.$refs.abundance_table);a.footable(),a.removeClass("hidden")}},updated:function(){if(this.$refs.abundance_table){var a=e(this.$refs.abundance_table);a.footable(),a.removeClass("hidden")}},methods:{loadData:function(){var e=this;console.log("%s loading data %s",this.id,new Date);var a=this,t=function(t){console.log("%s data loaded at %s",a.id,new Date),a.protein=t.data;var n=a.protein.externalId.split("."),o=i()(n,1);a.speciesId=o[0],a.species=s["a"][e.speciesId],a.dataset=Object(l["a"])(e.species);for(var r=!1,d=a.protein.abundances,u=0;u<d.length;u+=1)if("NA"!==d[u].formattedAbundance){r=!0;break}if(r||delete a.protein.abundances,e.uniprotLink=[],Object.prototype.hasOwnProperty.call(a.protein,"uniprotId"))a.uniprotLink.push("http://www.uniprot.org/uniprot/".concat(a.protein.uniprotId));else{var b=a.protein.externalId.substring(a.protein.externalId.indexOf(".")+1),p=c["a"].SPECIES_FULL_NAMES[a.speciesId],m="http://www.uniprot.org/uniprot/?sort=score";m="".concat(m,'&query=organism:"').concat(p,'"+AND+').concat(b),a.uniprotLink.push(m)}var _=encodeURIComponent("https://pax-db.org/payload/".concat(a.speciesId,"-payload-v18.json")),g="&amp;external_payload=".concat(_);a.stringdbProteinUrl="https://version-10-5.string-db.org/newstring_cgi/show_network_section.pl?caller_identity=paxdb-v4.0&amp;all_channels_on=1&amp;targetmode=proteins&amp;network_flavor=evidence"+g+"&amp;identifier=".concat(a.protein.externalId),a.proteinNetworkImage="https://version-10-5.string-db.org/api/image/network?identifier=".concat(a.protein.externalId)+g+"&amp;network_flavor=evidence",a.isFetching=!1};return this.$axios.get("http://api.pax-db.org/protein/".concat(this.id)).then(t).catch(function(e){a.$q.notify({color:"negative",position:"top",message:"Loading failed",icon:"report_problem"}),o["a"].rollbar.error(e)})}},components:{ProteinFamily:r["a"]}}}).call(this,t("9537"))},a075:function(e,a,t){"use strict";(function(e){t("7f7f"),t("7514");var n=t("d457"),i=t("3e0f"),o=t("16b0");a["a"]={name:"DatasetPage",data:function(){var e=parseInt(this.$route.params.datasetId,10),a=this.$route.params.speciesId,t=i["a"][a],n=t.datasets.find(function(a){return a.id==e});return{id:e,species:t,dataset:n,datasetFilenames:{278:"10090-Adrenal_gland_geiger_2013.txt",154:"10090-BRAIN-integrated.txt",343:"10090-BROWN_ADIPOSE_TISSUE-integrated.txt",291:"10090-Brain_cortex_geiger_2013.txt",409:"10090-Brain_medulla_geiger_2013.txt",238:"10090-Brown_fat_geiger_2013.txt",251:"10090-CELL_LINE-integrated.txt",218:"10090-Cerebellum_geiger_2013.txt",378:"10090-Colon_geiger_2013.txt",439:"10090-Diaphragm_geiger_2013.txt",6:"10090-Duodenum_geiger_2013.txt",78:"10090-Embryonic_tissue_geiger_2013.txt",115:"10090-Eye_geiger_2013.txt",149:"10090-GPM_2012_09_Mus_musculus.txt",481:"10090-GPM_201408.txt",73:"10090-HEART-integrated.txt",72:"10090-Heart_geiger_2013.txt",222:"10090-Ileum_geiger_2013.txt",9:"10090-Jejunum_geiger_2013.txt",440:"10090-KIDNEY-integrated.txt",427:"10090-Kidney_cortex_geiger_2013.txt",132:"10090-Kidney_medulla_geiger_2013.txt",254:"10090-LIVER-integrated.txt",74:"10090-LUNG-integrated.txt",488:"10090-Liver_geiger_2013.txt",339:"10090-Lung_geiger_2013.txt",421:"10090-Menschaert_protID_NSAF.txt",161:"10090-Menschaert_protID_emPAI.txt",400:"10090-Midbrain_geiger_2013.txt",346:"10090-MouseBuild243_Jun2010.peptides.new.txt",26:"10090-Mouse_PeptideAtlas_2011-06_Ens62.txt",148:"10090-Mouse_brain_Huttlin_cell2010.csv.txt",18:"10090-Mouse_brownfat_Huttlin_cell2010.csv.txt",459:"10090-Mouse_erythrocytes_Kruger2008_cell.txt",76:"10090-Mouse_gut_epithelium_Kruger2008_cell.txt",402:"10090-Mouse_heart_Huttlin_cell2010.csv.txt",268:"10090-Mouse_heart_Kruger2008_cell.txt",482:"10090-Mouse_kidney_Huttlin_cell2010.csv.txt",125:"10090-Mouse_liver_Huttlin_cell2010.csv.txt",446:"10090-Mouse_liver_Kruger2008_cell.txt",375:"10090-Mouse_liver_Wang2009_Proteomics_ClinApp.txt",67:"10090-Mouse_lung_Huttlin_cell2010.csv.txt",405:"10090-Mouse_pancreas_Huttlin_cell2010.csv.txt",277:"10090-Mouse_platelets_Kruger2008_cell.txt",452:"10090-Mouse_spleen_B_cells_Kruger2008_cell.txt",259:"10090-Mouse_spleen_Huttlin_cell2010.csv.txt",411:"10090-Mouse_testis_Huttlin_cell2010_peptide.csv.txt",436:"10090-Mus_musculus_SC_biomart_11364_E__2reps.txt",424:"10090-Mus_musculus_SC_biomart_28162_O_BTO0005136ESE14cell_1reps.txt",69:"10090-Muscle_geiger_2013.txt",345:"10090-Olfactory_bulb_geiger_2013.txt",328:"10090-Ovary_geiger_2013.txt",372:"10090-PANCREAS-integrated.txt",266:"10090-PA_plasma.txt",319:"10090-Pancreas_geiger_2013.txt",79:"10090-SPLEEN-integrated.txt",102:"10090-Salivary_gland_geiger_2013.txt",364:"10090-Spleeen_geiger_2013.txt",80:"10090-Stomach_geiger_2013.txt",373:"10090-Thymus_geiger_2013.txt",320:"10090-Uterus_geiger_2013.txt",196:"10090-WHOLE_ORGANISM-integrated.txt",219:"10090-White_fat_geiger_2013.txt",305:"10090-mouse_brain_Martens_2006_proteomics.txt",490:"10090-mouse_brain_Wang_2007_J_proteome_Res.txt",489:"10090-mouse_brain_kislinger2006_cell.txt",273:"10090-mouse_glomeruli_kidney_F.Waanders2009_PNAs.txt",309:"10090-mouse_haploidGermCells_Guo_2010_JProteome.txt",109:"10090-mouse_heart_kislinger2006_cell.txt",144:"10090-mouse_kidney_kislinger2006_cell.txt",417:"10090-mouse_liver_Shi2007_JourmalofProteome.txt",34:"10090-mouse_liver_kislinger2006_cell.txt",177:"10090-mouse_lung_kislinger2006_cell.txt",228:"10090-mouse_pancreas_F.Waanders2009_PNAs_combine.txt",262:"10090-mouse_placenta_kislinger2006_cell.txt",476:"10116-GPM_2012_09_Rattus_norvegicus.txt",408:"10116-GPM_201408.txt",462:"10116-PA_201311.txt",464:"10116-Rattus_norvegicus_SC_biomart_19302_E__6reps.txt",93:"10116-WHOLE_ORGANISM-integrated.txt",410:"1148-SynechocystisSp_strain_PRIDE.txt",176:"122586-GPM_201408.txt",70:"158878-GPM_201408.txt",415:"160490-GPM_201408.txt",360:"160490-PA_200702.txt",307:"160490-StreptococcusPyogenes_M1GAS_PRIDE.txt",159:"160490-WHOLE_ORGANISM-integrated.txt",280:"160490-atlas_build_101.txt",396:"169963-GPM_201408.txt",127:"192222-Campylobacter_jejuni_SC_biomart_17684_E__1reps.txt",19:"192222-Campylobacter_jejuni_SC_biomart_17684_O__1reps.txt",484:"192222-WHOLE_ORGANISM-integrated.txt",458:"198214-ShigellaFlexneri_2a_PRIDE.txt",88:"208964-GPM_201408.txt",103:"208964-Pseudomonas_aeruginosa_SC_biomart_22024_E__1reps.txt",130:"208964-WHOLE_ORGANISM-integrated.txt",166:"211586-GPM_201408.txt",454:"214092-Yersinia_pestis_SC_biomart_18099_O__3reps.txt",352:"214684-Cryptococcus_biofilm_Santi_2014.txt",111:"214684-Cryptococcus_planktonic_Santi_2014.txt",37:"214684-WHOLE_ORGANISM-integrated.txt",312:"224308-BacillusSubtilisSubsp_PRIDE.txt",493:"224308-GPM_201408.txt",256:"224308-WHOLE_ORGANISM-integrated.txt",347:"226186-GPM_201408.txt",422:"243159-Acidithiobac_PSM_biofilm_Vera_2013.txt",223:"243159-Acidithiobac_PSM_planktonic_Vera_2013.txt",113:"243159-WHOLE_ORGANISM-integrated.txt",272:"260799-GPM_201408.txt",460:"267671-Leptospira_build_121.peptides.cut.txt",195:"267671-PA_201308.txt",84:"267671-WHOLE_ORGANISM-integrated.txt",349:"267671-controlMS1_Leptospira_Malmstroem_2009.txt",354:"267671-controlSpectral_Leptospira_Malmstroem_2009.txt",35:"272623-GPM_201408.txt",348:"272623-Lactococcus_lactis_Lahtvee_2014.txt",194:"272623-WHOLE_ORGANISM-integrated.txt",75:"272624-GPM_201408.txt",437:"283166-Bhenselae_Albrethsen_2013.txt",47:"3055-GPM_201408.txt",52:"353153-LFQ_Intensity_Ade_deGodoy_2012.txt",370:"353153-LFQ_Intensity_Epi_deGodoy_2012.txt",215:"353153-LFQ_Intensity_Meta_deGodoy_2012.txt",190:"353153-LFQ_Intensity_Stress2h_deGodoy_2012.txt",5:"353153-LFQ_Intensity_Stress30_deGodoy_2012.txt",129:"353153-WHOLE_ORGANISM-integrated.txt",298:"3702-2.1ArabidopsisLeaf_waterDefict_SOW.txt",401:"3702-2.2ArabidopsisLeaf_waterDefict_SWD.txt",284:"3702-6.1JasmonateControls_controlLeaves.txt",233:"3702-6.2JasmonateControls_woundedLeaves.txt",270:"3702-7.1Large-scalePhosphoproteome_rosette_ED.txt",385:"3702-7.2Large-scalePhosphoproteome_rosette_EN.txt",2:"3702-7.3Large-scalePhosphoproteome_shoot.txt",163:"3702-Arabidopsis_capel_Baerenfaller_2011_IntegrBiol.txt",209:"3702-Arabidopsis_cotyledon_Baerenfaller_2011_IntegrBiol.txt",279:"3702-Arabidopsis_flower_Baerenfaller_2011_IntegrBiol.txt",413:"3702-Arabidopsis_flowerbud_Baerenfaller_2011_IntegrBiol.txt",283:"3702-Arabidopsis_juvenileLeaf_Baerenfaller_2011_IntegrBiol.txt",124:"3702-Arabidopsis_pollen_Grobei_2009_GenomeRes.txt",135:"3702-Arabidopsis_root_Baerenfaller_2011_IntegrBiol.txt",174:"3702-Arabidopsis_seed_Baerenfaller_2011_IntegrBiol.txt",213:"3702-Arabidopsis_silique_Baerenfaller_2011_IntegrBiol.txt",322:"3702-Arabidopsis_thaliana_SC_biomart_27037_E__1reps.txt",7:"3702-CARPEL-integrated.txt",341:"3702-COTYLEDON-integrated.txt",355:"3702-FLOWER-integrated.txt",374:"3702-FLOWERBUD-integrated.txt",14:"3702-GPM_201408.txt",447:"3702-JUVENILE_LEAF-integrated.txt",342:"3702-LEAF-integrated.txt",178:"3702-Motohashi_2012_avgWT_Prot_nSPC.txt",36:"3702-POLLEN-integrated.txt",444:"3702-ROOT-integrated.txt",258:"3702-ROSETTE-integrated.txt",304:"3702-SEED-integrated.txt",82:"3702-SHOOT-integrated.txt",324:"3702-SILIQUE-integrated.txt",323:"3702-WHOLE_ORGANISM-integrated.txt",122:"3702-arabidopsis1a_carpel_Baerenfaller_PRIDE.txt",152:"3702-arabidopsis1c_cotyledon_Baerenfaller_PRIDE.txt",91:"3702-arabidopsis1d_flower_Baerenfaller_PRIDE.txt",244:"3702-arabidopsis1e_flowerBud_Baerenfaller_PRIDE.txt",42:"3702-arabidopsis1f_juvenileLeaf_Baerenfaller_PRIDE.txt",173:"3702-arabidopsis1g_leaf_Baerenfaller_PRIDE.txt",191:"3702-arabidopsis1h_pollen_Baerenfaller_PRIDE.txt",153:"3702-arabidopsis1i_root_Baerenfaller_PRIDE.txt",302:"3702-arabidopsis1k_seed_Baerenfaller_PRIDE.txt",28:"3702-arabidopsis1l_shoot_Baerenfaller_PRIDE.txt",414:"3702-arabidopsis1m_silique_Baerenfaller_PRIDE.txt",89:"3702-arabidopsis2_estimationLeaf_PRIDE.txt",116:"3702-countCastellana.txt",412:"3702-peptide.Baerenfaller_2008_new.txt",428:"39947-GPM_2012_09_Oryza_sativa.txt",192:"4081-Solanum_lycopersicum_SC_biomart_18197_E__1reps.txt",321:"44689-GPM_201408.txt",169:"449447-MicrocystisAeruginosa_strainNIES-843_HUB5-3_PRIDE.txt",167:"449447-MicrocystisAeruginosa_strainNIES-843_PCC7005_PRIDE.txt",63:"449447-MicrocystisAeruginosa_strainNIES-843_PCC7806_PRIDE.txt",326:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCCBS_PRIDE.txt",237:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCMRC_PRIDE.txt",368:"449447-MicrocystisAeruginosa_strainNIES-843_UWOCCMRD_PRIDE.txt",188:"449447-WHOLE_ORGANISM-integrated.txt",65:"4577-LEAF-integrated.txt",426:"4577-Mays_SC_mature_rep1234_Facette_2013.txt",261:"4577-Mays_SC_zone1_rep1234_Facette_2013.txt",186:"4577-Mays_SC_zone2_rep1234_Facette_2013.txt",449:"4577-Mays_SC_zone3_rep1234_Facette_2013.txt",487:"4577-Mays_avgNSAF_ponnala_2014.txt",475:"4577-Mays_avgNadjSPC_ponnala_2014.txt",12:"4896-GPM_201408.txt",48:"4896-SchizosaccharomycesPombe_FissionYeast_PRIDE.txt",473:"4896-Schizosaccharomyces_pombe_SC_biomart_29627_E_BTO0000214cellculture_1reps.txt",286:"4896-Spombe_emPAI_Gunaratne_2013.txt",250:"4896-WHOLE_ORGANISM-integrated.txt",442:"4896-pombe_cell2012_protein.txt",387:"4896-proliferating_protID_copynr_Marguerat_2012.txt",429:"4896-quiescent_protID_copynr_Marguerat_2012.txt",450:"4932-GPM_2012_09_Saccharomyces_cerevisiae.txt",243:"4932-GPM_201408.txt",358:"4932-Ghaemmaghami_et_al_yeast_data.txt",25:"4932-Newman_et_al_yeast_data_SD.txt",300:"4932-Newman_et_al_yeast_data_YEPD.txt",242:"4932-PA_201303.txt",331:"4932-Saccharomyces_cerevisiae_SC_biomart_17916_O__1reps.txt",110:"4932-Saccharomyces_cerevisiae_SC_biomart_18097_O__1reps.txt",3:"4932-WHOLE_ORGANISM-integrated.txt",16:"4932-Yeast_2009-0522.txt",106:"4932-Yeast_GeneName_Intensity_Breker_2103.txt",203:"4932-Yeast_Lu_2007_YMD.txt",285:"4932-Yeast_Lu_2007_YPD.txt",61:"4932-Yeast_ORF_intensity_Denervaud_2013.txt",40:"4932-deGodoy_et_al_yeast_data.txt",128:"4932-peptidesatalas_build_April_2009_yeast.new.txt",184:"4932-yeast_Nagaraj_2012.txt",420:"5061-GPM_201408.txt",179:"511145-E_coli_Wisniewski_2014_cpc.txt",226:"511145-Ecoli_APEX_arike_2012.txt",236:"511145-Ecoli_Krug_2013.txt",43:"511145-Ecoli_Krug_2013_emPAI.txt",474:"511145-Ecoli_emPAI_arike_2012.txt",306:"511145-Ecoli_iBAQ_arike_2012.txt",164:"511145-Ecoli_strainK12_PRIDE.txt",432:"511145-Escherichia_coli_SC_biomart_12189_E__2reps.txt",86:"511145-Escherichia_coli_SC_biomart_17112_E__2reps.txt",146:"511145-Escherichia_coli_SC_biomart_18119_E__5reps.txt",379:"511145-Escherichia_coli_SC_biomart_29780_E_BTO0000214cellculture_2reps.txt",486:"511145-GPM_201408.txt",60:"511145-Lu_ecoli_2007.txt",351:"511145-Taniguchi_2010_Table_S6.txt",137:"511145-WHOLE_ORGANISM-integrated.txt",301:"511145-WT-lewis-peptide_count.txt",101:"511145-ecoli1_HCD_CID_PRIDE.txt",85:"511145-ecoli2_resolutionCombination_PRIDE.txt",13:"546414-DeinococcusDeserti_PRIDE.txt",198:"5691-GPM_201408.txt",50:"5833-Dd2_Tao_2014.txt",388:"5833-GPM_201408.txt",311:"5833-GiRBC_Tao_2014.txt",246:"5833-PA_201209.txt",131:"5833-WHOLE_ORGANISM-integrated.txt",297:"593117-ThermococcusGammatolerans_strain_PRIDE.txt",49:"6239-Celegans_Merrihew2008_GenomeRes_peptide.txt",425:"6239-GPM_2012_09_Caenorhabditis_elegans.txt",366:"6239-GPM_201408.txt",58:"6239-PA_201309.txt",204:"6239-Spectral_counting_C.elegans_PeptideAtals_Build_Aug_2011.txt",380:"6239-WHOLE_ORGANISM-integrated.txt",114:"6239-celegans_atalas_build_247.new.txt",181:"6239-new_celegans_Dauer_peptides.new.txt",282:"6239-new_celegans_L1_peptides.new.txt",22:"6239-new_celegans_embryo_peptides.new.txt",478:"64091-GPM_201408.txt",271:"64091-HalobacteriumBuild_Mar2008.peptides.new.txt",438:"64091-PA_201308.txt",138:"64091-WHOLE_ORGANISM-integrated.txt",369:"7165-GPM_201408.txt",201:"722438-Mycoplasma_pneumoniae_M129_Kuhner_et_al_Science2009.txt",10:"7227-Drosophila_2011-09_Ens62.txt",390:"7227-Drosophila_melanogaster_SC_biomart_31126_O_BTO0001489wholebody_119reps.txt",53:"7227-GPM_201408.txt",1:"7227-PA_201208.txt",220:"7227-WHOLE_ORGANISM-integrated.txt",252:"7227-drosophila.peptides_Brunner_2007.txt",296:"7227-drosophila_peptideatlas_July_2009.new.txt",105:"7227-drosophila_peptideatlas_June_2010.new.txt",384:"7227-fruitFly1_normalEmbryo_PRIDE.txt",20:"7227-fruitFly2_earlyEmbryoCell_PRIDE.txt",265:"7460-PA_201309.txt",104:"7460-WHOLE_ORGANISM-integrated.txt",51:"7460-honeybee_atlas_build_282_new.txt",264:"7955-EMBRYO-integrated.txt",151:"7955-GPM_2012_09_Danio_rerio.txt",165:"7955-GPM_201408.txt",362:"7955-PA_201403.txt",240:"7955-WHOLE_ORGANISM-integrated.txt",121:"7955-ZF_Brain_Kelkar_2014_SEQUEST.txt",38:"7955-ZF_Embryo120hr_Kelkar_2014_SEQUEST.txt",397:"7955-ZF_Embryo48hrs_Kelkar_2014_SEQUEST.txt",281:"7955-ZF_Eye_Kelkar_2014_SEQUEST.txt",469:"7955-ZF_GrossBody_Kelkar_2014_SEQUEST.txt",145:"7955-ZF_Head_Kelkar_2014_SEQUEST.txt",288:"7955-ZF_Heart_Kelkar_2014_SEQUEST.txt",295:"7955-ZF_IntPanc_Kelkar_2014_SEQUEST.txt",308:"7955-ZF_Kidney_Kelkar_2014_SEQUEST.txt",332:"7955-ZF_Liver_Kelkar_2014_SEQUEST.txt",290:"7955-ZF_Muscle_Kelkar_2014_SEQUEST.txt",54:"7955-ZF_Ovary_Kelkar_2014_SEQUEST.txt",445:"7955-ZF_Spleen_Kelkar_2014_SEQUEST.txt",100:"7955-ZF_Testis_Kelkar_2014_SEQUEST.txt",255:"7955-zf_velos_Kelkar_2014_SEQUEST.txt",62:"83332-GPM_201408.txt",147:"83332-M.tuberculosis_atlas_build_330.txt",157:"83332-Mtuberculosis_Albrethsen_2013.txt",133:"83332-My_PXD000483_Cortes_2013.txt",392:"83332-MycobacteriumTuberculosis.txt",156:"83332-PA_201307.txt",253:"83332-WHOLE_ORGANISM-integrated.txt",92:"8364-GPM_2012_09_Xenopus_tropicalis.txt",207:"85962-GPM_201408.txt",107:"85962-Helicobacter_pylori_SC_biomart_27551_E__5reps.txt",134:"85962-WHOLE_ORGANISM-integrated.txt",97:"882-Desulfo_Form_Exp_SC_zhang_2006.txt",263:"882-Desulfo_Form_Stat_SC_zhang_2006.txt",23:"882-Desulfo_Lac_Exp_SC_zhang_2006.txt",160:"882-Desulfo_Lac_Stat_SC_zhang_2006.txt",32:"882-WHOLE_ORGANISM-integrated.txt",234:"9031-GPM_2012_09_Gallus_gallus.txt",461:"9031-GPM_201408.txt",274:"9031-WHOLE_ORGANISM-integrated.txt",241:"9598-Chimp_iBAQ_Khan_2013.txt",99:"9606-A431_cellline_Lundberg_2010.txt",172:"9606-A549_Geiger_2012.txt",299:"9606-Adult_Adrenalgland_Kim_2014_SEQUEST.txt",293:"9606-Adult_Bcells_Kim_2014_SEQUEST.txt",71:"9606-Adult_CD4Tcells_Kim_2014_SEQUEST.txt",350:"9606-Adult_CD8Tcells_Kim_2014_SEQUEST.txt",96:"9606-Adult_Colon_Kim_2014_SEQUEST.txt",108:"9606-Adult_Esophagus_Kim_2014_SEQUEST.txt",480:"9606-Adult_Frontalcortex_Kim_2014_SEQUEST.txt",45:"9606-Adult_Gallbladder_Kim_2014_SEQUEST.txt",44:"9606-Adult_Heart_Kim_2014_SEQUEST.txt",170:"9606-Adult_Kidney_Kim_2014_SEQUEST.txt",212:"9606-Adult_Liver_Kim_2014_SEQUEST.txt",267:"9606-Adult_Lung_Kim_2014_SEQUEST.txt",4:"9606-Adult_Monocytes_Kim_2014_SEQUEST.txt",466:"9606-Adult_NKcells_Kim_2014_SEQUEST.txt",434:"9606-Adult_Ovary_Kim_2014_SEQUEST.txt",123:"9606-Adult_Pancreas_Kim_2014_SEQUEST.txt",383:"9606-Adult_Platelets_Kim_2014_SEQUEST.txt",386:"9606-Adult_Prostate_Kim_2014_SEQUEST.txt",435:"9606-Adult_Rectum_Kim_2014_SEQUEST.txt",363:"9606-Adult_Retina_Kim_2014_SEQUEST.txt",229:"9606-Adult_Spinalcord_Kim_2014_SEQUEST.txt",8:"9606-Adult_Testis_Kim_2014_SEQUEST.txt",17:"9606-Adult_Urinarybladder_Kim_2014_SEQUEST.txt",180:"9606-BRAIN-integrated.txt",260:"9606-CELL_LINE-integrated.txt",463:"9606-COLON-integrated.txt",407:"9606-Cellline_U2OS_Beck_2011.txt",357:"9606-ESOPHAGUS-integrated.txt",141:"9606-ES_(H1)_iTRAQ-117_Phanstiel_2011_gene.txt",117:"9606-ES_(H14)_iTRAQ-119_Phanstiel_2011_gene.txt",95:"9606-ES_(H7)_iTRAQ-115_Phanstiel_2011_gene.txt",126:"9606-ES_(H9)_iTRAQ-113_Phanstiel_2011_gene.txt",235:"9606-FEMALE_GONAD-integrated.txt",200:"9606-Fetal_Brain_Kim_2014_SEQUEST.txt",371:"9606-Fetal_Gut_Kim_2014_SEQUEST.txt",431:"9606-Fetal_Heart_Kim_2014_SEQUEST.txt",27:"9606-Fetal_Liver_Kim_2014_SEQUEST.txt",404:"9606-Fetal_Ovary_Kim_2014_SEQUEST.txt",314:"9606-Fetal_Placenta_Kim_2014_SEQUEST.txt",318:"9606-Fetal_Testis_Kim_2014_SEQUEST.txt",276:"9606-GALL_BLADDER-integrated.txt",140:"9606-GAMG_Geiger_2012.txt",197:"9606-GPM_2012_09_Homo_sapiens_-_ensembl.txt",162:"9606-GPM_201408.txt",150:"9606-HCT116_Koch_2014_emPAI.txt",143:"9606-HEART-integrated.txt",485:"9606-HEK293_Geiger_2012.txt",249:"9606-HeLa_Geiger_2012.txt",315:"9606-HepG2_Geiger_2012.txt",317:"9606-HsapiensBuild242_May2010.peptides.new.txt",381:"9606-Huh-7_Pando-Robles_2014.txt",330:"9606-Human_Chinese_liver_dblep.txt",118:"9606-Human_French_liver_dblep.txt",15:"9606-Human_Heart_Kline2008_JProteomeRes_peptide.txt",453:"9606-Human_PXD000396_hela_Kocher_2014.txt",66:"9606-Human_PXD000439_Balbin_2013_A460.txt",199:"9606-Human_PXD000439_Balbin_2013_A549.txt",289:"9606-Human_PXD000439_Balbin_2013_H1155.txt",391:"9606-Human_PXD000439_Balbin_2013_H1734.txt",287:"9606-Human_PXD000439_Balbin_2013_H1792.txt",31:"9606-Human_PXD000439_Balbin_2013_H2009.txt",448:"9606-Human_PXD000439_Balbin_2013_H2122.txt",316:"9606-Human_PXD000439_Balbin_2013_H23.txt",416:"9606-Human_PXD000439_Balbin_2013_H358.txt",344:"9606-Human_PXD000439_Balbin_2013_H441.txt",245:"9606-Human_PXD000439_Balbin_2013_H727.txt",227:"9606-Human_PXD000439_Balbin_2013_SKLU1.txt",193:"9606-Human_PXD000957_vitreous_humor_Murthy_2014_MASCOT.txt",155:"9606-Human_PeptideAtlas_2011-08_Ens62.new.txt",24:"9606-Human_fetal_liver_dblep..txt",55:"9606-Human_heart_ThinThinAye_MolecularBioSystems2010_APEX.txt",356:"9606-Human_heart_ThinThinAye_MolecularBioSystems2010_combine_MethodABC_I.txt",206:"9606-Human_iBAQ_Khan_2013.txt",87:"9606-Human_liver_Chinese_2010_J._Proteome_Res.txt",393:"9606-Jurkat_Geiger_2012.txt",214:"9606-K562_Geiger_2012.txt",336:"9606-KIDNEY-integrated.txt",11:"9606-LIVER-integrated.txt",158:"9606-LUNG-integrated.txt",183:"9606-LnCap_Geiger_2012.txt",382:"9606-MCF7_Geiger_2012.txt",361:"9606-PANCREAS-integrated.txt",389:"9606-PA_201408.txt",377:"9606-PA_brain_201308.txt",83:"9606-PA_heart_201408.txt",139:"9606-PA_kidney_201308.txt",325:"9606-PA_liver_201308.txt",210:"9606-PA_plasma.txt",247:"9606-PA_urine_201308.txt",310:"9606-PLACENTA-integrated.txt",171:"9606-PLASMA-integrated.txt",94:"9606-PLATELET-integrated.txt",68:"9606-PROSTATE_GLAND-integrated.txt",479:"9606-Panc_cellline_Kim_2014.txt",419:"9606-RECTUM-integrated.txt",334:"9606-RKO_Geiger_2012.txt",142:"9606-SALIVA-integrated.txt",367:"9606-SKIN-integrated.txt",333:"9606-TESTIS-integrated.txt",221:"9606-U251MG_cellline_Lundberg_2010.txt",353:"9606-U2OS_Geiger_2012.txt",81:"9606-U2OS_cellline_Lundberg_2010.txt",224:"9606-URINE-integrated.txt",477:"9606-UTERUS-integrated.txt",29:"9606-WHOLE_ORGANISM-integrated.txt",64:"9606-ascites_Wilhelm_2014_Maxquant.txt",189:"9606-cardia_Wilhelm_2014_Maxquant.txt",217:"9606-cerebral_cortex_Wilhelm_2014_Maxquant.txt",77:"9606-cervix_uteri_Wilhelm_2014_Maxquant.txt",467:"9606-colon_Wilhelm_2014_Maxquant.txt",216:"9606-earwax_Wilhelm_2014_Maxquant.txt",472:"9606-esophagus_Wilhelm_2014_Maxquant.txt",340:"9606-gall_bladder_Wilhelm_2014_Maxquant.txt",208:"9606-hair_follicle_Chymotrypsin_Wilhelm_2014_Maxquant.txt",41:"9606-hela_Nagaraj_2011.txt",327:"9606-hela_Nagaraj_2011_cpc.txt",455:"9606-hela_Nagaraj_2011_iBAQ.txt",430:"9606-human3_non-alkylatedPlasma_PRIDE.txt",185:"9606-human4_normal_PRIDE.txt",56:"9606-human5_salivaCollected_PRIDE.txt",90:"9606-human_brain_Martens_2006_proteomics.txt",471:"9606-human_brain_atlas_201106.txt",468:"9606-human_lungPeptides_Abdul-Salam_2010_Circulation_PeptideIonIntensityValues.txt",338:"9606-human_lungProteins_Abdul-Salam_2010_Circulation_PeptideIonIntensityValues.txt",492:"9606-human_plasma_atlas_201005.txt",433:"9606-human_platelet_Martens_2005_Proteomics.txt",57:"9606-human_urine_atalas_201006.txt",292:"9606-iBAQ_A549_Geiger_2012_uniprot.txt",337:"9606-iBAQ_GAMG_Geiger_2012_uniprot.txt",329:"9606-iBAQ_HEK293_Geiger_2012_uniprot.txt",182:"9606-iBAQ_HeLa_Geiger_2012_uniprot.txt",257:"9606-iBAQ_HepG2_Geiger_2012_uniprot.txt",175:"9606-iBAQ_Jurkat_Geiger_2012_uniprot.txt",294:"9606-iBAQ_K562_Geiger_2012_uniprot.txt",451:"9606-iBAQ_LnCap_Geiger_2012_uniprot.txt",441:"9606-iBAQ_MCF7_Geiger_2012_uniprot.txt",395:"9606-iBAQ_RKO_Geiger_2012_uniprot.txt",359:"9606-iBAQ_U2OS_Geiger_2012_uniprot.txt",232:"9606-iPS_(DF19.11)_iTRAQ-114_Phanstiel_2011_gene.txt",225:"9606-iPS_(DF19.7)_iTRAQ-116_Phanstiel_2011_gene.txt",418:"9606-iPS_(DF4.7)_iTRAQ-121_Phanstiel_2011_gene.txt",398:"9606-iPS_(DF6.9)_iTRAQ-118_Phanstiel_2011_gene.txt",376:"9606-kidney_Wilhelm_2014_Maxquant.txt",470:"9606-liver_Wilhelm_2014_Maxquant.txt",168:"9606-lung_Wilhelm_2014_Maxquant.txt",119:"9606-lymph_node_Wilhelm_2014_Maxquant.txt",335:"9606-milk_Wilhelm_2014_Maxquant.txt",406:"9606-nasopharynx_Wilhelm_2014_Maxquant.txt",205:"9606-oral_cavity_Wilhelm_2014_Maxquant.txt",275:"9606-ovary_Wilhelm_2014_Maxquant.txt",187:"9606-pancreas_Wilhelm_2014_Maxquant.txt",33:"9606-peptideatlaBuild_March2009.peptides.new.txt",313:"9606-placenta_Wilhelm_2014_Maxquant.txt",443:"9606-prostate_Wilhelm_2014_Maxquant.txt",465:"9606-rectum_Wilhelm_2014_Maxquant.txt",423:"9606-saliva_Wilhelm_2014_Maxquant.txt",39:"9606-salivary_gland_Wilhelm_2014_Maxquant.txt",59:"9606-seminal_vesicle_Wilhelm_2014_Maxquant.txt",112:"9606-skin_Wilhelm_2014_Maxquant.txt",136:"9606-spleen_Wilhelm_2014_Maxquant.txt",457:"9606-stomach_Wilhelm_2014_Maxquant.txt",239:"9606-testis_Wilhelm_2014_Maxquant.txt",491:"9606-thyroid_gland_Wilhelm_2014_Maxquant.txt",269:"9606-tonsil_Wilhelm_2014_Maxquant.txt",394:"9606-tube_Wilhelm_2014_Maxquant.txt",230:"9606-uterus_post-menopause_Wilhelm_2014_Maxquant.txt",98:"9606-uterus_pre-menopause_Wilhelm_2014_Maxquant.txt",303:"9606-vulva_Wilhelm_2014_Maxquant.txt",403:"9615-GPM_2012_09_Canis_familiaris.txt",120:"9796-PA_201304.txt",30:"9823-Pig_PeptideAtlas_2011-08_Ens62.txt",21:"9913-GPM_2012_09_Bos_taurus.txt",231:"9913-GPM_201408.txt",365:"9913-PA_201112.txt",456:"9913-WHOLE_ORGANISM-integrated.txt",202:"9913-cow_atlas_build_320.txt",46:"99287-GPM_201408.txt",211:"99287-SalmonellaTyphimurium_PRIDE.txt",248:"99287-WHOLE_ORGANISM-integrated.txt"},correlationTarget:""}},meta:function(){return{title:"".concat(this.dataset.name," - PAXdb")}},computed:{correlationPlot:function(){return this.correlationTarget?"https://pax-db.org/species/".concat(this.id,"/correlate/").concat(this.correlationTarget):""}},watch:{$route:function(e,a){var t=this;this.id=parseInt(e.params.datasetId,10),this.species=i["a"][e.params.speciesId],this.dataset=this.species.datasets.find(function(e){return e.id===t.id})}},mounted:function(){if(!(this.species.datasets.length<2)){var a=e(this.$refs.dataset_overview_table);a.footable(),a.removeClass("hidden")}},methods:{datasetSelectedHandler:function(e){e.id?this.correlationTarget=e.id:this.correlationTarget=o["a"].SPECIES_NAMES_IDS[e.official_name]},exportData:function(e){var a=this,t=a.$refs.abundancesTable.$refs.grid,n=t.getdisplayrows();if(0!==n.length){var i=t.getpaginginformation(),o={json:function(e){return JSON.stringify(e,0)},tsv:function(e){return h(e,"\t")},csv:function(e){return h(e,",")}},r=t.getdatainformation(),s=r.sortinformation,c=r.paginginformation,l="paxdb-abundances-".concat(a.species.id,"_").concat(a.dataset.id);l+="-".concat(s.sortcolumn,"_").concat(s.sortdirection.descending),l+="_".concat(c.pagenum,"_").concat(c.pagesize);for(var d=[],u=i.pagesize*i.pagenum;u<Math.min(n.length,(i.pagenum+1)*i.pagesize);u+=1){var b=n[u];d.push({index:b.index,proteinId:b.id,proteinName:b.name,abundance:b.abundance,annotation:b.annotation})}if(0!==d.length){var p=o[e](d),m=new Blob([p],{type:"".concat(e,";charset=utf-8;")});if(navigator.msSaveBlob)navigator.msSaveBlob(m,"".concat(l,".").concat(e));else{var _=document.createElement("a");if(void 0!==_.download){var g=URL.createObjectURL(m);_.setAttribute("href",g),_.setAttribute("download","".concat(l,".").concat(e)),_.style="visibility:hidden",document.body.appendChild(_),_.click(),document.body.removeChild(_)}}}}function h(a,t){var n=a.map(function(e){return[e.index,e.proteinId,e.proteinName,e.abundance,e.annotation].join(t)}),i="csv"===e?",":"\t";return n.unshift("#position".concat(i,"proteinId").concat(i,"proteinName").concat(i,"abundance").concat(i,"annotation")),n.join("\r\n")}}},components:{DatasetAbundancesTable:n["a"]}}}).call(this,t("9537"))},bdea:function(e,a,t){"use strict";(function(e){t("7f7f");var n=t("2b0e"),i=t("16b0");a["a"]={name:"ProteinsPage",data:function(){return{ids:this.$route.query.ids,isFetching:!0,proteins:"",species:"",speciesName:""}},created:function(){return this.loadData()},meta:function(){return{title:"".concat(this.species.name," proteins - PAXdb")}},watch:{$route:function(e,a){this.ids=this.$route.query.ids},ids:function(){this.loadData()}},mounted:function(){if(!this.isFetching){var a=e(this.$refs.data_table);a.footable(),a.removeClass("hidden")}},updated:function(){if(this.$refs.data_table){var a=e(this.$refs.data_table);a.footable(),a.removeClass("hidden")}},methods:{loadData:function(){var e=this,a=function(a){e.isFetching=!1,e.resource=a.data,e.speciesId=e.resource.speciesId,e.speciesName=i["a"].SPECIES_FULL_NAMES[e.speciesId]};return this.$axios.get("http://api.pax-db.org/proteins?ids=".concat(this.ids)).then(a).catch(function(a){e.$q.notify({color:"negative",position:"top",message:"Loading failed",icon:"report_problem"}),n["a"].rollbar.error(a)})},exportData:function(){var e="Protein\t".concat(this.resource.datasets.map(function(e){return e.name}).join("\t"),"\n"),a=this.resource.proteins.map(function(e){return"".concat(e.externalId,"\t").concat(e.name,"\t").concat(e.abundances.join("\t"))}).join("\n"),t="".concat(e).concat(a),n="paxdb-abundances-".concat(this.speciesId,"_").concat(this.ids,".tsv"),i=new Blob([t],{type:"text/tab-separated-values;charset=utf-8;"});if(navigator.msSaveBlob)navigator.msSaveBlob(i,n);else{var o=document.createElement("a");if(void 0!==o.download){var r=URL.createObjectURL(i);o.setAttribute("href",r),o.setAttribute("download",n),o.style="visibility:hidden",document.body.appendChild(o),o.click(),document.body.removeChild(o)}}}}}}).call(this,t("9537"))},c106:function(e,a,t){},c88b:function(e,a,t){"use strict";var n=t("fca6"),i=t.n(n);i.a},d1ac:function(e,a,t){"use strict";var n=t("3be7"),i=t.n(n);i.a},d457:function(e,a,t){"use strict";var n=function(){var e=this,a=e.$createElement,t=e._self._c||a;return t("JqxGrid",{ref:"grid",attrs:{width:"100%",source:e.dataAdapter,columns:e.columns,columnsresize:!0,sorting:!0,pageable:!0,pagesizeoptions:[10,20,50],editable:!1,selectionmode:"none",autoheight:!0},on:{cellclick:function(a){return e.clickHandler(a)}}})},i=[],o=(t("7f7f"),t("2b0e")),r=t("b8ed"),s=(t("8f3e"),{name:"AbundancesTable",props:["speciesId","datasetId","totalAbundances"],data:function(){var e=this,a={datatype:"json",url:"http://api.pax-db.org/dataset/".concat(this.speciesId,"/").concat(this.datasetId,"/abundances?start=0&end=30000&sort=-abundance"),root:"abundances",sortcolumn:"index",sortdirection:"asc",id:"index",abundances:[]};return{width:850,sortParam:"-abundance",start:0,end:10,source:a,totalrecords:e.totalAbundances,dataAdapter:new jqx.dataAdapter(a,{beforeLoadComplete:function(a){return e.abundances=a,a.forEach(function(e,a){e.index=a+1,e.protein='<a href="/protein/'.concat(e.id,'">').concat(e.name,"</a>")}),a},loadError:function(a,t,n){console.log("load abundances ERROR:".concat(JSON.stringify(n))),e.$q.notify({color:"negative",position:"top",message:"Loading abundances failed",icon:"report_problem"}),o["a"].rollbar.error(n)}}),columns:[{text:"",datafield:"index",width:"50px",sortable:!0,cellsrenderer:this.cellsrenderer},{text:"Protein",datafield:"protein",sortable:!1,width:"150px",cellsrenderer:this.cellsrenderer},{text:"Abundance",datafield:"abundance",sortable:!1,width:"100px",cellsalign:"right",cellsrenderer:this.cellsrenderer},{text:"Annotation",datafield:"annotation",cellsalign:"left",sortable:!1,cellsrenderer:this.cellsrenderer}]}},beforeCreate:function(){},methods:{clickHandler:function(e){if("protein"===e.args.datafield){e.stopPropagation(),e.preventDefault();var a=e.args.originalEvent.target.getAttribute("href");this.$router.push(a)}},cellsrenderer:function(e,a,t,n,i,o){var r=i?"text-align:".concat(i.cellsalign):"";return'<div style="overflow: hidden; text-overflow: ellipsis; padding: 10px;'.concat(r,';">').concat(t,"</div>")}},components:{JqxGrid:r["a"]}}),c=s,l=t("2877"),d=Object(l["a"])(c,n,i,!1,null,null,null);a["a"]=d.exports},ef6d:function(e,a,t){"use strict";var n=t("4991"),i=t.n(n);i.a},fca6:function(e,a,t){}},[[0,"runtime","vendor"]]]);